<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
  <head>
    <title>Planet Neuroscience</title>
    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet"> 
    <link href="https://fonts.googleapis.com/css?family=Comfortaa" rel="stylesheet">
    <!-- Google analytics -->
    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-60261100-2', 'auto');
      ga('send', 'pageview');

    </script>
  </head>

  <body>
    <nav class="navbar navbar-default navbar-static-top">
      <div class="container">
        <div class="navbar-header">
          <a class="navbar-brand" href="#">Planet Neuroscience</a>
        </div>
          <ul class="nav navbar-nav navbar-right">
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">More Neuroscience <span class="caret"></span></a>
              <ul class="dropdown-menu">
                <li><a href="http://neuroblog.fedoraproject.org/planet-neuroscientists/">Planet Neuroscientists</a></li>
                <li><a href="http://home.earthlink.net/~perlewitz/index.html">Computational Neuroscience on the web</a></li>
              </ul>
            </li>
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Options <span class="caret"></span></a>
              <ul class="dropdown-menu">
                <li><a href="https://github.com/neurofedora/planet-neuroscience/issues/new">Suggest a new feed</a></li>
                <li><a href="https://github.com/neurofedora/planet-neuroscience">View Planet source</a></li>
                <li role="separator" class="divider"></li>
                <li><a href="https://github.com/feedreader/pluto">View Pluto source</a></li>
              </ul>
            </li>
          </ul>
        </div><!--/.nav-collapse -->
      </div>
    </nav>
    <div class="container">
      <div class="page-header">
        <h1>Planet Neuroscience</h1>
        <h2>An aggregation of RSS feeds from various neuroscience journals.</h2>
        <p>
        last updated by <a href="https://feedreader.github.io">Pluto</a>
        on 2025-04-07 08:18:20 UTC 
        on behalf of <a href="https://neuro.fedoraproject.org" target="_blank"> the NeuroFedora SIG</a>.
        </p>
      </div>
    </div>

    <div class="container">
      <div class="col-md-9">
        <ul class="list-group">
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/q-bio.NC/>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/q-bio.NC/" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/q-bio.NC/&title=arXiv: Quantitative Biology: Neurons and Cognition" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2502.14671">Explanations of Large Language Models Explain Language Representations in the Brain</a></h4>

              
                arXiv:2502.14671v3 Announce Type: replace-cross 
Abstract: Large language models (LLMs) not only exhibit human-like performance but also share computational principles with the brain's language processing mechanisms. While prior research has focused on mapping LLMs' internal representations to neural activity, we propose a novel approach using explainable AI (XAI) to strengthen this link. Applying attribution methods, we quantify the influence of preceding words on LLMs' next-word predictions and use these explanations to predict fMRI data from participants listening to narratives. We find that attribution methods robustly predict brain activity across the language network, revealing a hierarchical pattern: explanations from early layers align with the brain's initial language processing stages, while later layers correspond to more advanced stages. Additionally, layers with greater influence on next-word prediction$\unicode{x2014}$reflected in higher attribution scores$\unicode{x2014}$demonstrate stronger brain alignment. These results underscore XAI's potential for exploring the neural basis of language and suggest brain alignment for assessing the biological plausibility of explanation methods.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/q-bio.NC/">arXiv: Quantitative Biology: Neurons and Cognition</a> on 2025-04-07 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2504.03031">Building functional and mechanistic models of cortical computation based on canonical cell type connectivity</a></h4>

              
                arXiv:2504.03031v1 Announce Type: cross 
Abstract: Neuronal circuits of the cerebral cortex are the structural basis of mammalian cognition. The same qualitative components and connectivity motifs are repeated across functionally specialized cortical areas and mammalian species, suggesting a single underlying algorithmic motif. Here, we propose a perspective on current knowledge of the cortical structure, from which we extract two core principles for computational modeling. The first principle is that cortical cell types fulfill distinct computational roles. The second principle is that cortical connectivity can be efficiently characterized by only a few canonical blueprints of connectivity between cell types. Starting with these two foundational principles, we outline a general framework for building functional and mechanistic models of cortical circuits.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-07 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2504.02972">Improved Compact Genetic Algorithms with Efficient Caching</a></h4>

              
                arXiv:2504.02972v1 Announce Type: new 
Abstract: Compact Genetic Algorithms (cGAs) are condensed variants of classical Genetic Algorithms (GAs) that use a probability vector representation of the population instead of the complete population. cGAs have been shown to significantly reduce the number of function evaluations required while producing outcomes similar to those of classical GAs. However, cGAs have a tendency to repeatedly generate the same chromosomes as they approach convergence, resulting in unnecessary evaluations of identical chromosomes. This article introduces the concept of caching in cGAs as a means of avoiding redundant evaluations of the same chromosomes. Our proposed approach operates equivalently to cGAs, but enhances the algorithm's time efficiency by reducing the number of function evaluations. We also present a data structure for efficient cache maintenance to ensure low overhead. The proposed caching approach has an asymptotically constant time complexity on average. The proposed method further generalizes the caching mechanism with higher selection pressure for elitism-based cGAs. We conduct a rigorous analysis based on experiments on benchmark optimization problems using two well-known cache replacement strategies. The results demonstrate that caching significantly reduces the number of function evaluations required while maintaining the same level of performance accuracy.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-07 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2504.03037">Low Rank Factorizations are Indirect Encodings for Deep Neuroevolution</a></h4>

              
                arXiv:2504.03037v1 Announce Type: new 
Abstract: Deep neuroevolution is a highly scalable alternative to reinforcement learning due to its unique ability to encode network updates in a small number of bytes. Recent insights from traditional deep learning indicate high-dimensional models possess intrinsic, low-rank structure. In this work, we introduce low-rank, factorized neuroevolution: an indirect encoding through which we can search a small space of low-rank factors that enforce underlying structure across a network's weights. We compare our approach with non-factorized networks of similar and smaller size to understand how much performance can be attributed to the smaller search space. We evaluate our method on a language modeling task using transformers, as well as continuous and discrete vision-based reinforcement learning tasks. Our study shows that low-rank, factorized neuroevolution outperforms or is competitive with non-factorized neuroevolution, performing notably well on language modeling. Our results also suggest deleterious factorized mutations have a stronger negative impact on performance than deleterious non-factorized mutations, which significantly reduces the runtime on environments with early termination for bad performers. More broadly, these results show how we can use insights from backpropgation-based methods to enhance neuroevolution
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-07 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2504.03576">Heterogeneous Resource Allocation for Ensuring End-to-End Quality of Service in Multi-hop Integrated Access and Backhaul Network</a></h4>

              
                arXiv:2504.03576v1 Announce Type: new 
Abstract: Faced with increasing network traffic demands, cell dense deployment is one of significant means to utilize spectrum resources efficiently to improve network capacity. Multi-hop integrated access and backhaul (IAB) architectures have emerged as a cost-effective solution for network densification. Meanwhile, dynamic time division duplex (D-TDD) is a promising solution to adapt to highly dynamic scenarios with asymmetric uplink and downlink traffic. Thus, dynamic resource allocation between backhaul and access links and high spectral efficiency under ensuring reliable transmission are two key objectives of IAB research. However, due to huge solution space, there are some challenges in multi-hop IAB with D-TDD if only an integrated optimization problem (IOP) is considered. To handle these challenges, we decompose the IOP into sub-problems to reduce the solution space. To tackle these sub-problems, we formulate them separately as the non-cooperative games and design the corresponding utility functions to guarantee the existence of Nash equilibrium solutions. Also, to achieve the system-wide solution, we propose a single-leader heterogeneous multi-follower Stackelberg-game-based resource allocation scheme, which can combine the solving results of all the sub-problems to get the IOP approximate solution. Simulation results show that the proposed scheme can improve throughput performance while meeting spectrum energy efficiency constraints.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-07 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2501.05407">On-line Policy Improvement using Monte-Carlo Search</a></h4>

              
                arXiv:2501.05407v1 Announce Type: cross 
Abstract: We present a Monte-Carlo simulation algorithm for real-time policy improvement of an adaptive controller. In the Monte-Carlo simulation, the long-term expected reward of each possible action is statistically measured, using the initial policy to make decisions in each step of the simulation. The action maximizing the measured expected reward is then taken, resulting in an improved policy. Our algorithm is easily parallelizable and has been implemented on the IBM SP1 and SP2 parallel-RISC supercomputers.
  We have obtained promising initial results in applying this algorithm to the domain of backgammon. Results are reported for a wide variety of initial policies, ranging from a random policy to TD-Gammon, an extremely strong multi-layer neural network. In each case, the Monte-Carlo algorithm gives a substantial reduction, by as much as a factor of 5 or more, in the error rate of the base players. The algorithm is also potentially useful in many other adaptive control applications in which it is possible to simulate the environment.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-07 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2504.02834">Explainable Dual-Attention Tabular Transformer for Soil Electrical Resistivity Prediction: A Decision Support Framework for High-Voltage Substation Construction</a></h4>

              
                arXiv:2504.02834v1 Announce Type: cross 
Abstract: This research introduces a novel dual-attention transformer architecture for predicting soil electrical resistivity, a critical parameter for high-voltage substation construction. Our model employs attention mechanisms operating across both features and data batches, enhanced by feature embedding layers that project inputs into higher-dimensional spaces. We implements Particle Swarm Optimization for hyperparameter tuning, systematically optimizing embedding dimensions, attention heads, and neural network architecture. The proposed architecture achieves superior predictive performance (Mean Absolute Percentage Error: 0.63%) compared to recent state of the art models for tabular data. Crucially, our model maintains explainability through SHapley Additive exPlanations value analysis, revealing that fine particle content and dry density are the most influential parameters affecting soil resistivity. We developes a web-based application implementing this model to provide engineers with an accessible decision support framework that bridges geotechnical and electrical engineering requirements for the Electricity Generating Authority of Thailand. This integrated approach satisfies both structural stability and electrical safety standards, improving construction efficiency and safety compliance in high-voltage infrastructure implementation.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-07 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2504.01915">Overcoming Deceptiveness in Fitness Optimization with Unsupervised Quality-Diversity</a></h4>

              
                arXiv:2504.01915v2 Announce Type: replace 
Abstract: Policy optimization seeks the best solution to a control problem according to an objective or fitness function, serving as a fundamental field of engineering and research with applications in robotics. Traditional optimization methods like reinforcement learning and evolutionary algorithms struggle with deceptive fitness landscapes, where following immediate improvements leads to suboptimal solutions. Quality-diversity (QD) algorithms offer a promising approach by maintaining diverse intermediate solutions as stepping stones for escaping local optima. However, QD algorithms require domain expertise to define hand-crafted features, limiting their applicability where characterizing solution diversity remains unclear. In this paper, we show that unsupervised QD algorithms - specifically the AURORA framework, which learns features from sensory data - efficiently solve deceptive optimization problems without domain expertise. By enhancing AURORA with contrastive learning and periodic extinction events, we propose AURORA-XCon, which outperforms all traditional optimization baselines and matches, in some cases even improving by up to 34%, the best QD baseline with domain-specific hand-crafted features. This work establishes a novel application of unsupervised QD algorithms, shifting their focus from discovering novel solutions toward traditional optimization and expanding their potential to domains where defining feature spaces poses challenges.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-07 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2412.10153">EVOS: Efficient Implicit Neural Training via EVOlutionary Selector</a></h4>

              
                arXiv:2412.10153v3 Announce Type: replace-cross 
Abstract: We propose EVOlutionary Selector (EVOS), an efficient training paradigm for accelerating Implicit Neural Representation (INR). Unlike conventional INR training that feeds all samples through the neural network in each iteration, our approach restricts training to strategically selected points, reducing computational overhead by eliminating redundant forward passes. Specifically, we treat each sample as an individual in an evolutionary process, where only those fittest ones survive and merit inclusion in training, adaptively evolving with the neural network dynamics. While this is conceptually similar to Evolutionary Algorithms, their distinct objectives (selection for acceleration vs. iterative solution optimization) require a fundamental redefinition of evolutionary mechanisms for our context. In response, we design sparse fitness evaluation, frequency-guided crossover, and augmented unbiased mutation to comprise EVOS. These components respectively guide sample selection with reduced computational cost, enhance performance through frequency-domain balance, and mitigate selection bias from cached evaluation. Extensive experiments demonstrate that our method achieves approximately 48%-66% reduction in training time while ensuring superior convergence without additional cost, establishing state-of-the-art acceleration among recent sampling-based strategies.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-07 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2502.17034">Evolution 6.0: Evolving Robotic Capabilities Through Generative Design</a></h4>

              
                arXiv:2502.17034v4 Announce Type: replace-cross 
Abstract: We propose a new concept, Evolution 6.0, which represents the evolution of robotics driven by Generative AI. When a robot lacks the necessary tools to accomplish a task requested by a human, it autonomously designs the required instruments and learns how to use them to achieve the goal. Evolution 6.0 is an autonomous robotic system powered by Vision-Language Models (VLMs), Vision-Language Action (VLA) models, and Text-to-3D generative models for tool design and task execution. The system comprises two key modules: the Tool Generation Module, which fabricates task-specific tools from visual and textual data, and the Action Generation Module, which converts natural language instructions into robotic actions. It integrates QwenVLM for environmental understanding, OpenVLA for task execution, and Llama-Mesh for 3D tool generation. Evaluation results demonstrate a 90% success rate for tool generation with a 10-second inference time, and action generation achieving 83.5% in physical and visual generalization, 70% in motion generalization, and 37% in semantic generalization. Future improvements will focus on bimanual manipulation, expanded task capabilities, and enhanced environmental interpretation to improve real-world adaptability.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-07 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58182-x">Mechanistic insights into the structure-based design of a CspZ-targeting Lyme disease vaccine</a></h4>

              
                <p>Nature Communications, Published online: 07 April 2025; <a href="https://www.nature.com/articles/s41467-025-58182-x">doi:10.1038/s41467-025-58182-x</a></p>The outer surface protein CspZ is a potential vaccine candidate of Borrelia burgdorferi for Lyme disease prevention. Here, using structure-based design, the authors generate a mutant CspZ protein with increased stability and improved immunogenicity, exposing protective epitopes.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58732-3">Molecular insights into the overall architecture of human rixosome</a></h4>

              
                <p>Nature Communications, Published online: 07 April 2025; <a href="https://www.nature.com/articles/s41467-025-58732-3">doi:10.1038/s41467-025-58732-3</a></p>Rixosome has critical roles in ribosome biogenesis and silencing of Polycomb target genes. Here authors Huang and Tong report the structures of several sub-complexes of rixosome and a model for the core scaffold of this complex.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04918-9">Electoral precinct-level database for Mexican municipal elections</a></h4>

              
                <p>Scientific Data, Published online: 07 April 2025; <a href="https://www.nature.com/articles/s41597-025-04918-9">doi:10.1038/s41597-025-04918-9</a></p>Electoral precinct-level database for Mexican municipal elections
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.04.647315v1?rss=1
">Longitudinal multimodal neuroimaging after traumatic brain injury</a></h4>

              
                Traumatic brain injury is a major cause of long term cognitive impairment, yet the mechanisms underlying recovery remain poorly understood. Neuroimaging methods such as diffusion MRI, functional MRI, and positron emission tomography (PET) provide insight into micro and macro scale changes post TBI, but the relationships between regional cellular and functional alterations remain unclear. In this study, we conducted a longitudinal, multimodal neuroimaging analysis quantifying TBI-related pathologies in four biomarkers, namely flumazenil PET derived binding potential, dMRI derived structural connectivity, and resting state fMRI derived functional connectivity and fractional amplitude of low frequency fluctuations in individuals with mild to severe brain injury at the subacute (4 to 6 months post injury) and chronic (1 year post injury) stages. Brain injury related regional pathologies, and their changes over time, were correlated across the four biomarkers. Our results reveal complex, dynamic changes over time. We found that flumazenil PET binding potential was significantly reduced in frontal and thalamic regions in brain injured subjects, consistent with neuronal loss, with partial recovery over time. Functional hyperconnectivity was observed in brain injured subjects initially but declined while remaining elevated compared to non-injured controls, whereas cortical structural hypoconnectivity persisted. Importantly, we observed that brain injury related alterations across MRI modalities became more strongly correlated with flumazenil PET at the chronic stage. Regions with chronic reductions in flumazenil PET binding also showed weaker structural node strength and lower amplitude of low frequency fluctuations, a relationship that was not found at the subacute stage. This observation could suggest a progressive convergence of structural and functional disruptions with neuronal loss over time. Additionally, regions with declining structural node strength also exhibited decreases in functional node strength, while these same regions showed increased amplitude of low frequency fluctuations over time. This pattern suggests that heightened intrinsic regional activity may serve as a compensatory mechanism in regions increasingly disconnected due to progressive axonal degradation. Altogether, these findings advance our understanding of how multimodal neuroimaging captures the evolving interplay between neuronal integrity, structural connectivity, and functional dynamics after brain injury. Clarifying these interrelationships could inform prognostic models and enhance knowledge of degenerative, compensatory, and recovery mechanisms in traumatic brain injury.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.05.647352v1?rss=1
">Spatiotemporal differences of GABAergic polarization and shunting during dendritic integration</a></h4>

              
                In the adult brain, GABA exerts either depolarizing or hyperpolarizing effects on neuronal membranes, depending on neuron type, subcellular location, and neuronal activity. Depolarizing GABA typically inhibits neurons through shunting, which is characterized by increased membrane conductance upon GABAA receptor activation; however, it can also excite neurons by recruiting voltage-dependent conductances. The net influence of these opposing actions of depolarizing GABA on glutamatergic synaptic inputs remains incompletely understood. Here, we examined the spatiotemporal characteristics of membrane polarization and shunting mediated by GABAA receptors and assessed their functional impact on the integration of GABAergic and glutamatergic inputs along dendrites. Using whole-cell current-clamp recordings in CA1 pyramidal neurons and dentate gyrus granule cells (GCs) from rat hippocampal slices, we mimicked GABAergic and glutamatergic inputs with local GABA puffs and glutamate spot-uncaging, respectively. A mathematical model further quantified the relative effects of local shunting and polarization. Depolarizing GABAergic postsynaptic responses (GPSRs) exhibited biphasic actions, exerting inhibitory effects at the synapse through shunting, and excitatory effects distally, where depolarization predominated. The excitatory component also persisted longer than the shunting inhibition. In contrast, hyperpolarizing GPSRs remained consistently inhibitory across both spatial and temporal dimensions. These findings highlight the complex spatiotemporal interplay between shunting and membrane polarization mediated by GABAergic inputs, providing new insights into dendritic computation and neuronal network dynamics.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00304-3?rss=yes">Quantifying the evolutionary paths to endomembranes</a></h4>

              
                Schavemaker and Lynch derived functions that relate the cell biology of endomembranes to cellular fitness. Applied to the pinocytosis of small-molecule nutrients and the insertion of membrane proteins by a proto-endoplasmic reticulum, the proto-endoplasmic reticulum is revealed to be the more likely path to complex endomembranes in the origin of eukaryotes.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58457-3">Aneuploidy confers a unique transcriptional and phenotypic profile to Candida albicans</a></h4>

              
                <p>Nature Communications, Published online: 06 April 2025; <a href="https://www.nature.com/articles/s41467-025-58457-3">doi:10.1038/s41467-025-58457-3</a></p>Mackey et al report the transcriptional signature and phenotypic features of aneuploid Candida albicans strains which impact infection and susceptibility to environmental stimuli.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58646-0">Atomic-scale observation of geometric reconstruction in a fluorine-intercalated infinite layer nickelate superlattice</a></h4>

              
                <p>Nature Communications, Published online: 06 April 2025; <a href="https://www.nature.com/articles/s41467-025-58646-0">doi:10.1038/s41467-025-58646-0</a></p>This study shows atomic-scale observations of geometric reconstruction in a fluorine-intercalated infinite layer nickelate superlattice, highlighting how fluorine influences the material’s structure and electronic properties.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58521-y">ESM-Ezy: a deep learning strategy for the mining of novel multicopper oxidases with superior properties</a></h4>

              
                <p>Nature Communications, Published online: 06 April 2025; <a href="https://www.nature.com/articles/s41467-025-58521-y">doi:10.1038/s41467-025-58521-y</a></p>ESM-Ezy combines the ESM-1b protein language model with similarity analysis to predict enzymatic functions in low-similarity sequences. It identifies high-performance biocatalysts, such as novel multicopper oxidases and L-asparaginases, with enhanced efficiency, stability, and industrial potential, advancing sustainable biotech solutions.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58625-5">Primordial neon and the deep mantle origin of kimberlites</a></h4>

              
                <p>Nature Communications, Published online: 06 April 2025; <a href="https://www.nature.com/articles/s41467-025-58625-5">doi:10.1038/s41467-025-58625-5</a></p>Some diamondiferous kimberlite lavas can originate from ancient domains located at Earth’s core-mantle boundary, according to analyses of noble gas geochemistry of magmatic fluids trapped inside kimberlite hosted olivines.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58314-3">LEOPARD: missing view completion for multi-timepoint omics data via representation disentanglement and temporal knowledge transfer</a></h4>

              
                <p>Nature Communications, Published online: 06 April 2025; <a href="https://www.nature.com/articles/s41467-025-58314-3">doi:10.1038/s41467-025-58314-3</a></p>Missing modality challenges longitudinal omics studies. Here, the authors introduce LEOPARD, a pioneering neural network that uses style transfer to re-entangle omics data, enabling robust imputation and unlocking fresh insights into predictive healthcare and biological temporal dynamics.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-56989-2">Benchmarking large language models for biomedical natural language processing applications and recommendations</a></h4>

              
                <p>Nature Communications, Published online: 06 April 2025; <a href="https://www.nature.com/articles/s41467-025-56989-2">doi:10.1038/s41467-025-56989-2</a></p>Baseline performance, benchmarks, and guidance for LLMs in biomedicine are limited. The authors assess four LLMs on 12 tasks, establish baselines, examine hallucinations, and provide recommendations for optimal LLM use.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-08010-1">Targeting alveolar epithelial cells with lipid micelle-encapsulated necroptosis inhibitors to alleviate acute lung injury</a></h4>

              
                <p>Communications Biology, Published online: 06 April 2025; <a href="https://www.nature.com/articles/s42003-025-08010-1">doi:10.1038/s42003-025-08010-1</a></p>A lipid micelle-encapsulated drug targeting MLKL in alveolar type II epithelial cells successfully applied in ALI treatment. This nanoparticle selectively inhibited necroptosis, protecting epithelial cells and reducing inflammatory injury
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-08015-w">Spatially informed graph transformers for spatially resolved transcriptomics</a></h4>

              
                <p>Communications Biology, Published online: 06 April 2025; <a href="https://www.nature.com/articles/s42003-025-08015-w">doi:10.1038/s42003-025-08015-w</a></p>Spatially informed Graph Transformer integrates gene expression and spatial context to accurately denoise data and identify fine-grained tissue domains.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-08014-x">GRASPs link Reelin to the Golgi during neocortical development to control neuronal migration and dendritogenesis</a></h4>

              
                <p>Communications Biology, Published online: 06 April 2025; <a href="https://www.nature.com/articles/s42003-025-08014-x">doi:10.1038/s42003-025-08014-x</a></p>GRASPs act as downstream effectors for Reelin, influencing the positioning and deployment of the Golgi apparatus, directing neuronal migration, and regulating dendrite formation during the development of the neocortex.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-07995-z">Uncovering critical transitions and molecule mechanisms in disease progressions using Gaussian graphical optimal transport</a></h4>

              
                <p>Communications Biology, Published online: 06 April 2025; <a href="https://www.nature.com/articles/s42003-025-07995-z">doi:10.1038/s42003-025-07995-z</a></p>Gaussian Graphical Optimal Transport (GGOT) captures network instability to analyze disease progression, identify tipping points, and key regulatory molecules, offering insights into critical transitions across complex diseases.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.04.647327v1?rss=1
">Memory responses in visual cortex track recall success after single-trial encoding</a></h4>

              
                Classic models of episodic memory propose that retrieval relies on the reactivation of previous perceptual representations in sensory cortex, a phenomenon known as cortical reinstatement. Supporting this idea, visual memory retrieval has been shown to evoke activity patterns in visual areas similar to those during encoding. However, recent work suggests that memory responses systematically diverge from perceptual ones, challenging this idea. Critically, these studies have focused on highly trained memories, leaving open whether similar effects arise in more naturalistic, single-shot memory scenarios, which are hallmarks of episodic memory. Here, we used fMRI and population receptive field (pRF) modeling to test whether spatially tuned memory responses emerge in early visual cortex after a single encoding event. We scanned 19 participants with fMRI while testing them on their recognition and spatial recall of peripheral objects seen only once. We observed spatially tuned responses in early visual cortex during both recognition and recall, even though spatial location was never explicitly probed during recognition. These responses were better tuned for successfully remembered items, indicating a relationship between neural tuning and behavioral memory performance. Moreover, spatial tuning at encoding predicted subsequent memory: responses for subsequently remembered objects were stronger near the object location and suppressed elsewhere, relative to forgotten items. Taken together, our findings show that a single experience is sufficient to enable spatially tuned reactivation in early visual cortex when remembering an item. Further, our results indicate an important role, during both encoding and retrieval, for early visual cortex representations in successful episodic memory.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.04.647222v1?rss=1
">Statistical Characterization of Cortical-Thalamic Dynamics Evoked by Cortical Stimulation in Mice</a></h4>

              
                Objective: Statistical models are powerful tools for describing biological phenomena such as neuronal spiking activity. Although these models have been widely used to study spontaneous and stimulated neuronal activity, they have not yet been applied to analyze responses to electrical cortical stimulation. In this study, we present an innovative approach to characterize neuronal responses to electrical stimulation in the mouse cortex, providing detailed insights into cortical-thalamic dynamics. Approach: Our method applies Mixture Models to analyze the Peri-Stimulus Time His togram of each neuron, predicting the probability of spiking at specific latencies following the onset of electrical stimuli. By applying this approach, we investigated neuronal re sponses to cortical stimulation recorded from the motor cortex, somatosensory cortex, and sensorimotor-related thalamic nuclei in the mouse brain. Main results: The characterization approach achieved high goodness of fit, and the model features were leveraged by applying machine learning methods for stimulus intensity decoding and classification of brain regions to which a neuron belongs given its response to the stimulus. The Random Forest model demonstrated the highest F1 scores, achieving 92.86% for stimulus intensity decoding and 84.35% for brain zone classification. Significance: This study presents a novel statistical framework for characterizing neu ronal responses to electrical cortical stimulation, providing quantitative insights into cortical thalamic dynamics. Our approach achieves high accuracy in stimulus decoding and brain region classification, providing valuable contributions for neuroscience research and neuro technology applications
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.05.647370v1?rss=1
">Novel in situ seeding immunodetection assay uncovers neuronal-driven alpha-synuclein seeding in Parkinson's disease</a></h4>

              
                Aggregates of alpha-synuclein (-syn) and tau propagate through template-induced misfolding in the brains of Parkinson's (PD) and Alzheimer's disease (AD) patients. Prion-like seeding is crucial in disease initiation and progression and represents a major target for drug-modifying therapies. The detection of aggregated -syn and tau seeding activity with seeding amplification assays (SAAs) have remarkable diagnostic and research potential. However, current SAAs rely on bulk tissue homogenates or fluids, losing critical spatial and cellular resolution. Here, we report our novel in situ seeding immunodetection (isSID) assay that enables the visualization of -syn and tau seeding activities with unprecedented morphological detail in intact biological samples. Using the isSID assay, we confirm seeding activity in the pathological aggregates of PD and AD, among others, while uncovering neuron-driven -syn seeding events that precede the onset of clinical symptoms in PD. Our findings provide new fundamental insights into the pathogenesis underlying neurodegeneration and establish an invaluable tool for studying protein aggregation dynamics, with potential applications in biomarker discovery, diagnostics and drug testing.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.05.647372v1?rss=1
">A transgene harboring the human Glucose Transporter1 (GLUT1) gene locus ameliorates disease in GLUT1 deficiency syndrome model mice.</a></h4>

              
                Proper brain function relies on an adequate supply of energy - mainly glucose - to power neuronal activity. Delivery of this nutrient to the neuropil is mediated by the Glucose Transporter1 (GLUT1) protein. Perturbing glucose supply to the brain is profoundly damaging and exemplified by the neurodevelopmental disorder, GLUT1 deficiency syndrome (GLUT1DS). Resulting from haploinsufficiency of the SLC2A1 (GLUT1) gene, GLUT1DS is characterized by intractable infantile-onset seizures and a disabling movement disorder. Ketogenic diets, which supply the brain with an alternate energy source, ketone bodies, are currently the preferred therapeutic option for Glut1DS patients but do not address the underlying cause, low brain glucose, of the disease. One intuitively appealing therapeutic strategy that does, involves restoring GLUT1 levels to the patient brain. Here, we demonstrate that transgenic expression of the human GLUT1 genomic locus in a mouse model of GLUT1DS raises brain GLUT1 levels and reduces disease burden. Augmenting GLUT1 levels in mutants correspondingly raised cerebrospinal fluid (CSF) glucose levels, improved motor performance and reduced the frequency of seizures characteristically observed in GLUT1DS. Interestingly, the increased GLUT1 in mutants harboring the human GLUT1 locus was at least partly the result of an increase in murine Slc2a1 (Glut1) activity, most likely the effect of a long non-coding RNA (lncRNA) embedded in the human transgene. Collectively, our work has not only shown that repleting human GLUT1 mitigates GLUT1DS but also has yielded transgenic mice that constitute a useful tool to test and optimize clinically promising agents designed to stimulate this gene for therapeutic purposes.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.05.647321v1?rss=1
">Dual Hypocretin Receptor Antagonism Reduces Oxycodone Seeking During Abstinence</a></h4>

              
                A major barrier in the treatment of opioid use disorder is persistent drug craving during abstinence. While opioid-based medications have been used to treat opioid use disorder for decades, there is an urgent need for novel, non-opioid-based pharmacotherapies. The hypocretin/orexin (hypocretin) system is a promising target for treating opioid use disorder due to its influence on motivation for drugs of abuse through actions on dopamine transmission. We recently showed that intermittent access (IntA) to oxycodone promoted sustained oxycodone seeking and alterations in dopamine transmission during abstinence. In the current studies, we investigated to what extent suvorexant, an FDA-approved dual hypocretin receptor antagonist, reduces oxycodone seeking and restores dopamine function during abstinence. Results indicated that IntA to oxycodone produced sustained cue-induced oxycodone seeking after a 14-day abstinence period, which was associated with reduced dopamine uptake in the nucleus accumbens core as we have previously shown. Treatment with suvorexant 24 h prior to a cue-induced seeking test significantly reduced oxycodone seeking and normalized aberrant dopamine uptake. These findings suggest that targeting hypocretin receptors may be a promising strategy for reducing opioid craving and associated neuroadaptations, thus lowering the risk of relapse.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.06.647399v1?rss=1
">Incubation of oxycodone craving is associated with CP-AMPAR upregulation in D1 and D2 receptor-expressing medium spiny neurons in nucleus accumbens core and shell</a></h4>

              
                A major problem in treating opioid use disorder is persistence of craving after protracted abstinence. This has been modeled in rodents using the incubation of craving model, in which cue-induced drug seeking increases over the first weeks of abstinence from drug self-administration and then remains high for an extended period. Incubation has been reported for several opioids, including oxycodone, but little is known about underlying synaptic plasticity. In contrast, it is well established that incubation of cocaine and methamphetamine craving depends on strengthening of glutamate synapses in the nucleus accumbens (NAc) through incorporation of calcium-permeable AMPARs (CP-AMPARs). CP-AMPARs have higher conductance than the calcium-impermeable AMPARs that mediate NAc excitatory transmission in drug-naive animals, as well as other distinct properties. Here we examined AMPAR transmission in medium spiny neurons (MSN) of NAc core and shell subregions in rats during forced abstinence from extended-access oxycodone self-administration. In early abstinence (prior to incubation), CP-AMPAR levels were low. After 17-33 days of abstinence (when incubation is stably plateaued), CP-AMPAR levels were significantly elevated in both subregions. These results explain the prior demonstration that infusion of a selective CP-AMPAR antagonist into NAc core or shell subregions prevents expression of oxycodone incubation. Then, using transgenic rats, we found CP-AMPAR upregulation on both D1 and D2 receptor-expressing MSN, which contrasts with selective upregulation on D1 MSN after cocaine and methamphetamine incubation. Overall, our results demonstrate a common role for CP-AMPAR upregulation in psychostimulant and oxycodone incubation, albeit with differences in MSN subtype-specificity.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.05.647390v1?rss=1
">Nonlinear integration of sensory and motor inputs by a single neuron in C. elegans.</a></h4>

              
                Context is important for sensory integration. Rather than simply considering sensory information independently, the brain integrates this information to inform behavior, however identifying this property at the single-neuron level is not trivial. In Caenorhabditis elegans, the paired interneurons AIBL and AIBR (AIB) have a compartmentalized organization of presynapses along its singular process. Sensory and sensory interneurons primarily synapse along the proximal process, while motor and motor interneurons synapse along the distal process. Since this neuron has graded potentials, the simplest model for AIB integration is simply a convolution of its presynaptic inputs. Through a series of experiments to manipulate sensory and motor input onto AIB, we find that while AIB activity is primarily a convolution of motor inputs, its sensory responses are not integrated independently. Instead, the gain in sensory input is a function of the temporal dynamics of motor input. Sensory information is reinforced when it matches the expected behavioral response. We find this property is also observed in other whole-brain datasets. Context-dependent behavioral responses to sensory input is well-documented. Here, we show this property can be localized to single neurons in the worm nervous system. This integration property likely plays an important role in context-dependent decision-making, as well as the highly variable dynamics of the worm nervous system.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.05.647323v1?rss=1
">Context modulates brain state dynamics and behavioral responses during narrative comprehension</a></h4>

              
                Narrative comprehension is inherently context-sensitive, yet the brain and cognitive mechanisms by which brief contextual priming shapes story interpretation remain unclear. Using hidden Markov modeling (HMM) of fMRI data, we identified dynamic brain states as participants listened to an ambiguous spoken story under two distinct narrative contexts (affair vs. paranoia). We uncovered both context-invariant states, engaging auditory, language, and default mode networks, and context-specific states characterized by differential recruitment of control, salience, and visual networks. Narrative context selectively modulated the influence of character speech and linguistic features on brain state expression, with the central character's speech enhancing activation in shared states but suppressing activation in context-specific ones. Independent behavioral analyses revealed parallel context-dependent effects, with character-driven features exerting strong, selectively modulated influences on participants' judgments of narrative evidence. These findings demonstrate that brief narrative priming actively reshapes brain state dynamics and feature sensitivity during story comprehension, revealing how context guides moment-by-moment interpretive processing in naturalistic settings.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00300-6?rss=yes">Amino acids in cancer: Understanding metabolic plasticity and divergence for better therapeutic approaches</a></h4>

              
                Tumors exhibit substantial metabolic complexity. Do et al. discuss the metabolic properties of tumors with a focus on amino acid metabolism, particularly how the heterogeneity and flexibility of amino acid metabolism in cancer influence therapeutic responses. Understanding these intricacies can help us develop strategies for metabolic reprogramming in cancer treatment.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00312-2?rss=yes">PLK1 blockade enhances the anti-tumor effect of MAPK inhibition in pancreatic ductal adenocarcinoma</a></h4>

              
                Zhao et al. demonstrate that elevated expression of Polo-like kinase 1 (PLK1) in PDAC is associated with resistance to MEK inhibition. MEK inhibition suppresses JNK activity, resulting in the inactivation of c-JUN, cell death. The combined inhibition of MEK and PLK1 lifts the repression of JNK, leading to increased cell death.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00277-3?rss=yes">STAT3 signaling enhances tissue expansion during postimplantation mouse development</a></h4>

              
                Azami et al. reveal that STAT3 deletion depletes epiblast cells at implantation, causing a persistent developmental delay of around a day. STAT3 null embryonic stem cells contribute to all embryonic tissues in chimeras except blood. Mixing with wild-type cells for in vitro erythroid differentiation causes the outcompetition of STAT3 null cells.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-01038-7">Obesity-drug pioneers and 13,508 physicists win US$3-million Breakthrough Prizes</a></h4>

              
                <p>Nature, Published online: 05 April 2025; <a href="https://www.nature.com/articles/d41586-025-01038-7">doi:10.1038/d41586-025-01038-7</a></p>Advances recognized by science’s most lucrative awards include Large Hadron Collider experiments and groundbreaking weight-loss treatments.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04868-2">An open-access database of nature-based carbon offset project boundaries</a></h4>

              
                <p>Scientific Data, Published online: 05 April 2025; <a href="https://www.nature.com/articles/s41597-025-04868-2">doi:10.1038/s41597-025-04868-2</a></p>An open-access database of nature-based carbon offset project boundaries
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04914-z">An improved long-term high-resolution surface pCO2 data product for the Indian Ocean using machine learning</a></h4>

              
                <p>Scientific Data, Published online: 05 April 2025; <a href="https://www.nature.com/articles/s41597-025-04914-z">doi:10.1038/s41597-025-04914-z</a></p>An improved long-term high-resolution surface <i>p</i>CO<sub>2</sub> data product for the Indian Ocean using machine learning
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04903-2">Seven years of time-tracking data capturing collaboration and failure dynamics: the Gryzzly dataset</a></h4>

              
                <p>Scientific Data, Published online: 05 April 2025; <a href="https://www.nature.com/articles/s41597-025-04903-2">doi:10.1038/s41597-025-04903-2</a></p>Seven years of time-tracking data capturing collaboration and failure dynamics: the Gryzzly dataset
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04854-8">Diploid chromosome-level genome assembly and annotation for Lycorma delicatula</a></h4>

              
                <p>Scientific Data, Published online: 05 April 2025; <a href="https://www.nature.com/articles/s41597-025-04854-8">doi:10.1038/s41597-025-04854-8</a></p>Diploid chromosome-level genome assembly and annotation for <i>Lycorma delicatula</i>
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04753-y">Enabling disaggregation of Asian American subgroups: a dataset of Wikidata names for disparity estimation</a></h4>

              
                <p>Scientific Data, Published online: 05 April 2025; <a href="https://www.nature.com/articles/s41597-025-04753-y">doi:10.1038/s41597-025-04753-y</a></p>Enabling disaggregation of Asian American subgroups: a dataset of Wikidata names for disparity estimation
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-07993-1">Direct comparisons of neural activity during placebo analgesia and nocebo hyperalgesia between humans and rats</a></h4>

              
                <p>Communications Biology, Published online: 05 April 2025; <a href="https://www.nature.com/articles/s42003-025-07993-1">doi:10.1038/s42003-025-07993-1</a></p>A novel framework for cross-species analysis reveals the conserved and species-specific neural circuitry underlying placebo analgesia and nocebo hyperalgesia.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-08001-2">Trans-ethnic GWAS meta-analysis of idiopathic spermatogenic failure highlights the immune-mediated nature of Sertoli cell-only syndrome</a></h4>

              
                <p>Communications Biology, Published online: 05 April 2025; <a href="https://www.nature.com/articles/s42003-025-08001-2">doi:10.1038/s42003-025-08001-2</a></p>A large-scale genetic study conducted on European and Asian populations reveals shared genetic variants associated with the most severe form of spermatogenic failure, offering valuable insights into the underlying mechanisms of male infertility.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-07534-w">TransBind allows precise detection of DNA-binding proteins and residues using language models and deep learning</a></h4>

              
                <p>Communications Biology, Published online: 05 April 2025; <a href="https://www.nature.com/articles/s42003-025-07534-w">doi:10.1038/s42003-025-07534-w</a></p>An alignment-free deep learning framework predicts DNA-binding proteins and residues from primary sequences, eliminating multiple sequence alignments (MSAs). It uses protein language models and class-weighted training to address data imbalance and improve accuracy.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-07940-0">Network-informed analysis of a multivariate trait-space reveals optimal trait selection</a></h4>

              
                <p>Communications Biology, Published online: 05 April 2025; <a href="https://www.nature.com/articles/s42003-025-07940-0">doi:10.1038/s42003-025-07940-0</a></p>Using network analysis, the authors propose a trait reduction procedure to identify the most informative traits while preserving the functional complexity of trait space.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.04.647154v1?rss=1
">The Mismatch Negativity compared: EEG, SQUID-MEG and novel 4Helium-OPMs</a></h4>

              
                Magneto-encephalography (MEG) provides a higher spatial resolution than electro-encephalography (EEG) to measure human auditory responses. However, conventional cryogenic MEG systems (SQUID-MEG) suffer from severe technological restrictions limiting, for instance, routine clinical use. Fortunately, a new generation of MEG sensors, optically pumped magnetometers (OPMs), have been developed to bridge the gap, combining the wearability of EEG with the benefits of MEG signal acquisition. We aim to assess their potential for studying auditory mismatch processing. The auditory Mismatch Negativity (MMN) is a well-characterized evoked component observable using a passive oddball paradigm with two-tone sound sequences. It has been extensively described using both EEG and MEG and is part of many EEG-based clinical applications, such as the assessment of patients with disorders of consciousness. MMN is therefore a relevant candidate to evaluate OPM performance. We use recently developed Helium-OPMs, which are high dynamic range MEG sensors that operate at room temperature. We compare their performance with cryogenic SQUID-MEG and EEG in a passive frequency oddball paradigm. Results show a significant MMN across subjects in all modalities as well as a high temporal similarity between modalities. Signal-to-noise ratios were also similar, and detection of significant individual MMN (within-subjects) using the OPM system was equal or better than EEG. Given that the OPM system tested here is a prototype comprised of only five sensors, these results are a promising step towards wearable MEG that combines the advantages of MEG and EEG.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.03.643725v1?rss=1
">A transcriptomic comparison of the HD10.6 human sensory neuron-derived cell line with primary and iPSC sensory neurons</a></h4>

              
                A key concern in early-stage analgesic discovery efforts is the extent to which mechanisms identified in rodents will translate to humans. To evaluate an alternative approach to the use of rodent dissociated DRG neurons for in vitro analyses of nociceptive signaling, we performed a transcriptomic analysis of the HD10.6 human dorsal root ganglion (DRG)-derived immortalized cell line. We conducted RNA-seq on proliferating and mature HD10.6 cells to characterize transcriptional changes associated with maturation. We then compared the transcriptomes of HD10.6 cells and several recently developed lines of human induced pluripotent stem cell-derived sensory neurons (iPSC-SN) to single-nucleus RNA-seq data from human DRGs. HD10.6 cells showed the highest correlation with 3 human sensory neuron subtypes associated with nociception and pruriception. Each of the iPSC-SN lines evaluated showed a distinct pattern of correlation with human sensory neuron subtypes. We identified G protein-coupled receptors (GPCRs) and ion channels that are expressed in both HD10.6 cells and human DRG neurons, as well as numerous genes that are expressed in human DRG but not in rodent, underscoring the need for human sensory neuron in vitro models. Proof-of-concept evaluations of protein kinase A, protein kinase C and Erk signaling provide examples of scalable assays using HD10.6 cells to investigate well-established GPCR signaling pathways. We conclude that HD10.6 cells provide a versatile model for exploring human neuronal signaling mechanisms.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.04.647230v1?rss=1
">Theoretical physical color gamuts define luminosity and naturalness perceptual limits in natural scenes</a></h4>

              
                Realism in augmented reality (AR) hinges on the seamless blending of virtual elements into real-world environments. One possible factor influencing this realism may be the physical gamut: an internal representation of all perceivable colors within a natural scene. Previous studies on luminosity thresholds suggest that this gamut, rooted in optimal colors theory, constrains perceptual judgments. While promising, such findings were based on abstract and two-dimensional stimuli only. Before extending this framework to more realistic AR scenarios, an essential next step is to assess whether the physical gamut theory also applies to naturalistic stimuli. This study addresses that gap. Our results reveal that the physical gamut remains a valid construct for natural objects viewed in realistic scenes. Moreover, observers' judgments of luminosity thresholds appear guided not only by a criterion of self-luminosity, but also by an implicit sense of naturalness. These insights pave the way for exploring AR realism through the lens of physical gamut theory.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.04.647288v1?rss=1
">A mechanism linking dopamine's roles in reinforcement, movement and motivation</a></h4>

              
                Dopamine neurons (DANs) play seemingly distinct roles in reinforcement, motivation, and movement, and DA-modulating therapies relieve symptoms across a puzzling spectrum of neurologic and psychiatric symptoms. Yet, the mechanistic relationship among these roles is unknown. Here, we show DA's tripartite roles are causally linked by a process in which phasic striatal DA rapidly and persistently recalibrates the propensity to move, a measure of vigor. Using a self-timed movement task, we found that single exposures to reward-related DA transients (both endogenous and exogenously-induced) exerted one-shot updates to movement timing--but in a surprising fashion. Rather than reinforce specific movement times, DA transients quantitatively changed movement timing on the next trial, with larger transients leading to earlier movements (and smaller to later), consistent with a stochastic search process that calibrates the frequency of movement. Both abrupt and gradual changes in external and internal contingencies--such as timing criterion, reward content, and satiety state--caused changes to the amplitude of DA transients that causally altered movement timing. The rapidity and bidirectionality of the one-shot effects are difficult to reconcile with gradual synaptic plasticity, and instead point to more flexible cellular mechanisms, such as DA-dependent modulation of neuronal excitability. Our findings shed light on how natural reinforcement, as well as DA-related disorders such as Parkinson's disease, could affect behavioral vigor.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.04.647285v1?rss=1
">Large-scale 3D EM connectomics dataset of mouse hippocampal area CA1</a></h4>

              
                The hippocampal formation is thought to be crucial for memory and learning, with subarea cornu ammonis 1 (CA1) considered to play a major role in spatial and episodic memory formation, and for evaluating the match between retrieved memories and current sensory information. While enormous progress has been made in classifying CA1 neurons based on molecular, morphological, and functional properties, and in identifying their role in behavioral tasks, a clear understanding of the underlying circuits is still missing. Here, we present the first large-scale three dimensional (3D) electron microscopy dataset of mouse CA1 at nanometer-scale resolution. The dataset is available online and can be readily used for circuit reconstructions, as demonstrated for inputs to CA1 superficial layers. Example volume segmentations show that automated reconstruction detection is feasible. Using these data, we find evidence against the long-held assumption of a homogeneous pyramidal cell population. Furthermore we find substantial possibly long-range axonal innervation of stratum- lacunosum interneurons, suggested previously to originate in L2 of MEC. These first analyses illustrate the usability of this dataset for finally clarifying the connectomic properties of mouse CA1, a key structure in mammalian brains.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.03.647045v1?rss=1
">Cortical markers of PAS-induced long-term potentiation and depression in the motor system: A TMS-EEG Registered Report</a></h4>

              
                Paired associative stimulation (PAS), a neurostimulation protocol combining transcranial magnetic stimulation (TMS) pulses to the primary motor cortex (M1) with electrical median nerve stimulation, may promote synaptic plasticity (long-term potentiation - LTP, long-term depression - LTD) in the motor system. To date, PAS effects have been mainly investigated at the corticospinal level in the human motor system. In the present Registered Report, we leveraged TMS and electroencephalography (TMS-EEG) co-registration to track the cortical dynamics related to M1-PAS, aiming to characterize the neurophysiological substrates better, grounding the effectiveness of such protocol. In two within-subject sessions, 30 healthy participants underwent the standard M1-PAS protocols inducing LTP (PASLTP) and LTD (PASLTD) while measuring motor-evoked potentials (MEPs) and TMS-evoked potentials (TEPs) from M1 stimulation before, immediately after, and 30 minutes from the end of the PAS, applied both at supra- (i.e., 110%) and sub- (i.e., 90%) resting motor threshold intensities. Besides replicating MEPs enhancement and inhibition after PASLTP and PASLTD, our results showed that the P30 and N100 M1-TEPs components were significantly modulated immediately following PASLTP and PASLTD administration. These effects were detectable only in suprathreshold conditions, suggesting that M1 subthreshold stimulation could not be optimal for tracking cortical effects of PAS. Furthermore, exploratory analyses showed that P60 amplitude at baseline successfully predicted the magnitude of P30 modulations after PASLTP administration. Our findings provide compelling evidence about the specificity of early TEP components in reflecting changes in M1 reactivity underpinning PAS effects. From a broader perspective, our study fosters evidence about using TMS-EEG biomarkers to track complex plastic changes induced in the human brain, exploiting neuromodulatory non-invasive brain stimulation protocols based on associative mechanisms, like PAS.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.04.647310v1?rss=1
">Loudness and sound category: Their distinct roles in shaping perceptual and physiological responses to soundscapes</a></h4>

              
                When compared to nature sounds, exposure to mechanical sounds evokes higher levels of perceptual and physiological arousal, prompting the recruitment of attentional and physiological resources to elicit adaptive responses. However, it is unclear whether these attributes are solely related to the sound intensity of mechanical sounds, since in most real-world scenarios, mechanical sounds are present at high intensities, or if other acoustic or semantic factors are also at play. We measured the Skin Conductance Response (SCR), reflecting sympathetic nervous system (SNS) activity as well as the pleasantness and eventfulness of the soundscape across two passive and active listening tasks in (N = 25) healthy subjects. The auditory stimuli were divided into two categories, nature, and mechanical sounds, and were manipulated to vary in three perceived loudness levels. As expected, we found that the sound category influenced perceived soundscape pleasantness and eventfulness. SCR was analysed by taking the mean level across the stimulus epoch, and also by quantifying its dynamic. We found that mean SCR was modulated by loudness only. SCR rise-time (a measure of speed of the skin response) correlated significantly with soundscape pleasantness and eventfulness for nature and mechanical sounds. This study highlights the importance of considering both loudness level and sound category in evaluating the perceptual soundscape, highlighting SCR as a valuable tool for such assessments.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.03.647083v1?rss=1
">Unbiased Population-Based Statistics to Obtain Pathologic Burden of Injury after Experimental TBI</a></h4>

              
                Reproducibility of scientific data is a current concern throughout the neuroscience field. There are multiple on-going efforts to help resolve this problem. Within the preclinical neuroimaging field, the continued use of a region-of interest (ROI) type approaches combined with the well-known spatial heterogeneity of traumatic brain injury pathology is a barrier to the replicability and repeatability of data. Here we propose the conjoint use of an unbiased analysis of the whole brain after injury together with a population-based statistical analysis of sham-control brains as one approach that has been used in clinical research to help resolve this issue. The approach produces two volumes of pathology that are outside the normal range of sham brains, and can be interpreted as whole brain burden of injury. Using diffusion weighted imaging derived scalars from a tensor analysis of data acquired from adult, male rats at 2, 9 days, 1 and 5 months after lateral fluid percussion injury (LFPI) and in shams (n=73 and 12, respectively), we compared a data-driven, z-score mapping method to a whole brain and white matter-specific analysis, as well as an ROI-based analysis with brain regions preselected by virtue of their large group effect sizes. We show that the data-driven approach is statistically robust, providing the advantage of a large group effect size typical of a ROI analysis of mean scalar values derived from the tensor in regions of gross injury, but without the large multi-region statistical correction required for interrogating multiple brain areas, and without the potential bias inherent with using preselected ROIs. We show that the technique correctly captures the expected longitudinal time-course of the diffusion scalar volumes based on the spatial extent of the pathology and the known temporal changes in scalar values in the LFPI model.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.01.646146v1?rss=1
">Reverse Double-Dipping: When Data Dips You, Twice--Stimulus-Driven Information Leakage in Naturalistic Neuroimaging</a></h4>

              
                This article elucidates a methodological pitfall of cross-validation for evaluating predictive models applied to naturalistic neuroimaging data--namely, 'reverse double-dipping' (RDD). In a broader context, this problem is also known as 'leakage in training examples', which poses challenges in detecting it in practice. This issue can occur when predictive modeling is employed with data from a conventional neuroscientific design, characterized by a limited set of stimuli repeated across trials and/or participants, resulting in spurious predictive performances due to overfitting to repeated signals, even in the presence of independent noise. Through comprehensive simulations and real-world examples following theoretical formulation, the article underscores how such information leakage can occur and how severely it could compromise the analysis when it is combined with widely spread informal reverse inference. The article concludes with practical recommendations for researchers to avoid RDD in their experiment design and analysis.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.04.647225v1?rss=1
">Ethologically relevant behavioural assay for investigating reach and grasp kinematics during whole-body motor control in mice</a></h4>

              
                Reach and grasp are critical components of skilled mammalian motor control and their detailed analysis in rodents has been key to deepening our understanding of prehension in the context of health and disease. However, most studies investigating these behaviours focus on isolating forelimb movements with little regard to the whole-body movements that are key for effective behaviour. To address this issue, we designed a novel behavioural approach to investigate reach and grasp during whole-body, vertical locomotion in mice. Using a customizable transparent climbing surface, we show that our behavioural approach can extract key kinematic features of climbing. Mouse climbing gait reflects aspects of quadrupedal locomotion, showing similar phase dependencies on increasing speed, including reduced stance (i.e. grasp) time and duty factor. Analysis of multi-limb coordination indicated that climbing revolves around anti-phasic forepaw movements with less consistency in interlimb coordination in the hindpaws. Fore- and hindpaws also differed in their reach trajectories and velocity profiles. The flexibility of this approach also allows for tailored climbing configurations, which we use to show that mice can adapt to and overcome vertical obstacles. By leveraging naturalistic climbing, our modular behavioural approach enables investigation of complex prehensile behaviours and facilitates new study into the neural circuits underlying whole-body skilled motor control.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.04.647260v1?rss=1
">Neuronal expression of S100B triggered by oligomeric Aβ peptide protects against cytoskeletal damage and synaptic loss</a></h4>

              
                Alzheimer's disease (AD) is a complex neurodegenerative disorder characterized by the intracellular deposition of Tau protein and extracellular deposition of amyloid-{beta} peptide (A{beta}). AD is also characterized by neuroinflammation and synapse loss, among others. The S100 family is a group of calcium-binding proteins with intra- and extracellular functions, that are important modulators of inflammatory responses. S100B, which is upregulated in AD patients and the most abundant member of this family, was shown to inhibit in vitro the aggregation and toxicity of A{beta}42, acting as a neuroprotective holdase-type chaperone. Although S100B is primarily produced by astrocytes, it is also expressed by various cells, including neurons. In this work, we investigated if S100B neuronal expression is triggered as a response to A{beta} toxic species, to provide protection during disease progression. We used the AD mouse model A{beta}PPswe/PS1A246E to show that neuronal S100B levels are significantly higher in 10-month-old animals, and cellular assays to demonstrate that A{beta} oligomers significantly increase S100B expression in SH-SY5Y cells, but not monomeric or fibrillar A{beta}. Using primary cultures of rat hippocampal neurons, we showed that S100B partially reverts A{beta}-induced cofilin-actin rods (synapse disruptors), and rescues the decrease in active synapses and post- (PSD-95) synaptic marker, imposed by A{beta} peptide. Altogether, these findings establish the neuroprotective activity of S100B in response to proteotoxic stress in cells, highlighting its chaperone function as a crucial factor in understanding proteostasis regulation in the diseased brain and identifying potential therapeutic targets.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.03.646684v1?rss=1
">Transcriptional downregulation of rhodopsin is associated with desensitization of rods to light-induced damage in a murine model of retinitis pigmentosa.</a></h4>

              
                Class I rhodopsin mutations are known for some of the most severe forms of vision impairments in dominantly inherited rhodopsin retinitis pigmentosa. They disrupt the VxPx transport signal, which is required for the proper localization of rhodopsin to the outer segments. While various studies have focused on the light-dependent toxicity of mutant rhodopsin, it remains unclear whether and how these mutations exert dominant-negative effects. Using the class I RhoQ344X rhodopsin knock-in mouse model, we characterized the expression of rhodopsin and other genes by RNA sequencing and qPCR. Those studies indicated that rhodopsin is the most prominently downregulated photoreceptor-specific gene in RhoQ344X/+ mice. Rhodopsin is downregulated significantly prior to the onset of rod degeneration, whereas downregulation of other phototransduction genes, transducin, and Pde6, occurs after the onset and correlate with the degree of rod cell loss. Those studies indicated that the mutant rhodopsin gene causes downregulation of wild-type rhodopsin, imposing an mRNA-level dominant negative effect. Moreover, it causes downregulation of the mutant mRNA itself, mitigating the toxicity. The observed dominant effect is likely common among rhodopsin retinitis pigmentosa as we found a similar rhodopsin downregulation in the major class II rhodopsin mutant model, RhoP23H/+ mice, in which mutant rhodopsin is prone to misfold. Potentially due to mitigated toxicity by reduced rhodopsin expression, RhoQ344X/+ mice did not exhibit light-dependent exacerbation of rod degeneration, even after continuous exposure of mice for 5 days at 3000 lux. Thus, this study describes a novel form of dominant negative effect in inherited neurodegenerative disorders.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.04.644835v1?rss=1
">Listening to the room: disrupting activity of dorsolateral prefrontal cortex impairs learning of room acoustics in human listeners</a></h4>

              
                Navigating complex sensory environments is critical to survival, and brain mechanisms have evolved to cope with the wide range of surroundings we encounter. To determine how listeners learn the statistical properties of acoustic spaces, we assessed their ability to perceive speech in a range of noisy and reverberant rooms. Listeners were also exposed to repetitive transcranial stimulation (rTMS) to disrupt the dorsolateral prefrontal cortex (dlPFC) activity, a region believed to play a role in statistical learning. Our data suggest listeners rapidly adapt to statistical characteristics of an environment to improve speech understanding. This ability is impaired when rTMS is applied bilaterally to the dlPFC. The data demonstrate that speech understanding in noise is best when exposed to a room with reverberant characteristics common to human-built environments, with performance declining for higher and lower reverberation times, including fully anechoic (non-reverberant) environments. Our findings provide evidence for a reverberation-sweet-spot and the presence of brain mechanisms that might have evolved to cope with the acoustic characteristics of listening environments encountered every day.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.03.31.646280v1?rss=1
">The Arousal-Regulated Filter: Modulating Feedforward and Recurrent Dynamics for Adaptive Neural Tracking</a></h4>

              
                Classic behavioral studies have identified an inverted-U relationship between arousal and performance, while neurophysiology studies have shown that the arousal-related neuromodulator norepinephrine (NE) increases the signal-to-noise ratio (SNR) of neural responses to extrinsic inputs. More recently, abstract computational models suggest that arousal signals uncertainty in predictive internal models and increases the influence of new observations. Here, I present the arousal-regulated filter (ARF), a novel computational model of neural state estimation designed to bridge abstract algorithms, behavioral findings, and basic neural mechanisms. According to the ARF, arousal selectively amplifies feedforward synapses relative to recurrent synapses, consistent with findings from neurophysiology. Computationally, the ARF integrates predictions of an internal model, implemented in recurrent connections, with extrinsic observations, relayed by feedforward projections, to adaptively track dynamic systems. The ARF resembles a Kalman filter but replaces dynamic updating of the Kalman gain matrix with modulation of a scalar gain parameter influenced by arousal, and incorporates a potentially nonlinear activation function. Computational simulations demonstrate ARFs versatility across binary, multi-unit categorical, and continuous neural network architectures that track diffusion and drift-diffusion processes. When the internal model is aligned with environmental dynamics, arousal exhibits an inverted-U relationship with accuracy due to a bias-variance tradeoff, consistent with behavioral results. However, when the internal model is misaligned with the true dynamics (i.e. the true state is unpredictable), increased arousal monotonically improves accuracy. Optimal arousal levels vary systematically, being lower in noisy sensory contexts and higher in volatile environments or when unmodeled dynamics exist. Thus, the ARF provides a unifying framework that links abstract computational algorithms, behavioral phenomena, and neurophysiological mechanisms. The model also offers potential insights into the computational effects of altered arousal states in mental health conditions, with potential implications for new approaches to assessment and treatment.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.01.645054v1?rss=1
">PrPC-induced signaling in human neurons activates phospholipase Cγ1 and an Arc/Arg3.1 response</a></h4>

              
                Synaptic dysfunction and loss correlate with cognitive decline in neurodegenerative diseases, including Alzheimer disease (AD) and prion disease. Neuronal hyperexcitability occurs in the early stages of AD and experimental prion disease, prior to the onset of dementia, yet the underlying drivers are unclear. Here we identify an increase in the immediate early gene, Arc/Arg3.1, in the human prion disease-affected frontal cortex, suggestive of neuronal hyperactivity. To investigate early signaling events initiated by prion aggregates (PrPSc) in human neurons, we stimulated PrPC in human iPSC-derived excitatory neurons (iNs) with a known PrPSc-mimetic antibody (POM1), which recapitulated the Arc/Arg3.1 response within two hours. Proteomics, RNAseq, and a phosphokinase array in iNs revealed alterations in the EGF receptor and increased phosphorylated phospholipase C (PLC)-{gamma}1 (Y783), which was also observed in the cerebral cortex of prion-infected mice. Thus, PrPC ligands can induce a PLC-{gamma}1 intracellular signaling cascade together with an Arc response, mimicking a neuronal activity response.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.03.647027v1?rss=1
">Modeling dynamic inflow effects in fMRI to quantify cerebrospinal fluid flow</a></h4>

              
                Cerebrospinal fluid (CSF) flow in the brain is tightly regulated and essential for brain health, and imaging techniques are needed to quantitatively establish the properties of this flow system. Flow-sensitive fMRI has recently emerged as a tool to measure large scale CSF flow dynamics with high sensitivity and temporal resolution; however, the measured signal is not quantitative. Here, we developed a dynamic model to simulate and infer time varying flow velocities from fMRI data. We validated the model in both human and phantom data, and used it to identify important properties of the fMRI inflow signal that inform how the signal should be interpreted. Additionally, we developed a physics-based deep learning framework to invert the model, which enables direct estimation of velocity using fMRI inflow data. This work allows new quantitative information to be obtained from fMRI, which will enable neuroimaging researchers to take advantage of the high sensitivity, high temporal resolution, and wide availability of fMRI to obtain flow signals that are physically interpretable.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.03.642851v1?rss=1
">Contribution of S1pr1-featured astrocyte subpopulation to cisplatin-induced neuropathic pain</a></h4>

              
                Chemotherapy-induced peripheral neuropathy accompanied by neuropathic pain (CIPN) is a major neurotoxicity of cisplatin, a platinum-based drug widely used for lung, ovarian, and testicular cancer treatment. CIPN causes drug discontinuation and severely impacts life quality with no FDA-approved interventions. We previously reported that platinum-based drugs increase levels of sphingosine 1-phosphate (S1P) in the spinal cord and drive CIPN through activating the S1P receptor subtype 1 (S1PR1). However, the mechanisms engaged downstream of S1PR1 remain poorly understood. Using single cell transcriptomics on mouse spinal cord, our findings uncovered subpopulation-specific responses to cisplatin associated with CIPN. Particularly, cisplatin increased the proportion of astrocytes with high expression levels of S1pr1 (S1pr1high astrocytes), specific to which a Wnt signaling pathway was identified. To this end, several genes involved in Wnt signaling, such as the fibroblast growth factor receptor 3 gene (Fgfr3), were highly expressed in S1pr1high astrocytes. The functional S1PR1 antagonist, ozanimod, prevented cisplatin-induced neuropathic pain and astrocytic upregulation of the Wnt signaling pathway genes. FGFR3 belongs to the FGF/FGFR family which often signals to activate Wnt signaling. Intrathecal injection of the FGFR3 antagonist, PD173074, prevented CIPN. These data not only highlight FGFR3 as one of the astrocytic targets of S1PR1 but raise the possibility that S1PR1-induced engagement of Wnt signaling in S1pr1high astrocytes may contribute to CIPN. Overall, our results provide a comprehensive mapping of cellular and molecular changes engaged in cisplatin-induced neuropathic pain and decipher novel S1PR1-based mechanisms of action.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.04.647241v1?rss=1
">A Tonotopic Regulatory Axis Governing Isoform-Specific MYO7A Expression in Cochlear Hair Cells</a></h4>

              
                Myo7a, a gene mutated in Usher syndrome and non-syndromic deafness, encodes an unconventional myosin essential for hair cell function. Our previous work revealed that cochlear hair cells express distinct Myo7a isoforms with distinct spatial and cell type-specific patterns. The canonical isoform (Myo7a-C) and a novel isoform (Myo7a-N) are co-expressed in outer hair cells (OHCs) but exhibit opposing tonotopic gradients, while inner hair cells (IHCs) primarily express Myo7a-C. These isoforms arise from distinct transcriptional start sites, indicating separate regulatory inputs. Here, we identify an intronic cis-regulatory element, EnhancerA, essential for tonotopically graded Myo7a expression. EnhancerA deletion reduces MYO7A protein levels, disrupts hair bundle morphogenesis, alters OHC mechanotransduction, and leads to hair cell degeneration and hearing loss. We further identify SIX2, a tonotopically expressed transcription factor that may interact with EnhancerA to regulate Myo7a-N in OHCs. These findings define a cis-trans regulatory axis critical for isoform-specific Myo7a expression and cochlear function.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.05.647237v1?rss=1
">An engine for systematic discovery of cause-effect relationships between brain structure and function</a></h4>

              
                Characterising how perturbations of brain architecture influence brain function is essential to understand the origins of brain dysfunction, and devise potential avenues of treatment. Here we introduce a computational engine for systematic causal discovery of the functional consequences of altering network architecture and local biophysics in the brain. We integrate multimodal anatomical and functional neuroimaging to implement over 2,000 in-silico brains, and provide mechanistic insight into the functional consequences of local lesions, global wiring, and empirically-derived maps of regional cytoarchitecture and chemoarchitecture. We comprehensively assess how each manipulation of brain macrostructure reshapes spatial and temporal signal coordination, information dynamics, and functional hierarchy - as well as spontaneous co-activation of meta-analytic cognitive circuits, and >6,000 dimensions of local neural dynamics. Our computational model systematically identifies which features of brain architecture have overlapping or antagonistic causal influence over each dimension of brain function, and how functional properties are traded off against each other across disorders and neuromodulation. We find that regions' functional vulnerability to lesions in silico recapitulates their vulnerability to neurodevelopmental and psychiatric in vivo, along a core-periphery organisation. We provide convergent evidence that the brain's wiring diagram is finely tuned to favour the hierarchical integration of information. Notably, our model successfully recapitulates known empirical results that have not been modelled before, including desynchronisation and flattening of the brain's functional hierarchy induced by psychedelic 5HT2A agonists. To catalyse future discoveries, we make this resource freely available to the neuroscience community through an interactive website (https://systematic-causal-mapping.up.railway.app/), where users can interrogate our systematic database of simulated cause-effect relationships. Altogether, we provide a powerful computational engine to predict the functional consequences of experimental or clinical interventions, and drive neuroscientific hypothesis-generation.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.05.644402v1?rss=1
">Functional synaptic connectivity of engrafted spinal cord neurons with locomotor circuitry in the injured spinal cord</a></h4>

              
                Spinal cord injury (SCI) results in significant neurological deficits, with no currently available curative therapies. Neural progenitor cell (NPC) transplantation has emerged as a promising approach for neural repair, as graft-derived neurons (GNs) can integrate into the host spinal cord and support axon regeneration. However, the mechanisms underlying functional recovery remain poorly understood. In this study, we investigate the synaptic integration of NPC-derived neurons into locomotor circuits, the projection patterns of distinct neuronal subtypes, and their potential to modulate motor circuit activity. Using transsynaptic tracing in a mouse thoracic contusion SCI model, we found that NPC-derived neurons form synaptic connections with host locomotor circuits, albeit at low frequencies. Furthermore, we mapped the axon projections of V0C and V2a interneurons, revealing distinct termination patterns within host spinal cord laminae. To assess functional integration, we employed chemogenetic activation of GNs, which induced muscle activity in a subset of transplanted animals. However, NPC transplantation alone did not significantly improve locomotor recovery, highlighting a key challenge in the field. Our findings suggest that while GNs can integrate into host circuits and modulate motor activity, synaptic connectivity remains a limiting factor in functional recovery. Future studies should focus on enhancing graft-host connectivity and optimizing transplantation strategies to maximize therapeutic benefits for SCI.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.eneuro.org/rss/current.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.eneuro.org/rss/current.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.eneuro.org/rss/current.xml&title=eNeuro" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="http://www.eneuro.org/cgi/content/short/12/4/ENEURO.0423-24.2025?rss=1">Maturational Stage-Dependent Contributions of the Cav3.2 T-Type Calcium Channel to Dentate Gyrus Granule Cell Excitability</a></h4>

              
                <p>T-Type calcium channels shape neuronal excitability driving burst firing, plasticity, and neuronal oscillations that influence circuit activity. The three biophysically distinct T-type channel subtypes (Cav3.1, Cav3.2, Cav3.3) are differentially expressed in the brain, contributing to divergent physiological processes. Cav3.2 channels are highly expressed in the dentate gyrus (DG) of the hippocampus, and mice lacking Cav3.2 [knock-out (KO)] exhibit impairments in hippocampal dependent learning and memory tasks, as well as attenuated development of pilocarpine induced epilepsy. Owing to neurogenesis, granule cells (GCs) are continuously added to the DG, generating a heterogeneous population of maturational stages with distinct excitability. While initial studies identified the role of Cav3.2 in mature GC burst firing, its functional relevance in the intrinsic excitability of different GC subpopulations has not yet been examined. In this study, we used juvenile Cav3.2 KO mice to examine the contributions of Cav3.2 channels to GC excitability at three different stages of maturation. We recorded from cells throughout the GC layer using their electrophysiological and morphological features to allocate GCs into immature, intermediate, and mature groups. In immature GCs, loss of Cav3.2 channels reduced the proportion of cells that fired low-threshold calcium spikes. Conversely, Cav3.2 KO increased excitability in regular spiking intermediate and mature GCs, enabling higher-frequency firing, with little impact on the frequency-dependent response. Overall, this study shows that Cav3.2 channels differentially regulate GC excitability throughout maturation and suggest that calcium influx via Cav3.2 may have maturation-dependent contributions to DG processes such as GC survival, integration, and memory encoding.</p>
              

              <p class="text-muted">in <a href ="https://www.eneuro.org/rss/current.xml">eNeuro</a> on 2025-04-04 16:30:15 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.eneuro.org/rss/current.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.eneuro.org/rss/current.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.eneuro.org/rss/current.xml&title=eNeuro" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="http://www.eneuro.org/cgi/content/short/12/4/ENEURO.0561-24.2025?rss=1">The Effect of Speech Masking on the Human Subcortical Response to Continuous Speech</a></h4>

              
                <p>Auditory masking&mdash;the interference of the encoding and processing of an acoustic stimulus imposed by one or more competing stimuli&mdash;is nearly omnipresent in daily life and presents a critical barrier to many listeners, including people with hearing loss, users of hearing aids and cochlear implants, and people with auditory processing disorders. The perceptual aspects of masking have been actively studied for several decades, and particular emphasis has been placed on masking of speech by other speech sounds. The neural effects of such masking, especially at the subcortical level, have been much less studied, in large part due to the technical limitations of making such measurements. Recent work has allowed estimation of the auditory brainstem response (ABR), whose characteristic waves are linked to specific subcortical areas, to naturalistic speech. In this study, we used those techniques to measure the encoding of speech stimuli that were masked by one or more simultaneous other speech stimuli. We presented listeners with simultaneous speech from one, two, three, or five simultaneous talkers, corresponding to a range of signal-to-noise ratios (clean, 0, &ndash;3, and &ndash;6&nbsp;dB), and derived the ABR to each talker in the mixture. Each talker in a mixture was treated in turn as a target sound masked by other talkers, making the response quicker to acquire. We found consistently across listeners that ABR Wave V amplitudes decreased and latencies increased as the number of competing talkers increased.</p>
              

              <p class="text-muted">in <a href ="https://www.eneuro.org/rss/current.xml">eNeuro</a> on 2025-04-04 16:30:15 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-234/v2?src=rss">Women's studies in the Latin American context: a bibliometric approach [version 2; peer review: 1 approved with reservations]</a></h4>

              
                Background Women’s studies in Latin America are transcendental because they make visible and challenge gender inequalities to achieve a more just and equitable society. However, despite this, there are still many research gaps, for which the objective is to examine the trends in research on women’s studies in Latin America. Methods An exploratory methodology based on bibliometric analysis is proposed to evaluate the scientific literature, based on the parameters of the PRISMA-2020 declaration. Results The bibliometric analysis reveals a growth in women’s studies in Latin America, reflecting its importance and relevance. Scientific production has experienced exponential growth, with leading researchers and journals. The United States and Canada lead scientific production. A change is observed in the topics addressed, focusing more on gender and equality. The thematic clusters identified highlight priority areas such as politics, institutions and representation. Conclusions Emerging keywords include neoliberalism, gender violence, political participation, female empowerment, and femicide, reflecting new concerns and challenges addressed in gender studies in Latin America.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-04 14:51:09 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-395/v1?src=rss">Serum sampling proximal to the joint or after exercise influences biomarker concentration and potentially improves understanding of joint microenvironment: a pilot case-control study [version 1; peer review: awaiting peer review]</a></h4>

              
                Background Osteoarthritis (OA) is a common, heterogeneous whole-joint disease contributing to significant morbidity with variable trajectories of symptom progression. Early detrimental joint changes may be identified using molecular biomarkers in serum or synovial fluid. When measuring the former, biomarkers are at risk of systemic dilution, with the latter technically challenging with increased associated risks. In addition, biomarker concentrations are also potentially influenced by extrinsic factors, including exercise. This case-control pilot study aims to understand the influence of sampling location and sub-maximal exercise on serum biomarker concentrations. Methods Recruited participants either had a recent lower-limb musculoskeletal injury (INJ), established knee OA (KOA) or were controls (CON). Serum was taken from their antecubital fossa (ACF, ‘Arm 1’), their great saphenous vein local to their ipsilateral knee (‘Knee’), and after a ten-minute non-weight-bearing exercise, their ACF again (‘Arm 2’). Serum biomarker assays were performed, with results analysed for differences between condition (CON v KOA v INJ), location (Arm 1 v Knee) and pre-post exercise (Arm 1 v Arm 2). Results Thirty-two participants were recruited (CON n=12, KOA n=10, INJ n=10), 81% male, median age 28 (IQR: 25-43) and BMI 23.9 (IQR: 22.5-26.3). Interleukin-(IL)-1β concentration was lower in the KOA group (p=0.004). IL-6 (1.65 and 0.96 ng/L) and leptin (12.65 and 7.01 ug/L) were higher in the Knee sample than Arm 1 (both p<0.001). COMP (173.06 and 190.27 ug/L, p=0.005) and CTX-II (0.68 and 0.83 ug/L, p=0.02) increased following exercise. Further differences were noted when the location and exercise intervention analyses were stratified by condition. Conclusions These novel sampling techniques and easy, accessible 10-minute exercise protocol are suitable for a wide range of settings. Both sampling location and precedent non-weight-bearing sub-maximal exercise influenced serum biomarker concentrations in this pilot case-control study. Optimising these sampling variables could improve the sensitivity of biomarker analysis following validation in a larger population.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-04 14:49:20 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012959">Seizure and redox rescue in a model of glucose transport deficiency</a></h4>

              
                <p>by Jay S. Coggan, Polina Shichkova, Henry Markram, Daniel Keller</p>

Disruptions of energy supply to the brain are associated with many neurodegenerative pathologies and are difficult to study due to numerous interlinked metabolic pathways. We explored the effects of diminished energy supply on brain metabolism using a computational model of the neuro-glia-vasculature ensemble, in the form of a neuron, an astrocyte and local blood supply. As a case study, we investigated the glucose transporter type-1 deficiency syndrome (GLUT1-DS), a childhood affliction characterized by impaired glucose utilization and associated with phenotypes including seizures. Compared to neurons, astrocytes exhibited markedly higher metabolite concentration variabilities for all but a few redox species. This effect could signal a role for astrocytes in absorbing the shock of blood nutrient fluctuations. Redox balances were disrupted in GLUT1-DS with lower levels of reducing equivalent carriers NADH and ATP. The best non-glucose nutrient or pharmacotherapies for re-establishing redox normalcy involved lactate, the keto-diet (β-hydroxybutyrate), NAD and Q10 supplementation, suggesting a possible glucose sparing mechanism. GLUT1-DS seizures resulted from after-discharge neuronal firing caused by post-stimulus ATP reductions and impaired Na<sup>+</sup>/K<sup>+</sup>-ATPase, which can be rescued by restoring either normal glucose or by relatively small increases in neuronal ATP.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012906">Quantifying the impact of contact tracing interview prioritisation strategies on disease transmission: A modelling study</a></h4>

              
                <p>by Logan Wu, Christopher M Baker, Nicholas Tierney, Kylie Carville, Jodie McVernon, Nick Golding, James M McCaw, Freya M Shearer</p>

Contact tracing is an important public health measure used to reduce transmission of infectious diseases. Contact tracers typically conduct telephone interviews with cases to identify contacts and direct them to quarantine, with the aim of preventing onward transmission. However, in situations where caseloads exceed the capacity of the public health system, timely interviews may not be feasible for all cases. Here we present a modelling framework for assessing the impact of different case interview prioritisation strategies on disease transmission. Our model is based on Australian contact tracing procedures and informed by contact tracing data on COVID-19 cases notified in Australia from 2020 to 2021. Our results demonstrate that last-in-first-out strategies (where cases with the most recent swab or notification dates are interviewed first) are more effective at reducing transmission than first-in-first-out strategies (where cases with the oldest swab or notification dates are interviewed first) or strategies with no explicit prioritisation. To maximise the public health benefit from a given case interview capacity, public health practitioners may consider our findings when designing case interview prioritisation protocols for outbreak response.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012904">scTrans: Sparse attention powers fast and accurate cell type annotation in single-cell RNA-seq data</a></h4>

              
                <p>by Zhiyi Zou, Ying Liu, Yuting Bai, Jiawei Luo, Zhaolei Zhang</p>

Cell type annotation is crucial in single-cell RNA sequencing data analysis because it enables significant biological discoveries and deepens our understanding of tissue biology. Given the high-dimensional and highly sparse nature of single-cell RNA sequencing data, most existing annotation tools focus on highly variable genes to reduce dimensionality and computational load. However, this approach inevitably results in information loss, potentially weakening the model’s generalization performance and adaptability to novel datasets. To mitigate this issue, we developed scTrans, a single cell Transformer-based model, which employs sparse attention to utilize all non-zero genes, thereby effectively reducing the input data dimensionality while minimizing information loss. We validated the speed and accuracy of scTrans by performing cell type annotation on 31 different tissues within the Mouse Cell Atlas. Remarkably, even with datasets nearing a million cells, scTrans efficiently perform cell type annotation in limited computational resources. Furthermore, scTrans demonstrates strong generalization capabilities, accurately annotating cells in novel datasets and generating high-quality latent representations, which are essential for precise clustering and trajectory analysis.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012888">Ten simple rules for getting started with knowledge mobilization</a></h4>

              
                <p>by Rime Diany, Laura K. Fitzgibbon-Collins, Sarah A. Gagliano Taliun</p>
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012874">Segmentation of dense and multi-species bacterial colonies using models trained on synthetic microscopy images</a></h4>

              
                <p>by Vincent Hickl, Abid Khan, René M. Rossi, Bruno F. B. Silva, Katharina Maniura-Weber</p>

The spread of microbial infections is governed by the self-organization of bacteria on surfaces. Bacterial interactions in clinically relevant settings remain challenging to quantify, especially in systems with multiple species or varied material properties. Quantitative image analysis methods based on machine learning show promise to overcome this challenge and support the development of novel antimicrobial treatments, but are limited by a lack of high-quality training data. Here, novel experimental and image analysis techniques for high-fidelity single-cell segmentation of bacterial colonies are developed. Machine learning-based segmentation models are trained solely using synthetic microscopy images that are processed to look realistic using a state-of-the-art image-to-image translation method (cycleGAN), requiring no biophysical modeling. Accurate single-cell segmentation is achieved for densely packed single-species colonies and multi-species colonies of common pathogenic bacteria, even under suboptimal imaging conditions and for both brightfield and confocal laser scanning microscopy. The resulting data provide quantitative insights into the self-organization of bacteria on soft surfaces. Thanks to their high adaptability and relatively simple implementation, these methods promise to greatly facilitate quantitative descriptions of bacterial infections in varied environments, and may be used for the development of rapid diagnostic tools in clinical settings.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012864">Logic-based modeling of biological networks with Netflux</a></h4>

              
                <p>by Alexander P. Clark, Mukti Chowkwale, Alexander Paap, Stephen Dang, Jeffrey J. Saucerman</p>

Molecular signaling networks drive a diverse range of cellular decisions, including whether to proliferate, how and when to die, and many processes in between. Such networks often connect hundreds of proteins, genes, and processes. Understanding these complex networks is aided by computational modeling, but these tools require extensive programming knowledge. In this article, we describe a user-friendly, programming-free network simulation tool called Netflux. Over the last decade, Netflux has been used to construct numerous predictive network models that have deepened our understanding of how complex biological networks make cell decisions. Here, we provide a Netflux tutorial that covers how to construct a network model and then simulate network responses to perturbations. Upon completion of this tutorial, you will be able to construct your own model in Netflux and simulate how perturbations to proteins and genes propagate through signaling and gene-regulatory networks.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012771">Why the growth of arboviral diseases necessitates a new generation of global risk maps and future projections</a></h4>

              
                <p>by Oliver J. Brady, Leonardo S. Bastos, Jamie M. Caldwell, Simon Cauchemez, Hannah E. Clapham, Illaria Dorigatti, Katy A. M. Gaythorpe, Wenbiao Hu, Laith Hussain-Alkhateeb, Michael A. Johansson, Ahyoung Lim, Velma K. Lopez, Richard James Maude, Jane P. Messina, Erin A. Mordecai, Andrew Townsend Peterson, Isabel Rodriquez-Barraquer, Ingrid B. Rabe, Diana P. Rojas, Sadie J. Ryan, Henrik Salje, Jan C. Semenza, Quan Minh Tran</p>
Author summary: Global risk maps are an important tool for assessing the global threat of mosquito and tick-transmitted arboviral diseases. Public health officials increasingly rely on risk maps to understand the drivers of transmission, forecast spread, identify gaps in surveillance, estimate disease burden, and target and evaluate the impact of interventions. Here, we describe how current approaches to mapping arboviral diseases have become unnecessarily siloed, ignoring the strengths and weaknesses of different data types and methods. This places limits on data and model output comparability, uncertainty estimation and generalisation that limit the answers they can provide to some of the most pressing questions in arbovirus control. We argue for a new generation of risk mapping models that jointly infer risk from multiple data types. We outline how this can be achieved conceptually and show how this new framework creates opportunities to better integrate epidemiological understanding and uncertainty quantification. We advocate for more co-development of risk maps among modellers and end-users to better enable risk maps to inform public health decisions. Prospective validation of risk maps for specific applications can inform further targeted data collection and subsequent model refinement in an iterative manner. If the expanding use of arbovirus risk maps for control is to continue, methods must develop and adapt to changing questions, interventions and data availability.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012593">TIPP3 and TIPP3-fast: Improved abundance profiling in metagenomics</a></h4>

              
                <p>by Chengze Shen, Eleanor Wedell, Mihai Pop, Tandy Warnow</p>

We present TIPP3 and TIPP3-fast, new tools for abundance profiling in metagenomic datasets. Like its predecessor, TIPP2, the TIPP3 pipeline uses a maximum likelihood approach to place reads into labeled taxonomies using marker genes, but it achieves superior accuracy to TIPP2 by enabling the use of much larger taxonomies through improved algorithmic techniques. We show that TIPP3 is generally more accurate than leading methods for abundance profiling in two important contexts: when reads come from genomes not already in a public database (i.e., novel genomes) and when reads contain sequencing errors. We also show that TIPP3-fast has slightly lower accuracy than TIPP3, but is also generally more accurate than other leading methods and uses a small fraction of TIPP3’s runtime. Additionally, we highlight the potential benefits of restricting abundance profiling methods to those reads that map to marker genes (i.e., using a filtered marker-gene based analysis), which we show typically improves accuracy. TIPP3 is freely available at https://github.com/c5shen/TIPP3.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012493">Molecular dynamics simulations of human cohesin subunits identify DNA binding sites and their potential roles in DNA loop extrusion</a></h4>

              
                <p>by Chenyang Gu, Shoji Takada, Giovanni B. Brandani, Tsuyoshi Terakawa</p>

The SMC complex cohesin mediates interphase chromatin structural formation in eukaryotic cells through DNA loop extrusion. Here, we sought to investigate its mechanism using molecular dynamics simulations. To achieve this, we first constructed the amino-acid-residue-resolution structural models of the cohesin subunits, SMC1, SMC3, STAG1, and NIPBL. By simulating these subunits with double-stranded DNA molecules, we predicted DNA binding patches on each subunit and quantified the affinities of these patches to DNA using their dissociation rate constants as a proxy. Then, we constructed the structural model of the whole cohesin complex and mapped the predicted high-affinity DNA binding patches on the structure. From the spatial relations of the predicted patches, we identified that multiple patches on the SMC1, SMC3, STAG1, and NIPBL subunits form a DNA clamping patch group. The simulations of the whole complex with double-stranded DNA molecules suggest that this patch group facilitates DNA bending and helps capture a DNA segment in the cohesin ring formed by the SMC1 and SMC3 subunits. In previous studies, these have been identified as critical steps in DNA loop extrusion. Therefore, this study provides experimentally testable predictions of DNA binding sites implicated in previously proposed DNA loop extrusion mechanisms and highlights the essential roles of the accessory subunits STAG1 and NIPBL in the mechanism.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/plosbiology/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/plosbiology/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/plosbiology/NewArticles&title=PLoS Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3003125">Correction: Predation drives complex eco-evolutionary dynamics in sexually selected traits</a></h4>

              
                <p>by Brian A. Lerch, Maria R. Servedio</p>
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/plosbiology/NewArticles">PLoS Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/plosbiology/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/plosbiology/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/plosbiology/NewArticles&title=PLoS Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3003101">Refining the sleep circuits one neuron at a time</a></h4>

              
                <p>by Giorgio F. Gilestro</p>

The neural basis of sleep regulation remains elusive. A new study in PLOS Biology refines the key neuronal circuits involved in the regulation of sleep in fruit flies, confirming <i>Drosophila melanogaster</i> as the model of choice for unraveling the systems neuroscience of such a mysterious phenomenon.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/plosbiology/NewArticles">PLoS Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/plosbiology/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/plosbiology/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/plosbiology/NewArticles&title=PLoS Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3002649">Combining nanobody labeling with STED microscopy reveals input-specific and layer-specific organization of neocortical synapses</a></h4>

              
                <p>by Yeasmin Akter, Grace Jones, Grant J. Daskivich, Victoria Shifflett, Karina J. Vargas, Martin Hruska</p>

The discovery of synaptic nanostructures revealed key insights into the molecular logic of synaptic function and plasticity. Yet, our understanding of how diverse synapses in the brain organize their nano-architecture remains elusive, largely due to the limitations of super-resolution imaging in complex brain tissue. Here, we characterized single-domain camelid nanobodies for the 3D quantitative multiplex imaging of synaptic nano-organization sing tau-STED nanoscopy in cryosections from the mouse primary somatosensory cortex. We focused on thalamocortical (TC) and corticocortical (CC) synapses along the apical-basal axis of layer five pyramidal neurons as models of functionally diverse glutamatergic synapses in the brain. Spines receiving TC input were larger than those receiving CC input in all layers examined. However, the nano-architecture of TC synapses varied with dendritic location. TC afferents on apical dendrites frequently contacted spines with multiple aligned PSD-95/Bassoon nanomodules of constant size. In contrast, TC spines on basal dendrites predominantly contained a single aligned nanomodule, with PSD-95 nanocluster sizes scaling proportionally with spine volume. The nano-organization of CC synapses did not change across cortical layers and resembled modular architecture defined in vitro. These findings highlight the nanoscale diversity of synaptic architecture in the brain, that is, shaped by both the source of afferent input and the subcellular localization of individual synaptic contacts.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/plosbiology/NewArticles">PLoS Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/13-447/v3?src=rss">Case Report: Methimazole-Induced Parotitis - An Unusual Presentation [version 3; peer review: 1 approved, 1 approved with reservations, 2 not approved]</a></h4>

              
                A 56-year-old female with a medical history of unspecified hyperthyroidism and a recent thyroid storm presented to the ED with tachycardia, hypertension, and bilateral enlarged parotid glands. During a previous hospitalization, she was diagnosed with unspecified hyperthyroidism and started on methimazole. During hospitalization, laboratory findings suggested Graves’ disease with an acute thyroid storm. The patient also complained of enlarged parotid glands bilaterally. CT tomography of the neck revealed no calculi of the parotid glands but showed extensive fatty replacement, possibly related to methimazole use. Treatment with propranolol and IV hydrocortisone improved thyroid function. Due to the suspicion of methimazole-induced parotitis, she was transitioned to a reduced methimazole dosage for treatment of Graves’ disease, which subsequently improved her parotitis. Methimazole, the standard initial treatment for Graves’ disease, is generally well-tolerated. It can cause adverse reactions; however, parotitis is very rare and has been documented in only a few case reports. Owing to the limited number of reports, its incidence is currently unknown. Here, we present a case of methimazole-induced parotitis as an unusual presentation of thyroid storm. Drug-induced reactions can only be considered once common causes of parotitis such as viral infection, obstruction, and autoimmune diseases are ruled out. Treatment involves dosage adjustments and supportive care. Methimazole-induced parotitis is often misdiagnosed and overlooked because of the lack of reported cases. This necessitates future research into the reaction mechanisms and optimal treatment.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-04 12:59:10 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-394/v1?src=rss">The association between HBA1C level and the outcome of hernia surgery in diabetic patients managed at a tertiary center: A retrospective cohort study [version 1; peer review: awaiting peer review]</a></h4>

              
                Background The literature indicates a link between HbA1C levels, infections, and postoperative complications across various surgical procedures. However, its specific association with hernioplasty outcomes remains uncertain. This study aims to evaluate the relationship between HbA1C levels and the outcomes of hernia surgery in diabetic patients treated at a tertiary care center. Methods Data were collected from 301 diabetic patients, categorized into controlled (HbA1C < 7%) and uncontrolled (HbA1C ≥ 7%) groups, who underwent hernioplasty at King Abdulaziz Medical City (KAMC) between September 1, 2017, and September 1, 2022. Patient records were analyzed to explore the association between diabetes and hernia surgery outcomes, including complications, infections, and recurrence, as well as to compare the outcomes of different surgical techniques (open and laparoscopic). Results Of the 301 patients, this study included only 140 (46.5%) diabetic individuals who underwent hernia surgery at the Ministry of National Guard Health Affairs (MNGHA) in Riyadh between 2017 and 2022. Preoperative uncontrolled HbA1C levels were not statistically significant for wound infection (p = 0.706) or hernia recurrence (p = 0.075) within a one-year follow-up period. However, a statistically significant difference was observed in the prevalence of hypertension between the controlled and uncontrolled HbA1C groups (p = 0.039). Conclusion This study indicates that HbA1C levels do not significantly influence complications or infections following hernia surgery, particularly regarding wound infections and recurrence. However, HTN was found to be statistically significant and warrants further investigation. Additionally, no statistical significance was identified between elevated BMI (>30 kg/m2) and HbA1C groups (p = 0.081), which may be attributed to the limited sample size of this study.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-04 12:17:22 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-393/v1?src=rss">Comparing two chemotherapeutic regimens VDC/IE (or VAC/IE) v/s VIDE for patients suffering with Ewing sarcoma: A systematic review on survivorship and regime difference across borders [version 1; peer review: awaiting peer review]</a></h4>

              
                Background Ewing sarcoma is the second most common primary bone cancer in adolescents and young adults, with a poor prognosis. It is usually treated with a combination of treatment options, including but not limited to chemotherapy, surgery, and radiation therapy. In this systematic review, we focus on comparing two different chemotherapeutic regimens i.e., VDC/IE (vincristine, doxorubicin, and cyclophosphamide alternating with ifosfamide and etoposide) and VIDE (vincristine, ifosfamide, doxorubicin, and etoposide), for patients with Ewing Sarcoma, with Overall survival as the primary outcome and event-free survival as the secondary outcome. Methods MEDLINE, Embase (via Ovid), and Web of Science were searched methodically to identify studies which satisfied our inclusion and exclusion criteria. After obtaining the studies, deduplication was performed (manually and automatically). Once duplicates were removed, the studies were scanned against the inclusion and exclusion criteria predefined in the registered Prospero protocol. The process of screening was carried out in two phases: in the first stage, the Titles and Abstracts were scanned, and those which were included were scanned in the second stage of full-eligibility screening. The studies obtained were assessed for Risk of Bias using the JBI’s checklist. Results Post Risk of Bias Assessment, three studies were deemed suitable for inclusion. Through analysis of the data and text of these studies, we conclude that the VDC/IE regime yields better survival outcomes when compared to the VIDE regime, and hence should tentatively be used as standard of care for treating Ewing Sarcoma.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-04 12:13:34 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00282.2024?af=R">Neuroscientific Approaches to Adolescent Social Media Use: A Review of Neural correlates and Potential Associations to Social Media Behaviors</a></h4>

              
                Journal of Neurophysiology, Ahead of Print. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-04 11:59:53 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00042.2025?af=R">Convergent effects of peptides on the initiation of feeding motor programs in the mollusk Aplysia</a></h4>

              
                Journal of Neurophysiology, Ahead of Print. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-04 11:39:57 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00108.2024?af=R">Third-order entropy for spatiotemporal neural network characterization</a></h4>

              
                Journal of Neurophysiology, <a href="https://journals.physiology.org/toc/jn/133/4">Volume 133, Issue 4</a>, Page 1234-1244, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-04 10:59:32 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00480.2023?af=R">Action complexity modulates motor performance in the emotional oddball task</a></h4>

              
                Journal of Neurophysiology, <a href="https://journals.physiology.org/toc/jn/133/4">Volume 133, Issue 4</a>, Page 1245-1255, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-04 10:59:31 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00016.2025?af=R">A novel approach to the assessment of motor-unit identification accuracy and its radical incongruence with conventional methodology</a></h4>

              
                Journal of Neurophysiology, <a href="https://journals.physiology.org/toc/jn/133/4">Volume 133, Issue 4</a>, Page 1256-1265, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-04 10:59:30 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00034.2025?af=R">Dual task reactive balance control in older adults with mild cognitive impairment: Does the cognitive task domain make a difference?</a></h4>

              
                Journal of Neurophysiology, Ahead of Print. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-04 10:49:58 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-88/v2?src=rss">Structuring data analysis projects in the Open Science era with Kerblam! [version 2; peer review: 2 approved]</a></h4>

              
                Background Structuring data analysis projects, that is, defining the layout of files and folders needed to analyze data using existing tools and novel code, largely follows personal preferences. Open Science calls for more accessible, transparent and understandable research. We believe that Open Science principles can be applied to the way data analysis projects are structured. Methods We examine the structure of several data analysis project templates by analyzing project template repositories present in GitHub. Through visualization of the resulting consensus structure, we draw observations regarding how the ecosystem of project structures is shaped, and what salient characteristics it has. Results Project templates show little overlap, but many distinct practices can be highlighted. We take them into account with the wider Open Science philosophy to draw a few fundamental Design Principles to guide researchers when designing a project space. We present Kerblam!, a project management tool that can work with such a project structure to expedite data handling, execute workflow managers, and share the resulting workflow and analysis outputs with others. Conclusions We hope that, by following these principles and using Kerblam!, the landscape of data analysis projects can become more transparent, understandable, and ultimately useful to the wider community.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-04 10:21:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/13-1128/v2?src=rss">Fostering healthcare system sustainability through efficient practices: Can adopting biosimilars ease the financial burden of rheumatoid arthritis? [version 2; peer review: 1 approved, 1 approved with reservations]</a></h4>

              
                Immune-mediated inflammatory diseases like rheumatoid arthritis (RA) have been successfully treated using biologic disease-modifying antirheumatic drugs. These medications are not utilized as first-line treatment, in part because of their high cost, but they are frequently seen to be cost-effective for RA patient populations that do not respond adequately to conventional disease-modifying antirheumatic drugs. Moreover, not all RA patients who meet clinical eligibility criteria can access biologics, not even as second-line therapy. Recently, there has been an increasing interest in biosimilars that are highly comparable to their originator biologics in terms of efficacy and safety but generally come at a lower price. This review summarizes the potential role of biosimilars in reducing RA expenditure and increasing RA patient access to biologic therapies. As the global landscape for biosimilars continues to evolve, it is essential to consider the unique challenges and opportunities in different healthcare systems. By leveraging the potential of biosimilars, healthcare systems can improve RA management, ease its economic burden and ensure that patients have access to effective and affordable treatments. The future of RA treatment lies in the integration of biosimilars into clinical practice, offering hope for more sustainable and equitable healthcare systems.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-04 10:06:47 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml&title=Physical Review E: Biological physics" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="http://link.aps.org/doi/10.1103/PhysRevE.111.044404">Microbial populations hardly ever grow logistically and never sublinearly</a></h4>

              
                <p>Author(s): José Camacho-Mateu, Aniello Lampo, Mario Castro, and José A. Cuesta</p><p>Challenging the conventional logistic model, this study shows that microbial growth follows a generalized <math xmlns="http://www.w3.org/1998/Math/MathML" display="inline"><mi>θ</mi></math>-logistic model, accounting for environmental fluctuations. The findings show microbial growth is never sublinear, with significant implications for macroecological patterns and the mechanisms governing microbial ecosystem stability.</p>
<p>#BiophysicsSpotlight #TimelyTopic #AdvancingField</p><img src="//cdn.journals.aps.org/journals/PRE/key_images/10.1103/PhysRevE.111.044404.png" width="200" height=\"100\"><br/><p>[Phys. Rev. E 111, 044404] Published Fri Apr 04, 2025</p>
              

              <p class="text-muted">in <a href ="http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml">Physical Review E: Biological physics</a> on 2025-04-04 10:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/12-142/v4?src=rss">Comparison of the antimicrobial reduction effect of photodynamic inactivation with the addition of chlorophyll and curcumin photosensitizer in Aggregatibacter actinomycetemcomitans and Enterococcus faecalis [version 4; peer review: 3 approved with reserva</a></h4>

              
                Background Aggregatibacter actinomycetemcomitans and Enterococcus faecalis are pathogenic bacteria of the oral cavity that cause various diseases such as periodontitis and endodontics. These bacteria are easily resistant to antibiotics. Photodynamic inactivation (PDI) is a method of inactivating microorganisms that utilizes light to activate a photosensitizer agent (PS) that produces reactive oxygen species causing cell lysis. Methods This study used the PDI method with a 405 nm diode laser at various energy density with the addition PS curcumin or chlorophyll Alfalfa, as much as 1.6 mg/ml on A. actinomycetemcomitans and E. faecalis bacteria. Results The study on E. faecalis bacteria showed that the energy density diode laser irradiation of 1.59 J/cm² gave the percentage of E. faecalis bacteria death 36.7% without PS, 69.30% with the addition of chlorophyll Medicago sativa L and 89.42% with the addition of curcumin. Meanwhile, the bacteria A. actinomycetemcomitans showed that the energy density diode laser irradiation of 1.59 J/cm² gave the percentage of bacterial death 35.81% without PS, 64.39% with the addition of chlorophyll Medicago sativa L and 89.82% with the addition of curcumin. PS was critical to the success of the PDI. Conclusions The addition of PS curcumin increased the effectiveness of reducing bacteria E. faecalis and A. actinomycetemcomitans compared to chlorophyll Medicago sativa L.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-04 08:09:26 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.ads1325?af=R">Nonlinear sound-sheet microscopy: Imaging opaque organs at the capillary and cellular scale</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/388/6742">Volume 388, Issue 6742</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.ads4539?af=R">Exogenous RNA surveillance by proton-sensing TRIM25</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/388/6742">Volume 388, Issue 6742</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.ads7913?af=R">Homogeneous-heterogeneous bifunctionality in Pd-catalyzed vinyl acetate synthesis</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/388/6742">Volume 388, Issue 6742</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adt0548?af=R">Osteoarthritis treatment via the GLP-1–mediated gut-joint axis targets intestinal FXR signaling</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/388/6742">Volume 388, Issue 6742</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adp1853?af=R">A geological timescale for bacterial evolution and oxygen adaptation</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/388/6742">Volume 388, Issue 6742</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adr3675?af=R">Human high-order thalamic nuclei gate conscious perception through the thalamofrontal loop</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/388/6742">Volume 388, Issue 6742</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adq2110?af=R">Integrating multiple evidence streams to understand insect biodiversity change</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/388/6742">Volume 388, Issue 6742</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ado8268?af=R">Robust semi-automatic vessel tracing in the human retinal image by an instance segmentation neural network</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ads9711?af=R">PIP2 promotes the incorporation of CD43, PSGL-1, and CD44 into nascent HIV-1 particles</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adu4391?af=R">A high-throughput experimentation platform for data-driven discovery in electrochemistry</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adt4666?af=R">Galvanostatic cycling of a micron-sized solid-state battery: Visually linking void evolution to electrochemistry</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adt9526?af=R">Topological elastic liquid diode</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adv7218?af=R">Experimental quantification of nuclear quantum effects on the hydrogen bond of liquid water</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adr7134?af=R">MIST: An interpretable and flexible deep learning framework for single–T cell transcriptome and receptor analysis</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ads6094?af=R">Architecture and conformational dynamics of the BAM-SurA holo insertase complex</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ads7475?af=R">Complex-valued matrix-vector multiplication using a scalable coherent photonic processor</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adu0601?af=R">Mg2+ influx mediated by TRPM7 triggers the initiation of muscle stem cell activation</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adr9038?af=R">Leveraging data mining, active learning, and domain adaptation for efficient discovery of advanced oxygen evolution electrocatalysts</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adt1691?af=R">Observation of the magnonic Dicke superradiant phase transition</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adv1878?af=R">Melt-mixed superlayer cocrystal formation using symmetric and unsymmetric organic semiconductors</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adq1575?af=R">Atf3 controls transitioning in female mitochondrial cardiomyopathy as identified by spatial and single-cell transcriptomics</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adu4268?af=R">An autonomous snapper featuring adaptive actuation and embodied intelligence</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ads7191?af=R">Cholinergic signaling to CA1 astrocytes controls fear extinction</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adr1695?af=R">Global inland-water oxygen cycle has changed in the Anthropocene</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adr4492?af=R">Hot Schrödinger cat states</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adr1716?af=R">Delivery of intraflagellar transport proteins to the ciliary base and assembly into trains</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adr4310?af=R">Simultaneous acclimation to nitrogen and iron scarcity in open ocean cyanobacteria revealed by sparse tensor decomposition of metatranscriptomes</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adr6415?af=R">Retromer promotes the lysosomal turnover of mtDNA</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ads5113?af=R">A gain route to reversed Cherenkov radiation</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ads4593?af=R">FairDiffusion: Enhancing equity in latent diffusion models via fair Bayesian perturbation</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adt6631?af=R">Diagnosing Parkinson’s disease via behavioral biometrics of keystroke dynamics</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adv2010?af=R">Modular synthesis of bis-α-chiral amines using Ellman sulfinamide for consecutive S-to-C chirality induction/transfer</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adq9845?af=R">Structural architecture of TolQ-TolR inner membrane protein complex from opportunistic pathogen Acinetobacter baumannii</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adu6562?af=R">Spin-chirality-driven second-harmonic generation in two-dimensional magnet CrSBr</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adr4161?af=R">Tree-ring stable isotopes from the European Alps reveal long-term summer drying over the Holocene</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adx6064?af=R">Erratum for the Research Article: “Light-stimulated micromotor swarms in an electric field with accurate spatial, temporal, and mode control” by Z. Liang et al.</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://www.cell.com/trends/neurosciences/inpress.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://www.cell.com/trends/neurosciences/inpress.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://www.cell.com/trends/neurosciences/inpress.rss&title=Trends in Neurosciences: In press" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/trends/neurosciences/fulltext/S0166-2236(25)00060-8?rss=yes">[11C]ER176 images brain inflammation across TSPO genotypes and colocalizes with tau</a></h4>

              
                Accurately measuring brain inflammation in Alzheimer’s disease (AD) is crucial due to the role of inflammatory processes in neurodegeneration. In a recent study, Appleton, Finn, et al. used [11C]ER176, a novel translocator protein 18 kDa (TSPO)-positron emission tomography (PET) tracer overcoming genotype-related binding issues, to show increased inflammation in early-onset AD, with patterns aligning more closely with tau pathology than amyloid deposition or atrophy.
              

              <p class="text-muted">in <a href ="http://www.cell.com/trends/neurosciences/inpress.rss">Trends in Neurosciences: In press</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00301-8?rss=yes">A single amino acid in the Salmonella effector SarA/SteE triggers supraphysiological activation of STAT3 for anti-inflammatory gene expression</a></h4>

              
                Gaggioli et al. perform a comparative study of STAT3 activation by the Salmonella effector SarA and the host cytokine co-receptor gp130. SarA activation is biased toward anti-inflammatory target genes because its STAT3 binding motif has evolved to match the tyrosine autophosphorylation motif of GSK-3.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00285-2?rss=yes">Structural basis of urate transport by glucose transporter 9</a></h4>

              
                Matsushita et al. report cryo-EM structures of a urate transporter, GLUT9, in apo and urate-bound states, revealing a unique recognition manner as compared to conventional GLUTs. Functional analysis revealed the importance of collective action by several residues in the urate pocket.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00371-7?rss=yes">The Placental Gene PEG10 Promotes Progression of Neuroendocrine Prostate Cancer</a></h4>

              
                (Cell Reports 12, 922–936; August 11, 2015)
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41586-025-08923-1">Author Correction: Record sea surface temperature jump in 2023–2024 unlikely but not unexpected</a></h4>

              
                <p>Nature, Published online: 04 April 2025; <a href="https://www.nature.com/articles/s41586-025-08923-1">doi:10.1038/s41586-025-08923-1</a></p>Author Correction: Record sea surface temperature jump in 2023–2024 unlikely but not unexpected
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41586-025-08788-4">Author Correction: Traversable wormhole dynamics on a quantum processor</a></h4>

              
                <p>Nature, Published online: 04 April 2025; <a href="https://www.nature.com/articles/s41586-025-08788-4">doi:10.1038/s41586-025-08788-4</a></p>Author Correction: Traversable wormhole dynamics on a quantum processor
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00970-y">Revealed: where rare and giant starfish hide from an enigmatic killer</a></h4>

              
                <p>Nature, Published online: 04 April 2025; <a href="https://www.nature.com/articles/d41586-025-00970-y">doi:10.1038/d41586-025-00970-y</a></p>The sunflower star, which can be bigger than a hubcap, escapes a wasting disease by hunkering down in fjords.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-01073-4">Neck deep in overdiagnosis: Books in brief</a></h4>

              
                <p>Nature, Published online: 04 April 2025; <a href="https://www.nature.com/articles/d41586-025-01073-4">doi:10.1038/d41586-025-01073-4</a></p>Andrew Robinson reviews five of the best science picks.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-01071-6">Trump team removes senior NIH chiefs in shock move</a></h4>

              
                <p>Nature, Published online: 04 April 2025; <a href="https://www.nature.com/articles/d41586-025-01071-6">doi:10.1038/d41586-025-01071-6</a></p>Decision comes amidst mass layoffs — plus, administration instructs US biomedical agency to study ‘regret’ after transgender people transition.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-01060-9">Tariffs hit science labs: Trump levies raise cost of supplies</a></h4>

              
                <p>Nature, Published online: 04 April 2025; <a href="https://www.nature.com/articles/d41586-025-01060-9">doi:10.1038/d41586-025-01060-9</a></p>Import taxes on staples such as microscopes, glassware and computer chips will affect institutions already feeling financial strain.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-01056-5">International PhD students make emergency plans in fear of US immigration raids</a></h4>

              
                <p>Nature, Published online: 04 April 2025; <a href="https://www.nature.com/articles/d41586-025-01056-5">doi:10.1038/d41586-025-01056-5</a></p>Arrests of scholars by US President Donald Trump’s administration prompts lawsuit alleging free-speech violations.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/neuro/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/neuro/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/neuro/rss/current&title=Nature Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41593-025-01938-x">Opening the deep learning box</a></h4>

              
                <p>Nature Neuroscience, Published online: 04 April 2025; <a href="https://www.nature.com/articles/s41593-025-01938-x">doi:10.1038/s41593-025-01938-x</a></p>Opening the deep learning box
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/neuro/rss/current">Nature Neuroscience</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/neuro/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/neuro/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/neuro/rss/current&title=Nature Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41593-025-01940-3">AI predicts risk of mental health disorders</a></h4>

              
                <p>Nature Neuroscience, Published online: 04 April 2025; <a href="https://www.nature.com/articles/s41593-025-01940-3">doi:10.1038/s41593-025-01940-3</a></p>AI predicts risk of mental health disorders
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/neuro/rss/current">Nature Neuroscience</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/neuro/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/neuro/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/neuro/rss/current&title=Nature Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41593-025-01939-w">Sugar-coating the BBB</a></h4>

              
                <p>Nature Neuroscience, Published online: 04 April 2025; <a href="https://www.nature.com/articles/s41593-025-01939-w">doi:10.1038/s41593-025-01939-w</a></p>Sugar-coating the BBB
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/neuro/rss/current">Nature Neuroscience</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nphoton/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nphoton/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nphoton/rss/current&title=Nature Photomics" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41566-025-01653-w">Space-time-topological events in photonic quantum walks</a></h4>

              
                <p>Nature Photonics, Published online: 04 April 2025; <a href="https://www.nature.com/articles/s41566-025-01653-w">doi:10.1038/s41566-025-01653-w</a></p>Combining space topology and time topology, topological states that are localized simultaneously in space and time are theoretically and experimentally demonstrated, potentially enabling the space-time topological shaping of light waves with applications in spatiotemporal wave control for imaging, communications and topological lasers.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nphoton/rss/current">Nature Photomics</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nphys/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nphys/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nphys/rss/current&title=Nature Physics" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41567-025-02865-1">Shape-recovering liquids</a></h4>

              
                <p>Nature Physics, Published online: 04 April 2025; <a href="https://www.nature.com/articles/s41567-025-02865-1">doi:10.1038/s41567-025-02865-1</a></p>Placing particles at the interface between immiscible fluids usually enhances emulsification. However, now it is shown that if the particles are ferromagnetic, emulsification is suppressed and a non-planar recoverable interfacial shape develops.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nphys/rss/current">Nature Physics</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04916-x">Chromosome-scale genome assembly and annotation of Xenocypris argentea</a></h4>

              
                <p>Scientific Data, Published online: 04 April 2025; <a href="https://www.nature.com/articles/s41597-025-04916-x">doi:10.1038/s41597-025-04916-x</a></p>Chromosome-scale genome assembly and annotation of <i>Xenocypris argentea</i>
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04902-3">Chromosome-level genome assembly and annotation of the sharpsnout seabream (Diplodus puntazzo)</a></h4>

              
                <p>Scientific Data, Published online: 04 April 2025; <a href="https://www.nature.com/articles/s41597-025-04902-3">doi:10.1038/s41597-025-04902-3</a></p>Chromosome-level genome assembly and annotation of the sharpsnout seabream (<i>Diplodus puntazzo</i>)
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/90596">Stratification of enterochromaffin cells by single-cell expression analysis</a></h4>

              
                Dynamic interactions between gut mucosal cells and the external environment are essential to maintain gut homeostasis. Enterochromaffin (EC) cells transduce both chemical and mechanical signals and produce 5-hydroxytryptamine to mediate disparate physiological responses. However, the molecular and cellular basis for functional diversity of ECs remains to be adequately defined. Here, we integrated single-cell transcriptomics with spatial image analysis to identify 14 EC clusters that are topographically organized along the gut. Subtypes predicted to be sensitive to the chemical environment and mechanical forces were identified that express distinct transcription factors and hormones. A <i>Piezo2<sup>+</sup></i> population in the distal colon was endowed with a distinctive neuronal signature. Using a combination of genetic, chemogenetic, and pharmacological approaches, we demonstrated <i>Piezo2<sup>+</sup></i> ECs are required for normal colon motility. Our study constructs a molecular map for ECs and offers a framework for deconvoluting EC cells with pleiotropic functions.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/103417">CXXC-finger protein 1 associates with FOXP3 to stabilize homeostasis and suppressive functions of regulatory T cells</a></h4>

              
                FOXP3-expressing regulatory T (T<sub>reg</sub>) cells play a pivotal role in maintaining immune homeostasis and tolerance, with their activation being crucial for preventing various inflammatory responses. However, the mechanisms governing the epigenetic program in T<sub>reg</sub> cells during their dynamic activation remain unclear. In this study, we demonstrate that CXXC-finger protein 1 (CXXC1) interacts with the transcription factor FOXP3 and facilitates the regulation of target genes by modulating H3K4me3 deposition. <i>Cxxc1</i> deletion in T<sub>reg</sub> cells leads to severe inflammatory disease and spontaneous T cell activation, with impaired immunosuppressive function. As a transcriptional regulator, CXXC1 promotes the expression of key T<sub>reg</sub> functional markers under steady-state conditions, which are essential for the maintenance of T<sub>reg</sub> cell homeostasis and their suppressive functions. Epigenetically, CXXC1 binds to the genomic regulatory regions of T<sub>reg</sub> program genes in mouse T<sub>reg</sub> cells, overlapping with FOXP3-binding sites. Given its critical role in T<sub>reg</sub> cell homeostasis, CXXC1 presents itself as a promising therapeutic target for autoimmune diseases.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/88335">Annihilation of action potentials induces electrical coupling between neurons</a></h4>

              
                Neurons generate and propagate electrical pulses called action potentials which annihilate on arrival at the axon terminal. We measure the extracellular electric field generated by propagating and annihilating action potentials and find that on annihilation, action potentials expel a local discharge. The discharge at the axon terminal generates an inhomogeneous electric field that immediately influences target neurons and thus provokes ephaptic coupling. Our measurements are quantitatively verified by a powerful analytical model which reveals excitation and inhibition in target neurons, depending on position and morphology of the source-target arrangement. Our model is in full agreement with experimental findings on ephaptic coupling at the well-studied Basket cell-Purkinje cell synapse. It is able to predict ephaptic coupling for any other synaptic geometry as illustrated by a few examples.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/97196">The rhizobial effector NopT targets Nod factor receptors to regulate symbiosis in Lotus japonicus</a></h4>

              
                It is well documented that type-III effectors are required by Gram-negative pathogens to directly target different host cellular pathways to promote bacterial infection. However, in the context of legume–rhizobium symbiosis, the role of rhizobial effectors in regulating plant symbiotic pathways remains largely unexplored. Here, we show that NopT, a YopT-type cysteine protease of <i>Sinorhizobium fredii</i> NGR234 directly targets the plant’s symbiotic signaling pathway by associating with two Nod factor receptors (NFR1 and NFR5 of <i>Lotus japonicus</i>). NopT inhibits cell death triggered by co-expression of NFR1/NFR5 in <i>Nicotiana benthamiana</i>. Full-length NopT physically interacts with NFR1 and NFR5. NopT proteolytically cleaves NFR5 both in vitro and in vivo, but can be inactivated by NFR1 as a result of phosphorylation. NopT plays an essential role in mediating rhizobial infection in <i>L. japonicus</i>. Autocleaved NopT retains the ability to cleave NFR5 but no longer interacts with NFR1. Interestingly, genomes of certain <i>Sinorhizobium</i> species only harbor <i>nopT</i> genes encoding truncated proteins without the autocleavage site. These results reveal an intricate interplay between rhizobia and legumes, in which a rhizobial effector protease targets NFR5 to suppress symbiotic signaling. NFR1 appears to counteract this process by phosphorylating the effector. This discovery highlights the role of a bacterial effector in regulating a signaling pathway in plants and opens up the perspective of developing kinase-interacting proteases to fine-tune cellular signaling processes in general.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/101080">Reconstructing the phylogeny and evolutionary history of freshwater fishes (Nemacheilidae) across Eurasia since early Eocene</a></h4>

              
                Eurasia has undergone substantial tectonic, geological, and climatic changes throughout the Cenozoic, primarily associated with tectonic plate collisions and a global cooling trend. The evolution of present-day biodiversity unfolded in this dynamic environment, characterised by intricate interactions of abiotic factors. However, comprehensive, large-scale reconstructions illustrating the extent of these influences are lacking. We reconstructed the evolutionary history of the freshwater fish family Nemacheilidae across Eurasia and spanning most of the Cenozoic on the base of 471 specimens representing 279 species and 37 genera plus outgroup samples. Molecular phylogeny using six genes uncovered six major clades within the family, along with numerous unresolved taxonomic issues. Dating of cladogenetic events and ancestral range estimation traced the origin of Nemacheilidae to Indochina around 48 mya. Subsequently, one branch of Nemacheilidae colonised eastern, central, and northern Asia, as well as Europe, while another branch expanded into the Burmese region, the Indian subcontinent, the Near East, and northeast Africa. These expansions were facilitated by tectonic connections, favourable climatic conditions, and orogenic processes. Conversely, aridification emerged as the primary cause of extinction events. Our study marks the first comprehensive reconstruction of the evolution of Eurasian freshwater biodiversity on a continental scale and across deep geological time.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/100393">Linalool combats Saprolegnia parasitica infections through direct killing of microbes and modulation of host immune system</a></h4>

              
                <i>Saprolegnia parasitica</i> is one of the most virulent oomycete species in freshwater aquatic environments, causing severe saprolegniasis and leading to significant economic losses in the aquaculture industry. Thus far, the prevention and control of saprolegniasis face a shortage of medications. Linalool, a natural antibiotic alternative found in various essential oils, exhibits promising antimicrobial activity against a wide range of pathogens. In this study, the specific role of linalool in protecting <i>S. parasitica</i> infection at both in vitro and in vivo levels was investigated. Linalool showed multifaceted anti-oomycetes potential by both of antimicrobial efficacy and immunomodulatory efficacy. For in vitro test, linalool exhibited strong anti-oomycetes activity and mode of action included: (1) Linalool disrupted the cell membrane of the mycelium, causing the intracellular components leak out; (2) Linalool prohibited ribosome function, thereby inhibiting protein synthesis and ultimately affecting mycelium growth. Surprisingly, meanwhile we found the potential immune protective mechanism of linalool in the in vivo test: (1) Linalool enhanced the complement and coagulation system which in turn activated host immune defense and lysate <i>S. parasitica</i> cells; (2) Linalool promoted wound healing, tissue repair, and phagocytosis to cope with <i>S. parasitica</i> infection; (3) Linalool positively modulated the immune response by increasing the abundance of beneficial Actinobacteriota; (4) Linalool stimulated the production of inflammatory cytokines and chemokines to lyse <i>S. parasitica</i> cells. In all, our findings showed that linalool possessed multifaceted anti-oomycetes potential which would be a promising natural antibiotic alternative to cope with <i>S. parasitica</i> infection in the aquaculture industry.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/100854">Mesenchymal Meis2 controls whisker development independently from trigeminal sensory innervation</a></h4>

              
                Hair follicle development is initiated by reciprocal molecular interactions between the placode-forming epithelium and the underlying mesenchyme. Cell fate transformation in dermal fibroblasts generates a cell niche for placode induction by activation of signaling pathways WNT, EDA, and FGF in the epithelium. These successive paracrine epithelial signals initiate dermal condensation in the underlying mesenchyme. Although epithelial signaling from the placode to mesenchyme is better described, little is known about primary mesenchymal signals resulting in placode induction. Using genetic approach in mice, we show that <i>Meis2</i> expression in cells derived from the neural crest is critical for whisker formation and also for branching of trigeminal nerves. While whisker formation is independent of the trigeminal sensory innervation, MEIS2 in mesenchymal dermal cells orchestrates the initial steps of epithelial placode formation and subsequent dermal condensation. MEIS2 regulates the expression of transcription factor <i>Foxd1</i>, which is typical of pre-dermal condensation. However, deletion of <i>Foxd1</i> does not affect whisker development. Overall, our data suggest an early role of mesenchymal MEIS2 during whisker formation and provide evidence that whiskers can normally develop in the absence of sensory innervation or <i>Foxd1</i> expression.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/103137">Small-molecule activation of TFEB alleviates Niemann–Pick disease type C via promoting lysosomal exocytosis and biogenesis</a></h4>

              
                Niemann–Pick disease type C (NPC) is a devastating lysosomal storage disease characterized by abnormal cholesterol accumulation in lysosomes. Currently, there is no treatment for NPC. Transcription factor EB (TFEB), a member of the microphthalmia transcription factors (MiTF), has emerged as a master regulator of lysosomal function and promoted the clearance of substrates stored in cells. However, it is not known whether TFEB plays a role in cholesterol clearance in NPC disease. Here, we show that transgenic overexpression of TFEB, but not TFE3 (another member of MiTF family) facilitates cholesterol clearance in various NPC1 cell models. Pharmacological activation of TFEB by sulforaphane (SFN), a previously identified natural small-molecule TFEB agonist by us, can dramatically ameliorate cholesterol accumulation in human and mouse NPC1 cell models. In NPC1 cells, SFN induces TFEB nuclear translocation via a ROS-Ca<sup>2+</sup>-calcineurin-dependent but MTOR-independent pathway and upregulates the expression of TFEB-downstream genes, promoting lysosomal exocytosis and biogenesis. While genetic inhibition of TFEB abolishes the cholesterol clearance and exocytosis effect by SFN. In the NPC1 mouse model, SFN dephosphorylates/activates TFEB in the brain and exhibits potent efficacy of rescuing the loss of Purkinje cells and body weight. Hence, pharmacological upregulating lysosome machinery via targeting TFEB represents a promising approach to treat NPC and related lysosomal storage diseases, and provides the possibility of TFEB agonists, that is, SFN as potential NPC therapeutic candidates.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-04-04 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-390/v1?src=rss">Accuracy of Systemic Inflammatory Indices in identifying Neonatal Sepsis in preterms in a tertiary care hospital in Mangalore [version 1; peer review: awaiting peer review]</a></h4>

              
                Background Both term and preterm neonates can develop sepsis, a potentially lethal and life-threatening condition, during the first 28 days of life. Neonatal sepsis accounts for 8% of all neonatal fatalities. This study aimed to evaluate the predictive power of lab-based diagnostic indices for neonatal sepsis in preterm infants. Methods The Systemic Inflammatory Indices of the two groups of preterms – one control group without sepsis and one case group with sepsis–were compared to assess their value in predicting Neonatal Sepsis. Data from 138 preterm neonates were used in the present study. Systemic Inflammatory Indices were calculated and compared from the collected data in both the case and control groups. Results Platelet count, Pan Immune Inflammation Value (PIV), Platelet to Lymphocyte Ratio (PLR) and Systemic Immune Inflammatory Index (SII) were found to be significant predictors of neonatal sepsis. Platelet count had the highest predictive value, with an AUC value of 0.715 and optimal cut-off value of 219500. It had a sensitivity of 75.4 and specificity of 65.2. Conclusion According to this study, sepsis in preterm infants can be predicted by using systemic inflammatory indices. This will aid in early sepsis diagnosis and management and, in turn, reduce neonatal morbidity and mortality associated with sepsis.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-03 17:07:28 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-389/v1?src=rss">Assesing Conversion Rate, Treatment Success and Mortality of Drug Induce Liver Injury from First Line Antituberculosis Regiment: A Systematic Review and Meta -Analysis [version 1; peer review: awaiting peer review]</a></h4>

              
                The prevalence of Drug Induce Liver Injury (DILI) in Tuberculosis (TB) patients is a critical concern, as it complicates the treatment regimen. The hepatotoxicity of first-line anti-TB drugs such as isoniazid and rifampicin can lead to significant liver damage, which in turn affects the metabolism of these medications, potentially resulting in treatment failure or increased mortality. Given the limited certainty surrounding the clinical outcomes of tuberculosis-associated drug-induced liver injury (TB-DILI), an in-depth evaluation of existing evidence is essential. This systematic review and meta-analysis aimed to assess the clinical outcomes in patients with TB-DILI. Comprehensive searches were conducted across the Scopus, Embase, and PubMed databases, adhering to inclusion criteria derived from the PICOS framework. Keywords related to "drug-induced liver injury" and "tuberculosis," as well as their synonyms, were employed in the search. The Newcastle-Ottawa Scale (NOS) was utilized to assess the risk of bias in observational studies. Data were independently extracted, and the quality of the included studies was evaluated. Relative risk (RR) and tests for heterogeneity were conducted, and results were visualized through RR estimates and forest plots. A total of five studies, encompassing 5,798 patients, were ultimately included in the analysis. This study indicates that TB DILI has no significant risk on 2 months conversion compared to TB Non – DILI patients with RR: 1.03; 95%CI: 0.97–1.03, p<0.31. However, treatment success (RR: 1.11; 95%CI: 1.05–1.18, p<0.0004) and mortality risk (RR: 2.62; 95%CI: 1.14–6.03, p<0.02) were significant in TB Non – DILI patients.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-03 17:06:10 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/13-1547/v2?src=rss">Using text-mining to measure the scientific impact and legacy of ELIXIR, a distributed research infrastructure for life science data [version 2; peer review: 1 approved, 1 approved with reservations]</a></h4>

              
                Background ELIXIR is a pan-European public-funded research infrastructure dedicated to life science data. As such, it must demonstrate public value to its funders and stakeholders. We present methods to inventory research publications linked to ELIXIR that have received funding and support, as well as related citation metrics, used as performance metrics for these audiences. Methods To overcome challenges inherent in ELIXIR’s distributed structure, and the fact that those publishing ELIXIR-supported work are typically working part-time on ELIXIR matters, a semi-automated approach, consisting of text-mining followed by manual curation, is presented. A country-level case study (ELIXIR Italy) refines and expands the methods, notably by introducing more granularity in the curation process (e.g. considering all national-level grants, examining affiliations to report publication per institute) and by additionally looking at the scientific impact of the resources developed and operated by the Italian Node of ELIXIR. Results Overall, the methods described in this article have shown to: (1) be repeatable with acceptable levels of accuracy and consistency (notably across curators); (2) require reasonable effort in terms of curation of monthly ‘harvests’ of publications (obtained by text-mining); and (3) to be well-adapted to ELIXIR’s distributed nature. Conclusions Concrete examples are provided of downstream uses of the inventoried publications and their citations, both for ELIXIR as a whole and for the Italian case study. Limitations of the methods are discussed, particularly the challenges associated with using an ‘Open literature’ database (Europe PMC) for the text-mining, and the constraints related to curation capacity. The methods, along with the valuable lessons learned during their development, are sufficiently generic and pragmatic enough to be readily adapted by other similar research infrastructures.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-03 17:04:18 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/13-1044/v2?src=rss">Methodology used to develop the minimum common data elements for surveillance and Reporting of Musculoskeletal Injuries in the MILitary (ROMMIL) statement [version 2; peer review: 1 approved, 2 approved with reservations]</a></h4>

              
                Background The objective was to summarize the methodology used to develop the international minimum data elements for surveillance and Reporting of Musculoskeletal Injuries in the MILitary (ROMMIL) statement. This is a recommended list of elements to be collected and reported when conducting injury surveillance research in military settings. Methods A Delphi methodology was employed to reach consensus. Preliminary steps included conducting a literature review and surveying a convenience sample of military stakeholders to 1) identify barriers and facilitators of military musculoskeletal injury (MSKI) prevention programs, 2) identify relevant knowledge gaps, and 3) establish future research priorities. A sequential three-round Delphi consensus survey followed, including relevant stakeholders from militaries around the world, using results to conduct an asynchronous knowledge user meeting (mixture of in-person and live video conference and recording) to explore the level of agreement among subject matter experts. Knowledge users, including former and current military service members, civilian practitioners working in military health networks, and international subject matter experts having experience with policy, execution, or clinical investigation of MSKI mitigation programs, MSKI diagnoses, and MSKI risk factors in military settings. For each round, participants scored questions on a Likert scale of 1-5. Scores ranged from No Importance (1) to Strong Importance (5). Results Literature review and surveys helped inform the scope of potential variables. Three rounds were necessary to reach minimum consensus. Ninety-five, 65, and 42 respondents participated in the first, second and third rounds, respectively. Conclusions Achieving consensus across relevant knowledge users representing military organizations globally can be challenging. This paper details the methodology employed to reach consensus for a core minimum data elements checklist for conducting MSKI research in military settings and improve data harmonization and scalability efforts. These methods can be used as a resource to assist in future consensus endeavors of similar nature.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-03 17:02:14 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/11-1107/v5?src=rss">Prevalence of common mental health problems and associated factors among university students visiting Supara mental health service: A cross-sectional study [version 5; peer review: 2 approved]</a></h4>

              
                Background: Early studies found that the mental health problems rate was relatively high in university students. We aimed to investigate the prevalence of mental problems and associated factors in university students. Methods: We conducted a cross-sectional descriptive study at Supara mental health service in the Faculty of Medicine Vajira Hospital between February 2020 to June 2021. The primary outcome was the prevalence of psychiatric diagnosis according to the 10th revision of the International Statistical Classification of Diseases and Related Health Problems (ICD-10). The secondary assessments included the Patient Health Questionnaire-9 (PHQ-9), 8 items from the Mini International Neuropsychiatric Interview (MINI) to assess suicidal risk (8Q), , and the Thai Mental Health Indicator (TMHI-15). The prevalence of mental health problems was presented by frequency and percentage. In addition, multivariable regression analysis was used to identify potential predictors of mental health problems. Results: A total of 184 participants (62% female; mean age = 22.49 years (SD 3.93) were recruited. The depressive disorders, adjustment disorders, and anxiety disorders rates were 57.1%, 15.2% and 13.6%, respectively. Grade point averages (GPAs) below 3.0 (OR=3.09, 95%CI: 1.17-8.14) and a family history of mental disorder (OR=3.40, 95%CI: 1.10-10.48) were significant associated factors of moderate to severe mental health problems. Detecting and screening these factors may help the university to provide early detection and treatment for students. Conclusions: Depressive disorders were the most common mental health disorders. Females, low GPAs and a family history of mental disorder were predictors of moderate to severe mental health problems.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-03 16:59:39 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/12-1108/v2?src=rss">A complementary approach to measure construct validity of WHO QOL BREF in Indian multidrug-resistant tuberculosis (MDR-TB) patients through psychometric property and factor analysis [version 2; peer review: 1 approved with reservations]</a></h4>

              
                Background Multidrug-resistant tuberculosis (MDR TB) affects the physical, psychosocial and inter-relationship structure and thus quality of life (QoL) of an individual. WHO QOL BREF is presumed to capture the QoL construct. This study investigates the diagnostic accuracy and construct of WHO QOL BREF from a psychometric perspective and complements and converges findings through classical test theory. Methods The instrument validity study was conducted in a district of Central India amongst the microbiologically confirmed MDR TB cohort of year 2017 (n=98). We calculated global and domain-specific Cronbach alpha and Inter-domain Pearson correlations. The dichotomized items were fitted through Rasch model for item endorsement, response pattern and for variation inconsistencies. Item Characteristic Curves and person item maps were also plotted. We performed DIF (Differential Item Functioning) to check the effect of subgroups on underlying traits. This was complimented with an Exploratory Factor Analysis (EFA) using oblique ProMax rotation. The optimum number of factors were identified by Scree plot and parallel analysis approach and the emerging factor structure was compared with the result obtained through the Rasch model. Results The global Cronbach’s alpha was 0.94 (95% CI 0.92-0.96). Social relationship domain had poor correlations with all three domains (r=0.42, r=0.41 r=0.58), higher beta values and less discrimination. DIF showed a differential response by gender. There was a visual clustering and a non-uniform distribution of items across the perceived QoL. A 3-factor model emerged through EFA and was reframed on the notion of self-concept. Items related to pain, medical aid had significant misfit and weak factor loading while items of sexual activity and social support had relatively poor performance in Infit, Wald, DIF on factor loading parameters. Conclusions The study indicates the possible deviation of scale from theorized dimensional construct in Indian MDR TB context more with the items of the social relationship domain.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-03 16:58:30 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/13-1134/v2?src=rss">Searching for non-English literature may be unnecessary for German HTA Reports [version 2; peer review: 2 approved with reservations, 1 not approved]</a></h4>

              
                Background Health technology assessment (HTA) reports are based on comprehensive information retrieval. Current standards discourage the use of search restrictions, such as publication date and language. Given limited resources, it was unclear whether the effort invested in screening and translating studies published in languages other than English provided relevant additional information compared with the inclusion of English-language publications alone. We therefore analysed the impact of non-English publications on the conclusions of HTA reports produced by the German HTA agency, the Institute for Quality and Efficiency in Health Care (IQWiG). Methods We determined whether non-English publications were included in all German HTA reports on non-drug interventions (published by IQWiG between 06/2007 to 08/2018) and on selected drug interventions. If at least one non-English publication was included, we assessed for each endpoint whether or not the exclusion of non-English publications changed the conclusion. If a non-English publication did not contain information relevant to the HTA report, we classified the publication as “not relevant”. Results Of 70 HTA reports, 38 (54%) included 126 non-English publications. In 4 reports (6%) with 50 endpoints investigated in 39 PICO questions, the exclusion of a total of 10 non-English publications led to a change in the conclusions for 13 endpoints (8 PICO questions). This was largely due to the fact that in many cases, non-English publications were the predominant or only literature available, resulting in a lack of analysable data after their exclusion. Conclusions In general, studies only published in non-English languages have little influence on the conclusions of German HTA reports. For the vast majority of topics, a language restriction to English seems justified. Studies published in non-English languages may be useful in exceptional cases, for example when an intervention is only available in certain countries.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-03 16:55:15 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-240/v2?src=rss">Is relief the most common reaction to abortion? Self-assessed intensity of emotions attributed to abortion in a national sample of women aged 41 to 45 [version 2; peer review: 2 approved]</a></h4>

              
                Background It is widely reported that the most prominent emotion following abortions is relief. This claim primarily rests on two studies of abortion clinic patients which had methodological and self-censure bias. Other studies have indicated that negative emotions are more common than positive emotions. The objective of this study is to obtain self-assessed data on the intensity of emotional responses to abortion and pregnancy loss in a random national sample. Methods Using visual analog scales, a random sample of 1,925 women aged 41 to 45 completed a survey in which respondents rated the degree to which they experienced emotional responses to their first abortion or natural pregnancy loss. The emotions assessed included relief, grief, depression, anxiety, guilt, emptiness, anger, regret, shame, unforgiveness of self, uncontrollable weeping, frequent thoughts of the child they could have had, and difficulty completing the grief process. Women were categorized into five groups based on pregnancy outcomes, and four abortion decision types: Wanted, Inconsistent, Unwanted, or Coerced. Results Among women with a history of abortion (n=409), negative emotions were reported more intensely than relief. Relief was the predominant emotion only among the 29.8% of women whose abortions were freely wanted and consistent with their own values and preferences. For all other groups, relief was low and negative emotions were more prominent. Emotions following natural pregnancy losses were similar to those following abortion, but less severe following wanted abortions and more severe following coerced abortions. Conclusions Relief is only common after freely wanted abortions. Most abortions are inconsistent or contrary to women’s own values. In these cases, strong negative emotions are far more dominant than relief. These results should inform pre-abortion screening, counseling supportive of women’s own values and preferences, and mental health support post-abortion.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-03 16:52:54 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-239/v2?src=rss">A Quasi-Experimental Study Protocol on the Impact of a Mobile Health Program for HIV Counseling and Testing in the Deaf Community [version 2; peer review: 1 approved with reservations]</a></h4>

              
                Background Deaf individuals are considered a high-risk population for health disparities. During the covid-19 epidemic, deaf and hearing loss persons also suffer from psychological issues, post-traumatic stress disorder, and seropositive HIV. Objective This study aims to examine the effectiveness of an mobile health educational program to increase mental health and HIV prevention among deaf community Methods This study employs a quasi-experimental design with a non-randomized controlled trial, involving single-blinded participants and a parallel group assignment, purpose for health service research, study phase 2-3. pronounced to escalate the sample size to 40 deaf per group, which is 80 total participants. Results The analysis of the data will be conducted utilizing the generalized estimation equation, with a confidence interval set at 95%. Significant differences, both between and within groups, will be identified at a threshold of P<.05. The findings of this study highlight the efficacy of a mobile educational program in enhancing mental health and preventing HIV within the deaf community. Furthermore, the outcomes of this research will augment existing knowledge regarding psychological distress, HIV prevention practices, and coping self-efficacy among individuals who are deaf. Conclusion The intervention group is expected to demonstrate significantly lower scores in psychological distress during both the immediate evaluation and the assessment conducted three months post-intervention, compared to the wait-list group. Additionally, the intervention group is anticipated to exhibit enhanced levels of HIV prevention practices and coping self-efficacy, resulting in a greater degree of adjustment. Clinical trial SLCTR/2024/039, 25 November 2024, https://slctr.lk/trials/slctr-2024-039
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-03 16:51:27 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/13-951/v3?src=rss">The effect of surface treatments on the bond strength of polyetheretherketone posts: a systematic review protocol [version 3; peer review: 2 approved with reservations, 1 not approved]</a></h4>

              
                Background Polyetheretherketone (PEEK) is widely used in the biomedical field due to its outstanding biological and mechanical properties. Originally employed as a temporary abutment in implantology, recent research has expanded its indications for more definitive applications, such as frameworks and dental post and core. However, PEEK’s inert nature and low surface energy pose challenges for adhesion, necessitating surface modifications. Various physical and chemical modification techniques, including acid etching (e.g., 98% sulfuric acid), sandblasting with alumina oxide (Al₂O₃), plasma treatment, laser irradiation, silanization, and air abrasion with silica-coated particles, have been proposed to enhance PEEK’s bonding performance. Despite numerous clinical investigations, standardized protocols for surface treatment remain lacking. This systematic review aims to assess the impact of surface treatments on the bonding performance of PEEK posts. Methods A detailed search of the literature will be conducted across several databases including PubMed, Scopus and clinical trial registries. Additional databases such as Cochrane Central, EMBASE, Web of Science and EBSCO will also be included. The search strategy will target controlled randomized studies and non-randomized clinical trials evaluating the impact of surface treatments on PEEK post adhesion strength. The Newcastle-Ottawa Scale (NOS) will be used to assess bias in non-randomized studies, while the Cochrane Risk of Bias (ROB II) tool will be employed for evaluating randomized controlled trials. Data extraction will focus on study design, treatment methods, outcomes and results. This systematic review protocol will adhere to the guidelines for systematic reviews outlined in the Preferred Reporting Items for Systematic Reviews and Meta-Analyses (PRISMA). Discussion The discussion will explore the implications of findings on clinical practice, highlighting the importance of enhancing PEEK’s bioactivity and surface energy to improve bonding efficacy in dental procedures. Moreover, it will suggest areas for future research to advance dental materials science, aiming to optimize the utilization of PEEK in dental applications Systematic review registration PROSPERO: CRD42024529783 (Registered on 08/04/2024).
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-03 16:49:48 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/13-1358/v2?src=rss">Identifying and Assessing Physiological Biomarkers for Food and Energy Intake: A Systematic Review with Meta-Analysis Protocol [version 2; peer review: 1 approved with reservations]</a></h4>

              
                Background Traditional dietary assessments are often inaccurate and prone to self-reporting biases. Tracking the physiological responses associated with eating and digestion events via wearable technologies may provide an effective approach for continuously monitoring food intake and estimating energy consumption. Eating and digestion are accompanied by a series of changes in the heart rate, skin temperature, blood oxygen saturation, and blood pressure. These changes can be tracked by wearable devices, such as smartwatches, which have been widely accepted in the market. This systematic review is the first to evaluate the effectiveness of tracking such physiological biomarkers in differentiating between high- and low-calorie meals, potentially paving the way for more accurate dietary monitoring. Methods Following the PRISMA-P guidelines, we will conduct a systematic literature search through MEDLINE, EMBASE, and PubMed for clinical trials that investigated physiological responses following meal intake in healthy subjects. Two independent reviewers will screen and select articles based on pre-defined eligibility criteria, with a third review to resolve any discrepancies. This will be followed by data extraction and quality assessment of the included studies. Statistical analyses, including meta-analyses, will be performed using R Studio software. Our primary outcome will be the comparison of physiological biomarkers before and after meal intake, while secondary outcomes will include comparisons of physiological biomarkers between high- and low-calorie meal consumption and the correlation between the caloric content of consumed meals and postprandial physiological changes. Discussion This systematic review and meta-analysis will identify physiological indicators for eating events and inform the design of wearable sensors that estimate food intake in healthy subjects. Systematic Review Registration PROSPERO Registration ID: CRD42024544353
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-03 16:47:32 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-133/v2?src=rss">Mortality involving and not involving COVID-19 among vaccinated vs. unvaccinated in England between Apr 21 and May 23 [version 2; peer review: 2 not approved]</a></h4>

              
                Background Comparing non-randomized groups, such as COVID-19 vaccinated and unvaccinated, even in the presence of seemingly relevant control variables, is challenging, but in this study, using English data, I show an achievable approach. Methods First, I estimated age-standardized all-cause mortality among vaccinated and unvaccinated ten years and older, covering 26 months from Apr 21 to May 23. Then, I estimated mortality not involving COVID-19, and finally, I differentiated the calculations. Results First, I found that all-cause mortality among unvaccinated was higher than among vaccinated. But, as the pattern was similar concerning mortality not involving COVID-19, the discrepancy is attributed mainly to unvaccinated having inferior health at the outset. There was nonetheless significant protection for vaccinated between July 21 and Jan 22. Absent of control variables as a means to compare non-randomized groups, I reached that finding by differentiating all-cause mortality from mortality not involving COVID-19. However, while mortality not involving COVID-19 decreased among unvaccinated compared to the first observation month, it was high among vaccinated, i.e., a relative increase in mortality among vaccinated. Conclusions An interpretation is that vaccination, despite temporary protection, increased mortality. Strengthening the interpretation was relatively high mortality among vaccinated not involving COVID-19 counterintuitively following periods of excess mortality. Further strengthening the interpretation was relatively high mortality not involving COVID-19 among vaccinated, corresponding with excess mortality during much of the same period. An implication of the study, which particularly has relevance for future pandemics, is that vaccinated may have a limited time window of protection and can even be exposed to detrimental health consequences. The pattern should be followed up over an extended period in future research. Also, future research should examine different age groups, vaccination types, and the number of doses given.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-03 15:12:51 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-388/v1?src=rss">Campus air quality dataset [version 1; peer review: awaiting peer review]</a></h4>

              
                Background Elevated levels of carbon dioxide (CO2) within academic settings can adversely affect the health and academic efficacy of both students and faculty. High concentrations of CO2 are correlated with reduced cognitive functioning, compromised decision-making abilities, diminished academic performance, and various health-related issues. The escalating apprehensions regarding the detrimental health consequences of air pollution have precipitated an increase in research focused on air quality assessment and amelioration. Method The investigation utilized Internet of Things (IoT) devices that were outfitted with sensors to gather data on various environmental parameters, such as temperature, humidity, CO2 concentrations, and light intensity. This data underwent analysis through the application of summary statistics to delineate the dataset and to visualize the distribution of variables via scatter matrix plots. Result The dataset obtained, which encompasses essential air quality and environmental parameters, is now accessible to the public through the Mendeley repository. The analytical findings illuminated significant characteristics of the data concerning CO2 levels and their prospective ramifications on the academic milieu. Conclusion The amalgamation of IoT technology with summary statistical analysis presents a promising methodology for the real-time surveillance of air quality. This approach yields critical insights into the health and academic ramifications of heightened CO2 levels within educational environments, underscoring the necessity for ongoing air quality monitoring to enhance campus conditions.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-03 14:38:30 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-387/v1?src=rss">The Effects of Information-Motivation-Behavioral Skills Model in Predicting Safer Sexual Dual Protection Behaviors Among Female University Students:  A Cross-sectional Study [version 1; peer review: awaiting peer review]</a></h4>

              
                Background The burden of sexually transmitted infections (STIs) and unintended pregnancy remains a significant public health concern, with young women being disproportionately affected. While both risks can be prevented simultaneously by the same safe sexual practice, such as abstinence or dual protection use, little is known about the patterns and predictors of a range of safer sexual dual protection behaviors among youth based on theoretical models in Ethiopia. This study aimed to examine the effects of information-motivation-behavioral skills (IMB) model in predicting various safer sexual dual protection behaviors, including primary sexual abstinence and dual-protection use among female university students in Ethiopia. Methods A cross-sectional study was conducted among female university students at MattuUniversity. Data were collected using a self-administered questionnaire and analyzed using SPSS version 23. Bivariate and multivariate analyses were conducted using structural equation modeling (SEM) with AMOS 23. Results Of the 1,020 participants, 624 (61.2%) practiced primary sexual abstinence, whereas 396 (38.8%) have ever had sexual intercourse, of whom only 76 (20.5%) used dual protection at their last sexual encounter. Results of the structural equation model analysis indicated that primary sexual abstinence was strongly correlated with motivation (β = 0.34, P < 0.001) and behavioralskills (β = 0.24, P < 0.001) and information (β = 0.11, P < 0.001). Among sexually active participants, dual protection use was strongly correlated with motivation (β = 0.23, P < 0.05) and behavioral skills (β = 0.24, P < 0.001), while it was moderately correlated with information (β = 0.04, P < 0.05). Approximately 28% of the variance in primary sexual abstinence and 27.4% of the variance in dual protection use was explained by the model constructs. Conclusions The findings indicate that the IMB model is useful for identifying powerful predictors of sexual abstinence and dual protection use among female university students, which has potential implications for designing interventions to improve knowledge, motivation, behavioral skills and practice of abstinence and/or dual protection use among youth at risk of both STI/HIV and unwanted pregnancy in settings with high HIV burdens.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-03 14:30:35 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012948">Integrating multiple spatial transcriptomics data using community-enhanced graph contrastive learning</a></h4>

              
                <p>by Wenqian Tu, Lihua Zhang</p>

Due to the rapid development of spatial sequencing technologies, large amounts of spatial transcriptomic datasets have been generated across various technological platforms or different biological conditions (e.g., control vs. treatment). Spatial transcriptomics data coming from different platforms usually has different resolutions. Moreover, current methods do not consider the heterogeneity of spatial structures within and across slices when modeling spatial transcriptomics data with graph-based methods. In this study, we propose a community-enhanced graph contrastive learning-based method named Tacos to integrate multiple spatial transcriptomics data. We applied Tacos to several real datasets coming from different platforms under different scenarios. Systematic benchmark analyses demonstrate Tacos’s superior performance in integrating different slices. Furthermore, Tacos can accurately denoise the spatially resolved transcriptomics data.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-03 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012902">CINner: Modeling and simulation of chromosomal instability in cancer at single-cell resolution</a></h4>

              
                <p>by Khanh N. Dinh, Ignacio Vázquez-García, Andrew Chan, Rhea Malhotra, Adam Weiner, Andrew W. McPherson, Simon Tavaré</p>

Cancer development is characterized by chromosomal instability, manifesting in frequent occurrences of different genomic alteration mechanisms ranging in extent and impact. Mathematical modeling can help evaluate the role of each mutational process during tumor progression, however existing frameworks can only capture certain aspects of chromosomal instability (CIN). We present CINner, a mathematical framework for modeling genomic diversity and selection during tumor evolution. The main advantage of CINner is its flexibility to incorporate many genomic events that directly impact cellular fitness, from driver gene mutations to copy number alterations (CNAs), including focal amplifications and deletions, missegregations and whole-genome duplication (WGD). We apply CINner to find chromosome-arm selection parameters that drive tumorigenesis in the absence of WGD in chromosomally stable cancer types from the Pan-Cancer Analysis of Whole Genomes (PCAWG, n=718 ). We found that the selection parameters predict WGD prevalence among different chromosomally unstable tumors, hinting that the selective advantage of WGD cells hinges on their tolerance for aneuploidy and escape from nullisomy. Analysis of inference results using CINner across cancer types in The Cancer Genome Atlas (n=8207) further reveals that the inferred selection parameters reflect the bias between tumor suppressor genes and oncogenes on specific genomic regions. Direct application of CINner to model the WGD proportion and fraction of genome altered (FGA) in PCAWG uncovers the increase in CNA probabilities associated with WGD in each cancer type. CINner can also be utilized to study chromosomally stable cancer types, by applying a selection model based on driver gene mutations and focal amplifications or deletions (chronic lymphocytic leukemia in PCAWG, n=95). Finally, we used CINner to analyze the impact of CNA probabilities, chromosome selection parameters, tumor growth dynamics and population size on cancer fitness and heterogeneity. We expect that CINner will provide a powerful modeling tool for the oncology community to quantify the impact of newly uncovered genomic alteration mechanisms on shaping tumor progression and adaptation.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-03 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012901">Playbook workflow builder: Interactive construction of bioinformatics workflows</a></h4>

              
                <p>by Daniel J.B. Clarke, John Erol Evangelista, Zhuorui Xie, Giacomo B. Marino, Anna I. Byrd, Mano R. Maurya, Sumana Srinivasan, Keyang Yu, Varduhi Petrosyan, Matthew E. Roth, Miroslav Milinkov, Charles Hadley King, Jeet Kiran Vora, Jonathon Keeney, Christopher Nemarich, William Khan, Alexander Lachmann, Nasheath Ahmed, Alexandra Agris, Juncheng Pan, Srinivasan Ramachandran, Eoin Fahy, Emmanuel Esquivel, Aleksandar Mihajlovic, Bosko Jevtic, Vuk Milinovic, Sean Kim, Patrick McNeely, Tianyi Wang, Eric Wenger, Miguel A. Brown, Alexander Sickler, Yuankun Zhu, Sherry L. Jenkins, Philip D. Blood, Deanne M. Taylor, Adam C. Resnick, Raja Mazumder, Aleksandar Milosavljevic, Shankar Subramaniam, Avi Ma’ayan</p>

The Playbook Workflow Builder (PWB) is a web-based platform to dynamically construct and execute bioinformatics workflows by utilizing a growing network of input datasets, semantically annotated API endpoints, and data visualization tools contributed by an ecosystem of collaborators. Via a user-friendly user interface, workflows can be constructed from contributed building-blocks without technical expertise. The output of each step of the workflow is added into reports containing textual descriptions, figures, tables, and references. To construct workflows, users can click on cards that represent each step in a workflow, or construct workflows via a chat interface that is assisted by a large language model (LLM). Completed workflows are compatible with Common Workflow Language (CWL) and can be published as research publications, slideshows, and posters. To demonstrate how the PWB generates meaningful hypotheses that draw knowledge from across multiple resources, we present several use cases. For example, one of these use cases prioritizes drug targets for individual cancer patients using data from the NIH Common Fund programs GTEx, LINCS, Metabolomics, GlyGen, and ExRNA. The workflows created with PWB can be repurposed to tackle similar use cases using different inputs. The PWB platform is available from: https://playbook-workflow-builder.cloud/.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-03 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012881">SpaMask: Dual masking graph autoencoder with contrastive learning for spatial transcriptomics</a></h4>

              
                <p>by Wenwen Min, Donghai Fang, Jinyu Chen, Shihua Zhang</p>

Understanding the spatial locations of cell within tissues is crucial for unraveling the organization of cellular diversity. Recent advancements in spatial resolved transcriptomics (SRT) have enabled the analysis of gene expression while preserving the spatial context within tissues. Spatial domain characterization is a critical first step in SRT data analysis, providing the foundation for subsequent analyses and insights into biological implications. Graph neural networks (GNNs) have emerged as a common tool for addressing this challenge due to the structural nature of SRT data. However, current graph-based deep learning approaches often overlook the instability caused by the high sparsity of SRT data. Masking mechanisms, as an effective self-supervised learning strategy, can enhance the robustness of these models. To this end, we propose SpaMask, dual masking graph autoencoder with contrastive learning for SRT analysis. Unlike previous GNNs, SpaMask masks a portion of spot nodes and spot-to-spot edges to enhance its performance and robustness. SpaMask combines Masked Graph Autoencoders (MGAE) and Masked Graph Contrastive Learning (MGCL) modules, with MGAE using node masking to leverage spatial neighbors for improved clustering accuracy, while MGCL applies edge masking to create a contrastive loss framework that tightens embeddings of adjacent nodes based on spatial proximity and feature similarity. We conducted a comprehensive evaluation of SpaMask on eight datasets from five different platforms. Compared to existing methods, SpaMask achieves superior clustering accuracy and effective batch correction.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-03 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012878">Predicting and comparing transcription start sites in single cell populations</a></h4>

              
                <p>by Shiwei Fu, Wei Vivian Li</p>

The advent of 5’ single-cell RNA sequencing (scRNA-seq) technologies offers unique opportunities to identify and analyze transcription start sites (TSSs) at a single-cell resolution. These technologies have the potential to uncover the complexities of transcription initiation and alternative TSS usage across different cell types and conditions. Despite the emergence of computational methods designed to analyze 5’ RNA sequencing data, current methods often lack comparative evaluations in single-cell contexts and are predominantly tailored for paired-end data, neglecting the potential of single-end data. This study introduces scTSS, a computational pipeline developed to bridge this gap by accommodating both paired-end and single-end 5’ scRNA-seq data. scTSS enables joint analysis of multiple single-cell samples, starting with TSS cluster prediction and quantification, followed by differential TSS usage analysis. It employs a Binomial generalized linear mixed model to accurately and efficiently detect differential TSS usage. We demonstrate the utility of scTSS through its application in analyzing transcriptional initiation from single-cell data of two distinct diseases. The results illustrate scTSS’s ability to discern alternative TSS usage between different cell types or biological conditions and to identify cell subpopulations characterized by unique TSS-level expression profiles.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-03 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012861">Spatial structure facilitates evolutionary rescue by drug resistance</a></h4>

              
                <p>by Cecilia Fruet, Ella Linxia Müller, Claude Loverdo, Anne-Florence Bitbol</p>

Bacterial populations often have complex spatial structures, which can impact their evolution. Here, we study how spatial structure affects the evolution of antibiotic resistance in a bacterial population. We consider a minimal model of spatially structured populations where all demes (i.e., subpopulations) are identical and connected to each other by identical migration rates. We show that spatial structure can facilitate the survival of a bacterial population to antibiotic treatment, starting from a sensitive inoculum. Specifically, the bacterial population can be rescued if antibiotic resistant mutants appear and are present when drug is added, and spatial structure can impact the fate of these mutants and the probability that they are present. Indeed, the probability of fixation of neutral or deleterious mutations providing drug resistance is increased in smaller populations. This promotes local fixation of resistant mutants in the structured population, which facilitates evolutionary rescue by drug resistance in the rare mutation regime. Once the population is rescued by resistance, migrations allow resistant mutants to spread in all demes. Our main result that spatial structure facilitates evolutionary rescue by antibiotic resistance extends to more complex spatial structures, and to the case where there are resistant mutants in the inoculum.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-03 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/plosbiology/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/plosbiology/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/plosbiology/NewArticles&title=PLoS Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3003102">The end of long-term ecological data?</a></h4>

              
                <p>by Daniel T. Blumstein</p>

Can we ever have too much ecological field data? Are data-sharing norms and the environmental costs of travel disincentivizing its collection? Allocating proper funding and resources to the collection of long-term ecological data is essential for studies of behavior and adaptation, which are particularly important in the face of anthropogenic change.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/plosbiology/NewArticles">PLoS Biology</a> on 2025-04-03 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/plosbiology/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/plosbiology/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/plosbiology/NewArticles&title=PLoS Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3003100">Gas1-high quiescent neural stem cells are multipotent and produce oligodendrocytes during aging and after demyelinating injury</a></h4>

              
                <p>by Chaoqiong Ding, Zhenzhong Pan, Xiang Yan, Ran Zhou, Huifang Li, Lu Chen, Yuan Wang, Yan Zhang</p>

Quiescent neural stem cells (qNSCs) in the adult mouse subventricular zone (SVZ) normally have limited capacity to generate glia. Gliogenic domains are present in both dorsal and ventral SVZ, with the ventral region featuring a subpopulation of Gli1<sup>+</sup> qNSCs. In dorsal SVZ, however, the molecular identity and developmental origin of oligodendrogenic qNSCs remains elusive. Here, through single-cell analysis and lineage tracing, we identify an undefined subpopulation of Gas1<sup>high</sup> qNSCs in dorsal SVZ, distinct from Gli1<sup>+</sup> qNSCs. These cells originate from embryonic Gas1<sup>high</sup> dorsal radial glia, and persist into the aged SVZ. Remarkably, they are multipotent and more gliogenic than Gas1<sup>low/−</sup> qNSCs, continuously generating oligodendrocytes in the adult and aged brain, and can be mobilized for myelin repair upon demyelination. Together, our study uncovers a subpopulation of dorsally derived, multipotent long-term qNSCs in the adult and aged SVZ with enhanced gliogenic potential, shedding light on the heterogeneity and plasticity of NSCs in normal, aging, and disease conditions.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/plosbiology/NewArticles">PLoS Biology</a> on 2025-04-03 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://onlinelibrary.wiley.com/action/showFeed?jc=10974547&type=etoc&feed=rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://onlinelibrary.wiley.com/action/showFeed?jc=10974547&type=etoc&feed=rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://onlinelibrary.wiley.com/action/showFeed?jc=10974547&type=etoc&feed=rss&title=Journal of Neuroscience Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://onlinelibrary.wiley.com/doi/10.1002/jnr.70034?af=R">MRI‐T2 Relaxometry is Increased in Mild Traumatic Brain Injury: Indications of Acute Brain Abnormalities After Injury</a></h4>

              
                <img src="https://onlinelibrary.wiley.com/cms/asset/9558d5df-8b54-46f0-9858-263cc4da3ac8/jnr70034-toc-0001-m.png"
     alt="MRI-T2 Relaxometry is Increased in Mild Traumatic Brain Injury: Indications of Acute Brain Abnormalities After Injury"/>
<p>Central nervous system disorders impair information transmission by causing synapse loss and dysfunction. Synaptic plasticity, involving the dynamic strengthening or weakening of synapses, enables brain adaptation. Regulating this process can alleviate disease progression and support synaptic recovery, introducing innovative treatment strategies that restore neural connectivity and improve overall brain function.</p>
<br/>
<h2>ABSTRACT</h2>
<p>Mild traumatic brain injury (mTBI) is a common condition, particularly pervasive in contact sports environments. A range of symptoms can accompany this type of injury and negatively impact people's lives. As mTBI diagnosis and recovery largely rely on subjective reports, more objective injury markers are needed. The current study compared structural brain MRI-T2 relaxometry between a group of 40 male athletes with mTBI within 14 days of injury and 40 age-matched male controls. Voxel-averaged T2 relaxometry within the gray matter was increased for the mTBI group compared to controls (<i>p</i> &lt; 0.001), with statistically significant increased T2 relaxometry particularly in superior cortical regions. Our findings indicate subtle brain abnormalities can be identified in acute mTBI using MRI-T2 relaxometry. These brain abnormalities may reflect inflammation present in the brain and could constitute an objective injury marker to supplement current subjective methods that dominate clinical decisions regarding diagnosis and prognosis. Future research should validate this potential marker with other data types, such as blood biomarkers or histological samples.</p>
              

              <p class="text-muted">in <a href ="https://onlinelibrary.wiley.com/action/showFeed?jc=10974547&type=etoc&feed=rss">Journal of Neuroscience Research</a> on 2025-04-03 13:05:56 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00199.2024?af=R">The influence of postural threat-induced anxiety on locomotor learning and updating: a mini-review</a></h4>

              
                Journal of Neurophysiology, <a href="https://journals.physiology.org/toc/jn/133/4">Volume 133, Issue 4</a>, Page 1205-1207, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-03 11:45:26 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00268.2024?af=R">Visual properties of action consequences and not environments affect context-specific motor learning</a></h4>

              
                Journal of Neurophysiology, <a href="https://journals.physiology.org/toc/jn/133/4">Volume 133, Issue 4</a>, Page 1176-1190, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-03 11:45:25 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00093.2025?af=R">Reply to Castro and Carvalho</a></h4>

              
                Journal of Neurophysiology, <a href="https://journals.physiology.org/toc/jn/133/4">Volume 133, Issue 4</a>, Page 1148-1149, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-03 11:45:22 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00308.2024?af=R">Postural strategy for two potential targets considering motor costs for postural stabilization and probabilistic information</a></h4>

              
                Journal of Neurophysiology, <a href="https://journals.physiology.org/toc/jn/133/4">Volume 133, Issue 4</a>, Page 1191-1204, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-03 11:45:21 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00561.2024?af=R">String-pulling by the common marmoset</a></h4>

              
                Journal of Neurophysiology, <a href="https://journals.physiology.org/toc/jn/133/4">Volume 133, Issue 4</a>, Page 1222-1233, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-03 11:45:20 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00041.2024?af=R">Choice anticipation as gated accumulation of sensory predictions</a></h4>

              
                Journal of Neurophysiology, <a href="https://journals.physiology.org/toc/jn/133/4">Volume 133, Issue 4</a>, Page 1159-1175, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-03 11:45:19 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00206.2024?af=R">Changes in gait asymmetry may be caused by adaptation of spinal reflexes</a></h4>

              
                Journal of Neurophysiology, <a href="https://journals.physiology.org/toc/jn/133/4">Volume 133, Issue 4</a>, Page 1208-1215, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-03 11:45:18 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00039.2025?af=R">Subcortical responses to continuous speech under bimodal divided attention</a></h4>

              
                Journal of Neurophysiology, <a href="https://journals.physiology.org/toc/jn/133/4">Volume 133, Issue 4</a>, Page 1216-1221, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-03 11:45:18 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00607.2024?af=R">Microbiome’s effect on white matter in autism</a></h4>

              
                Journal of Neurophysiology, <a href="https://journals.physiology.org/toc/jn/133/4">Volume 133, Issue 4</a>, Page 1150-1158, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-03 11:45:17 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00582.2024?af=R">Synaptic dynamics linked to widespread elevation of H-reflex before peripheral denervation in amyotrophic lateral sclerosis</a></h4>

              
                Journal of Neurophysiology, <a href="https://journals.physiology.org/toc/jn/133/4">Volume 133, Issue 4</a>, Page 1146-1147, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-03 11:45:15 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml&title=Physical Review E: Biological physics" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="http://link.aps.org/doi/10.1103/PhysRevE.111.044403">Migration feedback induces emergent ecotypes and abrupt transitions in evolving populations</a></h4>

              
                <p>Author(s): Casey O. Barkan and Shenshen Wang</p><p>We explore the connection between migration patterns and emergent behaviors of evolving populations in spatially heterogeneous environments. Despite extensive studies in systems of ecological and medical importance, a unifying framework that clarifies this connection and makes concrete predictions r…</p><br/><p>[Phys. Rev. E 111, 044403] Published Thu Apr 03, 2025</p>
              

              <p class="text-muted">in <a href ="http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml">Physical Review E: Biological physics</a> on 2025-04-03 10:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro&title=PNAS" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.pnas.org/doi/abs/10.1073/pnas.2417117122?af=R">Critical dynamics predicts cognitive performance and provides a common framework for heterogeneous mechanisms impacting cognition</a></h4>

              
                Proceedings of the National Academy of Sciences, Volume 122, Issue 14, April 2025. <br/>SignificanceCognitive function emerges from cortical network dynamics and is often impaired in neuropsychiatric disorders like epilepsy. Physics and information theory suggest that brain networks operate optimally at a critical state between order and ...
              

              <p class="text-muted">in <a href ="https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro">PNAS</a> on 2025-04-03 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro&title=PNAS" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.pnas.org/doi/abs/10.1073/pnas.2422709122?af=R">An unusual potassium conductance protects Caenorhabditis elegans pharyngeal muscle rhythms against environmental noise</a></h4>

              
                Proceedings of the National Academy of Sciences, Volume 122, Issue 14, April 2025. <br/>SignificanceWe present evidence from computational modeling that an unusual K+conductance endows the pharyngeal feeding organ of the nematodeCaenorhabditis eleganswith critical robustness against environmental noise that could otherwise disrupt its ...
              

              <p class="text-muted">in <a href ="https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro">PNAS</a> on 2025-04-03 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adx8687?af=R">Erratum for the Research Article “Whale song shows language-like statistical structure” by I. Arnon et al.</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/388/6742">Volume 388, Issue 6742</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-04-03 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adw9439?af=R">Beyond the binary: Navigating AI’s uncertain future in Africa</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/388/6742">Volume 388, Issue 6742</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-04-03 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adw0718?af=R">When trainees seek other paths</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/388/6742">Volume 388, Issue 6742</a>, Page 39-39, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-04-03 05:59:02 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adw2209?af=R">Convince me, control me</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/388/6742">Volume 388, Issue 6742</a>, Page 38-38, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-04-03 05:59:02 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adp5011?af=R">Vaccine-enhanced competition permits rational bacterial strain replacement in the gut</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/388/6742">Volume 388, Issue 6742</a>, Page 74-81, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-04-03 05:59:02 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adq6531?af=R">Meningeal regulatory T cells inhibit nociception in female mice</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/388/6742">Volume 388, Issue 6742</a>, Page 96-104, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-04-03 05:59:02 UTC.
              </p>
            </li>
          <!-- each item -->
        </ul>
      </div>
      <div class="col-md-3">
        <div class="panel panel-default">
          <div class="panel-heading">
            <h3 class="panel-title">Feed list</h3>
          </div>
          <ul class="list-group">
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/computational-neuroscience'>Frontiers in Computational Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/aging-neuroscience'>Frontiers in Ageing Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/evolutionary-neuroscience/rss'>Frontiers in Evolutionary Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/human-neuroscience/rss'>Frontiers in Human Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/molecular-neuroscience/rss'>Frontiers in Molecular Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/neuroenergetics/rss'>Frontiers in Neurogenetics</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/neuroengineering/rss'>Frontiers in Neuroengineering</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/systems-neuroscience/rss'>Frontiers in Systems Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/synaptic-neuroscience/rss'>Frontiers in Synaptic Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/cellular-neuroscience/rss'>Frontiers in Cellular Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/integrative-neuroscience/rss'>Frontiers in Integrative Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/neural-circuits/rss'>Frontiers in Neural Circuits</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/neuroanatomy/rss'>Frontiers in Neuroanatomy</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/neuroinformatics/rss'>Frontiers in Neuroinformatics</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://www.cell.com/trends/neurosciences/inpress.rss'>Trends in Neurosciences: In press</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://www.cell.com/neuron/current.rss'>Neuron: Current Issue</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://www.cell.com/neuron/inpress.rss'>Neuron: In press</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.cell.com/cell-reports/current.rss'>Cell Reports: Current Issue</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.cell.com/cell-reports/inpress.rss'>Cell Reports: In press</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.plos.org/ploscompbiol/NewArticles'>PLoS Computational Biology</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.plos.org/plosbiology/NewArticles'>PLoS Biology</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/nature/rss/current'>Nature</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/neuro/rss/current'>Nature Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/nrn/rss/current'>Nature Reviews</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/nphoton/rss/current'>Nature Photomics</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/nmeth/rss/current'>Nature Methods</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/ncomms/rss/current'>Nature Communications</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/nphys/rss/current'>Nature Physics</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.nature.com/sdata.rss'>Nature scientific data</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.nature.com/commsbio.rss'>Nature communications biology</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://elifesciences.org/rss/recent.xml'>eLife</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://onlinelibrary.wiley.com/action/showFeed?jc=15318249&type=etoc&feed=rss'>Annals of Neurology</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://onlinelibrary.wiley.com/feed/10969861/most-recent'>Journal of Comparative Neurology</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://onlinelibrary.wiley.com/action/showFeed?jc=10974547&type=etoc&feed=rss'>Journal of Neuroscience Research</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.jneurosci.org/rss/current.xml'>Journal of Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://bmcneurosci.biomedcentral.com/articles/most-recent/rss.xml'>BMC Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro'>PNAS</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://direct.mit.edu/neco'>Neural Computation</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://direct.mit.edu/netn'>Network Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience'>bioRxiv: Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://export.arxiv.org/rss/q-bio.NC/'>arXiv: Quantitative Biology: Neurons and Cognition</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://export.arxiv.org/rss/cs.NE'>arXiv: Computer Science: Neural and Evolutionary Computing</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://rss.sciencedirect.com/publication/science/09594388'>Current Opinion in Neurobiology</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://rss.sciencedirect.com/publication/science/08936080'>Neural Networks</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://academic.oup.com/rss/site_5122/3011.xml'>Cerebral Cortex</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.eneuro.org/rss/current.xml'>eNeuro</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn'>Journal of Neurophysiology</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science'>Science</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv'>Science Advances</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://onlinelibrary.wiley.com/feed/10981063/most-recent'>Hippocampus</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://rescience.github.io/feed'>ReScience C</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml'>Physical Review E: Biological physics</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://mathematical-neuroscience.springeropen.com/articles/most-recent/rss.xml'>The Journal of Mathematical Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://behavioralandbrainfunctions.biomedcentral.com/articles/most-recent/rss.xml'>Behavioural and Brain Functions</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.mdpi.com/rss/journal/brainsci'>Brain Sciences</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://f1000research.com/published/rss'>F1000Research</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://journals.sagepub.com/action/showFeed?ui=0&mi=ehikzz&ai=2b4&jc=nroa&type=etoc&feed=rss'>The Neuroscientist</a>
            </li>
           
           </ul>
        </div>
      </div>
    </div>
    <footer id="site-footer">
      <p class="text-center text-muted">All content on this page is owned by their respective owners. The source code used to generate this page can be found <a href="https://github.com/neurofedora/planet-neuroscience">here</a>.</p>
      </div>
    </footer>
    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <script type="text/javascript">
      $( document ).ready(function(){
        $("img").addClass("img-responsive");
        $("figure").addClass("img-responsive");
      });
    </script>
    <!-- Include all compiled plugins (below), or include individual files as needed -->
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
  </body>
</html>

