<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
  <head>
    <title>Planet Neuroscience</title>
    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet"> 
    <link href="https://fonts.googleapis.com/css?family=Comfortaa" rel="stylesheet">
    <!-- Google analytics -->
    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-60261100-2', 'auto');
      ga('send', 'pageview');

    </script>
  </head>

  <body>
    <nav class="navbar navbar-default navbar-static-top">
      <div class="container">
        <div class="navbar-header">
          <a class="navbar-brand" href="#">Planet Neuroscience</a>
        </div>
          <ul class="nav navbar-nav navbar-right">
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">More Neuroscience <span class="caret"></span></a>
              <ul class="dropdown-menu">
                <li><a href="http://neuroblog.fedoraproject.org/planet-neuroscientists/">Planet Neuroscientists</a></li>
                <li><a href="http://home.earthlink.net/~perlewitz/index.html">Computational Neuroscience on the web</a></li>
              </ul>
            </li>
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Options <span class="caret"></span></a>
              <ul class="dropdown-menu">
                <li><a href="https://github.com/neurofedora/planet-neuroscience/issues/new">Suggest a new feed</a></li>
                <li><a href="https://github.com/neurofedora/planet-neuroscience">View Planet source</a></li>
                <li role="separator" class="divider"></li>
                <li><a href="https://github.com/feedreader/pluto">View Pluto source</a></li>
              </ul>
            </li>
          </ul>
        </div><!--/.nav-collapse -->
      </div>
    </nav>
    <div class="container">
      <div class="page-header">
        <h1>Planet Neuroscience</h1>
        <h2>An aggregation of RSS feeds from various neuroscience journals.</h2>
        <p>
        last updated by <a href="https://feedreader.github.io">Pluto</a>
        on 2025-03-25 08:17:55 UTC 
        on behalf of <a href="https://neuro.fedoraproject.org" target="_blank"> the NeuroFedora SIG</a>.
        </p>
      </div>
    </div>

    <div class="container">
      <div class="col-md-9">
        <ul class="list-group">
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://onlinelibrary.wiley.com/feed/10981063/most-recent>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://onlinelibrary.wiley.com/feed/10981063/most-recent" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://onlinelibrary.wiley.com/feed/10981063/most-recent&title=Hippocampus" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://onlinelibrary.wiley.com/doi/10.1002/hipo.70010?af=R">Associations Between Episodic Memory and Hippocampal Volume in Late Adulthood</a></h4>

              
                <h2>ABSTRACT</h2>
<p>Different tasks of episodic memory (EM) are only moderately correlated with each other. Furthermore, various EM tasks exhibit disproportional relationships with the hippocampus. This study examined the covariance structure of EM tasks and assessed whether this structure relates differently to hippocampal volume (HV) in a sample of 648 cognitively unimpaired older adults (mean age = 69.88). A confirmatory factor analysis (CFA) and linear regression models were used to test the associations between the observed factors of EM and HV. A model with three first-order subfactors (immediate verbal recall, delayed verbal recall, and visuospatial) derived from a second-order EM domain factor satisfied model fit (<i>χ</i>
<sup>2</sup>
<i>p</i> value ≥ 0.05, CFI &gt; 0.90, RMSEA &lt; 0.08, SRMR &lt; 0.08). Total, left, and right HV explained a similar amount of variance in all EM subfactors. CA1, CA3, subiculum, and entorhinal cortex volume were associated with all subfactors, while CA2 and dentate gyrus volume were not associated with EM. These results suggest that EM tasks are measuring the same construct, but different complex processes contribute to EM. Furthermore, HV accounted for a small portion of the variance in EM, suggesting that HV might not be a useful marker of EM in cognitively unimpaired older adults. Finally, this study provides evidence that various hippocampal subfield volumes may not be purely associated with any one aspect of EM processing.</p>
              

              <p class="text-muted">in <a href ="https://onlinelibrary.wiley.com/feed/10981063/most-recent">Hippocampus</a> on 2025-03-25 05:59:53 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/q-bio.NC/>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/q-bio.NC/" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/q-bio.NC/&title=arXiv: Quantitative Biology: Neurons and Cognition" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.18754">Dynamically Learning to Integrate in Recurrent Neural Networks</a></h4>

              
                arXiv:2503.18754v1 Announce Type: new 
Abstract: Learning to remember over long timescales is fundamentally challenging for recurrent neural networks (RNNs). While much prior work has explored why RNNs struggle to learn long timescales and how to mitigate this, we still lack a clear understanding of the dynamics involved when RNNs learn long timescales via gradient descent. Here we build a mathematical theory of the learning dynamics of linear RNNs trained to integrate white noise. We show that when the initial recurrent weights are small, the dynamics of learning are described by a low-dimensional system that tracks a single outlier eigenvalue of the recurrent weights. This reveals the precise manner in which the long timescale associated with white noise integration is learned. We extend our analyses to RNNs learning a damped oscillatory filter, and find rich dynamical equations for the evolution of a conjugate pair of outlier eigenvalues. Taken together, our analyses build a rich mathematical framework for studying dynamical learning problems salient for both machine learning and neuroscience.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/q-bio.NC/">arXiv: Quantitative Biology: Neurons and Cognition</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/q-bio.NC/>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/q-bio.NC/" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/q-bio.NC/&title=arXiv: Quantitative Biology: Neurons and Cognition" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.18804">Vision: looking and seeing through our brain's information bottleneck</a></h4>

              
                arXiv:2503.18804v1 Announce Type: new 
Abstract: Our brain recognizes only a tiny fraction of sensory input, due to an information processing bottleneck. This blinds us to most visual inputs. Since we are blind to this blindness, only a recent framework highlights this bottleneck by formulating vision as mainly looking and seeing. Looking selects a tiny fraction of visual information for progression through the bottleneck, mainly by shifting gaze to center an attentional spotlight. Seeing decodes, i.e., recognizes, objects within the selected information. Since looking often occurs before seeing and evokes limited awareness, humans have the impression of seeing whole scenes clearly. According to the new framework, the bottleneck starts from the output of the primary visual cortex (V1) to downstream brain areas. This is motivated by the evidence-backed V1 Saliency Hypothesis (V1SH) that V1 creates a saliency map of the visual field to guide looking. Massive visual information loss downstream from V1 makes seeing vulnerable to ambiguity and illusions (errors). To overcome this, feedback from downstream to upstream areas such as V1 queries for additional relevant information. An integral part of this framework is the central-peripheral dichotomy (CPD) theory proposing that vision in the peripheral and central visual fields are specialized for looking (deciding where to shift the gaze) and seeing, respectively, and that the feedback query to aid seeing is mainly directed to the central visual field. This V1SH-Bottleneck-CPD framework predicts that the peripheral visual field, lacking feedback queries, is more vulnerable to illusions, and that such illusions become visible in the central visual field when the feedback query is compromised. We present theoretical predictions, experimental confirmations, a Feedforward-Feedback-Verify-and-reWeight (FFVW) algorithm for seeing through the bottleneck.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/q-bio.NC/">arXiv: Quantitative Biology: Neurons and Cognition</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/q-bio.NC/>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/q-bio.NC/" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/q-bio.NC/&title=arXiv: Quantitative Biology: Neurons and Cognition" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.17546">Communities in the Kuramoto Model: Dynamics and Detection via Path Signatures</a></h4>

              
                arXiv:2503.17546v1 Announce Type: cross 
Abstract: The behavior of multivariate dynamical processes is often governed by underlying structural connections that relate the components of the system. For example, brain activity which is often measured via time series is determined by an underlying structural graph, where nodes represent neurons or brain regions and edges represent cortical connectivity. Existing methods for inferring structural connections from observed dynamics, such as correlation-based or spectral techniques, may fail to fully capture complex relationships in high-dimensional time series in an interpretable way. Here, we propose the use of path signatures a mathematical framework that encodes geometric and temporal properties of continuous paths to address this problem. Path signatures provide a reparametrization-invariant characterization of dynamical data and, in particular, can be used to compute the lead matrix which reveals lead-lag phenomena. We showcase our approach on time series from coupled oscillators in the Kuramoto model defined on a stochastic block model graph, termed the Kuramoto stochastic block model (KSBM). Using mean-field theory and Gaussian approximations, we analytically derive reduced models of KSBM dynamics in different temporal regimes and theoretically characterize the lead matrix in these settings. Leveraging these insights, we propose a novel signature-based community detection algorithm, achieving exact recovery of structural communities from observed time series in multiple KSBM instances. Our results demonstrate that path signatures provide a novel perspective on analyzing complex neural data and other high-dimensional systems, explicitly exploiting temporal functional relationships to infer underlying structure.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/q-bio.NC/">arXiv: Quantitative Biology: Neurons and Cognition</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/q-bio.NC/>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/q-bio.NC/" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/q-bio.NC/&title=arXiv: Quantitative Biology: Neurons and Cognition" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.18223">MammAlps: A multi-view video behavior monitoring dataset of wild mammals in the Swiss Alps</a></h4>

              
                arXiv:2503.18223v1 Announce Type: cross 
Abstract: Monitoring wildlife is essential for ecology and ethology, especially in light of the increasing human impact on ecosystems. Camera traps have emerged as habitat-centric sensors enabling the study of wildlife populations at scale with minimal disturbance. However, the lack of annotated video datasets limits the development of powerful video understanding models needed to process the vast amount of fieldwork data collected. To advance research in wild animal behavior monitoring we present MammAlps, a multimodal and multi-view dataset of wildlife behavior monitoring from 9 camera-traps in the Swiss National Park. MammAlps contains over 14 hours of video with audio, 2D segmentation maps and 8.5 hours of individual tracks densely labeled for species and behavior. Based on 6135 single animal clips, we propose the first hierarchical and multimodal animal behavior recognition benchmark using audio, video and reference scene segmentation maps as inputs. Furthermore, we also propose a second ecology-oriented benchmark aiming at identifying activities, species, number of individuals and meteorological conditions from 397 multi-view and long-term ecological events, including false positive triggers. We advocate that both tasks are complementary and contribute to bridging the gap between machine learning and ecology. Code and data are available at: https://github.com/eceo-epfl/MammAlps
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/q-bio.NC/">arXiv: Quantitative Biology: Neurons and Cognition</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/q-bio.NC/>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/q-bio.NC/" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/q-bio.NC/&title=arXiv: Quantitative Biology: Neurons and Cognition" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.18619">Reading Decisions from Gaze Direction during Graphics Turing Test of Gait Animation</a></h4>

              
                arXiv:2503.18619v1 Announce Type: cross 
Abstract: We investigated gaze direction during movement observation. The eye movement data were collected during an experiment, in which different models of movement production (based on movement primitives, MPs) were compared in a two alternatives forced choice task (2AFC).
  Participants observed side-by-side presentation of two naturalistic 3D-rendered human movement videos, where one video was based on motion captured gait sequence, the other one was generated by recombining the machine-learned MPs to approximate the same movement. The task was to discriminate between these movements while their eye movements were recorded. We are complementing previous binary decision data analyses with eye tracking data. Here, we are investigating the role of gaze direction during task execution. We computed the shared information between gaze features and decisions of the participants, and between gaze features and correct answers.
  We found that eye movements reflect the decision of participants during the 2AFC task, but not the correct answer. This result is important for future experiments, which should take advantage of eye tracking to complement binary decision data.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/q-bio.NC/">arXiv: Quantitative Biology: Neurons and Cognition</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/q-bio.NC/>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/q-bio.NC/" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/q-bio.NC/&title=arXiv: Quantitative Biology: Neurons and Cognition" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2411.15234">Adaptive Intelligence: leveraging insights from adaptive behavior in animals to build flexible AI systems</a></h4>

              
                arXiv:2411.15234v3 Announce Type: replace 
Abstract: Biological intelligence is inherently adaptive -- animals continually adjust their actions based on environmental feedback. However, creating adaptive artificial intelligence (AI) remains a major challenge. The next frontier is to go beyond traditional AI to develop "adaptive intelligence," defined here as harnessing insights from biological intelligence to build agents that can learn online, generalize, and rapidly adapt to changes in their environment. Recent advances in neuroscience offer inspiration through studies that increasingly focus on how animals naturally learn and adapt their world models. In this Perspective, I will review the behavioral and neural foundations of adaptive biological intelligence, the parallel progress in AI, and explore brain-inspired approaches for building more adaptive algorithms.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/q-bio.NC/">arXiv: Quantitative Biology: Neurons and Cognition</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/q-bio.NC/>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/q-bio.NC/" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/q-bio.NC/&title=arXiv: Quantitative Biology: Neurons and Cognition" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2501.07696">Inferring collective synchrony observing spiking of one or several neurons</a></h4>

              
                arXiv:2501.07696v2 Announce Type: replace 
Abstract: We tackle a quantification of synchrony in a large ensemble of interacting neurons from the observation of spiking events. In a simulation study, we efficiently infer the synchrony level in a neuronal population from a point process reflecting spiking of a small number of units and even from a single neuron. We introduce a synchrony measure (order parameter) based on the Bartlett covariance density; this quantity can be easily computed from the recorded point process. This measure is robust concerning missed spikes and, if computed from observing several neurons, does not require spike sorting. We illustrate the approach by modeling populations of spiking or bursting neurons, including the case of sparse synchrony.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/q-bio.NC/">arXiv: Quantitative Biology: Neurons and Cognition</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/q-bio.NC/>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/q-bio.NC/" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/q-bio.NC/&title=arXiv: Quantitative Biology: Neurons and Cognition" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.12334">When neural implant meets multimodal LLM: A dual-loop system for neuromodulation and naturalistic neuralbehavioral research</a></h4>

              
                arXiv:2503.12334v2 Announce Type: replace 
Abstract: We propose a novel dual-loop system that synergistically combines responsive neurostimulation (RNS) implants with artificial intelligence-driven wearable devices for treating post-traumatic stress disorder (PTSD) and enabling naturalistic brain research. In PTSD Therapy Mode, an implanted closed-loop neural device monitors amygdala activity and provides on-demand stimulation upon detecting pathological theta oscillations, while an ensemble of wearables (smart glasses, smartwatches, smartphones) uses multimodal large language model (LLM) analysis of sensory data to detect environmental or physiological PTSD triggers and deliver timely audiovisual interventions. Logged events from both the neural and wearable loops are analyzed to personalize trigger detection and progressively transition patients to non-invasive interventions. In Neuroscience Research Mode, the same platform is adapted for real-world brain activity capture. Wearable-LLM systems recognize naturalistic events (social interactions, emotional situations, compulsive behaviors, decision making) and signal implanted RNS devices (via wireless triggers) to record synchronized intracranial data during these moments. This approach builds on recent advances in mobile intracranial EEG recording and closed-loop neuromodulation in humans (BRAIN Initiative, 2023) (Mobbs et al., 2021). We discuss how our interdisciplinary system could revolutionize PTSD therapy and cognitive neuroscience by enabling 24/7 monitoring, context-aware intervention, and rich data collection outside traditional labs. The vision is a future where AI-enhanced devices continuously collaborate with the human brain, offering therapeutic support and deep insights into neural function, with the resulting real-world context rich neural data, in turn, accelerating the development of more biologically-grounded and human-centric AI.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/q-bio.NC/">arXiv: Quantitative Biology: Neurons and Cognition</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/q-bio.NC/>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/q-bio.NC/" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/q-bio.NC/&title=arXiv: Quantitative Biology: Neurons and Cognition" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.12392">Oscillatory Signatures of Parkinson's Disease: Central and Parietal EEG Alterations Across Multiple Frequency Bands</a></h4>

              
                arXiv:2503.12392v2 Announce Type: replace 
Abstract: This study investigates EEG as a potential early biomarker by applying deep learning techniques to resting-state EEG recordings from 31 subjects (15 with PD and 16 healthy controls). EEG signals underwent preprocessing to remove tremor artifacts before classification with CNNs using wavelet-based electrode triplet images. Our analysis across different brain regions and frequency bands showed distinct spatial-spectral patterns of PD-related neural oscillations. We identified high classification accuracy (76%) using central electrodes (C3, Cz, C4) with full-spectrum 0.4-62.4 Hz analysis and 74% accuracy in right parietal regions (P8, CP6, P4) with 10-second windows. Bilateral centro-parietal regions showed strong performance (67%) in the theta band (4.0-7.79 Hz), while multiple areas demonstrated some sensitivity (65%) in the alpha band (7.8-15.59 Hz). We also observed a distinctive topographical pattern of gamma band (40-62.4 Hz) alterations specifically localized to central-parietal regions, which remained consistent across different temporal windows. In particular, we observed pronounced right-hemisphere involvement across several frequency bands. Unlike previous studies that achieved higher accuracies by potentially including tremor artifacts, our approach isolates genuine neurophysiological alterations in cortical activity. These findings suggest that specific EEG-based oscillatory patterns, especially in central and parietal regions and across multiple frequency bands, may provide diagnostic information for PD, potentially before the onset of motor symptoms.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/q-bio.NC/">arXiv: Quantitative Biology: Neurons and Cognition</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.18114">Feature Learning beyond the Lazy-Rich Dichotomy: Insights from Representational Geometry</a></h4>

              
                arXiv:2503.18114v1 Announce Type: cross 
Abstract: The ability to integrate task-relevant information into neural representations is a fundamental aspect of both biological and artificial intelligence. To enable theoretical analysis, recent work has examined whether a network learns task-relevant features (rich learning) or resembles a random feature model (or a kernel machine, i.e., lazy learning). However, this simple lazy-versus-rich dichotomy overlooks the possibility of various subtypes of feature learning that emerge from different architectures, learning rules, and data properties. Furthermore, most existing approaches emphasize weight matrices or neural tangent kernels, limiting their applicability to neuroscience because they do not explicitly characterize representations.
  In this work, we introduce an analysis framework based on representational geometry to study feature learning. Instead of analyzing what are the learned features, we focus on characterizing how task-relevant representational manifolds evolve during the learning process. In both theory and experiment, we find that when a network learns features useful for solving a task, the task-relevant manifolds become increasingly untangled. Moreover, by tracking changes in the underlying manifold geometry, we uncover distinct learning stages throughout training, as well as different learning strategies associated with training hyperparameters, uncovering subtypes of feature learning beyond the lazy-versus-rich dichotomy. Applying our method to neuroscience and machine learning, we gain geometric insights into the structural inductive biases of neural circuits solving cognitive tasks and the mechanisms underlying out-of-distribution generalization in image classification. Our framework provides a novel geometric perspective for understanding and quantifying feature learning in both artificial and biological neural networks.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2407.00957">Expressivity of Neural Networks with Random Weights and Learned Biases</a></h4>

              
                arXiv:2407.00957v3 Announce Type: replace 
Abstract: Landmark universal function approximation results for neural networks with trained weights and biases provided the impetus for the ubiquitous use of neural networks as learning models in neuroscience and Artificial Intelligence (AI). Recent work has extended these results to networks in which a smaller subset of weights (e.g., output weights) are tuned, leaving other parameters random. However, it remains an open question whether universal approximation holds when only biases are learned, despite evidence from neuroscience and AI that biases significantly shape neural responses. The current paper answers this question. We provide theoretical and numerical evidence demonstrating that feedforward neural networks with fixed random weights can approximate any continuous function on compact sets. We further show an analogous result for the approximation of dynamical systems with recurrent neural networks. Our findings are relevant to neuroscience, where they demonstrate the potential for behaviourally relevant changes in dynamics without modifying synaptic weights, as well as for AI, where they shed light on recent fine-tuning methods for large language models, like bias and prefix-based approaches.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.17397">Availability of Perfect Decomposition in Statistical Linkage Learning for Unitation-based Function Concatenations</a></h4>

              
                arXiv:2503.17397v1 Announce Type: new 
Abstract: Statistical Linkage Learning (SLL) is a part of many state-of-the-art optimizers. The purpose of SLL is to discover variable interdependencies. It has been shown that the effectiveness of SLL-using optimizers is highly dependent on the quality of SLL-based problem decomposition. Thus, understanding what kind of problems are hard or easy to decompose by SLL is important for practice. In this work, we analytically estimate the size of a population sufficient for obtaining a perfect decomposition in case of concatenations of certain unitation-based functions. The experimental study confirms the accuracy of the proposed estimate. Finally, using the proposed estimate, we identify those problem types that may be considered hard for SLL-using optimizers.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.17763">Lifelong Evolution of Swarms</a></h4>

              
                arXiv:2503.17763v1 Announce Type: new 
Abstract: Adapting to task changes without forgetting previous knowledge is a key skill for intelligent systems, and a crucial aspect of lifelong learning. Swarm controllers, however, are typically designed for specific tasks, lacking the ability to retain knowledge across changing tasks. Lifelong learning, on the other hand, focuses on individual agents with limited insights into the emergent abilities of a collective like a swarm. To address this gap, we introduce a lifelong evolutionary framework for swarms, where a population of swarm controllers is evolved in a dynamic environment that incrementally presents novel tasks. This requires evolution to find controllers that quickly adapt to new tasks while retaining knowledge of previous ones, as they may reappear in the future. We discover that the population inherently preserves information about previous tasks, and it can reuse it to foster adaptation and mitigate forgetting. In contrast, the top-performing individual for a given task catastrophically forgets previous tasks. To mitigate this phenomenon, we design a regularization process for the evolutionary algorithm, reducing forgetting in top-performing individuals. Evolving swarms in a lifelong fashion raises fundamental questions on the current state of deep lifelong learning and on the robustness of swarm controllers in dynamic environments.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.18002">Neuromorphic Principles for Efficient Large Language Models on Intel Loihi 2</a></h4>

              
                arXiv:2503.18002v1 Announce Type: new 
Abstract: Large language models (LLMs) deliver impressive performance but require large amounts of energy. In this work, we present a MatMul-free LLM architecture adapted for Intel's neuromorphic processor, Loihi 2. Our approach leverages Loihi 2's support for low-precision, event-driven computation and stateful processing. Our hardware-aware quantized model on GPU demonstrates that a 370M parameter MatMul-free model can be quantized with no accuracy loss. Based on preliminary results, we report up to 3x higher throughput with 2x less energy, compared to transformer-based LLMs on an edge GPU, with significantly better scaling. Further hardware optimizations will increase throughput and decrease energy consumption. These results show the potential of neuromorphic hardware for efficient inference and pave the way for efficient reasoning models capable of generating complex, long-form text rapidly and cost-effectively.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.18061">Reinforcement Learning-based Self-adaptive Differential Evolution through Automated Landscape Feature Learning</a></h4>

              
                arXiv:2503.18061v1 Announce Type: new 
Abstract: Recently, Meta-Black-Box-Optimization (MetaBBO) methods significantly enhance the performance of traditional black-box optimizers through meta-learning flexible and generalizable meta-level policies that excel in dynamic algorithm configuration (DAC) tasks within the low-level optimization, reducing the expertise required to adapt optimizers for novel optimization tasks. Though promising, existing MetaBBO methods heavily rely on human-crafted feature extraction approach to secure learning effectiveness. To address this issue, this paper introduces a novel MetaBBO method that supports automated feature learning during the meta-learning process, termed as RLDE-AFL, which integrates a learnable feature extraction module into a reinforcement learning-based DE method to learn both the feature encoding and meta-level policy. Specifically, we design an attention-based neural network with mantissa-exponent based embedding to transform the solution populations and corresponding objective values during the low-level optimization into expressive landscape features. We further incorporate a comprehensive algorithm configuration space including diverse DE operators into a reinforcement learning-aided DAC paradigm to unleash the behavior diversity and performance of the proposed RLDE-AFL. Extensive benchmark results show that co-training the proposed feature learning module and DAC policy contributes to the superior optimization performance of RLDE-AFL to several advanced DE methods and recent MetaBBO baselines over both synthetic and realistic BBO scenarios. The source codes of RLDE-AFL are available at https://github.com/GMC-DRL/RLDE-AFL.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.18066">Accurate Peak Detection in Multimodal Optimization via Approximated Landscape Learning</a></h4>

              
                arXiv:2503.18066v1 Announce Type: new 
Abstract: Detecting potential optimal peak areas and locating the accurate peaks in these areas are two major challenges in Multimodal Optimization problems (MMOPs). To address them, much efforts have been spent on developing novel searching operators, niching strategies and multi-objective problem transformation pipelines. Though promising, existing approaches more or less overlook the potential usage of landscape knowledge. In this paper, we propose a novel optimization framework tailored for MMOPs, termed as APDMMO, which facilitates peak detection via fully leveraging the landscape knowledge and hence capable of providing strong optimization performance on MMOPs. Specifically, we first design a novel surrogate landscape model which ensembles a group of non-linear activation units to improve the regression accuracy on diverse MMOPs. Then we propose a free-of-trial peak detection method which efficiently locates potential peak areas through back-propagation on the learned surrogate landscape model. Based on the detected peak areas, we employ SEP-CMAES for local search within these areas in parallel to further improve the accuracy of the found optima. Extensive benchmarking results demonstrate that APDMMO outperforms several up-to-date baselines. Further ablation studies verify the effectiveness of the proposed novel designs. The source-code is available at ~\href{}{https://github.com/GMC-DRL/APDMMO}.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.18849">Application of Physics-Informed Neural Networks for Solving the Inverse Advection-Diffusion Problem to Localize Pollution Sources</a></h4>

              
                arXiv:2503.18849v1 Announce Type: new 
Abstract: This paper investigates the application of Physics-Informed Neural Networks (PINNs) for solving the inverse advection-diffusion problem to localize pollution sources. The study focuses on optimizing neural network architectures to accurately model pollutant dispersion dynamics under diverse conditions, including scenarios with weak and strong winds and multiple pollution sources. Various PINN configurations are evaluated, showing the strong dependence of solution accuracy on hyperparameter selection. Recommendations for efficient PINN configurations are provided based on these comparisons. The approach is tested across multiple scenarios and validated using real-world data that accounts for atmospheric variability. The results demonstrate that the proposed methodology achieves high accuracy in source localization, showcasing the stability and potential of PINNs for addressing environmental monitoring and pollution management challenges under complex weather conditions.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.17394">Temporal Flexibility in Spiking Neural Networks: Towards Generalization Across Time Steps and Deployment Friendliness</a></h4>

              
                arXiv:2503.17394v1 Announce Type: cross 
Abstract: Spiking Neural Networks (SNNs), models inspired by neural mechanisms in the brain, allow for energy-efficient implementation on neuromorphic hardware. However, SNNs trained with current direct training approaches are constrained to a specific time step. This "temporal inflexibility" 1) hinders SNNs' deployment on time-step-free fully event-driven chips and 2) prevents energy-performance balance based on dynamic inference time steps. In this study, we first explore the feasibility of training SNNs that generalize across different time steps. We then introduce Mixed Time-step Training (MTT), a novel method that improves the temporal flexibility of SNNs, making SNNs adaptive to diverse temporal structures. During each iteration of MTT, random time steps are assigned to different SNN stages, with spikes transmitted between stages via communication modules. After training, the weights are deployed and evaluated on both time-stepped and fully event-driven platforms. Experimental results show that models trained by MTT gain remarkable temporal flexibility, friendliness for both event-driven and clock-driven deployment (nearly lossless on N-MNIST and 10.1% higher than standard methods on CIFAR10-DVS), enhanced network generalization, and near SOTA performance. To the best of our knowledge, this is the first work to report the results of large-scale SNN deployment on fully event-driven scenarios.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.17756">Bandwidth Reservation for Time-Critical Vehicular Applications: A Multi-Operator Environment</a></h4>

              
                arXiv:2503.17756v1 Announce Type: cross 
Abstract: Onsite bandwidth reservation requests often face challenges such as price fluctuations and fairness issues due to unpredictable bandwidth availability and stringent latency requirements. Requesting bandwidth in advance can mitigate the impact of these fluctuations and ensure timely access to critical resources. In a multi-Mobile Network Operator (MNO) environment, vehicles need to select cost-effective and reliable resources for their safety-critical applications. This research aims to minimize resource costs by finding the best price among multiple MNOs. It formulates multi-operator scenarios as a Markov Decision Process (MDP), utilizing a Deep Reinforcement Learning (DRL) algorithm, specifically Dueling Deep Q-Learning. For efficient and stable learning, we propose a novel area-wise approach and an adaptive MDP synthetic close to the real environment. The Temporal Fusion Transformer (TFT) is used to handle time-dependent data and model training. Furthermore, the research leverages Amazon spot price data and adopts a multi-phase training approach, involving initial training on synthetic data, followed by real-world data. These phases enable the DRL agent to make informed decisions using insights from historical data and real-time observations. The results show that our model leads to significant cost reductions, up to 40%, compared to scenarios without a policy model in such a complex environment.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.18008">Personalized Language Models via Privacy-Preserving Evolutionary Model Merging</a></h4>

              
                arXiv:2503.18008v1 Announce Type: cross 
Abstract: Personalization in large language models (LLMs) seeks to tailor models to individual user or user group preferences. Prompt-based methods augment queries with user preference information, whereas training-based methods directly encode preferences into model parameters for more effective personalization. Despite achieving some success in personalizing LLMs, prior methods often fail to directly optimize task-specific metrics and lack explicit privacy-preservation mechanisms. To address these limitations, we propose Privacy-Preserving Model Merging via Evolutionary Algorithms (PriME), a novel approach to personalization that employs gradient-free methods to directly optimize task-specific metrics while preserving user privacy. By incorporating privacy preservation into optimization, PriME produces a personalized module that effectively captures the target user's preferences while minimizing the privacy risks for the users sharing their private information. Experiments on the LaMP benchmark show that PriME outperforms both prompt-based and training-based methods, achieving up to a 45% performance improvement over the prior art. Further analysis shows that PriME achieves a significantly better privacy-utility trade-off, highlighting the potential of evolutionary approaches for privacy-preserving LLM personalization.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.18060">Surrogate Learning in Meta-Black-Box Optimization: A Preliminary Study</a></h4>

              
                arXiv:2503.18060v1 Announce Type: cross 
Abstract: Recent Meta-Black-Box Optimization (MetaBBO) approaches have shown possibility of enhancing the optimization performance through learning meta-level policies to dynamically configure low-level optimizers. However, existing MetaBBO approaches potentially consume massive function evaluations to train their meta-level policies. Inspired by the recent trend of using surrogate models for cost-friendly evaluation of expensive optimization problems, in this paper, we propose a novel MetaBBO framework which combines surrogate learning process and reinforcement learning-aided Differential Evolution algorithm, namely Surr-RLDE, to address the intensive function evaluation in MetaBBO. Surr-RLDE comprises two learning stages: surrogate learning and policy learning. In surrogate learning, we train a Kolmogorov-Arnold Networks (KAN) with a novel relative-order-aware loss to accurately approximate the objective functions of the problem instances used for subsequent policy learning. In policy learning, we employ reinforcement learning (RL) to dynamically configure the mutation operator in DE. The learned surrogate model is integrated into the training of the RL-based policy to substitute for the original objective function, which effectively reduces consumed evaluations during policy learning. Extensive benchmark results demonstrate that Surr-RLDE not only shows competitive performance to recent baselines, but also shows compelling generalization for higher-dimensional problems. Further ablation studies underscore the effectiveness of each technical components in Surr-RLDE. We open-source Surr-RLDE at https://github.com/GMC-DRL/Surr-RLDE.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.18096">Informer in Algorithmic Investment Strategies on High Frequency Bitcoin Data</a></h4>

              
                arXiv:2503.18096v1 Announce Type: cross 
Abstract: The article investigates the usage of Informer architecture for building automated trading strategies for high frequency Bitcoin data. Three strategies using Informer model with different loss functions: Root Mean Squared Error (RMSE), Generalized Mean Absolute Directional Loss (GMADL) and Quantile loss, are proposed and evaluated against the Buy and Hold benchmark and two benchmark strategies based on technical indicators. The evaluation is conducted using data of various frequencies: 5 minute, 15 minute, and 30 minute intervals, over the 6 different periods. Although the Informer-based model with Quantile loss did not outperform the benchmark, two other models achieved better results. The performance of the model using RMSE loss worsens when used with higher frequency data while the model that uses novel GMADL loss function is benefiting from higher frequency data and when trained on 5 minute interval it beat all the other strategies on most of the testing periods. The primary contribution of this study is the application and assessment of the RMSE, GMADL, and Quantile loss functions with the Informer model to forecast future returns, subsequently using these forecasts to develop automated trading strategies. The research provides evidence that employing an Informer model trained with the GMADL loss function can result in superior trading outcomes compared to the buy-and-hold approach.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.18531">Parental Guidance: Efficient Lifelong Learning through Evolutionary Distillation</a></h4>

              
                arXiv:2503.18531v1 Announce Type: cross 
Abstract: Developing robotic agents that can perform well in diverse environments while showing a variety of behaviors is a key challenge in AI and robotics. Traditional reinforcement learning (RL) methods often create agents that specialize in narrow tasks, limiting their adaptability and diversity. To overcome this, we propose a preliminary, evolution-inspired framework that includes a reproduction module, similar to natural species reproduction, balancing diversity and specialization. By integrating RL, imitation learning (IL), and a coevolutionary agent-terrain curriculum, our system evolves agents continuously through complex tasks. This approach promotes adaptability, inheritance of useful traits, and continual learning. Agents not only refine inherited skills but also surpass their predecessors. Our initial experiments show that this method improves exploration efficiency and supports open-ended learning, offering a scalable solution where sparse reward coupled with diverse terrain environments induces a multi-task setting.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2503.18858">Dynamics of Insect Paraintelligence: How a Mindless Colony of Ants Meaningfully Moves a Beetle</a></h4>

              
                arXiv:2503.18858v1 Announce Type: cross 
Abstract: In this work, a new concept called Vector Dissipation of Randomness (VDR) is developed and formalized. It describes the mechanism by which complex multicomponent systems transition from chaos to order through the filtering of random directions, accumulation of information in the environment, and self-organization of agents. VDR explains how individual random strategies can evolve into collective goaldirected behavior, leading to the emergence of an ordered structure without centralized control. To test the proposed model, a numerical simulation of the "ant and beetle" system was conducted, in which agents (ants) randomly choose movement directions, but through feedback mechanisms and filtering of weak strategies, they form a single coordinated vector of the beetles movement. VDR is a universal mechanism applicable to a wide range of self-organizing systems, including biological populations, decentralized technological networks, sociological processes, and artificial intelligence algorithms. For the first time, an equation of the normalized emergence function in the processing of vector dissipation of randomness in the Ant and Beetle system has been formulated. The concept of paraintelligence was introduced for the first time. Insect paraintelligence is interpreted as a rational functionality that is close to or equivalent to intelligent activity in the absence of reflexive consciousness and selfawareness.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2401.04628">Multi-Neuron Representations of Hierarchical Concepts in Spiking Neural Networks</a></h4>

              
                arXiv:2401.04628v3 Announce Type: replace 
Abstract: We describe how hierarchical concepts can be represented in three types of layered neural networks. The aim is to support recognition of the concepts when partial information about the concepts is presented, and also when some of the neurons in the network might fail. Our failure model involves initial random failures. The three types of networks are: feed-forward networks with high connectivity, feed-forward networks with low connectivity, and layered networks with low connectivity and with both forward edges and "lateral" edges within layers. In order to achieve fault-tolerance, the representations all use multiple representative neurons for each concept. We show how recognition can work in all three of these settings, and quantify how the probability of correct recognition depends on several parameters, including the number of representatives and the neuron failure probability. We also discuss how these representations might be learned, in all three types of networks. For the feed-forward networks, the learning algorithms are similar to ones used in [4], whereas for networks with lateral edges, the algorithms are generally inspired by work on the assembly calculus [3, 6, 7].
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2502.10436">MERGE$^3$: Efficient Evolutionary Merging on Consumer-grade GPUs</a></h4>

              
                arXiv:2502.10436v2 Announce Type: replace 
Abstract: Evolutionary model merging enables the creation of high-performing multi-task models but remains computationally prohibitive for consumer hardware. We introduce MERGE$^3$, an efficient framework that makes evolutionary merging feasible on a single GPU by reducing fitness computation costs 50$\times$ while preserving performance. MERGE$^3$ achieves this by Extracting a reduced dataset for evaluation, Estimating model abilities using Item Response Theory (IRT), and Evolving optimal merges via IRT-based performance estimators. Our method enables state-of-the-art multilingual and cross-lingual merging, transferring knowledge across languages with significantly lower computational overhead. We provide theoretical guarantees and an open-source library, democratizing high-quality model merging.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2408.01536">Active Learning for Neural PDE Solvers</a></h4>

              
                arXiv:2408.01536v2 Announce Type: replace-cross 
Abstract: Solving partial differential equations (PDEs) is a fundamental problem in science and engineering. While neural PDE solvers can be more efficient than established numerical solvers, they often require large amounts of training data that is costly to obtain. Active learning (AL) could help surrogate models reach the same accuracy with smaller training sets by querying classical solvers with more informative initial conditions and PDE parameters. While AL is more common in other domains, it has yet to be studied extensively for neural PDE solvers. To bridge this gap, we introduce AL4PDE, a modular and extensible active learning benchmark. It provides multiple parametric PDEs and state-of-the-art surrogate models for the solver-in-the-loop setting, enabling the evaluation of existing and the development of new AL methods for neural PDE solving. We use the benchmark to evaluate batch active learning algorithms such as uncertainty- and feature-based methods. We show that AL reduces the average error by up to 71% compared to random sampling and significantly reduces worst-case errors. Moreover, AL generates similar datasets across repeated runs, with consistent distributions over the PDE parameters and initial conditions. The acquired datasets are reusable, providing benefits for surrogate models not involved in the data generation.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2410.02249">Spiking Neural Network as Adaptive Event Stream Slicer</a></h4>

              
                arXiv:2410.02249v3 Announce Type: replace-cross 
Abstract: Event-based cameras are attracting significant interest as they provide rich edge information, high dynamic range, and high temporal resolution. Many state-of-the-art event-based algorithms rely on splitting the events into fixed groups, resulting in the omission of crucial temporal information, particularly when dealing with diverse motion scenarios (\eg, high/low speed).In this work, we propose SpikeSlicer, a novel-designed plug-and-play event processing method capable of splitting events stream adaptively.SpikeSlicer utilizes a low-energy spiking neural network (SNN) to trigger event slicing. To guide the SNN to fire spikes at optimal time steps, we propose the Spiking Position-aware Loss (SPA-Loss) to modulate the neuron's state. Additionally, we develop a Feedback-Update training strategy that refines the slicing decisions using feedback from the downstream artificial neural network (ANN). Extensive experiments demonstrate that our method yields significant performance improvements in event-based object tracking and recognition. Notably, SpikeSlicer provides a brand-new SNN-ANN cooperation paradigm, where the SNN acts as an efficient, low-energy data processor to assist the ANN in improving downstream performance, injecting new perspectives and potential avenues of exploration. Our code is available at https://github.com/AndyCao1125/SpikeSlicer.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2502.17034">Evolution 6.0: Evolving Robotic Capabilities Through Generative Design</a></h4>

              
                arXiv:2502.17034v3 Announce Type: replace-cross 
Abstract: We propose a new concept, Evolution 6.0, which represents the evolution of robotics driven by Generative AI. When a robot lacks the necessary tools to accomplish a task requested by a human, it autonomously designs the required instruments and learns how to use them to achieve the goal. Evolution 6.0 is an autonomous robotic system powered by Vision-Language Models (VLMs), Vision-Language Action (VLA) models, and Text-to-3D generative models for tool design and task execution. The system comprises two key modules: the Tool Generation Module, which fabricates task-specific tools from visual and textual data, and the Action Generation Module, which converts natural language instructions into robotic actions. It integrates QwenVLM for environmental understanding, OpenVLA for task execution, and Llama-Mesh for 3D tool generation. Evaluation results demonstrate a 90% success rate for tool generation with a 10-second inference time, and action generation achieving 83.5% in physical and visual generalization, 70% in motion generalization, and 37% in semantic generalization. Future improvements will focus on bimanual manipulation, expanded task capabilities, and enhanced environmental interpretation to improve real-world adaptability.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-03-25 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://onlinelibrary.wiley.com/feed/10969861/most-recent>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://onlinelibrary.wiley.com/feed/10969861/most-recent" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://onlinelibrary.wiley.com/feed/10969861/most-recent&title=Journal of Comparative Neurology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://onlinelibrary.wiley.com/doi/10.1002/cne.70045?af=R">Timing Matters: Lessons From Perinatal Neurogenesis in the Olfactory Bulb</a></h4>

              
                <img src="https://onlinelibrary.wiley.com/cms/asset/ee9b76b8-4d99-44f8-81b3-d633394146a4/cne70045-gra-0001-m.png"
     alt="Timing Matters: Lessons From Perinatal Neurogenesis in the Olfactory Bulb"/>
<p>The timing of neurogenesis influences the location of olfactory bulb interneurons. Early born granule cells populate predominately the superficial granule cell layer and external plexiform layer, whereas later born granule cells are predominately in the deeper sublayers. These different locations likely establish a foundation for differential interactions with subpopulations of olfactory bulb projection neurons. 

</p>
<br/>
<h2>ABSTRACT</h2>
<p>In the olfactory bulb (OB), odorant receptor-specific input converges into glomeruli. Subsequently, the coding of odor information is fine-tuned by local synaptic circuits within the glomeruli and the deeper external plexiform layer (EPL) in the OB. Deciphering the organization of inhibitory granule cells (GCs) in the EPL relative to the secondary dendrites of projection neurons is pivotal for understanding odor processing. We conducted a detailed investigation of GCs, focusing on the timing of neurogenesis, laminar distribution, and synaptogenesis between GCs and projection neurons. In summary, GCs develop following a developmental continuum with an outside-in maturation pattern from embryogenesis to adulthood. GCs born 1 week after birth display a unique sublayer-specific distribution pattern, marking a transition between embryonic or neonatal and adult stages. Integration into reciprocal synaptic circuits occurred 10 days post-neurogenesis. We conclude that the timing of neurogenesis dictates the anatomical configuration of GCs within the OB, which, in turn, regulates preferential synaptic integration with either mitral cell or tufted cell secondary dendrites.</p>
              

              <p class="text-muted">in <a href ="https://onlinelibrary.wiley.com/feed/10969861/most-recent">Journal of Comparative Neurology</a> on 2025-03-25 02:44:31 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58216-4">Molecular basis for human respiratory syncytial virus transcriptional regulator NS1 interactions with MED25</a></h4>

              
                <p>Nature Communications, Published online: 25 March 2025; <a href="https://www.nature.com/articles/s41467-025-58216-4">doi:10.1038/s41467-025-58216-4</a></p>This study provides the structural and biochemical characterization of the RSV NS1/MED25 ACID complex, revealing how RSV exploits fuzzy interfaces between coactivators and transcription factors, including ATF3, to modulate host gene transcription.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-03-25 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-57497-z">Dynamic synthesis and transport of phenazine-1-carboxylic acid to boost extracellular electron transfer rate</a></h4>

              
                <p>Nature Communications, Published online: 25 March 2025; <a href="https://www.nature.com/articles/s41467-025-57497-z">doi:10.1038/s41467-025-57497-z</a></p>Electron shuttle-mediated extracellular electron transfer (EET) is most critical in determining the efficiency of chemical-to-power of exoelectrogens. Here, the authors show that addition of phenazine-1-carboxylic acid at ~80 µM leads to the improved power generation in Shewenella oneidensis MR-1.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-03-25 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-17/v3?src=rss">Trends, key contributors, and emerging issues in honey and breast cancer: A bibliometric analysis from 2014 to 2024 [version 3; peer review: 1 approved, 1 approved with reservations]</a></h4>

              
                Background Honey, a natural product with diverse bioactive compounds, has been increasingly explored for its potential anticancer properties. This study aims to comprehensively analyze the scientific literature on the relationship between honey and breast cancer. Methods A bibliometric analysis was conducted using the ScienceDirect database to identify publications from 2014 to 2024. Data on publication trends, author collaboration, and keyword analysis were extracted to gain insight into the research landscape. Keyword analysis identified nine distinct clusters, indicating diverse research directions regarding the role of honey in breast cancer treatment. Results In Key journals such as the Journal of Ethnopharmacology and Food Chemistry have been at the forefront of disseminating research findings in this domain, demonstrating a strongly interdisciplinary approach that bridges traditional medicine and modern scientific inquiry. The increasing interest in the anticancer properties of honey, as evidenced by the growing number of studies, underlines its potential as a promising natural agent for breast cancer prevention and treatment. Conclusions This study provides a comprehensive overview of the current knowledge landscape and highlights emerging issues that require further exploration.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-03-24 17:11:18 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-322/v1?src=rss">An extensive review of SAR remote sensing in mode transportation studies: Major findings and future recommendations [version 1; peer review: awaiting peer review]</a></h4>

              
                Background The availability of synthetic aperture radar (SAR) remote sensing technology and platform has been widely used in the study of transportation. It includes all three modes of air, water, and land. This review aims to determine the importance of SAR remote sensing technology in a specific mode of transportation focus area. Methods For this reason, an extensive literature review was conducted. This review used the Web of Science, the IEEEXplore, and the ScienceDirect database. The systematic search strategy was developed for query-related research papers. The rules were then proposed to filter more related research papers. Then the selected papers were classified into five classes (mode, container, infrastructure, geographic distribution, and pattern of publication). Finally, a descriptive statistical analysis was conducted. Results Many studies have been done in the last three decades for mode transportation. Based on the mode of transportation and its container, the water mode of transportation and ship were the most studied. It is due to the contrast differences between the ship as the detected object and the sea as the background. While based on the infrastructure the airport was the most studied object, followed by the railway and harbour. Most of the studies on using SAR as the mode of transportation were conducted in the northern part of the equator. Currently, neural networks and deep learning algorithms are introduced to detect the mode of transportation using SAR remote sensing datasets. Conclusion Future research is expected to detect ships in a more heterogeneous background. More studies in moving object detection using SAR are also expected in the future.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-03-24 17:02:48 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-321/v1?src=rss">The distribution of groundwater uranium in Chintamani village, Karnataka, India [version 1; peer review: awaiting peer review]</a></h4>

              
                Background Chintamani village, Chikkaballapura district, Karnataka, India was found to possess high aquifer uranium concentrations. Ge- ologically, Chintamani village is located on bedrock that is rich in elements like potassium (K) that naturally contain high levels of radioactive elements, such as uranium and thorium, due to the presence of alkali-feldspar granites and gneisses. Aquifer de- pletion has caused the concentration of these elements in groundwater to increase over time, posing a potential health hazard to the residents of Chintamani village. Methods Here, we report the sampling of groundwater from 12 borewells located in Chintamani village in between the period of August 2024 to December 2024. We observed groundwater uranium concentrations of 0.02 ppm to 8.64 ppm. Data for borewell depth, the quantity of total dissolved solids (TDS), and the elemental composition of TDS is also reported. We observed a statistically significant spatial distribution of uranium concentrations in Chintamani village. Borewells possessing the highest observed concentrations of uranium were clustered towards the northwestern region of the village. Conclusions This dataset is expected to serve as a resource for guiding potential remediation efforts in these locations.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-03-24 16:53:58 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-320/v1?src=rss">Political bias in historiography - an experimental investigation of preferences for publication as a function of political orientation [version 1; peer review: awaiting peer review]</a></h4>

              
                Background This study examines the influence of political preferences on historians’ assessments of the publishability of contemporary history abstracts by investigating whether historians favor abstracts that align with their political orientation. Methods In an online experiment, 75 historians evaluated 17 fictitious contemporary history abstracts from 17 pairs, each presented with either a progressive or conservative stance. The participants made initial intuitive assessments regarding the publishability of each abstract and later provided more considered responses, also rating their Feeling of Rightness (FOR) regarding their initial judgments. Results The results revealed a significant interaction effect between an abstract’s political stance and historians’ political orientation, consistent with the observation that right-wing historians prefer conservative abstracts, left-wing historians prefer progressive abstracts, and moderate historians show no preference for either. Overall, participants preferred progressive abstracts, largely reflecting a majority of left-leaning historians in our sample. Moreover, after reconsidering their responses and providing FOR ratings, participants’ initial decisions did not significantly change. Conclusions Our study suggests that political preferences influence research evaluations and are not diminished by more deliberate processing, as demonstrated through the case study of historians.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-03-24 15:55:26 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/11-53/v4?src=rss">Changing the method of consent to increase the numbers of deceased donors in Saudi Arabia: the autonomy paradox [version 4; peer review: 1 approved with reservations, 3 not approved]</a></h4>

              
                Background In Saudi Arabia, the gap between the demand for and availability of organs persists, with a total of 13,731 patients on the waiting list in 2019. Family refusal is a major obstacle limiting donation since their consent must be obtained prior to the retrieval of organs. The cause of family refusal is mainly due to their lack of knowledge of their loved ones’ wish to become a donor. This paper aimed to compare three systems of obtaining consent in terms of effectiveness, respect for autonomy, and the cultural role of families in Saudi Arabia to ensure feasibility and effectiveness in increasing the number of donors. Policy alternatives and implications The consent systems include informed consent, presumed consent, and mandated choice. The mandated choice policy might be the optimal solution since it is the most likely to balance the respect for individual autonomy and the cultural role of families in Saudi Arabia. Conclusions and recommendations Mandated choice could be an option that ensures respect for personal autonomy. Nevertheless, more research should be conducted to support adopting this policy in the Saudi context. Additionally, a recommendation to decision makers is to utilize the Tawakkalna app to send alerts to the next of kin when a user registers as a donor with the user’s consent.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-03-24 15:32:51 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-319/v1?src=rss">Identification and Functional Characterization of microRNAs in Ichthyosporea [version 1; peer review: awaiting peer review]</a></h4>

              
                Background microRNAs (miRNAs) are key post-transcriptional regulators of gene expression in animals, yet their origins and early functions remain elusive. In this study, we explore the diversity and functionality of miRNAs in Ichthyosporea, a group of unicellular organisms that represent some of the closest relatives of animals. Methods We have performed miRNA identification by integrating deep small RNA sequencing with draft genome assemblies across six species of Ichthyosporea, including Abeoforma whisleri, Pirum gemmata, and Sphaeroforma sp. To assess the functional roles of these miRNAs, we have used computational target prediction and degradome sequencing. Results We identified a rich and varied repertoire of miRNAs across Ichthyosporea. Many of these miRNAs appear to be lineage-specific, while a subset is conserved among closely related species, suggesting both rapid evolutionary turnover and the persistence of ancient regulatory elements. Moreover, predicted target genes span a wide range of transcripts, implicating miRNAs in diverse cellular processes. Additionally, degradome sequencing reveal that in S. arctica, several miRNAs are likely to induce mRNA cleavage. Conclusions These findings not only extend previous reports of miRNA presence in Ichthyosporea but also underscore the deep evolutionary roots of miRNA-mediated gene regulation in the holozoan lineage. The miRNA induced target cleavage in S. arctica is also observed in cnidarians and supports the hypothesis that mRNA cleavage may represent the ancestral function of animal miRNAs.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-03-24 15:28:37 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://onlinelibrary.wiley.com/feed/10969861/most-recent>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://onlinelibrary.wiley.com/feed/10969861/most-recent" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://onlinelibrary.wiley.com/feed/10969861/most-recent&title=Journal of Comparative Neurology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://onlinelibrary.wiley.com/doi/10.1002/cne.70042?af=R">A Rapid Heat‐Enhanced Golgi‐Cox Staining Method for Detailed Neuroanatomical Analysis Coupled With Immunostaining</a></h4>

              
                <img src="https://onlinelibrary.wiley.com/cms/asset/ad98dd15-0147-4b56-82b4-e751309294f3/cne70042-gra-0001-m.png"
     alt="A Rapid Heat-Enhanced Golgi-Cox Staining Method for Detailed Neuroanatomical Analysis Coupled With Immunostaining"/>
<p>Optimized Golgi-Cox staining reduces the staining duration to 24 h for 100-µm-thick mouse brain sections, the optimized protocol exhibits excellent compatibility with immunostaining and transgenic models. This approach enables comprehensive multimodal analysis with a single sample and enhances Golgi-Cox staining's applicability in modern neuroscience.

</p>
<br/>
<h2>ABSTRACT</h2>
<p>The Golgi-Cox staining technique is renowned for its ability to delineate neuronal architecture with remarkable precision. However, the traditional protocol's lengthy processing timeline and limited compatibility with immunostaining and transgenic labeling have hindered its widespread adoption in modern neuroscience research.</p>
<p>In the current study, we found that adjusting the incubation temperature to 55°C significantly reduced the staining duration to a mere 24 h for 100 µm-thick sections of mouse brain tissue. Importantly, our optimized protocol is compatible with immunostaining techniques and transgenic mouse models. In addition, using a lipopolysaccharides-induced mouse model of depression, we found a reduction in dendritic spines labeled by Golgi-Cox staining and an increase in the number of microglial cells labeled by immunofluorescence in the same samples, in addition, cross-talk between Golgi-Cox-stained neurons and microglial fibers were observed.</p>
<p>In conclusion, the modified Golgi-Cox staining technique allows for the acquisition of a more comprehensive set of data from the same biological tissue with increased efficiency. This advancement promises to improve methodologies in histopathology and neurobiology, making advanced applications of Golgi-Cox staining more accessible in contemporary neuroscience research.</p>
              

              <p class="text-muted">in <a href ="https://onlinelibrary.wiley.com/feed/10969861/most-recent">Journal of Comparative Neurology</a> on 2025-03-24 15:08:39 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-318/v1?src=rss">Fostering open science and responsible research practices: A pre-post study [version 1; peer review: awaiting peer review]</a></h4>

              
                Background Educational initiatives could foster the adoption of open science (OS) and responsible research practices (RRPs). This single group pre-post study evaluated the impact of an educational intervention on increasing the adherence, knowledge and perceptions about adopting OS practices and RRPs among graduate researchers at a Brazilian University. Methods Graduate students from a southern Brazilian university were invited to participate in a course addressing OS and RRPs. The intervention was an online interactive course on OS and RRPs. The number of OS outputs, including Open Science Framework (OSF) accounts, study registrations, protocols, analysis plans, data sets, preprints, and the number of projects published by each participant were collected before and after the intervention. Additionally, a self-administered online questionnaire was applied before and after the intervention to evaluate participants’ perceptions on RRPs, OS practices and on the current researchers’ evaluation system. Results Eighty-four students finished the course and 80 agreed to participate in the study. The number of OSF accounts increased from 7 to 78 after the course, and the number of projects increased from 7 to 10, six months after the intervention. No registrations, protocols, analysis plans, data sets, or preprints were found after 6 and 12 months, respectively. The participants’ perceptions of the current research evaluation system and on the OS practices and RRPs changed positively with the intervention. Also, the intention to adopt practices like registration, protocol and preprint publications has noticeably increased after the course. Conclusions The number of participants’ OSF outputs showed little or no improvement after the intervention. The most important impact difference could be identified in terms of the participants’ perceptions and intentions to adhere to such practices in the future.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-03-24 10:00:07 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml&title=Physical Review E: Biological physics" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="http://link.aps.org/doi/10.1103/PhysRevE.111.034408">Interaction network structures in competitive ecosystems</a></h4>

              
                <p>Author(s): David A. Kessler and Nadav M. Shnerb</p><p>We present a numerical analysis of local community assembly through weak migration from a regional species pool. At equilibrium, the local community consists of a subset (“clique”) of species from the regional community. Our analysis, based on numerical integration of the generalized Lotka-Volterra …</p><br/><p>[Phys. Rev. E 111, 034408] Published Mon Mar 24, 2025</p>
              

              <p class="text-muted">in <a href ="http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml">Physical Review E: Biological physics</a> on 2025-03-24 10:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://onlinelibrary.wiley.com/feed/10981063/most-recent>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://onlinelibrary.wiley.com/feed/10981063/most-recent" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://onlinelibrary.wiley.com/feed/10981063/most-recent&title=Hippocampus" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://onlinelibrary.wiley.com/doi/10.1002/hipo.70011?af=R">Brain Networks Differ According to Levels of Interference in Spatiotemporal Processing</a></h4>

              
                <h2>ABSTRACT</h2>
<p>The ability to form different neural representations for similar inputs is a central process of episodic memory. Although the dorsal dentate gyrus and CA3 have been indicated as important in this phenomenon, the neuronal circuits underlying spatiotemporal memory processing with different levels of spatial similarity are still elusive. In this study, we measured the expression of the immediate early gene c-Fos to evaluate brain areas activated when rats recalled the temporal order of object locations in a task, with either high or low levels of spatial interference. Animals showed spatiotemporal memory in both conditions once they spent more time exploring the older object locations relative to the more recent ones. We found no difference in the levels of c-Fos expression between high and low spatial interference. However, the levels of c-Fos expression in CA2 positively correlated with the discrimination index in the low spatial interference condition. More importantly, functional network connectivity analysis revealed a wider and more interconnected neuronal circuit in conditions of high than in low spatial interference. Our study advances the understanding of brain networks recruited in episodic memory with different degrees of spatial similarity.</p>
              

              <p class="text-muted">in <a href ="https://onlinelibrary.wiley.com/feed/10981063/most-recent">Hippocampus</a> on 2025-03-24 07:05:45 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://www.cell.com/trends/neurosciences/inpress.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://www.cell.com/trends/neurosciences/inpress.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://www.cell.com/trends/neurosciences/inpress.rss&title=Trends in Neurosciences: In press" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/trends/neurosciences/fulltext/S0166-2236(25)00058-X?rss=yes">The long and the short of TDP-43</a></h4>

              
                In a recent study, Dykstra and colleagues show that shortened TAR DNA Binding Protein 43 (sTDP-43) isoforms are generated as by-products of TDP-43 autoregulation. sTDP-43 levels are regulated through nonsense-mediated decay and proteasomal and autophagic degradation, and elicit toxicity through dominant negative effects on TDP-43 splicing activity. These results identify mechanisms contributing to sTDP-43 accumulation and toxicity in disease.
              

              <p class="text-muted">in <a href ="http://www.cell.com/trends/neurosciences/inpress.rss">Trends in Neurosciences: In press</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://www.cell.com/neuron/inpress.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://www.cell.com/neuron/inpress.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://www.cell.com/neuron/inpress.rss&title=Neuron: In press" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/neuron/fulltext/S0896-6273(25)00171-0?rss=yes">A pontine center in descending pain control</a></h4>

              
                The pain experience is modulated by descending pathways. Using an intersectional strategy, Li et al. map anatomical and molecular-spatial features of nociceptive descending pathways. Their findings uncover a pontine region, previously linked to micturition, as a powerful station for descending pain modulation, exposing an attractive target for combating pain.
              

              <p class="text-muted">in <a href ="http://www.cell.com/neuron/inpress.rss">Neuron: In press</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://www.cell.com/neuron/inpress.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://www.cell.com/neuron/inpress.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://www.cell.com/neuron/inpress.rss&title=Neuron: In press" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/neuron/fulltext/S0896-6273(25)00167-9?rss=yes">Sleep stages antagonistically modulate reactivation drift</a></h4>

              
                Bollmann et al. track reactivated CA1 assemblies representing spatial memories during 16–20 h of sleep/rest. Assemblies initially reflect recently learned spatial memories but are gradually transformed into those seen during the memory recall session following rest. Whereas slow-wave sleep accelerates the assembly drift, REM epochs counteract it.
              

              <p class="text-muted">in <a href ="http://www.cell.com/neuron/inpress.rss">Neuron: In press</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00242-6?rss=yes">Comprehensive nucleoside analysis of archaeal RNA modification profiles reveals an m7G in the conserved P loop of 23S rRNA</a></h4>

              
                Tsai et al. describe comprehensive nucleoside profiles of archaeal RNAs and identify temperature-responsive modifications using LC-MS/MS. They establish a workflow for RNA-modifying enzyme discovery by associating enzyme predictions with global nucleoside profiles. This approach reveals a methyltransferase responsible for installing an m7G in the peptidyl transferase center of 23S rRNA.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41586-025-08887-2">Generalizing arene C−H alkylations by radical−radical cross-coupling</a></h4>

              
                <p>Nature, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41586-025-08887-2">doi:10.1038/s41586-025-08887-2</a></p>Generalizing arene C−H alkylations by radical−radical cross-coupling
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00921-7">Reviving the biodiversity around an ancient palace</a></h4>

              
                <p>Nature, Published online: 24 March 2025; <a href="https://www.nature.com/articles/d41586-025-00921-7">doi:10.1038/d41586-025-00921-7</a></p>Juan Ramón Fernández Cardenete tends the Alhambra’s many water features and gardens.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00897-4">Glaciers are not just blocks of ice — plans to save them mustn’t overlook their hidden life</a></h4>

              
                <p>Nature, Published online: 24 March 2025; <a href="https://www.nature.com/articles/d41586-025-00897-4">doi:10.1038/d41586-025-00897-4</a></p>As glaciers begin to disappear, technological fixes to slow or halt ice melt are emerging. But regulations are urgently required before these fixes are used widely.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00920-8">RNA function follows form – why is it so hard to predict?</a></h4>

              
                <p>Nature, Published online: 24 March 2025; <a href="https://www.nature.com/articles/d41586-025-00920-8">doi:10.1038/d41586-025-00920-8</a></p>AlphaFold’s highly accurate structural models transformed protein biology,but RNA lags behind.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00864-z">Marathon runners tap brain-cell insulation for racetime fuel</a></h4>

              
                <p>Nature, Published online: 24 March 2025; <a href="https://www.nature.com/articles/d41586-025-00864-z">doi:10.1038/d41586-025-00864-z</a></p>MRI scans suggest runners’ brains might use the fatty substance myelin as fuel. The finding could lead to treatments for neurological diseases.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00869-8">Does sharing first authorship on a paper carry a penalty? What the research says</a></h4>

              
                <p>Nature, Published online: 24 March 2025; <a href="https://www.nature.com/articles/d41586-025-00869-8">doi:10.1038/d41586-025-00869-8</a></p>Study dispels myth that order of names in a paper’s author list dictates perception of success.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00919-1">Shaking up earthquake safety in Nepal with a folk song</a></h4>

              
                <p>Nature, Published online: 24 March 2025; <a href="https://www.nature.com/articles/d41586-025-00919-1">doi:10.1038/d41586-025-00919-1</a></p>Geophysicist Shiba Subedi combines science and songwriting to raise awareness and debunk earthquake myths.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00695-y">Why Africans should be telling the story of human origins</a></h4>

              
                <p>Nature, Published online: 24 March 2025; <a href="https://www.nature.com/articles/d41586-025-00695-y">doi:10.1038/d41586-025-00695-y</a></p>Yohannes Haile-Selassie wants to shift the trajectory of palaeoanthropology in fossil-rich Ethiopia away from its long colonial heritage.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00787-9">What CERN does next matters for science and for international cooperation</a></h4>

              
                <p>Nature, Published online: 24 March 2025; <a href="https://www.nature.com/articles/d41586-025-00787-9">doi:10.1038/d41586-025-00787-9</a></p>The world’s largest particle-physics laboratory is approaching a pivotal moment in its history.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00863-0">Paralysed man stands again after receiving ‘reprogrammed’ stem cells</a></h4>

              
                <p>Nature, Published online: 24 March 2025; <a href="https://www.nature.com/articles/d41586-025-00863-0">doi:10.1038/d41586-025-00863-0</a></p>Another man also regained some movement, but two others experienced minimal improvement.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00896-5">The expanding Universe — do ongoing tensions leave room for new physics?</a></h4>

              
                <p>Nature, Published online: 24 March 2025; <a href="https://www.nature.com/articles/d41586-025-00896-5">doi:10.1038/d41586-025-00896-5</a></p>One century after Edwin Hubble revealed his astonishing discovery of a cosmos beyond the Milky Way, the most precise measurements still can’t agree on how fast galaxies are moving.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00804-x">Live-cell super-resolution microscopy reveals how molecules enter and exit the nucleus</a></h4>

              
                <p>Nature, Published online: 24 March 2025; <a href="https://www.nature.com/articles/d41586-025-00804-x">doi:10.1038/d41586-025-00804-x</a></p>The nuclear pore complex is the gateway to the nucleus of cells. Now an in vivo imaging technique can track the way in which molecules move through this complex.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00809-6">Winds on an ultrahot planet challenge climate models</a></h4>

              
                <p>Nature, Published online: 24 March 2025; <a href="https://www.nature.com/articles/d41586-025-00809-6">doi:10.1038/d41586-025-00809-6</a></p>The ultrahot exoplanet WASP‑121 b has strong winds that transport material in different directions depending on the altitude in the planet’s atmosphere. High-resolution observations have mapped these winds for the first time, challenging the current understanding of atmospheric dynamics.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00903-9">How to know whether a conference is right for you</a></h4>

              
                <p>Nature, Published online: 24 March 2025; <a href="https://www.nature.com/articles/d41586-025-00903-9">doi:10.1038/d41586-025-00903-9</a></p>Some low-profile meetings are helpful for parts of the scientific community. Others look like more of a cash grab.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00859-w">‘Anxiety is palpable’: detention of researchers at US border spurs travel worries</a></h4>

              
                <p>Nature, Published online: 24 March 2025; <a href="https://www.nature.com/articles/d41586-025-00859-w">doi:10.1038/d41586-025-00859-w</a></p>Several high-profile incidents in the past month have scientists on edge.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/neuro/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/neuro/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/neuro/rss/current&title=Nature Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41593-025-01921-6">Neural ensembles that encode nocifensive mechanical and heat pain in mouse spinal cord</a></h4>

              
                <p>Nature Neuroscience, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41593-025-01921-6">doi:10.1038/s41593-025-01921-6</a></p>Zhang et al. identify unimodal neural representations in the spinal cord of cutaneous mechanical and heat stimuli gated by a shared feed-forward local inhibitory neuron type and a neural transition during neuropathy, with increased excitatory drive.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/neuro/rss/current">Nature Neuroscience</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/neuro/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/neuro/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/neuro/rss/current&title=Nature Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41593-025-01893-7">Phase of firing does not reflect temporal order in sequence memory of humans and recurrent neural networks</a></h4>

              
                <p>Nature Neuroscience, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41593-025-01893-7">doi:10.1038/s41593-025-01893-7</a></p>The temporal order of events in working memory is thought to be reflected by ordered neuronal firing at different phases. Here the authors show that this is not the case and that phase order is linked to stimulus timing and oscillation frequency.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/neuro/rss/current">Nature Neuroscience</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/neuro/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/neuro/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/neuro/rss/current&title=Nature Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41593-025-01904-7">Integrating multimodal data to understand cortical circuit architecture and function</a></h4>

              
                <p>Nature Neuroscience, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41593-025-01904-7">doi:10.1038/s41593-025-01904-7</a></p>This paper discusses how experimental and computational studies integrating multimodal data, such as RNA expression, connectivity and neural activity, are advancing our understanding of the architecture, mechanisms and function of cortical circuits.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/neuro/rss/current">Nature Neuroscience</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nrn/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nrn/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nrn/rss/current&title=Nature Reviews" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41583-025-00914-5">How microglia contribute to the induction and maintenance of neuropathic pain</a></h4>

              
                <p>Nature Reviews Neuroscience, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41583-025-00914-5">doi:10.1038/s41583-025-00914-5</a></p>Neuropathic pain caused by nerve damage results in neuronal pathway changes and immune cell engagement. In this Review, Malcangio and Sideris-Lampretsas discuss how microglia respond to and modulate neuronal activity and suggest that microglia–neuron pathways offer novel approaches for the attenuation of neuropathic pain.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nrn/rss/current">Nature Reviews</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nmeth/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nmeth/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nmeth/rss/current&title=Nature Methods" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41592-025-02634-1">Simultaneous single-cell DNA damage profiling and RNA sequencing by Paired-Damage-seq</a></h4>

              
                <p>Nature Methods, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41592-025-02634-1">doi:10.1038/s41592-025-02634-1</a></p>We developed Paired-Damage-seq to profile oxidative DNA damage and single-stranded DNA damage jointly with gene-expression analysis in single cells. By dissecting cell-type-specific genome vulnerability, our approach provides a method to pinpoint hotspots of molecular fidelity loss during dynamic biological processes.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nmeth/rss/current">Nature Methods</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nmeth/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nmeth/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nmeth/rss/current&title=Nature Methods" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41592-025-02632-3">Single-cell parallel analysis of DNA damage and transcriptome reveals selective genome vulnerability</a></h4>

              
                <p>Nature Methods, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41592-025-02632-3">doi:10.1038/s41592-025-02632-3</a></p>This work presents an in situ labeling-by-repair strategy, Paired-Damage-seq, to achieve joint profiling of oxidative and single-stranded DNA damage alongside gene expression in single cells.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nmeth/rss/current">Nature Methods</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58288-2">Author Correction: Pan-cancer multi-omic model of LINE-1 activity reveals locus heterogeneity of retrotransposition efficiency</a></h4>

              
                <p>Nature Communications, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41467-025-58288-2">doi:10.1038/s41467-025-58288-2</a></p>Author Correction: Pan-cancer multi-omic model of LINE-1 activity reveals locus heterogeneity of retrotransposition efficiency
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58224-4">Giant magnetoresistance induced by spin-dependent orbital coupling in Fe3GeTe2/graphene heterostructures</a></h4>

              
                <p>Nature Communications, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41467-025-58224-4">doi:10.1038/s41467-025-58224-4</a></p>Graphene is known to exhibit a magnetoresistance, however, in pristine graphene, the magnetoresistance is highly temperature sensitive. Here, by combining graphene with Fe3GeTe2, Huang et al find a strongly enhanced magnetoresistance that is temperature insensitive.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58198-3">Polarity modulation in compositionally tunable Bi2O2Se thin films</a></h4>

              
                <p>Nature Communications, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41467-025-58198-3">doi:10.1038/s41467-025-58198-3</a></p>Bi2O2Se is a promising 2D semiconductor with high electron mobility and native high-k dielectric layers, but its p-type doping remains challenging. Here, the authors report a low-temperature substitutional doping method to fabricate 2D Bi2O2Se p-n junctions and p-type transistors
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58115-8">A combined adjuvant and ferritin nanocage based mucosal vaccine against Streptococcus pneumoniae induces protective immune responses in a murine model</a></h4>

              
                <p>Nature Communications, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41467-025-58115-8">doi:10.1038/s41467-025-58115-8</a></p>Protein nanocages are multimeric molecular structures that can be engineered to mimic conformations of microorganisms. Here the authors display antigen and adjuvant on ferritin nanocage and use as a mucosal vaccine showing induction of protective immunity in a murine model of Streptococcus pneumoniae infection.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-57957-6">Benchmarking metagenomic binning tools on real datasets across sequencing platforms and binning modes</a></h4>

              
                <p>Nature Communications, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41467-025-57957-6">doi:10.1038/s41467-025-57957-6</a></p>Metagenomic binning is the process of grouping metagenomic sequences by their organism of origin. Here the authors benchmark 13 metagenomic binning tools using seven data-binning combinations on five real-world datasets.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58200-y">Synergistic activation of the human phosphate exporter XPR1 by KIDINS220 and inositol pyrophosphate</a></h4>

              
                <p>Nature Communications, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41467-025-58200-y">doi:10.1038/s41467-025-58200-y</a></p>XPR1 is the sole known phosphate exporter in human. Here, the authors provide structural evidence on how KIDINS220 and inositol pyrophosphate synergistically regulate the conformation and phosphate efflux activity of XPR1.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nphys/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nphys/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nphys/rss/current&title=Nature Physics" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41567-025-02839-3">Active membrane deformations of a minimal synthetic cell</a></h4>

              
                <p>Nature Physics, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41567-025-02839-3">doi:10.1038/s41567-025-02839-3</a></p>How cells actively change their shape is an open question. Now, a reconstituted minimal cytoskeleton composed of microtubules and molecular motors is shown to produce membrane fluctuations that drive active shape changes in synthetic cells.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nphys/rss/current">Nature Physics</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nphys/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nphys/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nphys/rss/current&title=Nature Physics" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41567-025-02807-x">Light-induced cortical excitability reveals programmable shape dynamics in starfish oocytes</a></h4>

              
                <p>Nature Physics, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41567-025-02807-x">doi:10.1038/s41567-025-02807-x</a></p>Optogenetically induced chemo-mechanical excitations are used to drive and study shape deformations in starfish oocytes. Understanding and eventually controlling such waves is important for the development of synthetic cells.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nphys/rss/current">Nature Physics</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04848-6">Open-source Raman spectra of chemical compounds for active pharmaceutical ingredient development</a></h4>

              
                <p>Scientific Data, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41597-025-04848-6">doi:10.1038/s41597-025-04848-6</a></p>Open-source Raman spectra of chemical compounds for active pharmaceutical ingredient development
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04810-6">Chromosome-level assembly of Prunus serrula Franch genome</a></h4>

              
                <p>Scientific Data, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41597-025-04810-6">doi:10.1038/s41597-025-04810-6</a></p>Chromosome-level assembly of <i>Prunus serrula</i> Franch genome
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04832-0">Single-cell light-sheet fluorescence 3D images of tumour-stroma spheroid multicultures</a></h4>

              
                <p>Scientific Data, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41597-025-04832-0">doi:10.1038/s41597-025-04832-0</a></p>Single-cell light-sheet fluorescence 3D images of tumour-stroma spheroid multicultures
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04824-0">A complete telomere-to-telomere chromosome-level genome assembly of X-ray tetra (Pristella maxillaris)</a></h4>

              
                <p>Scientific Data, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41597-025-04824-0">doi:10.1038/s41597-025-04824-0</a></p>A complete telomere-to-telomere chromosome-level genome assembly of X-ray tetra (<i>Pristella maxillaris</i>)
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04813-3">Chromosome-level de novo genome assembly of wild, anoxia-tolerant crucian carp, Carassius carassius</a></h4>

              
                <p>Scientific Data, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41597-025-04813-3">doi:10.1038/s41597-025-04813-3</a></p>Chromosome-level <i>de novo</i> genome assembly of wild, anoxia-tolerant crucian carp, <i>Carassius carassius</i>
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04816-0">An enhanced national-scale urban tree canopy cover dataset for the United States</a></h4>

              
                <p>Scientific Data, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41597-025-04816-0">doi:10.1038/s41597-025-04816-0</a></p>An enhanced national-scale urban tree canopy cover dataset for the United States
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04802-6">SCANIA Component X dataset: a real-world multivariate time series dataset for predictive maintenance</a></h4>

              
                <p>Scientific Data, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41597-025-04802-6">doi:10.1038/s41597-025-04802-6</a></p>SCANIA Component X dataset: a real-world multivariate time series dataset for predictive maintenance
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04789-0">Mobility of Erasmus+ students in Europe: Geolocated individual and aggregate mobility flows from 2014 to 2022</a></h4>

              
                <p>Scientific Data, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s41597-025-04789-0">doi:10.1038/s41597-025-04789-0</a></p>Mobility of Erasmus+ students in Europe: Geolocated individual and aggregate mobility flows from 2014 to 2022
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-07818-1">Toxic bait abandonment by an invasive ant is driven by aversive memories</a></h4>

              
                <p>Communications Biology, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s42003-025-07818-1">doi:10.1038/s42003-025-07818-1</a></p>Ant foragers actively abandon toxic baits, hampering control efforts. This abandonment process is driven at least in part by learning and memory, while aversive pheromones and passive mechanisms do not seem to play a role.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-07846-x">Integrative genomics sheds light on the immunogenetics of tuberculosis in cattle</a></h4>

              
                <p>Communications Biology, Published online: 24 March 2025; <a href="https://www.nature.com/articles/s42003-025-07846-x">doi:10.1038/s42003-025-07846-x</a></p>Integration of genome-wide SNP, peripheral blood transcriptome, and GWAS data highlights genes with expression patterns associated with bovine tuberculosis susceptibility in cattle.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/93373">Loss of CTRP10 results in female obesity with preserved metabolic health</a></h4>

              
                Obesity is a major risk factor for type 2 diabetes, dyslipidemia, cardiovascular disease, and hypertension. Intriguingly, there is a subset of metabolically healthy obese (MHO) individuals who are seemingly able to maintain a healthy metabolic profile free of metabolic syndrome. The molecular underpinnings of MHO, however, are not well understood. Here, we report that CTRP10/C1QL2-deficient mice represent a unique female model of MHO. CTRP10 modulates weight gain in a striking and sexually dimorphic manner. Female, but not male, mice lacking CTRP10 develop obesity with age on a low-fat diet while maintaining an otherwise healthy metabolic profile. When fed an obesogenic diet, female <i>Ctrp10</i> knockout (KO) mice show rapid weight gain. Despite pronounced obesity, <i>Ctrp10</i> KO female mice do not develop steatosis, dyslipidemia, glucose intolerance, insulin resistance, oxidative stress, or low-grade inflammation. Obesity is largely uncoupled from metabolic dysregulation in female KO mice. Multi-tissue transcriptomic analyses highlighted gene expression changes and pathways associated with insulin-sensitive obesity. Transcriptional correlation of the differentially expressed gene (DEG) orthologs in humans also shows sex differences in gene connectivity within and across metabolic tissues, underscoring the conserved sex-dependent function of CTRP10. Collectively, our findings suggest that CTRP10 negatively regulates body weight in females, and that loss of CTRP10 results in benign obesity with largely preserved insulin sensitivity and metabolic health. This female MHO mouse model is valuable for understanding sex-biased mechanisms that uncouple obesity from metabolic dysfunction.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/99678">Spatial localization of hippocampal replay requires dopamine signaling</a></h4>

              
                Sequenced reactivations of hippocampal neurons called replays, concomitant with sharp-wave ripples in the local field potential, are critical for the consolidation of episodic memory, but whether replays depend on the brain’s reward or novelty signals is unknown. Here, we combined chemogenetic silencing of dopamine neurons in ventral tegmental area (VTA) and simultaneous electrophysiological recordings in dorsal hippocampal CA1, in freely behaving male rats experiencing changes to reward magnitude and environmental novelty. Surprisingly, VTA silencing did not prevent ripple increases where reward was increased, but caused dramatic, aberrant ripple increases where reward was unchanged. These increases were associated with increased reverse-ordered replays. On familiar tracks this effect disappeared, and ripples tracked reward prediction error (RPE), indicating that non-VTA reward signals were sufficient to direct replay. Our results reveal a novel dependence of hippocampal replay on dopamine, and a role for a VTA-independent RPE signal that is reliable only in familiar environments.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/101973">ORMDL3 restrains type I interferon signaling and anti-tumor immunity by promoting RIG-I degradation</a></h4>

              
                Mounting evidence has demonstrated the genetic association of ORMDL sphingolipid biosynthesis regulator 3 (ORMDL3) gene polymorphisms with bronchial asthma and a diverse set of inflammatory disorders. However, its role in type I interferon (type I IFN) signaling remains poorly defined. Herein, we report that ORMDL3 is a negative modulator of the type I IFN signaling by interacting with mitochondrial antiviral signaling protein (MAVS) and subsequently promoting the proteasome-mediated degradation of retinoic acid-inducible gene I (RIG-I). Immunoprecipitation coupled with mass spectrometry (IP-MS) assays uncovered that ORMDL3 binds to ubiquitin-specific protease 10 (USP10), which forms a complex with and stabilizes RIG-I through decreasing its K48-linked ubiquitination. ORMDL3 thus disrupts the interaction between USP10 and RIG-I, thereby promoting RIG-I degradation. Additionally, subcutaneous syngeneic tumor models in C57BL/6 mice revealed that inhibition of ORMDL3 enhances anti-tumor efficacy by augmenting the proportion of cytotoxic CD8 positive T cells and IFN production in the tumor microenvironment (TME). Collectively, our findings reveal the pivotal roles of ORMDL3 in maintaining antiviral innate immune responses and anti-tumor immunity.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-03-24 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://www.cell.com/neuron/inpress.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://www.cell.com/neuron/inpress.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://www.cell.com/neuron/inpress.rss&title=Neuron: In press" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/neuron/fulltext/S0896-6273(25)00133-3?rss=yes">Single-cell spatial transcriptomic atlas of the whole mouse brain</a></h4>

              
                Han et al. used snRNA-seq and Stereo-seq to generate a detailed mouse brain atlas with over 4 million spatially resolved cells and 29,655 genes. Their findings include region- enriched cell clusters and genes, spatiotemporal dynamics of transcription factor regulons, and a gene-based parcellation framework, providing a comprehensive resource for brain architecture.
              

              <p class="text-muted">in <a href ="http://www.cell.com/neuron/inpress.rss">Neuron: In press</a> on 2025-03-23 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00217-7?rss=yes">Optimal functioning of the Lpt bridge depends on a ternary complex between the lipocalin YedD and the LptDE translocon</a></h4>

              
                Gennaris et al. identify YedD as a component of the Lpt system in Escherichia coli. Cryo-EM analysis reveals YedD interacting with the outer membrane LptDE translocon, with its binding proposed to stabilize the Lpt bridge. Deletion of yedD exacerbates LPS transport defects, highlighting its critical role in optimizing Lpt function.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-03-23 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00222-0?rss=yes">CaV2.1 mediates presynaptic dysfunction induced by amyloid β oligomers</a></h4>

              
                There is an urgent need for new targets for Alzheimer’s disease therapy. Jeans et al. identify a presynaptic signaling pathway engaged by Aβ oligomers that drives glutamatergic synaptic dysfunction via pathological enhancement of CaV2.1 Ca2+ channel activity and synaptic vesicle exocytosis. Lowering CaV2.1 expression restores normal synaptic function.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-03-23 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-07841-2">The dynamics of prebiotic take-off: the transfer of functional RNA communities from mineral surfaces to vesicles</a></h4>

              
                <p>Communications Biology, Published online: 23 March 2025; <a href="https://www.nature.com/articles/s42003-025-07841-2">doi:10.1038/s42003-025-07841-2</a></p>In silico simulations reveal compositional and dynamic differences between surface-bound and vesicle-enclosed prebiotic replicator communities with metabolic cooperation. The transition from surface to vesicle is found feasible and beneficial.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-03-23 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-07868-5">Both slow wave and rapid eye movement sleep contribute to emotional memory consolidation</a></h4>

              
                <p>Communications Biology, Published online: 23 March 2025; <a href="https://www.nature.com/articles/s42003-025-07868-5">doi:10.1038/s42003-025-07868-5</a></p>Examining the roles of rapid eye movement and slow wave sleep in emotional memory consolidation through targeted memory reactivation.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-03-23 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://onlinelibrary.wiley.com/action/showFeed?jc=15318249&type=etoc&feed=rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://onlinelibrary.wiley.com/action/showFeed?jc=15318249&type=etoc&feed=rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://onlinelibrary.wiley.com/action/showFeed?jc=15318249&type=etoc&feed=rss&title=Annals of Neurology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://onlinelibrary.wiley.com/doi/10.1002/ana.27241?af=R">Neurology in the Twenty‐First Century</a></h4>

              
                
              

              <p class="text-muted">in <a href ="https://onlinelibrary.wiley.com/action/showFeed?jc=15318249&type=etoc&feed=rss">Annals of Neurology</a> on 2025-03-22 11:10:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://onlinelibrary.wiley.com/action/showFeed?jc=15318249&type=etoc&feed=rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://onlinelibrary.wiley.com/action/showFeed?jc=15318249&type=etoc&feed=rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://onlinelibrary.wiley.com/action/showFeed?jc=15318249&type=etoc&feed=rss&title=Annals of Neurology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://onlinelibrary.wiley.com/doi/10.1002/ana.27237?af=R">Neuroimaging Biomarkers for Friedreich Ataxia: A Cross‐Sectional Analysis of the TRACK‐FA Study</a></h4>

              
                <h2>Objective</h2>
<p>We aimed to quantify differences in the brain and spinal cord between Friedreich ataxia and controls, stratified by age and disease stage, including for the first time in young children.</p>
<h2>Methods</h2>
<p>TRACK-FA is the largest prospective, longitudinal, multi-modal neuroimaging study in Friedreich ataxia to date. We assessed individuals with Friedreich ataxia and controls, 5 to 42 years, at 7 sites across 4 continents. The 17 imaging primary outcome measures (POMs) were selected from metrics that showed a significant longitudinal change in previous small-scale studies. These included brain and spinal cord morphometry (structural magnetic resonance imaging [MRI]) and microstructure (diffusion MRI); brain iron levels (quantitative susceptibility mapping); and spinal cord biochemistry (magnetic resonance spectroscopy). This study is registered with <a target="_blank"
   title="Link to external resource"
   href="http://clinicaltrials.gov">ClinicalTrials.gov</a> (NCT04349514).</p>
<h2>Results</h2>
<p>Between February 2021 and August 2023, we assessed 169 individuals with Friedreich ataxia and 95 controls. Compared to controls, individuals with Friedreich ataxia had lower volume of dentate nucleus and superior cerebellar peduncles; smaller cross-sectional area of spinal cord; lower fractional anisotropy and higher diffusivity in spinal cord and superior cerebellar peduncles; and lower total N-acetyl-aspartate/myo-inositol ratio in spinal cord. Morphometric differences in spinal cord and superior cerebellar peduncles increased dramatically with age during childhood, with rapid development in controls, but not in Friedreich ataxia. Many imaging POMs showed significant associations with clinical severity.</p>
<h2>Interpretation</h2>
<p>Our findings provide strong imaging evidence of impaired development of spinal cord and superior cerebellar peduncles during childhood in Friedreich ataxia and open the way for the use of neuroimaging biomarkers in clinical trials. ANN NEUROL 2025</p>
              

              <p class="text-muted">in <a href ="https://onlinelibrary.wiley.com/action/showFeed?jc=15318249&type=etoc&feed=rss">Annals of Neurology</a> on 2025-03-22 11:09:41 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00229-3?rss=yes">Inward transport of organelles drives outward migration of the spindle during C. elegans meiosis</a></h4>

              
                Aquino et al. find that kinesin-driven transport of mitochondria toward the center of a C. elegans oocyte is sufficient to drive movement of the meiotic spindle in the opposite direction to the periphery of the oocyte.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-03-22 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-07897-0">ZDHHC9-mediated CD38 palmitoylation stabilizes CD38 expression and promotes pancreatic cancer growth</a></h4>

              
                <p>Communications Biology, Published online: 22 March 2025; <a href="https://www.nature.com/articles/s42003-025-07897-0">doi:10.1038/s42003-025-07897-0</a></p>Understanding the palmitoylation of cluster of differentiation 38 (CD38), a transmembrane protein involved in aging and cancer, and accordingly the design of a competitive peptide are helpful for cancer treatment.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-03-22 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-07866-7">Open-transmit and flexible receiver array for high resolution ultrahigh-field fMRI of the human sensorimotor cortex</a></h4>

              
                <p>Communications Biology, Published online: 22 March 2025; <a href="https://www.nature.com/articles/s42003-025-07866-7">doi:10.1038/s42003-025-07866-7</a></p>An open-transmit and 24-channel flexible receiver head coil enhances ultrahigh-field fMRI, enabling sub-millimeter resolution for somatosensory and motor cortex mapping, advancing neuroscience and clinical neurology research.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-03-22 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-07827-0">Structural and virological identification of neutralizing antibody footprint provides insights into therapeutic antibody design against SARS-CoV-2 variants</a></h4>

              
                <p>Communications Biology, Published online: 22 March 2025; <a href="https://www.nature.com/articles/s42003-025-07827-0">doi:10.1038/s42003-025-07827-0</a></p>A SARS-CoV-2 antibody, NT-108, exhibits potent therapeutic effects in vivo. Cryo-EM structure of the spike complexed with single-chain Fv of NT108 reveals its neutralizing mechanism, providing insight into antibody therapeutic design.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-03-22 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-07848-9">GPR35 prevents osmotic stress induced cell damage</a></h4>

              
                <p>Communications Biology, Published online: 22 March 2025; <a href="https://www.nature.com/articles/s42003-025-07848-9">doi:10.1038/s42003-025-07848-9</a></p>GPR35 controls the transmembrane ion gradient by regulating Na/K-ATPase function with knock-on effects on osmosis, cellular stress and glutamine uptake.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-03-22 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://academic.oup.com/rss/site_5122/3011.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://academic.oup.com/rss/site_5122/3011.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://academic.oup.com/rss/site_5122/3011.xml&title=Cerebral Cortex" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://academic.oup.com/cercor/article/doi/10.1093/cercor/bhaf062/8090426?rss=1">A deep learning model for characterizing altered gyro-sulcal functional connectivity in abstinent males with methamphetamine use disorder and associated emotional symptoms</a></h4>

              
                <span class="paragraphSection"><div class="boxTitle">Abstract</div>Failure to manage emotional withdrawal symptoms can exacerbate relapse to methamphetamine use. Understanding the neuro-mechanisms underlying methamphetamine overuse and the associated emotional withdrawal symptoms is crucial for developing effective clinical strategies. This study aimed to investigate the distinct functional contributions of fine-scale gyro-sulcal signaling in the psychopathology of patients with methamphetamine use disorder and its associations with emotional symptoms. We recruited 48 male abstinent methamphetamine use disorders and 48 age- and gender-matched healthy controls, obtaining their resting-state functional magnetic resonance imaging data along with scores on anxiety and depressive symptoms. The proposed deep learning model, a spatio-temporal graph convolutional network utilizing gyro-sulcal subdivisions, achieved the highest average classification accuracy in distinguishing resting-state functional magnetic resonance imaging data of methamphetamine use disorders from healthy controls. Within this model, nodes in the lateral orbitofrontal cortex, and the habitual and executive control networks, contributed most significantly to the classification. Additionally, emotional symptom scores were negatively correlated with the sum of negative functional connectivity in the right caudal anterior cingulate sulcus and the functional connectivity between the left putamen and pallidum in methamphetamine use disorders. These findings provide novel insights into the differential functions of gyral and sulcal regions, enhancing our understanding of the neuro-mechanisms underlying methamphetamine use disorders.</span>
              

              <p class="text-muted">in <a href ="https://academic.oup.com/rss/site_5122/3011.xml">Cerebral Cortex</a> on 2025-03-22 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.eneuro.org/rss/current.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.eneuro.org/rss/current.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.eneuro.org/rss/current.xml&title=eNeuro" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="http://www.eneuro.org/cgi/content/short/12/3/ENEURO.0449-24.2025?rss=1">Alpha-Frequency Stimulation Enhances Synchronization of Alpha Oscillations with Default Mode Network Connectivity</a></h4>

              
                <p>Alpha (8&ndash;12&nbsp;Hz) oscillations and default mode network (DMN) activity dominate the brain's intrinsic activity in the temporal and spatial domains, respectively. They are thought to play crucial roles in the spatiotemporal organization of the complex brain system. Relatedly, both have been implicated, often concurrently, in diverse neuropsychiatric disorders, with accruing electroencephalogram (EEG)/magnetoencephalogram and functional magnetic resonance imaging (fMRI) data linking these two neural activities both at rest and during key cognitive operations. Prominent theories and extant findings thus converge to suggest a mechanistic relationship between alpha oscillations and the DMN. Here, we leveraged simultaneous EEG&ndash;fMRI data acquired before and after alpha-frequency transcranial alternating current stimulation (<I>&alpha;</I>-tACS) and observed that <I>&alpha;</I>-tACS tightened the dynamic coupling between spontaneous fluctuations in alpha power and DMN connectivity (especially, in the posterior DMN, between the posterior cingulate cortex and the bilateral angular gyrus). In comparison, no significant changes were observed for temporal correlations between power in other oscillatory frequencies and connectivity in other major networks. These results thus suggest an inherent coupling between alpha and DMN activity in humans. Importantly, these findings highlight the efficacy of <I>&alpha;</I>-tACS in regulating the DMN, a clinically significant network that is challenging to target directly with noninvasive methods.</p>
              

              <p class="text-muted">in <a href ="https://www.eneuro.org/rss/current.xml">eNeuro</a> on 2025-03-21 16:30:33 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.eneuro.org/rss/current.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.eneuro.org/rss/current.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.eneuro.org/rss/current.xml&title=eNeuro" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="http://www.eneuro.org/cgi/content/short/12/3/ENEURO.0578-24.2025?rss=1">Aniracetam Ameliorates Attention Deficit Hyperactivity Disorder Behavior in Adolescent Mice</a></h4>

              
                <p>Attention-deficit/hyperactivity disorder (ADHD) is a neurodevelopmental disorder that affects 8&ndash;12% of children globally. Hyperactivity-related behaviors, as well as inattention and impulsivity, are regarded as the nuclear symptoms of ADHD. At present, its etiologies and risk factors are unknown. Previous research linked TARP -8 deficiency to ADHD-like behaviors in mice, including hyperactivity, impulsivity, and memory deficits. Aniracetam, a nootropic drug, enhances cognition by modulating cholinergic activity and glutamate receptors, offering neuroprotective effects. This study examined TARP -8 knockout (KO) mice at 4 and 8 weeks, assessing behaviors through locomotor activity, cliff avoidance, novel object recognition, and contextual fear conditioning tests. TARP -8 KO mice exhibited hyperactivity, reduced recognition memory, and impaired short-term memory and long-term memory. Aniracetam administration improved these behavioral deficits, suggesting its potential as a therapeutic agent for ADHD. The findings align with ADHD&rsquo;s pathophysiology, resembling the neurological impairments in TARP -8 KO mice. Aniracetam shows promise as a novel treatment for ADHD symptoms, highlighting its therapeutic value.</p>
              

              <p class="text-muted">in <a href ="https://www.eneuro.org/rss/current.xml">eNeuro</a> on 2025-03-21 16:30:33 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-317/v1?src=rss">Electroencephalographic signatures during pregnancy and their role on parenting-related measures: a scoping review protocol [version 1; peer review: awaiting peer review]</a></h4>

              
                Objective This scoping review aims to map and synthesise the current research evidence targeting the electroencephalography (EEG) neural correlates during pregnancy and its association with parenting-related measures during pregnancy and postpartum period. Introduction Pregnancy is characterised by a wide range of biological changes associated with adaptation to parenthood. A growing body of literature has examined the neural correlates of pregnancy using EEG, revealing distinct patterns in pregnant females, with these EEG metrics changing throughout gestation and postpartum. Due to the heterogeneity of the evidence, the current literature lacks an organised synthesis, making it difficult to understand the neural correlates during pregnancy and their association with parenting-related measures during pregnancy and postpartum period. Inclusion criteria Studies will be included if they contain a quantitative EEG metric in their assessment in pregnant women. Studies will be included if they involve clinical or community samples. No sociodemographic, obstetric, or health exclusion criteria will be applied. Methods The scoping review will be conducted following the Joanna Briggs Institute’s (JBI) guidelines and will be reported following Preferred Reporting Items for Systematic Reviews and Meta-Analyses extension for Scoping Reviews (PRISMA-ScR) guidelines. Searches will be conducted in 7 databases (Cochrane for Trials and Reviews, PsycInfo, PubMed, Psychological and Behavioural Science Collection, Scopus and Web of Science Core Collection) for articles published in English, Portuguese, or Spanish with no limitation on the region or publication time frame. Two independent reviewers will screen each record following a standardised flowchart using asreview lab. Data extraction will be performed by the two reviewers using charting tables in Excel. Disagreements at any step will be resolved via consensus or by a third reviewer. Results will be reported using tables and graphs along with a descriptive analysis, according to the research questions.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-03-21 14:44:16 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-316/v1?src=rss">Whole Genome Characterization of Potential Isolates for Next-Generation SARS-CoV-2 Vaccine Development Utilizing Structural Protein of Multiple Variants [version 1; peer review: awaiting peer review]</a></h4>

              
                Background SARS-CoV has caused a high number of COVID-19 cases in Indonesia. COVID-19 cases continue to be reported, including incidents of the latest variant of SARS-CoV-2, Omicron. The development of vaccines remains essential to address potential future outbreaks. Methods We performed whole genome sequencing using Oxford Nanopore Technology and phylogenetic and mutational analysis using Molecular Evolutionary Genetics Analysis to identify potential isolates as a new platform vaccine candidate. Results Nine isolates have the potential to serve as candidates for the next generation COVID-19 vaccine. The sequencing test results of the nine selected isolates revealed that two samples were identified as the alpha variant, two as the delta variant, and five as the Omicron variant. Alpha variant has been identified as B.1 lineage, the delta variant with lineage AY.23, and the omicron variant with three lineages including BA.1.1, BA.5.2, and BQ.1.23 lineages. Phylogenetic analysis revealed five distinct classes: 20A, 21J, 21K, 22B, and 22E. The quantification of SARS-CoV-2 amino acid mutations in structural proteins revealed that the spike protein had the highest mutation percentage at 44.56%, followed by the N protein at 9.05%, the M protein at 5.54%, and the E protein at 1.78%. Conclusions It was concluded that nine isolates had potential to be developed as candidates for the next generation SARS-CoV-2 vaccine. These isolates include three variants: alpha, delta, and omicron.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-03-21 14:40:37 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/13-815/v3?src=rss">Factors Influencing Climate-Smart Agriculture Practices Adoption and Crop Productivity among Smallholder Farmers in Nyimba District, Zambia [version 3; peer review: 1 approved, 1 approved with reservations, 1 not approved]</a></h4>

              
                Climate change significantly affects smallholder farmers, whose livelihoods are closely tied to the environment. This study explores factors influencing the adoption of climate-smart agriculture (CSA) practices and their impact on crop productivity among small-scale farmers in Nyimba District, Zambia. Data were collected from 194 households across 12 villages, and logistic regression and propensity score matching analyses were employed to identify key factors and evaluate CSA’s effects on crop yields. Findings revealed that CSA adoption is influenced by factors such as education level, household size, fertilizer use, age, gender, farming experience, livestock ownership, income, farmland size, marital status, and access to climate-related information. CSA adopters experienced a 20.20% increase in overall crop yields compared to non-adopters, with a 21.50% increase in maize yields specifically. The study underscores the need for targeted interventions to support CSA adoption through education, improved dissemination of climate information, and access to critical resources such as improved seeds and financial services. This research offers insights for policymakers and extension services to develop evidence-based strategies enhancing resilience and productivity among smallholder farmers in response to climate challenges.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-03-21 14:38:53 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-315/v1?src=rss">Dark Matter: From Dark Quarks to Dark Black Holes [version 1; peer review: awaiting peer review]</a></h4>

              
                This paper explores the theoretical existence and implications of dark quarks and their role in forming dark hadrons, dark neutron stars, and dark black holes. By simplifying the relationship between hadron charge and quark number, we propose a more transparent and theoretically elegant framework for understanding the charge structure of fundamental particles. We introduce a classification system that distinguishes between bright and dark particles based on their charge and color charge properties, leading to potential extensions in particle physics models. Furthermore, we investigate the nature of dark black holes, differentiating them from bright black holes, and discuss their potential observational signatures in astrophysics. This study provides a theoretical foundation for future explorations of dark matter and its interactions in the universe.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-03-21 14:37:10 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/plosbiology/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/plosbiology/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/plosbiology/NewArticles&title=PLoS Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3003005">Navigation strategies in Caenorhabditis elegans are differentially altered by learning</a></h4>

              
                <p>by Kevin S Chen, Anuj K Sharma, Jonathan W Pillow, Andrew M Leifer</p>

Learned olfactory-guided navigation is a powerful platform for studying how a brain generates goal-directed behaviors. However, the quantitative changes that occur in sensorimotor transformations and the underlying neural circuit substrates to generate such learning-dependent navigation is still unclear. Here we investigate learned sensorimotor processing for navigation in the nematode <i>Caenorhabditis elegans</i> by measuring and modeling experience-dependent odor and salt chemotaxis. We then explore the neural basis of learned odor navigation through perturbation experiments. We develop a novel statistical model to characterize how the worm employs two behavioral strategies: a biased random walk and weathervaning. We infer weights on these strategies and characterize sensorimotor kernels that govern them by fitting our model to the worm’s time-varying navigation trajectories and precise sensory experiences. After olfactory learning, the fitted odor kernels reflect how appetitive and aversive trained worms up- and down-regulate both strategies, respectively. The model predicts an animal’s past olfactory learning experience with  > 90<i>%</i> accuracy given finite observations, outperforming a classical chemotaxis metric. The model trained on natural odors further predicts the animals’ learning-dependent response to optogenetically induced odor perception. Our measurements and model show that behavioral variability is altered by learning—trained worms exhibit less variable navigation than naive ones. Genetically disrupting individual interneuron classes downstream of an odor-sensing neuron reveals that learned navigation strategies are distributed in the network. Together, we present a flexible navigation algorithm that is supported by distributed neural computation in a compact brain.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/plosbiology/NewArticles">PLoS Biology</a> on 2025-03-21 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/13-1182/v2?src=rss">Economic factors influencing the empowerment of Peruvian women [version 2; peer review: 1 approved, 1 approved with reservations]</a></h4>

              
                Objective To determine whether economic factors are crucial in empowering women, guiding them towards growth and development opportunities, achieving empowerment, and contributing to two sustainable development goals of the 2030 development agenda: ending poverty and achieving gender equality. Methodology The research was foundational, with a phenomenological and hermeneutic design. The applied technique was in-depth interviews with 12 women who had started a business within the last five years in a region of Peru. Results It is evident that economic factors are decisive in business experiences and decisions, highlighting the necessity of having contingency funds to prevent operational impacts. Through entrepreneurship, women achieved economic independence, enabling them to support their families and impacting their empowerment. It concludes that to promote economic opportunity equality, addressing financing needs, encouraging economic independence, strengthening family empowerment, improving customer management, and facilitating access to government funds are essential. Conclusions The narrative of the participants provides a solid foundation for designing specific policies and support programs that boost the economic empowerment of women entrepreneurs and encourage their active participation in the business sphere.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-03-21 12:19:12 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://onlinelibrary.wiley.com/action/showFeed?jc=15318249&type=etoc&feed=rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://onlinelibrary.wiley.com/action/showFeed?jc=15318249&type=etoc&feed=rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://onlinelibrary.wiley.com/action/showFeed?jc=15318249&type=etoc&feed=rss&title=Annals of Neurology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://onlinelibrary.wiley.com/doi/10.1002/ana.27239?af=R">Letter on Wine Glass Sign in Bulbar‐Onset Amyotrophic Lateral Sclerosis</a></h4>

              
                
              

              <p class="text-muted">in <a href ="https://onlinelibrary.wiley.com/action/showFeed?jc=15318249&type=etoc&feed=rss">Annals of Neurology</a> on 2025-03-21 11:59:38 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/12-849/v3?src=rss">Development and acceptability of a cereal bar with Atta sexdens ant flour [version 3; peer review: 1 approved, 2 approved with reservations]</a></h4>

              
                In Peru, insect consumption, as a nutritional complement or as the main source in the diet, is limited to the regions of our the central jungle where Atta sexdens ants are consumed. An energy bar based on Andean grains with Atta sexdens ant flour was formulated. The ants were obtained from the department of San Martin, district of Rioja, province of Rioja. Four different formulations were prepared with different Atta sexdens ant flour concentrations: 15%, 20%, 25%, and 30%. Moisture, total fat, ash, proteins, fiber, carbohydrates, instrumental texture, and organoleptic analysis (taste, texture, and color) were performed. The developed cereal energy bar presented a 10 g protein content in 100 g of the final product. Acceptability was evaluated in adolescents and young adult populations. The developed cereal bars presented a high protein content, adequate organoleptic properties and high acceptability. However, this must be considered in the context of the rising cost of protein-rich foods.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-03-21 11:42:54 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/11-1393/v2?src=rss">Effectiveness of a mHealth application on remote monitoring and self-management of persons with hypertension in a coastal taluk of Udupi district: A study protocol for a community-based Cluster Randomized trial [version 2; peer review: 1 approved with res</a></h4>

              
                Background Hypertension is a significant risk factor for cardiovascular disease, contributing to global mortality and disability. Approximately 30% of Indian adults are diagnosed with hypertension. Evidence supports that self-monitoring and blood pressure self-management can lower systolic BP by an average of 3.2 mmHg. mHealth applications facilitate remote monitoring and self-management, yet existing applications in India lack customisation for user needs, limiting their usability. This study aims to develop and evaluate a novel, user-friendly mHealth application tailored to hypertensive individuals. Methods This study follows an Agile development design, an iterative software development approach that allows continuous feedback and refinement. The research will be conducted in three phases over an anticipated duration of 27 months. Phase 1 (6 months) will involve in-depth interviews and focus group discussions to identify essential features for a customised Android-based mHealth application. Phase 2 (9 months) will involve developing the application using Android Studio following Agile principles. Phase 3 (12 months) will be a community-based cluster randomised trial conducted in 12 villages to evaluate the application’s effectiveness. Villages will be randomised into intervention and control groups. Each group will include 118 participants. The intervention group will use the mHealth application, while the control group will follow the standard hypertension management regimen. Villages and participants will be selected based on specific criteria, including population size, availability of healthcare facilities, and smartphone accessibility among hypertensive patients. Results In the proposed study, if the intervention is helpful, hypertension patients in the community can be encouraged to use the mHealth application. If found effective, this application is anticipated to improve hypertensive patients’ health status, knowledge, and self-care approach. Registration: Clinical Trials Registry - India (CTRI/2022/03/041544).
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-03-21 09:49:29 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adk3248?af=R">Regulatory T cells constrain T cells of shared specificity to enforce tolerance during infection</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/387/6740">Volume 387, Issue 6740</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.ads5704?af=R">Epithelial polarization by the planar cell polarity complex is exclusively non–cell autonomous</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/387/6740">Volume 387, Issue 6740</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adl2373?af=R">Unifying spatial scaling laws of biodiversity and ecosystem stability</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/387/6740">Volume 387, Issue 6740</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.ado8316?af=R">Synaptic architecture of a memory engram in the mouse hippocampus</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/387/6740">Volume 387, Issue 6740</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.ads2659?af=R">The Sikkim flood of October 2023: Drivers, causes, and impacts of a multihazard cascade</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/387/6740">Volume 387, Issue 6740</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ads6312?af=R">Time-resolved tracking of cellulose biosynthesis and assembly during cell wall regeneration in live Arabidopsis protoplasts</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adr4004?af=R">Stereoregular radical polymers enable selective spin transfer</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ads5458?af=R">SIP2 is the master transcription factor of Plasmodium merozoite formation</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adu1153?af=R">Simultaneous head-mounted imaging of neural and hemodynamic activities at high spatiotemporal resolution in freely behaving mice</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adt0318?af=R">Self-powered electrotactile textile haptic glove for enhanced human-machine interface</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adv0095?af=R">Chiral ruthenium complex/Ph2P(2-furyl)–catalyzed asymmetric nucleophilic addition of aryl aldehyde hydrazones to simple ketones</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adq0332?af=R">Distinct attentional characteristics of neurons with visual feature coding in the primate brain</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adt7899?af=R">Purely organic room-temperature phosphorescence sensitizers for highly efficient hyperfluorescence OLEDs</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adi9885?af=R">Pathway-instructed therapeutic selection of ruxolitinib reduces neuroinflammation in fungal postinfectious inflammatory syndrome</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adt3909?af=R">Lensless imaging with a programmable Fresnel zone aperture</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ado2223?af=R">Skin regional specification and higher-order HoxC regulation</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adr5973?af=R">Ultrafast energy transfer beyond the Förster approximation in organic photovoltaic blends with non-fullerene acceptors</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ads4415?af=R">Soft, stretchable conductive hydrogels for high-performance electronic implants</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adt5888?af=R">Particle-armored liquid robots</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adt2117?af=R">CD38-mediated metabolic reprogramming promotes the stability and suppressive function of regulatory T cells in tumor</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ads3689?af=R">Imaging and simulation of surface plasmon polaritons on layered 2D MXenes</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ads8149?af=R">Paramagnetic susceptibility measured by magnetic resonance imaging as an in vivo biomarker for iron pathology in epilepsy</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adt9159?af=R">Ultranarrow-linewidth wavelength-vortex metasurface holography</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ads1388?af=R">Nanosecond-pulsed electroluminescence from high current–driven quantum-dot light-emitting diodes</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adt3518?af=R">A low-temperature, one-step synthesis for monazite can transform monazite into a readily usable advanced nuclear waste form</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adn8402?af=R">AMPK-dependent Parkin activation suppresses macrophage antigen presentation to promote tumor progression</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adq9952?af=R">High-temperature surface state in Kondo insulator U3Bi4Ni3</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ads7083?af=R">Eliminating nonuniform geometric effects for long-term stable electrochemical extraction of high-purity titanium</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adq7741?af=R">Distortion-free sampling of ultrabroadband terahertz electric fields by interfacial spin accumulation</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ads3660?af=R">Unveiling the complexity of Arnold’s tongues in a breathing-soliton laser</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adr1538?af=R">Nicotinamide riboside and nicotinamide mononucleotide facilitate NAD+ synthesis via enterohepatic circulation</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ads9581?af=R">Selective promotion of sensory innervation–mediated immunoregulation for tissue repair</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ads9182?af=R">A metabolic shift to the serine pathway induced by lipids fosters epigenetic reprogramming in nontransformed breast cells</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adt5369?af=R">Climate warming and heatwaves accelerate global lake deoxygenation</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adp2386?af=R">Evidence and quantification of memory effects in competitive first-passage events</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adp9052?af=R">Emergence of a senescent and inflammatory pulmonary CD4+ T cell population prior to lung allograft failure</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adx1775?af=R">Erratum for the Research Article: “Ex vivo expansion and hydrogel-mediated in vivo delivery of tissue-resident memory T cells for immunotherapy” by S. Li et al.</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adx1795?af=R">Erratum for the Research Article: “Combinatorial screening of biochemical and physical signals for phenotypic regulation of stem cell–based cartilage tissue engineering” by J. Lee et al.</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/12">Volume 11, Issue 12</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-03-21 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://www.cell.com/trends/neurosciences/inpress.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://www.cell.com/trends/neurosciences/inpress.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://www.cell.com/trends/neurosciences/inpress.rss&title=Trends in Neurosciences: In press" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/trends/neurosciences/fulltext/S0166-2236(25)00040-2?rss=yes">Somatic mosaicism and interneuron involvement in mTORopathies</a></h4>

              
                Somatic mutations in genes regulating mechanistic target of rapamycin (mTOR) pathway signaling can cause epilepsy, autism, and cognitive dysfunction. Research has predominantly focused on mTOR regulation of excitatory neurons in these conditions; however, dysregulated mTOR signaling among interneurons may also be critical. In this review, we discuss clinical evidence for interneuron involvement, and potential mechanisms, known and hypothetical, by which interneurons might come to directly harbor pathogenic mutations. To understand how mTOR hyperactive interneurons might drive dysfunction, we review studies in which mTOR signaling has been selectively disrupted among interneurons and interneuron progenitors in mouse model systems. Complex cellular mosaicism and dual roles for mTOR (hyper)activation in mediating disease pathogenesis and homeostatic responses raise challenging questions for effective treatment of these disorders.
              

              <p class="text-muted">in <a href ="http://www.cell.com/trends/neurosciences/inpress.rss">Trends in Neurosciences: In press</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://www.cell.com/trends/neurosciences/inpress.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://www.cell.com/trends/neurosciences/inpress.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://www.cell.com/trends/neurosciences/inpress.rss&title=Trends in Neurosciences: In press" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/trends/neurosciences/fulltext/S0166-2236(25)00037-2?rss=yes">Awake replay: off the clock but on the job</a></h4>

              
                Hippocampal replay is widely thought to support two key cognitive functions: online decision-making and offline memory consolidation. In this review, we take a closer look at the hypothesized link between awake replay and online decision-making in rodents, and find only marginal evidence in support of this role. By contrast, the consolidation view is bolstered by new computational ideas and recent data, suggesting that (i) replay performs offline fictive learning for later goal-oriented behavior; and (ii) replay tags memories prior to sleep, prioritizing them for consolidation. Based on these recent advances, we favor an updated and refined role for awake replay – that is, supporting prioritized offline learning and tagging outside the hippocampus – rather than a direct, online role in guiding behavior.
              

              <p class="text-muted">in <a href ="http://www.cell.com/trends/neurosciences/inpress.rss">Trends in Neurosciences: In press</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://www.cell.com/trends/neurosciences/inpress.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://www.cell.com/trends/neurosciences/inpress.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://www.cell.com/trends/neurosciences/inpress.rss&title=Trends in Neurosciences: In press" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/trends/neurosciences/fulltext/S0166-2236(25)00036-0?rss=yes">A role for respiration in coordinating sleep oscillations and memory consolidation</a></h4>

              
                Memory consolidation is thought to rely on the interplay of sleep-related brain oscillations. Drawing on recent findings that highlight the influence of respiration on these rhythms, we outline a framework positioning respiration as pacemaker for sleep’s memory function. By orchestrating the cardinal non-rapid eye movement (NREM) oscillations, namely slow oscillations, spindles, and sharp wave-ripples, respiration may coordinate the hippocampo-cortical crosstalk essential for memory consolidation.
              

              <p class="text-muted">in <a href ="http://www.cell.com/trends/neurosciences/inpress.rss">Trends in Neurosciences: In press</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00227-X?rss=yes">The metabolic sensor LKB1 regulates ILC3 homeostasis and mitochondrial function</a></h4>

              
                ILC3s are essential for intestinal defense; however, how metabolism affects their function has remained elusive. Fonseca-Pereira et al. show that LKB1 controls ILC3 function in a cell-intrinsic manner following its activation by upstream cytokine signals. LKB1 inhibits ROS production and PD-1 expression to promote mitochondrial function and sustain IL-22 production.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00290-6?rss=yes">The Natural Compound Notopterol Binds and Targets JAK2/3 to Ameliorate Inflammation and Arthritis</a></h4>

              
                (Cell Reports 32, 108158; September 15, 2020)
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41586-025-08904-4">Author Correction: Heritable polygenic editing: the next frontier in genomic medicine?</a></h4>

              
                <p>Nature, Published online: 21 March 2025; <a href="https://www.nature.com/articles/s41586-025-08904-4">doi:10.1038/s41586-025-08904-4</a></p>Author Correction: Heritable polygenic editing: the next frontier in genomic medicine?
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41586-025-08905-3">Author Correction: AI models collapse when trained on recursively generated data</a></h4>

              
                <p>Nature, Published online: 21 March 2025; <a href="https://www.nature.com/articles/s41586-025-08905-3">doi:10.1038/s41586-025-08905-3</a></p>Author Correction: AI models collapse when trained on recursively generated data
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41586-025-08872-9">Publisher Correction: A metagenomic ‘dark matter’ enzyme catalyses oxidative cellulose conversion</a></h4>

              
                <p>Nature, Published online: 21 March 2025; <a href="https://www.nature.com/articles/s41586-025-08872-9">doi:10.1038/s41586-025-08872-9</a></p>Publisher Correction: A metagenomic ‘dark matter’ enzyme catalyses oxidative cellulose conversion
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00769-x">Ship-pollution cuts have an electrifying effect: less lightning at sea</a></h4>

              
                <p>Nature, Published online: 21 March 2025; <a href="https://www.nature.com/articles/d41586-025-00769-x">doi:10.1038/d41586-025-00769-x</a></p>A crackdown on sulfur in ship fuel is linked to a steep drop in thunderclouds in the Indian Ocean and South China Sea.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00840-7">‘Boggles the mind’: US defence department slashes research on emerging threats</a></h4>

              
                <p>Nature, Published online: 21 March 2025; <a href="https://www.nature.com/articles/d41586-025-00840-7">doi:10.1038/d41586-025-00840-7</a></p>Terminated projects include studies on the implications of artificial intelligence in combat and how extremism spreads online.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00849-y">‘All this is in crisis’: US universities curtail staff and spending as Trump cuts take hold</a></h4>

              
                <p>Nature, Published online: 21 March 2025; <a href="https://www.nature.com/articles/d41586-025-00849-y">doi:10.1038/d41586-025-00849-y</a></p>Furloughs and more staff reductions loom as academic institutions contend with the prospect of even deeper cuts to federal funding.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00828-3">NIH has cut one mRNA-vaccine grant. Will more follow?</a></h4>

              
                <p>Nature, Published online: 21 March 2025; <a href="https://www.nature.com/articles/d41586-025-00828-3">doi:10.1038/d41586-025-00828-3</a></p>Concern rises among researchers after agency officials ask for a list of projects funded to study the jabs.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/neuro/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/neuro/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/neuro/rss/current&title=Nature Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41593-025-01941-2">Author Correction: Mitochondrial respiratory complex IV deficiency recapitulates amyotrophic lateral sclerosis</a></h4>

              
                <p>Nature Neuroscience, Published online: 21 March 2025; <a href="https://www.nature.com/articles/s41593-025-01941-2">doi:10.1038/s41593-025-01941-2</a></p>Author Correction: Mitochondrial respiratory complex IV deficiency recapitulates amyotrophic lateral sclerosis
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/neuro/rss/current">Nature Neuroscience</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nphoton/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nphoton/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nphoton/rss/current&title=Nature Photomics" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41566-025-01644-x">Elucidating the photodegradation pathways of polymer donors for organic solar cells with seven months of outdoor operational stability</a></h4>

              
                <p>Nature Photonics, Published online: 21 March 2025; <a href="https://www.nature.com/articles/s41566-025-01644-x">doi:10.1038/s41566-025-01644-x</a></p>A systematic study of 15 non-fullerene-based organic solar cells elucidates loss mechanisms and enables an encapsulated device to retain 91% of its initial efficiency after seven months of outdoor operation in Saudi Arabian climate.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nphoton/rss/current">Nature Photomics</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nphoton/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nphoton/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nphoton/rss/current&title=Nature Photomics" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41566-025-01633-0">Three-dimensional photonic integration for ultra-low-energy, high-bandwidth interchip data links</a></h4>

              
                <p>Nature Photonics, Published online: 21 March 2025; <a href="https://www.nature.com/articles/s41566-025-01633-0">doi:10.1038/s41566-025-01633-0</a></p>Dense three-dimensional integration of photonics and electronics results in a high-speed (800 Gb s−1) data interface for semiconductor chips that features 80 communication channels and consumes only tens of femtojoules per transmitted bit.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nphoton/rss/current">Nature Photomics</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nmeth/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nmeth/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nmeth/rss/current&title=Nature Methods" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41592-025-02621-6">Connectivity of single neurons classifies cell subtypes in mouse brains</a></h4>

              
                <p>Nature Methods, Published online: 21 March 2025; <a href="https://www.nature.com/articles/s41592-025-02621-6">doi:10.1038/s41592-025-02621-6</a></p>This Resource presents a method to define connectivity types of neurons based on a spatially registered large database containing more than 20,000 neuronal reconstructions. A brain connectivity map is also generated using such connectivity features.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nmeth/rss/current">Nature Methods</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nphys/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nphys/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nphys/rss/current&title=Nature Physics" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41567-025-02811-1">Deterministic remote entanglement using a chiral quantum interconnect</a></h4>

              
                <p>Nature Physics, Published online: 21 March 2025; <a href="https://www.nature.com/articles/s41567-025-02811-1">doi:10.1038/s41567-025-02811-1</a></p>Large quantum computers are likely to require methods of connecting devices by transmitting and absorbing photons. Entanglement between two superconducting qubit devices has now been established using a waveguide with tunable directionality.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nphys/rss/current">Nature Physics</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/106838">Correction: Sodium taurocholate cotransporting polypeptide is a functional receptor for human hepatitis B and D virus</a></h4>

              
                -/-
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/92525">Emergence of ion-channel-mediated electrical oscillations in Escherichia coli biofilms</a></h4>

              
                Bacterial biofilms are communities of bacteria usually attached to solid strata and often differentiated into complex structures. Communication across biofilms has been shown to involve chemical signaling and, more recently, electrical signaling in Gram-positive biofilms. We report for the first time, community-level synchronized membrane potential dynamics in three-dimensional <i>Escherichia coli</i> biofilms. Two hyperpolarization events are observed in response to light stress. The first requires mechanically sensitive ion channels (MscK, MscL, and MscS) and the second needs the Kch-potassium channel. The channels mediated both local spiking of single <i>E. coli</i> biofilms and long-range coordinated electrical signaling in <i>E. coli</i> biofilms. The electrical phenomena are explained using Hodgkin-Huxley and 3D fire-diffuse-fire agent-based models. These data demonstrate that electrical wavefronts based on potassium ions are a mechanism by which signaling occurs in Gram-negative biofilms and as such may represent a conserved mechanism for communication across biofilms.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/92892">The neural correlates of novelty and variability in human decision-making under an active inference framework</a></h4>

              
                Active inference integrates perception, decision-making, and learning into a united theoretical framework, providing an efficient way to trade off exploration and exploitation by minimizing (expected) free energy. In this study, we asked how the brain represents values and uncertainties (novelty and variability), and resolves these uncertainties under the active inference framework in the exploration-exploitation trade-off. Twenty-five participants performed a contextual two-armed bandit task, with electroencephalogram (EEG) recordings. By comparing the model evidence for active inference and reinforcement learning models of choice behavior, we show that active inference better explains human decision-making under novelty and variability, which entails exploration or information seeking. The EEG sensor-level results show that the activity in the frontal, central, and parietal regions is associated with novelty, while the activity in the frontal and central brain regions is associated with variability. The EEG source-level results indicate that the expected free energy is encoded in the frontal pole and middle frontal gyrus and uncertainties are encoded in different brain regions but with overlap. Our study dissociates the expected free energy and uncertainties in active inference theory and their neural correlates, speaking to the construct validity of active inference in characterizing cognitive processes of human decisions. It provides behavioral and neural evidence of active inference in decision processes and insights into the neural mechanism of human decisions under uncertainties.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/102019">Zinc finger homeobox-3 (ZFHX3) orchestrates genome-wide daily gene expression in the suprachiasmatic nucleus</a></h4>

              
                The mammalian suprachiasmatic nucleus (SCN), situated in the ventral hypothalamus, directs daily cellular and physiological rhythms across the body. The SCN clockwork is a self-sustaining transcriptional-translational feedback loop (TTFL) that in turn coordinates the expression of clock-controlled genes (CCGs) directing circadian programmes of SCN cellular activity. In the mouse, the transcription factor, ZFHX3 (zinc finger homeobox-3), is necessary for the development of the SCN and influences circadian behaviour in the adult. The molecular mechanisms by which ZFHX3 affects the SCN at transcriptomic and genomic levels are, however, poorly defined. Here, we used chromatin immunoprecipitation sequencing to map the genomic localization of ZFHX3-binding sites in SCN chromatin. To test for function, we then conducted comprehensive RNA sequencing at six distinct times-of-day to compare the SCN transcriptional profiles of control and ZFHX3-conditional null mutants. We show that the genome-wide occupancy of ZFHX3 occurs predominantly around gene transcription start sites, co-localizing with known histone modifications, and preferentially partnering with clock transcription factors (CLOCK, BMAL1) to regulate clock gene(s) transcription. Correspondingly, we show that the conditional loss of ZFHX3 in the adult has a dramatic effect on the SCN transcriptome, including changes in the levels of transcripts encoding elements of numerous neuropeptide neurotransmitter systems while attenuating the daily oscillation of the clock TF <i>Bmal1</i>. Furthermore, various TTFL genes and CCGs exhibited altered circadian expression profiles, consistent with an advanced in daily behavioural rhythms under 12 h light–12 h dark conditions. Together, these findings reveal the extensive genome-wide regulation mediated by ZFHX3 in the central clock that orchestrates daily timekeeping in mammals.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-03-21 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-314/v1?src=rss">Spectrum of lower respiratory tract infections and antimicrobial resistance pattern in head and neck cancer patients undergoing chemoradiation [version 1; peer review: awaiting peer review]</a></h4>

              
                Background Head and neck cancer (HNCs) is a major health issue worldwide, and India has contributed to approximately 2.4 lakh new cases in 2022. Definitive chemoradiation is the standard treatment for locally advanced disease but carries a risk of lower respiratory tract infections (LRTI) that add to morbidity, hospitalization, cost, and possible delay in treatment. The increasing incidence of antimicrobial resistance (AMR) has also contributed to management burden. This study aimed to assess the microbiological profiles and antimicrobial resistance patterns of lower respiratory tract infections in patients with head and neck cancer receiving chemoradiation. Methods Patients who underwent definitive radiotherapy with or without chemotherapy and who developed LRTIs were included in the study. Sputum and tracheostomy suction tip cultures were obtained and processed using standard microbiological techniques such as Gram staining, biochemical tests, and VITEK-2 automated systems. Antimicrobial susceptibility was tested according to the Clinical and Laboratory Standards Institute (CLSI) and EUCAST recommendations. Clinical and treatment-related factors were documented and compared using SPSS version 23.0, with descriptive statistics, chi-square tests, t-tests, ANOVA, and logistic regression models. Results Pseudomonas aeruginosa was the most frequently isolated pathogen (35.0%), followed by Klebsiella pneumoniae (16.7%), and Acinetobacter baumannii (10.0%). The pathogens were strongly resistant to fluoroquinolones and third-generation cephalosporins but were susceptible to carbapenems and aminoglycosides in the majority of isolates. Pseudomonas aeruginosa was the most frequent pathogen in all age groups and chemotherapy regimens (p<0.001). Conclusion LRTIs in patients with HNC treated with chemoradiation were mainly caused by multidrug-resistant Pseudomonas aeruginosa and Klebsiella pneumoniae. Resistance patterns are crucial for directing empirical antibiotic therapy, minimizing treatment delays, and enhancing clinical outcomes.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-03-20 17:56:42 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-313/v1?src=rss">Demographic, clinical, pathologic and treatment pattern of soft tissue sarcoma at Tikur Anbessa specialized hospital, Ethiopia - retrospective cross-sectional study [version 1; peer review: awaiting peer review]</a></h4>

              
                Background Soft-tissue sarcomas are an uncommon group of neoplasms that can be fatal, especially in metastatic settings. Thus, this study aimed to assess the demographic, pathological, clinical, and treatment patterns of soft tissue sarcoma at the Tikur Anbesa Specialized Hospital in Ethiopia. Methods A retrospective cross-sectional study was conducted on patients with soft tissue sarcomas who visited the Tihur Anbesa Specialized Hospital, Ethiopia, between August 2017 and August 2021GC. A total of 190 samples were taken. Data from health management information system logbooks and medical records were collected, with only histopathology-confirmed cases included, while those with incomplete data were excluded. Analysis was performed using SPSS version 25, and ethical clearance was obtained from Addis Ababa University College of Health Sciences. Results A total of 320 patients were registered between August 2017 and August 2021; 190 patients were available for this study, and 130 were lost. The median age was 32 years of age and male sex was commonly affected, accounting 61.6%, and female sex accounted 38.4%. The common sites were extremities, accounting for 48.9% of cases. The Commonest presenting symptoms were mass and pain, accounting for 93.2% and 77.9% of cases, respectively. The common histological subtypes were undifferentiated sarcoma, rhabdomyosarcoma, and synovial sarcoma, accounting 50%, 15.8%, and 6.3% cases, respectively. The grade of the lesions was described in 67.3% of patients. Of the lesions whose grade was described, 56.8% were high grade lesions. In all cases, 58.3% of the patients underwent surgery as the primary treatment modality. Only 50.5% of cases were started treatment with curative intent, and 49.5% of the patients started their treatment with palliative treatment. Conclusions The pattern of soft tissue sarcomas showed a heterogeneous distribution in terms of demography, clinical presentation, treatment, and pathological subtypes. The advanced stage of initial clinical presentation and substandard work-up and therapy make it unique from cases reported in other parts of the world.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-03-20 17:53:58 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.eneuro.org/rss/current.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.eneuro.org/rss/current.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.eneuro.org/rss/current.xml&title=eNeuro" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="http://www.eneuro.org/cgi/content/short/12/3/ENEURO.0103-24.2024?rss=1">Paired Stimulation of Different Digits for 30 min Does Not Produce Long-Term Plastic Changes in the Human Cutaneomuscular Reflex</a></h4>

              
                <p>Cutaneomuscular reflexes (CMRs) can be recorded in the hand muscle of human subjects after stimulation of a digital nerve. We hypothesized that repeated synchronous stimulation of nerves from two digits may lead to long-term plastic changes in CMR, by the mechanisms of spike-timing&ndash;dependent plasticity (STDP). To test this idea, we conducted experiments in 27 healthy human volunteers. After baseline measurement of CMR, one of four 30-min-long stimulation conditions were tested; the CMR was then remeasured. The four conditions were simultaneous index finger and thumb stimulation; asynchronous index finger and thumb stimulation; thumb 5&nbsp;ms before index finger stimulation; and thumb-only stimulation. Neither the early (E1) nor late excitatory (E2) components of the CMR showed consistent changes after any stimulation condition. The inhibitory (I1) component was slightly reduced in all cases. To understand why paired stimulation did not produce long-term changes, we conducted a further experiment. In this, we measured the CMR in response to simultaneous stimulation of index finger and thumb, compared with a prediction expected if the responses summed linearly. This revealed sublinear summation, possibly indicating partial response saturation after stimulation of only one digit. We argue such a pattern prevents paired stimuli from generating especially reliable and well-timed outputs relative to synaptic inputs in downstream neurons, which is required to produce plasticity by STDP.</p>
              

              <p class="text-muted">in <a href ="https://www.eneuro.org/rss/current.xml">eNeuro</a> on 2025-03-20 16:30:36 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.eneuro.org/rss/current.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.eneuro.org/rss/current.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.eneuro.org/rss/current.xml&title=eNeuro" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="http://www.eneuro.org/cgi/content/short/12/3/ENEURO.0370-24.2025?rss=1">Postmovement Beta Synchronization Induced by Speed Effects on IHI from the Ipsilateral to Contralateral Motor Cortex</a></h4>

              
                <p>Beta event-related spectral perturbation, including bilateral movement-related beta desynchronization (MRBD) and postmovement beta synchronization (PMBS), can be evoked by unilateral speed movement. A potential correlation might exist between power (de)synchronization and interhemispheric coherence during movement execution. However, during the PMBS phase, the existence of interhemispheric coupling and the effect of speed on it are largely undiscovered. To answer this question, we investigated eight healthy, right-handed volunteers using a combination of electroencephalography, transcranial magnetic stimulation, and electromyography. We explored interhemispheric (directed) coherence during isotonic right index finger abduction movements at two speeds: ballistic and self-paced. We discovered that (1) interhemispheric coherence was greater during the PMBS than during the MRBD period. Furthermore, ballistic movement induced a larger coherence during the PMBS period, but not during the MRBD period. (2) In the MRBD phase, directed coherence from the contralateral motor cortex (CM1) to the ipsilateral motor cortex (IM1) was larger, with a reverse tendency observed during the PMBS period. Additionally, in ballistic movement, directed coherence from IM1 to CM1 was stronger and positively correlated with coherence, with no effect of speed on directed coherence detected in the MRBD phase. To explore the causality of interhemispheric coherence during the PMBS period, we investigated the interhemispheric inhibition (IHI) from IM1 to CM1 at different speeds. A stronger IHI from IM1 to CM1 at PMBS peak time was demonstrated, which was enhanced during ballistic movement. Additionally, IHI was negatively correlated with PMBS, and movement speed was positively associated with interhemispheric coupling during the PMBS period and IHI from IM1 to CM1.</p>
              

              <p class="text-muted">in <a href ="https://www.eneuro.org/rss/current.xml">eNeuro</a> on 2025-03-20 16:30:36 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012893">How immunity shapes the long-term dynamics of influenza H3N2</a></h4>

              
                <p>by Oliver Eales, Freya M. Shearer, James M. McCaw</p>

Since its emergence in 1968, influenza A H3N2 has caused yearly epidemics in temperate regions. While infection confers immunity against antigenically similar strains, new antigenically distinct strains that evade existing immunity regularly emerge (‘antigenic drift’). Immunity at the individual level is complex, depending on an individual's lifetime infection history. An individual's first infection with influenza typically elicits the greatest response with subsequent infections eliciting progressively reduced responses (‘antigenic seniority’). The combined effect of individual-level immune responses and antigenic drift on the epidemiological dynamics of influenza are not well understood. Here we develop an integrated modelling framework of influenza transmission, immunity, and antigenic drift to show how individual-level exposure, and the build-up of population level immunity, shape the long-term epidemiological dynamics of H3N2. Including antigenic seniority in the model, we observe that following an initial decline after the pandemic year, the average annual attack rate increases over the next 80 years, before reaching an equilibrium, with greater increases in older age-groups. Our analyses suggest that the average attack rate of H3N2 is still in a growth phase. Further increases, particularly in the elderly, may be expected in coming decades, driving an increase in healthcare demand due to H3N2 infections.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-03-20 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012843">Unknotting RNA: A method to resolve computational artifacts</a></h4>

              
                <p>by Simón Poblete, Mikolaj Mlynarczyk, Marta Szachniuk</p>

RNA 3D structure prediction often encounters entanglements, computational artifacts that complicate structural models, resulting in their exclusion from further studies despite the potentially accurate prediction of regions outside the entanglement. This study presents a protocol aimed at resolving such issues in RNA models while preserving the overall 3D fold and structural integrity. By employing the SPQR coarse-grained model and short Molecular Dynamics simulations, the protocol imposes energy terms that enable selective modifications to disentangle structures without causing significant distortions. The method was validated on 195 entangled RNA models from CASP15 and RNA-Puzzles, successfully resolving over 70% of interlaces and approximately 40% of lassos, with minimal impact on the original geometry but notable improvement in ClashScore. The efficiency of untangling conformations that are unequivocally classified as artifacts is 81%. Certain cases, particularly those involving dense packing of atoms or complex secondary structures, posed challenges that limited the efficiency of the method. In this paper, we present quantitative results from the application of the protocol and discuss examples of both successfully disentangled and unresolved structures. We show a viable approach for refining models previously deemed unsuitable due to topological artifacts.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-03-20 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012818">Gauge fixing for sequence-function relationships</a></h4>

              
                <p>by Anna Posfai, Juannan Zhou, David M. McCandlish, Justin B. Kinney</p>

Quantitative models of sequence-function relationships are ubiquitous in computational biology, e.g., for modeling the DNA binding of transcription factors or the fitness landscapes of proteins. Interpreting these models, however, is complicated by the fact that the values of model parameters can often be changed without affecting model predictions. Before the values of model parameters can be meaningfully interpreted, one must remove these degrees of freedom (called “gauge freedoms” in physics) by imposing additional constraints (a process called “fixing the gauge”). However, strategies for fixing the gauge of sequence-function relationships have received little attention. Here we derive an analytically tractable family of gauges for a large class of sequence-function relationships. These gauges are derived in the context of models with all-order interactions, but an important subset of these gauges can be applied to diverse types of models, including additive models, pairwise-interaction models, and models with higher-order interactions. Many commonly used gauges are special cases of gauges within this family. We demonstrate the utility of this family of gauges by showing how different choices of gauge can be used both to explore complex activity landscapes and to reveal simplified models that are approximately correct within localized regions of sequence space. The results provide practical gauge-fixing strategies and demonstrate the utility of gauge-fixing for model exploration and interpretation.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-03-20 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/plosbiology/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/plosbiology/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/plosbiology/NewArticles&title=PLoS Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3003076">Enhanced memory despite severe sleep loss in Drosophila insomniac mutants</a></h4>

              
                <p>by Sheng Huang, Chengji Piao, Zhiying Zhao, Christine B. Beuschel, Oriane Turrel, David Toppe, Stephan J. Sigrist</p>

Sleep is crucial for cognitive functions and life span across species. While sleep homeostasis and cognitive processes are linked through cellular and synaptic plasticity, the signaling pathways connecting them remain unclear. Here, we show that <i>Drosophila insomniac</i> (<i>inc</i>) short sleep mutants, which lack an adaptor protein for the autism-associated Cullin-3 ubiquitin ligase, exhibited enhanced Pavlovian aversive olfactory learning and memory, unlike other sleep mutants with normal or reduced memory. Through a genetic modifier screen, we found that a mild reduction of Protein Kinase A (PKA) signaling specifically rescued the sleep and longevity phenotypes of <i>inc</i> mutants. However, this reduction further increased their excessive memory and mushroom body overgrowth. Since <i>inc</i> mutants displayed higher PKA signaling, we propose that <i>inc</i> loss-of-function suppresses sleep via increased PKA activity, which also constrains the excessive memory of <i>inc</i> mutants. Our data identify a signaling cascade for balancing sleep and memory functions, and provide a plausible explanation for the sleep phenotypes of <i>inc</i> mutants, suggesting that memory hyperfunction can provoke sleep deficits.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/plosbiology/NewArticles">PLoS Biology</a> on 2025-03-20 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/plosbiology/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/plosbiology/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/plosbiology/NewArticles&title=PLoS Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3003061">In vivo sonic hedgehog pathway antagonism temporarily results in ancestral proto-feather-like structures in the chicken</a></h4>

              
                <p>by Rory L. Cooper, Michel C. Milinkovitch</p>

The morphological intricacies of avian feathers make them an ideal model for investigating embryonic patterning and morphogenesis. In particular, the sonic hedgehog (Shh) pathway is an important mediator of feather outgrowth and branching. However, functional in vivo evidence regarding its role during feather development remains limited. Here, we demonstrate that an intravenous injection of sonidegib, a potent Shh pathway inhibitor, at embryonic day 9 (E9) temporarily produces striped domains (instead of spots) of <i>Shh</i> expression in the skin, arrests morphogenesis, and results in unbranched and non-invaginated feather buds—akin to proto-feathers—in embryos until E14. Although feather morphogenesis partially recovers, hatched treated chickens exhibit naked skin regions with perturbed follicles. Remarkably, these follicles are subsequently reactivated by seven weeks post-hatching. Our RNA-sequencing data and rescue experiment using Shh-agonism confirm that sonidegib specifically down-regulates Shh pathway activity. Overall, we provide functional evidence for the role of the Shh pathway in mediating feather morphogenesis and confirm its role in the evolutionary emergence and diversification of feathers.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/plosbiology/NewArticles">PLoS Biology</a> on 2025-03-20 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml&title=Physical Review E: Biological physics" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="http://link.aps.org/doi/10.1103/PhysRevE.111.034407">From noisy cell size control to population growth: When variability can be beneficial</a></h4>

              
                <p>Author(s): Arthur Genthon</p><p>This paper demonstrates a method to determine the dependence of population growth on single-cell variability. The authors consider fluctuations in single-cell growth rate, size added, and size partitioning. Results reveal how different growth control mechanisms affect population growth.</p>
<p>#BiophysicsSpotlight</p><img src="//cdn.journals.aps.org/journals/PRE/key_images/10.1103/PhysRevE.111.034407.png" width="200" height=\"100\"><br/><p>[Phys. Rev. E 111, 034407] Published Thu Mar 20, 2025</p>
              

              <p class="text-muted">in <a href ="http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml">Physical Review E: Biological physics</a> on 2025-03-20 10:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-312/v1?src=rss">What Are the Characteristics of PhD-holding Government Officials Responsible for Science and Technology Policy in Japan? [version 1; peer review: awaiting peer review]</a></h4>

              
                Background The Japanese central government’s human resource system is a “closed career-based system,” meaning that some ministries/agencies appoint government officials with PhDs in the field of Natural Science to positions responsible for policymaking and comprehensive coordination, similar to administrative officials who are executive candidates, rather than the positions requiring scientific and specialized knowledge. This study aims to clarify the characteristics of PhD-holding government officials, involved in the policymaking process, who possess expertise based on their doctoral degree in the field of Natural Sciences, yet are placed within the same personnel system as administrative officials. The study also aims to provide insights for future personnel systems. Methods A questionnaire survey was distributed among government officials responsible for science and technology policy. The questionnaire focused on tasks such as overall coordination, policymaking, and other general administrative duties. The analysis utilized descriptive statistics based on the perception data collected through the survey, and hypotheses derived from prior studies on administrative expertise were tested through statistical analysis using ordinary least squares (OLS). Result The results of the questionnaire survey revealed no significant differences between PhD-holding officials and others in terms of their perceptions of coordination tasks or the importance of evidence; however, regarding policymaking, PhD-holding officials showed a notable tendency to find greater fulfillment in formulating new policies. The OLS analysis, with the number of new policy proposals as the dependent variable, showed that the presence of a doctoral degree was found to have a significant impact. Conclusions This study suggests that PhD-holding officials utilize their specialized literacy when proposing new policies. The findings imply that leveraging doctoral-holding personnel with not only expertise in scientific knowledge but also specialized literacy and general skills in public institutions could lead to the creation of new innovations.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-03-20 09:28:15 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-311/v1?src=rss">Mitigating retail rice price volatility for sustainable supply chains: an optimization and regression-based approach [version 1; peer review: awaiting peer review]</a></h4>

              
                Background This study addresses the challenge of stabilizing rice retail prices in Indonesia, a critical staple food across many Asian countries. The government prioritizes price stability to prevent sharp increases that could lead to social unrest due to rice shortages. Approaches to price control are categorized as direct or indirect. Direct controls involve immediate interventions, such as increasing rice stocks through imports to rapidly influence market prices. Indirect controls comprises longer-term measures, like enhancing domestic production capacity to achieve gradual price stability. This study proposes optimization model to determine the optimal volume of rice imports required to minimize bimonthly changes in rice retail prices. Methods A linear programming model is formulated to minimize bimonthly price changes, subject to constraints including local production capacity, import limits, rice flow balance, and demand fulfillment. The monthly retail price is modeled using a compound linear regression approach with ten explanatory variables: rice production, the rupiah-dollar exchange rate, GDP per capita, agricultural land area, harvested dry rice price per kilogram, ground dry rice price per kilogram, domestic rice procurement, rice imports, rice stocks, and monthly rice distribution. The explanatory variable is forecasted using methods best suited to its historical pattern. Results The model was tested using data from 2020 to 2023. The results indicate that bimonthly rice prices increases can be effectively controlled, with maximum inflation rates maintained between 0.42% and 0.53% and a standard deviation ranging from 0.39% to 0.53%. These values are significantly lower than the anticipated inflation rate of 2–3%. Conclusions The optimization model effectively determines the required volume of rice imports to minimize bimonthly retail price fluctuations. By regulating import volumes, excessive price increases can be prevented. Enhanced data-driven forecasting with granular historical data may further improve the accuracy of retail rice price predictions and strengthen price stabilization initiatives.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-03-20 09:26:39 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro&title=PNAS" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.pnas.org/doi/abs/10.1073/pnas.2409325122?af=R">Trans-synaptic modulation of cholinergic circuits tunes opioid reinforcement</a></h4>

              
                Proceedings of the National Academy of Sciences, Volume 122, Issue 12, March 2025. <br/>SignificanceThis study explores the molecular and synaptic changes in the nucleus accumbens occurring upon opioid exposure, crucial for understanding reward processing and addiction. We spotlight the pivotal role of a cell adhesion molecule, ELFN1, for ...
              

              <p class="text-muted">in <a href ="https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro">PNAS</a> on 2025-03-20 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adx3819?af=R">Erratum for the Report “A Werner syndrome stem cell model unveils heterochromatin alterations as a driver of human aging” by W. Zhang et al.</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/387/6740">Volume 387, Issue 6740</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-03-20 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adw5211?af=R">Artificial intelligence learns to reason</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/387/6740">Volume 387, Issue 6740</a>, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-03-20 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00622.2024?af=R">Comprehensive analysis of human dendritic spine morphology and density</a></h4>

              
                Journal of Neurophysiology, <a href="https://journals.physiology.org/toc/jn/133/4">Volume 133, Issue 4</a>, Page 1086-1102, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-03-20 06:20:40 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adv8256?af=R">Uncertainty abounds, what now?</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/387/6740">Volume 387, Issue 6740</a>, Page 1261-1261, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-03-20 05:58:02 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adw6760?af=R">The animal apothecaries</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/387/6740">Volume 387, Issue 6740</a>, Page 1260-1260, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-03-20 05:58:02 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.ado8680?af=R">Apoplastic barriers are essential for nodule formation and nitrogen fixation in Lotus japonicus</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/387/6740">Volume 387, Issue 6740</a>, Page 1281-1286, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-03-20 05:58:02 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adq4921?af=R">Cognitive perception of circulating oxygen in seals is the reason they don’t drown</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/387/6740">Volume 387, Issue 6740</a>, Page 1276-1280, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-03-20 05:58:02 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adr7075?af=R">Error-detected quantum operations with neutral atoms mediated by an optical cavity</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/387/6740">Volume 387, Issue 6740</a>, Page 1301-1305, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-03-20 05:58:02 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.ads8738?af=R">In-cell architecture of the mitochondrial respiratory chain</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/387/6740">Volume 387, Issue 6740</a>, Page 1296-1301, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-03-20 05:58:02 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adq3278?af=R">Macroecological rules predict how biomass scales with species richness in nature</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/387/6740">Volume 387, Issue 6740</a>, Page 1272-1276, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-03-20 05:58:02 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adp7290?af=R">Direct effects of mountain uplift and topography on biodiversity</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/387/6740">Volume 387, Issue 6740</a>, Page 1287-1291, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-03-20 05:58:02 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adt7570?af=R">Hippocampal encoding of memories in human infants</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/387/6740">Volume 387, Issue 6740</a>, Page 1316-1320, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-03-20 05:58:02 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adk1637?af=R">Recent gains in global terrestrial carbon stocks are mostly stored in nonliving pools</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/387/6740">Volume 387, Issue 6740</a>, Page 1291-1295, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-03-20 05:58:02 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adx4658?af=R">My crisis Ph.D.</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/387/6740">Volume 387, Issue 6740</a>, Page 1322-1322, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-03-20 05:58:02 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adx5175?af=R">In Other Journals</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/387/6740">Volume 387, Issue 6740</a>, Page 1265-1266, March 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-03-20 05:58:02 UTC.
              </p>
            </li>
          <!-- each item -->
        </ul>
      </div>
      <div class="col-md-3">
        <div class="panel panel-default">
          <div class="panel-heading">
            <h3 class="panel-title">Feed list</h3>
          </div>
          <ul class="list-group">
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/computational-neuroscience'>Frontiers in Computational Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/aging-neuroscience'>Frontiers in Ageing Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/evolutionary-neuroscience/rss'>Frontiers in Evolutionary Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/human-neuroscience/rss'>Frontiers in Human Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/molecular-neuroscience/rss'>Frontiers in Molecular Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/neuroenergetics/rss'>Frontiers in Neurogenetics</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/neuroengineering/rss'>Frontiers in Neuroengineering</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/systems-neuroscience/rss'>Frontiers in Systems Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/synaptic-neuroscience/rss'>Frontiers in Synaptic Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/cellular-neuroscience/rss'>Frontiers in Cellular Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/integrative-neuroscience/rss'>Frontiers in Integrative Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/neural-circuits/rss'>Frontiers in Neural Circuits</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/neuroanatomy/rss'>Frontiers in Neuroanatomy</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/neuroinformatics/rss'>Frontiers in Neuroinformatics</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://www.cell.com/trends/neurosciences/inpress.rss'>Trends in Neurosciences: In press</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://www.cell.com/neuron/current.rss'>Neuron: Current Issue</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://www.cell.com/neuron/inpress.rss'>Neuron: In press</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.cell.com/cell-reports/current.rss'>Cell Reports: Current Issue</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.cell.com/cell-reports/inpress.rss'>Cell Reports: In press</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.plos.org/ploscompbiol/NewArticles'>PLoS Computational Biology</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.plos.org/plosbiology/NewArticles'>PLoS Biology</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/nature/rss/current'>Nature</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/neuro/rss/current'>Nature Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/nrn/rss/current'>Nature Reviews</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/nphoton/rss/current'>Nature Photomics</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/nmeth/rss/current'>Nature Methods</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/ncomms/rss/current'>Nature Communications</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/nphys/rss/current'>Nature Physics</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.nature.com/sdata.rss'>Nature scientific data</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.nature.com/commsbio.rss'>Nature communications biology</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://elifesciences.org/rss/recent.xml'>eLife</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://onlinelibrary.wiley.com/action/showFeed?jc=15318249&type=etoc&feed=rss'>Annals of Neurology</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://onlinelibrary.wiley.com/feed/10969861/most-recent'>Journal of Comparative Neurology</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://onlinelibrary.wiley.com/action/showFeed?jc=10974547&type=etoc&feed=rss'>Journal of Neuroscience Research</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.jneurosci.org/rss/current.xml'>Journal of Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://bmcneurosci.biomedcentral.com/articles/most-recent/rss.xml'>BMC Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro'>PNAS</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://direct.mit.edu/neco'>Neural Computation</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://direct.mit.edu/netn'>Network Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience'>bioRxiv: Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://export.arxiv.org/rss/q-bio.NC/'>arXiv: Quantitative Biology: Neurons and Cognition</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://export.arxiv.org/rss/cs.NE'>arXiv: Computer Science: Neural and Evolutionary Computing</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://rss.sciencedirect.com/publication/science/09594388'>Current Opinion in Neurobiology</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://rss.sciencedirect.com/publication/science/08936080'>Neural Networks</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://academic.oup.com/rss/site_5122/3011.xml'>Cerebral Cortex</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.eneuro.org/rss/current.xml'>eNeuro</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn'>Journal of Neurophysiology</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science'>Science</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv'>Science Advances</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://onlinelibrary.wiley.com/feed/10981063/most-recent'>Hippocampus</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://rescience.github.io/feed'>ReScience C</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml'>Physical Review E: Biological physics</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://mathematical-neuroscience.springeropen.com/articles/most-recent/rss.xml'>The Journal of Mathematical Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://behavioralandbrainfunctions.biomedcentral.com/articles/most-recent/rss.xml'>Behavioural and Brain Functions</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.mdpi.com/rss/journal/brainsci'>Brain Sciences</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://f1000research.com/published/rss'>F1000Research</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://journals.sagepub.com/action/showFeed?ui=0&mi=ehikzz&ai=2b4&jc=nroa&type=etoc&feed=rss'>The Neuroscientist</a>
            </li>
           
           </ul>
        </div>
      </div>
    </div>
    <footer id="site-footer">
      <p class="text-center text-muted">All content on this page is owned by their respective owners. The source code used to generate this page can be found <a href="https://github.com/neurofedora/planet-neuroscience">here</a>.</p>
      </div>
    </footer>
    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <script type="text/javascript">
      $( document ).ready(function(){
        $("img").addClass("img-responsive");
        $("figure").addClass("img-responsive");
      });
    </script>
    <!-- Include all compiled plugins (below), or include individual files as needed -->
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
  </body>
</html>

