<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
  <head>
    <title>Planet Neuroscience</title>
    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet"> 
    <link href="https://fonts.googleapis.com/css?family=Comfortaa" rel="stylesheet">
    <!-- Google analytics -->
    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-60261100-2', 'auto');
      ga('send', 'pageview');

    </script>
  </head>

  <body>
    <nav class="navbar navbar-default navbar-static-top">
      <div class="container">
        <div class="navbar-header">
          <a class="navbar-brand" href="#">Planet Neuroscience</a>
        </div>
          <ul class="nav navbar-nav navbar-right">
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">More Neuroscience <span class="caret"></span></a>
              <ul class="dropdown-menu">
                <li><a href="http://neuroblog.fedoraproject.org/planet-neuroscientists/">Planet Neuroscientists</a></li>
                <li><a href="http://home.earthlink.net/~perlewitz/index.html">Computational Neuroscience on the web</a></li>
              </ul>
            </li>
            <li class="dropdown">
              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Options <span class="caret"></span></a>
              <ul class="dropdown-menu">
                <li><a href="https://github.com/neurofedora/planet-neuroscience/issues/new">Suggest a new feed</a></li>
                <li><a href="https://github.com/neurofedora/planet-neuroscience">View Planet source</a></li>
                <li role="separator" class="divider"></li>
                <li><a href="https://github.com/feedreader/pluto">View Pluto source</a></li>
              </ul>
            </li>
          </ul>
        </div><!--/.nav-collapse -->
      </div>
    </nav>
    <div class="container">
      <div class="page-header">
        <h1>Planet Neuroscience</h1>
        <h2>An aggregation of RSS feeds from various neuroscience journals.</h2>
        <p>
        last updated by <a href="https://feedreader.github.io">Pluto</a>
        on 2025-04-08 08:19:00 UTC 
        on behalf of <a href="https://neuro.fedoraproject.org" target="_blank"> the NeuroFedora SIG</a>.
        </p>
      </div>
    </div>

    <div class="container">
      <div class="col-md-9">
        <ul class="list-group">
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/q-bio.NC/>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/q-bio.NC/" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/q-bio.NC/&title=arXiv: Quantitative Biology: Neurons and Cognition" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2504.04329">Modeling the Dynamics of Attentional Gamma Oscillations During the Encoding Process of Noise-Mixed Speech Signals</a></h4>

              
                arXiv:2504.04329v1 Announce Type: new 
Abstract: The brain's bottom-up loop for processing speech influx involves both the selective attention and the encoding of specific speech information. Previous human studies have found that such attention can be represented by the cortical gamma-rhythm oscillations. However, the underlying mechanisms remain unclear. To address this issue, this paper proposes a neural network model that incorporates speech signal input, the cochlea, the thalamus, and a balanced excitatory-inhibitory cortical neural network, with the aim of connecting real speech signals to brain cortical responses. Using this model, we explored neural oscillation patterns in response to mixed speech stimuli and background noise. The findings revealed that the peak of gamma oscillation decreased as the frequency of the pure-tone stimuli diminished. This suggests a strong correlation and coding role of gamma oscillation peaks in auditory attention. Similar results were confirmed by analyzing the rhythmic oscillations of EEG data in response to pure-tone signals. Further results indicated that dynamic gamma oscillations are involved in the encoding capacity of continuous speech input. The coding entropy of the dynamic series was found to be proportional to the complexity of the content. This suggests that gamma oscillations play multiple roles, not only in sustaining the bottom-up attentional state but also in potentially conveying specific information from external speech inputs. Finally, we found that enhancing the excitatory-inhibitory balance level properly could improve auditory attention. This finding provides a potential endogenous explanation for the dynamic switching process of brain attention in processing auditory signals.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/q-bio.NC/">arXiv: Quantitative Biology: Neurons and Cognition</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/q-bio.NC/>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/q-bio.NC/" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/q-bio.NC/&title=arXiv: Quantitative Biology: Neurons and Cognition" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2504.03681">End-to-End Deep Learning for Real-Time Neuroimaging-Based Assessment of Bimanual Motor Skills</a></h4>

              
                arXiv:2504.03681v1 Announce Type: cross 
Abstract: The real-time assessment of complex motor skills presents a challenge in fields such as surgical training and rehabilitation. Recent advancements in neuroimaging, particularly functional near-infrared spectroscopy (fNIRS), have enabled objective assessment of such skills with high accuracy. However, these techniques are hindered by extensive preprocessing requirements to extract neural biomarkers. This study presents a novel end-to-end deep learning framework that processes raw fNIRS signals directly, eliminating the need for intermediate preprocessing steps. The model was evaluated on datasets from three distinct bimanual motor tasks--suturing, pattern cutting, and endotracheal intubation (ETI)--using performance metrics derived from both training and retention datasets. It achieved a mean classification accuracy of 93.9% (SD 4.4) and a generalization accuracy of 92.6% (SD 1.9) on unseen skill retention datasets, with a leave-one-subject-out cross-validation yielding an accuracy of 94.1% (SD 3.6). Contralateral prefrontal cortex activations exhibited task-specific discriminative power, while motor cortex activations consistently contributed to accurate classification. The model also demonstrated resilience to neurovascular coupling saturation caused by extended task sessions, maintaining robust performance across trials. Comparative analysis confirms that the end-to-end model performs on par with or surpasses baseline models optimized for fully processed fNIRS data, with statistically similar (p<0.05) or improved prediction accuracies. By eliminating the need for extensive signal preprocessing, this work provides a foundation for real-time, non-invasive assessment of bimanual motor skills in medical training environments, with potential applications in robotics, rehabilitation, and sports.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/q-bio.NC/">arXiv: Quantitative Biology: Neurons and Cognition</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/q-bio.NC/>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/q-bio.NC/" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/q-bio.NC/&title=arXiv: Quantitative Biology: Neurons and Cognition" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2504.04560">Beyond catastrophic forgetting in associative networks with self-interactions</a></h4>

              
                arXiv:2504.04560v1 Announce Type: cross 
Abstract: Spin-glass models of associative memories are a cornerstone between statistical physics and theoretical neuroscience. In these networks, stochastic spin-like units interact through a synaptic matrix shaped by local Hebbian learning. In absence of self-interactions (i.e., autapses), the free energy reveals catastrophic forgetting of all stored patterns when their number exceeds a critical memory load. Here, we bridge the gap with biology by considering networks of deterministic, graded units coupled via the same Amari-Hopfield synaptic matrix, while retaining autapses. Contrary to the assumption that self-couplings play a negligible role, we demonstrate that they qualitatively reshape the energy landscape, confining the recurrent dynamics to the subspace hosting the stored patterns. This allows for the derivation of an exact overlap-dependent Lyapunov function, valid even for networks with finite size. Moreover, self-interactions generate an auxiliary internal field aligned with the target memory pattern, widening the repertoire of accessible attractor states. Consequently, pure recall states act as robust associative memories for any memory load, beyond the critical threshold for catastrophic forgetting observed in spin-glass models -- all without requiring nonlocal learning prescriptions or significant reshaping of the Hebbian synaptic matrix.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/q-bio.NC/">arXiv: Quantitative Biology: Neurons and Cognition</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/q-bio.NC/>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/q-bio.NC/" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/q-bio.NC/&title=arXiv: Quantitative Biology: Neurons and Cognition" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2109.11632">Retinotopic Mechanics derived using classical physics</a></h4>

              
                arXiv:2109.11632v3 Announce Type: replace 
Abstract: The concept of a cell$'$s receptive field is a bedrock in systems neuroscience, and the classical static description of the receptive field has had enormous success in explaining the fundamental mechanisms underlying visual processing. Borne out by the spatio-temporal dynamics of visual sensitivity to probe stimuli in primates, I build on top of this static account with the introduction of a new computational field of research, retinotopic mechanics. At its core, retinotopic mechanics assumes that during active sensing receptive fields are not static but can shift beyond their classical extent. Specifically, the canonical computations and the neural architecture that supports these computations are inherently mediated by a neurobiologically inspired force field (e.g.,$R_s\propto \sim 1 /\Delta M$). For example, when the retina is displaced because of a saccadic eye movement from one point in space to another, cells across retinotopic brain areas are tasked with discounting the retinal disruptions such active surveillance inherently introduces. This neural phenomenon is known as spatial constancy. Using retinotopic mechanics, I propose that to achieve spatial constancy or any active visually mediated task, retinotopic cells, namely their receptive fields, are constrained by eccentricity dependent elastic fields. I propose that elastic fields are self-generated by the visual system and allow receptive fields the ability to predictively shift beyond their classical extent to future post-saccadic location such that neural sensitivity which would otherwise support intermediate eccentric locations likely to contain retinal disruptions is transiently blunted.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/q-bio.NC/">arXiv: Quantitative Biology: Neurons and Cognition</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/q-bio.NC/>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/q-bio.NC/" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/q-bio.NC/&title=arXiv: Quantitative Biology: Neurons and Cognition" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2408.08408">Oja's plasticity rule overcomes several challenges of training neural networks under biological constraints</a></h4>

              
                arXiv:2408.08408v3 Announce Type: replace 
Abstract: Deep neural networks have achieved impressive performance through carefully engineered training strategies. Nonetheless, such methods lack parallels in biological neural circuits, relying heavily on non-local credit assignment, precise initialization, normalization layers, batch processing, and large datasets. Biologically plausible plasticity rules, such as random feedback alignment, often suffer from instability and unbounded weight growth without these engineered methods, while Hebbian-type schemes fail to provide goal-oriented credit. In this study, we demonstrate that incorporating Oja's plasticity rule into error-driven training yields stable, efficient learning in feedforward and recurrent architectures, obviating the need for carefully engineered tricks. Our results show that Oja's rule preserves richer activation subspaces, mitigates exploding or vanishing signals, and improves short-term memory in recurrent networks. Notably, meta-learned local plasticity rules incorporating Oja's principle not only match but surpass standard backpropagation in data-scarce regimes. These findings reveal a biologically grounded pathway bridging engineered deep networks and plausible synaptic mechanisms.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/q-bio.NC/">arXiv: Quantitative Biology: Neurons and Cognition</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/q-bio.NC/>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/q-bio.NC/" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/q-bio.NC/&title=arXiv: Quantitative Biology: Neurons and Cognition" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2504.01887">Dual mechanism of Anti-Seizure Medications in controlling seizure activity</a></h4>

              
                arXiv:2504.01887v2 Announce Type: replace 
Abstract: Background: Anti-seizure medications (ASMs) can reduce seizure duration, but their precise modes of action are unclear. Specifically, it is unknown whether ASMs shorten seizures by simply compressing existing seizure activity into a shorter time frame or by selectively suppressing certain seizure activity patterns.
  Methods: We analysed intracranial EEG (iEEG) recordings of 457 seizures from 28 people with epilepsy undergoing ASM tapering. Beyond measuring seizure occurrence and duration, we categorized distinct seizure activity patterns (states) based on spatial and frequency power characteristics and related these to different ASM levels.
  Results: We found that reducing ASM levels led to increased seizure frequency (r = 0.87, p < 0.001) and longer seizure duration ($\beta$ = -0.033, p < 0.001), consistent with prior research. Further analysis revealed two distinct mechanisms in which seizures became prolonged:
  Emergence of new seizure patterns - In approx. 40% of patients, ASM tapering unmasked additional seizure activity states, and seizures containing these 'taper-emergent states' were substantially longer (r = 0.49, p < 0.001).
  Prolongation of existing seizure patterns - Even in seizures without taper-emergent states, lower ASM levels still resulted in approx. 12-224% longer durations depending on the ASM dosage and tapering ($\beta$ = -0.049, p < 0.001).
  Conclusion: ASMs influence seizures through two mechanisms: they (i) suppress specific seizure activity patterns (states) in an all-or-nothing fashion and (ii) curtail the duration of other seizure patterns. These findings highlight the complex role of ASMs in seizure modulation and could inform personalized dosing strategies for epilepsy management. These findings may also have implications in understanding the effects of ASMs on cognition and mood.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/q-bio.NC/">arXiv: Quantitative Biology: Neurons and Cognition</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/q-bio.NC/>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/q-bio.NC/" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/q-bio.NC/&title=arXiv: Quantitative Biology: Neurons and Cognition" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2406.09067">Interpreting the structure of multi-object representations in vision encoders</a></h4>

              
                arXiv:2406.09067v3 Announce Type: replace-cross 
Abstract: In this work, we interpret the representations of multi-object scenes in vision encoders through the lens of structured representations. Structured representations allow modeling of individual objects distinctly and their flexible use based on the task context for both scene-level and object-specific tasks. These capabilities play a central role in human reasoning and generalization, allowing us to abstract away irrelevant details and focus on relevant information in a compact and usable form. We define structured representations as those that adhere to two specific properties: binding specific object information into discrete representation units and segregating object representations into separate sets of tokens to minimize cross-object entanglement. Based on these properties, we evaluated and compared image encoders pre-trained on classification (ViT), large vision-language models (CLIP, BLIP, FLAVA), and self-supervised methods (DINO, DINOv2). We examine the token representations by creating object-decoding tasks that measure the ability of specific tokens to capture individual objects in multi-object scenes from the COCO dataset. This analysis provides insights into how object-wise representations are distributed across tokens and layers within these vision encoders. Our findings highlight significant differences in the representation of objects depending on their relevance to the pre-training objective, with this effect particularly pronounced in the CLS token (often used for downstream tasks). Meanwhile, networks and layers that exhibit more structured representations retain better information about individual objects. To guide practical applications, we propose formal measures to quantify the two properties of structured representations, aiding in selecting and adapting vision encoders for downstream tasks.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/q-bio.NC/">arXiv: Quantitative Biology: Neurons and Cognition</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/q-bio.NC/>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/q-bio.NC/" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/q-bio.NC/&title=arXiv: Quantitative Biology: Neurons and Cognition" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2409.06325">Dense networks of integrate-and-fire neurons: Spatially-extended mean-field limit of the empirical measure</a></h4>

              
                arXiv:2409.06325v3 Announce Type: replace-cross 
Abstract: The dynamics of spatially-structured networks of $N$ interacting stochastic neurons can be described by deterministic population equations in the mean-field limit. While this is known, a general question has remained unanswered: does synaptic weight scaling suffice, by itself, to guarantee the convergence of network dynamics to a deterministic population equation, even when networks are not assumed to be homogeneous or spatially structured? In this work, we consider networks of stochastic integrate-and-fire neurons with arbitrary synaptic weights satisfying a $O(1/N)$ scaling condition. Borrowing results from the theory of dense graph limits, or graphons, we prove that, as $N\to\infty$, and up to the extraction of a subsequence, the empirical measure of the neurons' membrane potentials converges to the solution of a spatially-extended mean-field partial differential equation (PDE). Our proof requires analytical techniques that go beyond standard propagation of chaos methods. In particular, we introduce a weak metric that depends on the dense graph limit kernel and we show how the weak convergence of the initial data can be obtained by propagating the regularity of the limit kernel along the dual-backward equation associated with the spatially-extended mean-field PDE. Overall, this result invites us to reinterpret spatially-extended population equations as universal mean-field limits of networks of neurons with $O(1/N)$ synaptic weight scaling.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/q-bio.NC/">arXiv: Quantitative Biology: Neurons and Cognition</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2504.03671">HiAER-Spike: Hardware-Software Co-Design for Large-Scale Reconfigurable Event-Driven Neuromorphic Computing</a></h4>

              
                arXiv:2504.03671v1 Announce Type: new 
Abstract: In this work, we present HiAER-Spike, a modular, reconfigurable, event-driven neuromorphic computing platform designed to execute large spiking neural networks with up to 160 million neurons and 40 billion synapses - roughly twice the neurons of a mouse brain at faster-than real-time. This system, which is currently under construction at the UC San Diego Supercomputing Center, comprises a co-designed hard- and software stack that is optimized for run-time massively parallel processing and hierarchical address-event routing (HiAER) of spikes while promoting memory-efficient network storage and execution. Our architecture efficiently handles both sparse connectivity and sparse activity for robust and low-latency event-driven inference for both edge and cloud computing. A Python programming interface to HiAER-Spike, agnostic to hardware-level detail, shields the user from complexity in the configuration and execution of general spiking neural networks with virtually no constraints in topology. The system is made easily available over a web portal for use by the wider community. In the following we provide an overview of the hard- and software stack, explain the underlying design principles, demonstrate some of the system's capabilities and solicit feedback from the broader neuromorphic community.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2504.04477">Attention-Driven LPLC2 Neural Ensemble Model for Multi-Target Looming Detection and Localization</a></h4>

              
                arXiv:2504.04477v1 Announce Type: new 
Abstract: Lobula plate/lobula columnar, type 2 (LPLC2) visual projection neurons in the fly's visual system possess highly looming-selective properties, making them ideal for developing artificial collision detection systems. The four dendritic branches of individual LPLC2 neurons, each tuned to specific directional motion, enhance the robustness of looming detection by utilizing radial motion opponency. Existing models of LPLC2 neurons either concentrate on individual cells to detect centroid-focused expansion or utilize population-voting strategies to obtain global collision information. However, their potential for addressing multi-target collision scenarios remains largely untapped. In this study, we propose a numerical model for LPLC2 populations, leveraging a bottom-up attention mechanism driven by motion-sensitive neural pathways to generate attention fields (AFs). This integration of AFs with highly nonlinear LPLC2 responses enables precise and continuous detection of multiple looming objects emanating from any region of the visual field. We began by conducting comparative experiments to evaluate the proposed model against two related models, highlighting its unique characteristics. Next, we tested its ability to detect multiple targets in dynamic natural scenarios. Finally, we validated the model using real-world video data collected by aerial robots. Experimental results demonstrate that the proposed model excels in detecting, distinguishing, and tracking multiple looming targets with remarkable speed and accuracy. This advanced ability to detect and localize looming objects, especially in complex and dynamic environments, holds great promise for overcoming collision-detection challenges in mobile intelligent machines.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2504.04527">Memetic Search for Green Vehicle Routing Problem with Private Capacitated Refueling Stations</a></h4>

              
                arXiv:2504.04527v1 Announce Type: new 
Abstract: The green vehicle routing problem with private capacitated alternative fuel stations (GVRP-PCAFS) extends the traditional green vehicle routing problem by considering refueling stations limited capacity, where a limited number of vehicles can refuel simultaneously with additional vehicles must wait. This feature presents new challenges for route planning, as waiting times at stations must be managed while keeping route durations within limits and reducing total travel distance. This article presents METS, a novel memetic algorithm (MA) with separate constraint-based tour segmentation (SCTS) and efficient local search (ELS) for solving GVRP-PCAFS. METS combines global and local search effectively through three novelties. For global search, the SCTS strategy splits giant tours to generate diverse solutions, and the search process is guided by a comprehensive fitness evaluation function to dynamically control feasibility and diversity to produce solutions that are both diverse and near-feasible. For local search, ELS incorporates tailored move operators with constant-time move evaluation mechanisms, enabling efficient exploration of large solution neighborhoods. Experimental results demonstrate that METS discovers 31 new best-known solutions out of 40 instances in existing benchmark sets, achieving substantial improvements over current state-of-the-art methods. Additionally, a new large-scale benchmark set based on real-world logistics data is introduced to facilitate future research.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2504.04551">Dynamic Neural Field Modeling of Visual Contrast for Perceiving Incoherent Looming</a></h4>

              
                arXiv:2504.04551v1 Announce Type: new 
Abstract: Amari's Dynamic Neural Field (DNF) framework provides a brain-inspired approach to modeling the average activation of neuronal groups. Leveraging a single field, DNF has become a promising foundation for low-energy looming perception module in robotic applications. However, the previous DNF methods face significant challenges in detecting incoherent or inconsistent looming features--conditions commonly encountered in real-world scenarios, such as collision detection in rainy weather. Insights from the visual systems of fruit flies and locusts reveal encoding ON/OFF visual contrast plays a critical role in enhancing looming selectivity. Additionally, lateral excitation mechanism potentially refines the responses of loom-sensitive neurons to both coherent and incoherent stimuli. Together, these offer valuable guidance for improving looming perception models. Building on these biological evidence, we extend the previous single-field DNF framework by incorporating the modeling of ON/OFF visual contrast, each governed by a dedicated DNF. Lateral excitation within each ON/OFF-contrast field is formulated using a normalized Gaussian kernel, and their outputs are integrated in the Summation field to generate collision alerts. Experimental evaluations show that the proposed model effectively addresses incoherent looming detection challenges and significantly outperforms state-of-the-art locust-inspired models. It demonstrates robust performance across diverse stimuli, including synthetic rain effects, underscoring its potential for reliable looming perception in complex, noisy environments with inconsistent visual cues.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2504.03649">Diagnostic Method for Hydropower Plant Condition-based Maintenance combining Autoencoder with Clustering Algorithms</a></h4>

              
                arXiv:2504.03649v1 Announce Type: cross 
Abstract: The French company EDF uses supervisory control and data acquisition systems in conjunction with a data management platform to monitor hydropower plant, allowing engineers and technicians to analyse the time-series collected. Depending on the strategic importance of the monitored hydropower plant, the number of time-series collected can vary greatly making it difficult to generate valuable information from the extracted data. In an attempt to provide an answer to this particular problem, a condition detection and diagnosis method combining clustering algorithms and autoencoder neural networks for pattern recognition has been developed and is presented in this paper. First, a dimension reduction algorithm is used to create a 2-or 3-dimensional projection that allows the users to identify unsuspected relationships between datapoints. Then, a collection of clustering algorithms regroups the datapoints into clusters. For each identified cluster, an autoencoder neural network is trained on the corresponding dataset. The aim is to measure the reconstruction error between each autoencoder model and the measured values, thus creating a proximity index for each state discovered during the clustering stage.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2504.03715">Multi-Objective Quality-Diversity in Unstructured and Unbounded Spaces</a></h4>

              
                arXiv:2504.03715v1 Announce Type: cross 
Abstract: Quality-Diversity algorithms are powerful tools for discovering diverse, high-performing solutions. Recently, Multi-Objective Quality-Diversity (MOQD) extends QD to problems with several objectives while preserving solution diversity. MOQD has shown promise in fields such as robotics and materials science, where finding trade-offs between competing objectives like energy efficiency and speed, or material properties is essential. However, existing methods in MOQD rely on tessellating the feature space into a grid structure, which prevents their application in domains where feature spaces are unknown or must be learned, such as complex biological systems or latent exploration tasks. In this work, we introduce Multi-Objective Unstructured Repertoire for Quality-Diversity (MOUR-QD), a MOQD algorithm designed for unstructured and unbounded feature spaces. We evaluate MOUR-QD on five robotic tasks. Importantly, we show that our method excels in tasks where features must be learned, paving the way for applying MOQD to unsupervised domains. We also demonstrate that MOUR-QD is advantageous in domains with unbounded feature spaces, outperforming existing grid-based methods. Finally, we demonstrate that MOUR-QD is competitive with established MOQD methods on existing MOQD tasks and achieves double the MOQD-score in some environments. MOUR-QD opens up new opportunities for MOQD in domains like protein design and image generation.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2504.03800">Decision SpikeFormer: Spike-Driven Transformer for Decision Making</a></h4>

              
                arXiv:2504.03800v1 Announce Type: cross 
Abstract: Offline reinforcement learning (RL) enables policy training solely on pre-collected data, avoiding direct environment interaction - a crucial benefit for energy-constrained embodied AI applications. Although Artificial Neural Networks (ANN)-based methods perform well in offline RL, their high computational and energy demands motivate exploration of more efficient alternatives. Spiking Neural Networks (SNNs) show promise for such tasks, given their low power consumption. In this work, we introduce DSFormer, the first spike-driven transformer model designed to tackle offline RL via sequence modeling. Unlike existing SNN transformers focused on spatial dimensions for vision tasks, we develop Temporal Spiking Self-Attention (TSSA) and Positional Spiking Self-Attention (PSSA) in DSFormer to capture the temporal and positional dependencies essential for sequence modeling in RL. Additionally, we propose Progressive Threshold-dependent Batch Normalization (PTBN), which combines the benefits of LayerNorm and BatchNorm to preserve temporal dependencies while maintaining the spiking nature of SNNs. Comprehensive results in the D4RL benchmark show DSFormer's superiority over both SNN and ANN counterparts, achieving 78.4% energy savings, highlighting DSFormer's advantages not only in energy efficiency but also in competitive performance. Code and models are public at https://wei-nijuan.github.io/DecisionSpikeFormer.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2504.03891">Efficient FPGA-accelerated Convolutional Neural Networks for Cloud Detection on CubeSats</a></h4>

              
                arXiv:2504.03891v1 Announce Type: cross 
Abstract: We present the implementation of four FPGA-accelerated convolutional neural network (CNN) models for onboard cloud detection in resource-constrained CubeSat missions, leveraging Xilinx's Vitis AI (VAI) framework and Deep Learning Processing Unit (DPU), a programmable engine with pre-implemented, parameterizable IP cores optimized for deep neural networks, on a Zynq UltraScale+ MPSoC. This study explores both pixel-wise (Pixel-Net and Patch-Net) and image-wise (U-Net and Scene-Net) models to benchmark trade-offs in accuracy, latency, and model complexity. Applying channel pruning, we achieved substantial reductions in model parameters (up to 98.6%) and floating-point operations (up to 90.7%) with minimal accuracy loss. Furthermore, the VAI tool was used to quantize the models to 8-bit precision, ensuring optimized hardware performance with negligible impact on accuracy. All models retained high accuracy post-FPGA integration, with a cumulative maximum accuracy drop of only 0.6% after quantization and pruning. The image-wise Scene-Net and U-Net models demonstrated strong real-time inference capabilities, achieving frame rates per second of 57.14 and 37.45, respectively, with power consumption of around 2.5 W, surpassing state-of-the-art onboard cloud detection solutions. Our approach underscores the potential of DPU-based hardware accelerators to expand the processing capabilities of small satellites, enabling efficient and flexible onboard CNN-based applications.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2504.05108">Algorithm Discovery With LLMs: Evolutionary Search Meets Reinforcement Learning</a></h4>

              
                arXiv:2504.05108v1 Announce Type: cross 
Abstract: Discovering efficient algorithms for solving complex problems has been an outstanding challenge in mathematics and computer science, requiring substantial human expertise over the years. Recent advancements in evolutionary search with large language models (LLMs) have shown promise in accelerating the discovery of algorithms across various domains, particularly in mathematics and optimization. However, existing approaches treat the LLM as a static generator, missing the opportunity to update the model with the signal obtained from evolutionary exploration. In this work, we propose to augment LLM-based evolutionary search by continuously refining the search operator - the LLM - through reinforcement learning (RL) fine-tuning. Our method leverages evolutionary search as an exploration strategy to discover improved algorithms, while RL optimizes the LLM policy based on these discoveries. Our experiments on three combinatorial optimization tasks - bin packing, traveling salesman, and the flatpack problem - show that combining RL and evolutionary search improves discovery efficiency of improved algorithms, showcasing the potential of RL-enhanced evolutionary strategies to assist computer scientists and mathematicians for more efficient algorithm design.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2403.01900">Universality of reservoir systems with recurrent neural networks</a></h4>

              
                arXiv:2403.01900v2 Announce Type: replace 
Abstract: Approximation capability of reservoir systems whose reservoir is a recurrent neural network (RNN) is discussed. We show what we call uniform strong universality of RNN reservoir systems for a certain class of dynamical systems. This means that, given an approximation error to be achieved, one can construct an RNN reservoir system that approximates each target dynamical system in the class just via adjusting its linear readout. To show the universality, we construct an RNN reservoir system via parallel concatenation that has an upper bound of approximation error independent of each target in the class.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2406.01309">REvolve: Reward Evolution with Large Language Models using Human Feedback</a></h4>

              
                arXiv:2406.01309v3 Announce Type: replace 
Abstract: Designing effective reward functions is crucial to training reinforcement learning (RL) algorithms. However, this design is non-trivial, even for domain experts, due to the subjective nature of certain tasks that are hard to quantify explicitly. In recent works, large language models (LLMs) have been used for reward generation from natural language task descriptions, leveraging their extensive instruction tuning and commonsense understanding of human behavior. In this work, we hypothesize that LLMs, guided by human feedback, can be used to formulate reward functions that reflect human implicit knowledge. We study this in three challenging settings -- autonomous driving, humanoid locomotion, and dexterous manipulation -- wherein notions of ``good" behavior are tacit and hard to quantify. To this end, we introduce REvolve, a truly evolutionary framework that uses LLMs for reward design in RL. REvolve generates and refines reward functions by utilizing human feedback to guide the evolution process, effectively translating implicit human knowledge into explicit reward functions for training (deep) RL agents. Experimentally, we demonstrate that agents trained on REvolve-designed rewards outperform other state-of-the-art baselines.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2501.11477">Image Classification Method using Dynamic Quantum Inspired Genetic Algorithm</a></h4>

              
                arXiv:2501.11477v4 Announce Type: replace 
Abstract: This study presents a dynamic Quantum-Inspired Genetic Algorithm (D-QIGA) for feature selection, leveraging quantum principles like superposition and rotation gates to enhance exploration and exploitation. D-QIGA introduces adaptive mechanisms and a lengthening chromosome strategy to avoid local optima and improve optimization. Tested on benchmark and real-world problems, it significantly outperforms traditional Genetic Algorithms, achieving over 99.99% classification accuracy compared to GA's 95%.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2309.02712">Unveiling the frontiers of deep learning: innovations shaping diverse domains</a></h4>

              
                arXiv:2309.02712v2 Announce Type: replace-cross 
Abstract: Deep learning (DL) allows computer models to learn, visualize, optimize, refine, and predict data. To understand its present state, examining the most recent advancements and applications of deep learning across various domains is essential. However, prior reviews focused on DL applications in only one or two domains. The current review thoroughly investigates the use of DL in four different broad fields due to the plenty of relevant research literature in these domains. This wide range of coverage provides a comprehensive and interconnected understanding of DL's influence and opportunities, which is lacking in other reviews. The study also discusses DL frameworks and addresses the benefits and challenges of utilizing DL in each field, which is only occasionally available in other reviews. DL frameworks like TensorFlow and PyTorch make it easy to develop innovative DL applications across diverse domains by providing model development and deployment platforms. This helps bridge theoretical progress and practical implementation. Deep learning solves complex problems and advances technology in many fields, demonstrating its revolutionary potential and adaptability. CNN LSTM models with attention mechanisms can forecast traffic with 99 percent accuracy. Fungal diseased mango leaves can be classified with 97.13 percent accuracy by the multi layer CNN model. However, deep learning requires rigorous data collection to analyze and process large amounts of data because it is independent of training data. Thus, large scale medical, research, healthcare, and environmental data compilation are challenging, reducing deep learning effectiveness. Future research should address data volume, privacy, domain complexity, and data quality issues in DL datasets.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://export.arxiv.org/rss/cs.NE>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://export.arxiv.org/rss/cs.NE" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://export.arxiv.org/rss/cs.NE&title=arXiv: Computer Science: Neural and Evolutionary Computing" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://arxiv.org/abs/2408.16389">Addressing common misinterpretations of KART and UAT in neural network literature</a></h4>

              
                arXiv:2408.16389v4 Announce Type: replace-cross 
Abstract: This note addresses the Kolmogorov-Arnold Representation Theorem (KART) and the Universal Approximation Theorem (UAT), focusing on their common and frequent misinterpretations in many papers related to neural network approximation. Our remarks aim to support a more accurate understanding of KART and UAT among neural network specialists. In addition, we explore the minimal number of neurons required for universal approximation, showing that KART's lower bounds extend to standard multilayer perceptrons, even with smooth activation functions.
              

              <p class="text-muted">in <a href ="http://export.arxiv.org/rss/cs.NE">arXiv: Computer Science: Neural and Evolutionary Computing</a> on 2025-04-08 04:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00862-1">154 million lives and counting: 5 charts reveal the power of vaccines</a></h4>

              
                <p>Nature, Published online: 08 April 2025; <a href="https://www.nature.com/articles/d41586-025-00862-1">doi:10.1038/d41586-025-00862-1</a></p>But public-health specialists worry that misinformation and funding cuts could affect vaccination rates.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-04-08 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58535-6">Transcriptomic and morphologic vascular aberrations underlying FCDIIb etiology</a></h4>

              
                <p>Nature Communications, Published online: 08 April 2025; <a href="https://www.nature.com/articles/s41467-025-58535-6">doi:10.1038/s41467-025-58535-6</a></p>Focal cortical dysplasia type II causes epileptic seizures and is associated with various types of neuronal abnormalities. This study uses patient tissue transcriptomics and animal models to reveal that vascular malformations are a key part of the pathology by creating ischemic-hypoxic microenvironments.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-04-08 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-233/v2?src=rss">Machine learning applications in risk management: Trends and research agenda [version 2; peer review: 1 approved with reservations, 1 not approved]</a></h4>

              
                Abstract Risk management has become a foundational aspect in numerous industries, propelling the implementation of machine learning technologies for impact assessment, prevention, and decision-making processes. Nevertheless, lacunae in the extant literature persist, particularly with regard to the identification of emergent trends and transversal applications. This study addresses this limitation through a bibliometric analysis of scientific production in Scopus and Web of Science, adhering to the PRISMA-2020 declaration. The findings reveal a substantial growth in publications on machine learning applied to risk management, with an increase of 98.99% between 2018 and 2023. China, South Korea, and the United States are identified as the primary research-producing countries. The analysis also identifies emerging trends, such as the application of machine learning in the evaluation of urban trees and the management of risks associated with the pandemic of severe acute respiratory syndrome (SARS-CoV-2). Key terms include random forest, support vector machines (SVM), and credit risk assessment, while terms such as prediction, postpartum depression, big data, and security emerge as new areas of study. Furthermore, there is a transition from traditional approaches such as stacking to advanced deep learning and feature selection techniques, reflecting the evolution of the discipline.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 16:44:43 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-410/v1?src=rss">Climate change mitigation and workers’ interests: why framing a Green New Deal as redistributive and security-enhancing is key to popularity [version 1; peer review: awaiting peer review]</a></h4>

              
                Background There is urgent need for comprehensive climate change policies to mitigate impacts and protect the interests of those most vulnerable to its worst effects. The Labour Government has rejected its own 2021 £28bn annual investment in climate change policies on account of commitment to economic restraint and public opinion. Not only does this pose real risks to the UK’s ability to respond to climate change, it may also reduce a range of social and economic benefits. Methods We report findings of innovative mixed-methods survey analysis of public perceptions of an illustrative Green New Deal within three surveys (1) n=693; 2) n=10; 3) n=2,200) of adult UK residents conducted between November 2023-January 2024. Results We analyse the findings of survey 3 to show that levels of support for a Green New Deal are high across parties and demographic groups, and increase further when voters are presented with narrative justifications adversarially co-produced with opponents – termed ‘haters’ – of the policy. We find clear associations between risk of destitution and various other socioeconomic characteristics, health status and levels of support. We present innovative Structural Equation Modelling (SEM) of these associations and find moderately strong positive correlations with levels of support for key infrastructural policies. Conclusions This article presents further evidence in support of the notion that exposure to risk of destitution, which varies by age, is a key determinant of policy preference at a time in which political affiliation is increasingly fluid and the prima facie need for a Green New Deal is considerable. This suggests that, in order to understand preferences and to present responses to challenges, there is good reason to focus on material outcomes. Given the importance of a Green New Deal to enhancing financial security, progressive politicians have every reason to commit to substantive reform.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 16:36:51 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-409/v1?src=rss">Exploring the Impact of Student-Staff Partnerships in Higher Education: A Realist Review Protocol. [version 1; peer review: awaiting peer review]</a></h4>

              
                Background Student-staff partnership (SSP) in higher education (HE) is rooted in democratic education and critical pedagogy, advocating for active student engagement. While widely recognised for fostering student success, SSP initiatives vary in definition, implementation, and impact. In Ireland, national policy has positioned SSP as central to student engagement, yet there is limited synthesis of the contextual factors and mechanisms influencing its success. Existing literature highlights benefits such as increased engagement and enhanced learning but often overlooks challenges, including power dynamics, resistance to change, and inconsistent institutional commitment. Aim This protocol outlines the methodology for a realist review, forming the first stage of a broader realist inquiry into SSP initiatives. The review will synthesise existing evidence to understand how, why, and in what contexts SSP initiatives succeed or face challenges in HE. Specifically, it will identify the contextual factors, mindsets, and engagement strategies that enable meaningful partnerships, as well as mechanisms for overcoming barriers. The review will generate initial programme theories (IPTs) to inform a subsequent realist evaluation within University College Dublin (UCD). Method A realist review methodology will be used to explore the interactions between context, mechanisms, and outcomes in SSP initiatives. The study will follow a five-stage realist review process: scoping the literature, developing IPTs, systematically reviewing evidence, synthesising findings, and refining theories with expert input. The Context-Mechanism-Outcome Configuration (CMOC) framework will guide the analysis. Conclusion This protocol sets out the approach for developing evidence-informed programme theories on SSP. These theories will underpin a subsequent realist evaluation within UCD to refine SSP implementation strategies. Findings will inform institutional strategies, policy development, and academic practice, with dissemination through academic and practitioner-focused outputs.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 16:33:41 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/13-907/v2?src=rss">Evaluation of NETosis (Neutrophil Extracellular Traps) biomarkers in smokers and nonsmokers periodontitis patients [version 2; peer review: 1 approved with reservations, 1 not approved]</a></h4>

              
                Background To evaluate the NETosis biomarkers citrullinated histone H3 (citH3), neutrophil elastase (ELA), calprotectin (CALPRO), and myeloperoxidase (MPO) as indicators of inflammation in the severe stages of periodontitis III and IV in both (smokers and nonsmokers) patients, and to determine the correlation between NETosis biomarkers and clinical periodontal parameters. Methods This study recruited male subjects with an age range of (20-60) years; 60 were stage III and stage IV periodontitis patients, 30 were cigarette smoker, and 30 were nonsmokers. After applying the inclusion and exclusion criteria to evaluate their eligibility for recruitment, 25 control subjects with a healthy periodontal status and good oral hygiene maintenance were included. Unstimulated saliva was obtained and evaluated using an enzyme-linked immunosorbent assay, and the following periodontal parameters were documented: [plaque index, bleeding on probing, periodontal pocket depth, and clinical attachment loss]. Results The mean levels of all salivary NETosis biomarkers citH3, ELA, CALPRO, and MPO were elevated in the periodontitis groups (smokers and nonsmokers) than in controls. Moreover, the mean NETosis biomarker‘s mean levels were significantly higher in smoker than in nonsmokers. In addition, the correlations were significant between CALPRO and CitH3 in smokers and between ELA and CitH3 in nonsmokers. Conclusions The results of this study showed that the chosen salivary biomarkers of NETosis revealed elevated clinical accuracy in differentiating the studied periodontitis groups (smokers and nonsmokers) from controls. In addition, cigarette smoking increases the risk of periodontitis, and neutrophils in smokers with periodontitis exhibited more susceptibility to form neutrophil extracellular traps when compared with nonsmokers.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 16:15:45 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012972">Flux-sum coupling analysis of metabolic network models</a></h4>

              
                <p>by Mihriban Seyis, Zahra Razaghi-Moghadam, Zoran Nikoloski</p>

Metabolites acting as substrates and regulators of all biochemical reactions play an important role in maintaining the functionality of cellular metabolism. Despite advances in the constraint-based framework for metabolic modeling at a genome-scale, we lack reliable proxies for metabolite concentrations that can be efficiently determined and that allows us to investigate the relationship between concentrations of metabolites in specified metabolic states in the absence of measurements. Here, we introduce a constraint-based approach, the flux-sum coupling analysis (FSCA), which facilitates the study of the interdependencies between metabolite concentrations by determining coupling relationships based on the flux-sum of metabolites. Application of FSCA on metabolic models of <i>Escherichia coli</i>, <i>Saccharomyces cerevisiae</i>, and <i>Arabidopsis thaliana</i> showed that the three coupling relations are present in all models and pinpointed similarities in coupled metabolite pairs. Using the available concentration measurements of <i>E. coli</i> metabolites, we demonstrated that the coupling relationships identified by FSCA can capture the qualitative associations between metabolite concentrations and that flux-sum is a reliable proxy for metabolite concentration. Therefore, FSCA provides a novel tool for exploring and understanding the intricate interdependencies between the concentration of metabolites, advancing the understanding of metabolic regulation, and improving flux-centered systems biology approaches.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-07 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012961">Neural energy coding patterns of dopaminergic neural microcircuit and its impairment in major depressive disorder: A computational study</a></h4>

              
                <p>by Yuanxi Li, Bing Zhang, Jinqi Liu, Rubin Wang</p>

Numerous experiments have found that the behavioral characteristics of major depressive disorder (MDD) animals are usually associated with abnormal neural activity patterns and brain energy metabolism. However, the relationship among the behavioral characteristics, neural activity patterns and brain energy metabolism remains unknown. In this paper, we computationally investigated this relationship, especially studied how neural energy coding patterns changed in MDD brains, in the VTA-NAc-mPFC dopaminergic pathway of the reward system based on our biological neural network model and neural energy calculation model. Interestingly, our results suggested that the neural energy consumption of the whole VTA-NAc-mPFC microcircuit in MDD group was significantly reduced, which was mainly attributed to the decreasing neural energy consumption in the mPFC region. This observation theoretically supported the view of low-level energy consumption in MDD. We also investigated the neural energy consumption patterns of various neuronal types in our VTA-NAc-mPFC microcircuit under the influence of different dopamine concentrations, and found that there were some specific impairments in MDD, which provided some potential biomarkers for MDD diagnosis. More specifically, we found that the actual neural energy consumption of medium spiny neurons (MSNs) in the NAc region was increased in the MDD group, whereas pyramidal neurons in the mPFC region exhibited higher actual neural energy consumption in the NC group. Additionally, in both neuron types, the actual neural energy required to generate an action potential was higher in the MDD group, suggesting that, given the same energy budget, these neurons in the MDD group tended to generate fewer action potentials. To further explore the relationship between neural coding patterns and neural energy coding patterns in the VTA-NAc-mPFC microcircuit, we in addition calculated P-V correlation for each neuronal types, defined as the Pearson’s correlation coefficient between membrane potential and neural power. The results showed that the membrane potential and neural power were not perfectly correlated (P-V correlations were ranged from 0.6 to 0.9), and dopamine concentration inputs affected the P-V correlations of the MSN, pyramidal neurons and CB interneurons in the mPFC region. It suggested that the joint application of the neural coding theory and neural energy coding theory will be superior to the application of any single theory, and this joint application could help discovery new mechanisms in neurocircuits of MDD. Overall, our study not only uncovered the neural energy coding patterns for the VTA-NAc-mPFC neural microcircuit, but also presented a novel pipeline for the study of MDD based on the neural coding theory and neural energy coding theory.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-07 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012951">A burst-dependent thalamocortical substrate for perceptual awareness</a></h4>

              
                <p>by Christopher J. Whyte, Eli J. Müller, Jaan Aru, Matthew Larkum, Yohan John, Brandon R. Munn, James M. Shine</p>

Contemporary models of perceptual awareness lack tractable neurobiological constraints. Inspired by recent cellular recordings in a mouse model of tactile threshold detection, we constructed a biophysical model of perceptual awareness that incorporated essential features of thalamocortical anatomy and cellular physiology. Our model reproduced, and mechanistically explains, the key <i>in vivo</i> neural and behavioural signatures of perceptual awareness in the mouse model, as well as the response to a set of causal perturbations. We generalised the same model (with identical parameters) to a more complex task – visual rivalry – and found that the same thalamic-mediated mechanism of perceptual awareness determined perceptual dominance. This led to the generation of a set of novel, and directly testable, electrophysiological predictions. Analyses of the model based on dynamical systems theory show that perceptual awareness in simulations of both threshold detection and visual rivalry arises from the emergent systems-level dynamics of thalamocortical loops.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-07 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012909">Role of pore dilation in molecular transport through the nuclear pore complex: Insights from polymer scaling theory</a></h4>

              
                <p>by Atsushi Matsuda, Mohammad R K Mofrad</p>

The nuclear pore complex (NPC), a channel within the nuclear envelope filled with intrinsically disordered proteins, regulates the transport of macromolecules between the nucleus and the cytoplasm. Recent studies have highlighted the NPC’s ability to adjust its diameter in response to the membrane tension, underscoring the importance of exploring how variations in pore size influence molecular transport through the NPC. In this study, we investigated the relationship between pore size and transport rate and proposed a mathematical model describing this connection. We began by theoretically analyzing how the pore size scales with the characteristic dimensions of the mesh-like structure within the pore. By introducing key assumptions about how the meshwork structure influences molecular diffusion, we derived a mathematical expression for the transport rate based on the size of the pore and the transported molecules. To validate our model, we conducted Brownian dynamics simulations using a coarse-grained representation of the NPC. These simulations, performed across a range of pore sizes, demonstrated strong agreement with our model’s predictions, confirming its accuracy and applicability. Our model is specifically tailored for small-to-medium-sized molecules, approximately 5 nanometers in size, making it relevant to a wide range of transcription factors and signaling molecules. It also extends to molecules with weak and transient interactions with FG-Nups, such as importin-<i>β</i>. By presenting this model formula, our study offers a quantitative framework for analyzing the effects of pore dilation on nucleocytoplasmic transport.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-07 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012885">Analysis of targeted and whole genome sequencing of PacBio HiFi reads for a comprehensive genotyping of gene-proximal and phenotype-associated Variable Number Tandem Repeats</a></h4>

              
                <p>by Sara Javadzadeh, Aaron Adamson, Jonghun Park, Se-Young Jo, Yuan-Chun Ding, Mehrdad Bakhtiari, Vikas Bansal, Susan L. Neuhausen, Vineet Bafna</p>

Variable Number Tandem repeats (VNTRs) refer to repeating motifs of size greater than five bp. VNTRs are an important source of genetic variation, and have been associated with multiple Mendelian and complex phenotypes. However, the highly repetitive structures require reads to span the region for accurate genotyping. Pacific Biosciences HiFi sequencing spans large regions and is highly accurate but relatively expensive. Therefore, targeted sequencing approaches coupled with long-read sequencing have been proposed to improve efficiency and throughput. In this paper, we systematically explored the trade-off between targeted and whole genome HiFi sequencing for genotyping VNTRs. We curated a set of 10 , 787 gene-proximal (G-)VNTRs, and 48 phenotype-associated (P-)VNTRs of interest. Illumina reads only spanned 46% of the G-VNTRs and 71% of P-VNTRs, motivating the use of HiFi sequencing. We performed targeted sequencing with hybridization by designing custom probes for 9,999 VNTRs and sequenced 8 samples using HiFi and Illumina sequencing, followed by adVNTR genotyping. We compared these results against HiFi whole genome sequencing (WGS) data from 28 samples in the Human Pangenome Reference Consortium (HPRC). With the targeted approach only 4,091 (41%) G-VNTRs and only 4 (8%) of P-VNTRs were spanned with at least 15 reads. A smaller subset of 3,579 (36%) G-VNTRs had higher median coverage of at least 63 spanning reads. The spanning behavior was consistent across all 8 samples. Among 5,638 VNTRs with low-coverage ( < 15), 67% were located within GC-rich regions ( > 60%). In contrast, the 40X WGS HiFi dataset spanned 98% of all VNTRs and 49 (98%) of P-VNTRs with at least 15 spanning reads, albeit with lower coverage. Spanning reads were sufficient for accurate genotyping in both cases. Our findings demonstrate that targeted sequencing provides consistently high coverage for a small subset of low-GC VNTRs, but WGS is more effective for broad and sufficient sampling of a large number of VNTRs.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-07 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012862">Nanometer scale difference in myofilament lattice structure of muscle alters muscle function in a spatially explicit model</a></h4>

              
                <p>by Travis Carver Tune, Simon Sponberg</p>

Crossbridge binding, state transitions, and force in active muscle is dependent on the radial spacing between the myosin-containing thick filament and the actin-containing thin filament in the filament lattice. This radial spacing has been previously shown through spatially explicit modeling and experimental efforts to greatly affect quasi-static, isometric, force production in muscle. It has recently been suggested that this radial spacing might also be able to drive differences in mechanical function, or net work, under dynamic oscillations like those which occur in muscles <i>in vivo</i>. However, previous spatially explicit models either had no radial spacing dependence, meaning the radial spacing could not be investigated, or did include radial spacing dependence but could not reproduce <i>in vivo</i> net work during dynamic oscillations and only investigated isometric contractions. Here we show the first spatially explicit model to include radial crossbridge dependence which can produce mechanical function similar to real muscle. Using this spatially explicit model of a half sarcomere, we show that when oscillated at strain amplitudes and frequencies like those in the main flight muscles of the hawkmoth <i>Manduca sexta</i>, mechanical function (net work) does depend on the lattice spacing. In addition, since the trajectory of lattice spacing changes during dynamic oscillation can vary from organism to organism, we can prescribe a trajectory of lattice spacing changes in the spatially explicit half sarcomere model and investigate the extent to which the time course of lattice spacing changes can affect mechanical function. We simulated a half sarcomere undergoing dynamic oscillations and prescribed the Poisson’s ratio of the lattice to be either 0 (constant lattice spacing) or 0.5 (isovolumetric lattice spacing changes). We also simulated net work using lattice spacing data taken from <i>M. sexta</i> which has a variable Poisson’s ratio. Our simulation results indicate that the lattice spacing can change the mechanical function of muscle, and that in some cases a 1 nm difference can switch the net work of the half sarcomere model from positive (motor-like) to negative (brake-like).
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-07 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012848">The multiscale self-similarity of the weighted human brain connectome</a></h4>

              
                <p>by Laia Barjuan, Muhua Zheng, M Ángeles Serrano</p>

Anatomical connectivity between different brain regions can be mapped to a network representation, the connectome, where the intensities of the links, the weights, influence resilience and functional processes. Yet, many features associated with these weights are not fully understood, particularly their multiscale organization. In this paper, we elucidate the architecture of weights, including weak ties, in multiscale human brain connectomes reconstructed from empirical data. Our findings reveal multiscale self-similarity, including the ordering of weak ties, in every individual connectome and group representative. This phenomenon is captured by a renormalization technique based on a geometric network model that replicates the observed structure of connectomes across all length scales, using the same connectivity law and weighting function for both weak and strong ties. The observed symmetry represents a signature of criticality in the weighted connectivity of the human brain and raises important questions for future research, such as the existence of symmetry breaking at some scale or whether it is preserved in cases of neurodegeneration or psychiatric disorder.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-07 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/plosbiology/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/plosbiology/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/plosbiology/NewArticles&title=PLoS Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3003114">Priorities for rapid and cost-effective vaccines to improve outbreak responses</a></h4>

              
                <p>by Sarah C. Gilbert</p>

The use of vaccine platform technologies greatly reduces the time and money required to develop a novel vaccine against an infectious disease, but much work is still needed if vaccines are to protect the world from the next pandemic.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/plosbiology/NewArticles">PLoS Biology</a> on 2025-04-07 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/plosbiology/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/plosbiology/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/plosbiology/NewArticles&title=PLoS Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3003092">The zona incerta negatively regulates the red nucleus during movement cued by sound signals</a></h4>

              
                <p>by Liang Chen, Xinxing Wang, Hanxiao Liu, Chenzhao He, Allen P. F. Chen, Lu Chen, Thomas A. Kim, Qiaojie Xiong</p>

Auditory signal-cued behaviors rely on a sophisticated neural network. While extensive research has focused on auditory processing and decision-making, the neural circuits governing motor coordination for goal-directed actions remain poorly understood. The red nucleus (RN) is essential for motor coordination, whereas the zona incerta (ZI) plays a key role in modulating sensorimotor circuits. Using tetrode recordings and optogenetics, we investigated the ZI-RN circuit’s role in an auditory-cued decision task. RN neurons were preferentially activated when mice moved to the contralateral port for a reward, and optogenetic activation biased choices toward the contralateral side. Notably, parvalbumin-positive ZI neurons projected to the RN and negatively regulated movement coordination. These findings reveal an inhibitory ZI-RN circuit that shapes auditory-cued, goal-directed movement.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/plosbiology/NewArticles">PLoS Biology</a> on 2025-04-07 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/plosbiology/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/plosbiology/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/plosbiology/NewArticles&title=PLoS Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3003087">Whisking and locomotion are jointly represented in superior colliculus neurons</a></h4>

              
                <p>by Suma Chinta, Scott R. Pluta</p>

Active sensation requires the brain to interpret external stimuli against an ongoing estimate of body position. While internal estimates of body position are often ascribed to the cerebral cortex, we examined the midbrain superior colliculus (SC), due to its close relationship with the sensory periphery as well as higher, motor-related brain regions. Using high-density electrophysiology and movement tracking, we discovered that the on-going kinematics of whisker motion and locomotion speed accurately predict the firing rate of mouse SC neurons. Neural activity was best predicted by movements occurring either in the past, present, or future, indicating that the SC population continuously estimates a trajectory of self-motion. A combined representation of slow and fast whisking features predicted absolute whisker angle at high temporal resolution. Sensory reafference played at least a partial role in shaping this feature tuning. Taken together, these data indicate that the SC contains a joint representation of whisking and locomotor features that is potentially useful in guiding complex orienting movements involving the face and limbs.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/plosbiology/NewArticles">PLoS Biology</a> on 2025-04-07 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/plosbiology/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/plosbiology/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/plosbiology/NewArticles&title=PLoS Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3003077">The SPFH complex HflK-HflC regulates aerobic respiration in bacteria</a></h4>

              
                <p>by María Isabel Pérez-López, Paul Lubrano, Georgia Angelidou, Sarah Hoch, Timo Glatter, Nicole Paczia, Hannes Link, Victor Sourjik</p>

The bacterial HflK-HflC membrane complex is a member of the highly conserved family of SPFH proteins, which are present in all domains of life and include eukaryotic stomatins, flotillins, and prohibitins. These proteins organize cell membranes and are involved in various processes. However, the exact physiological functions of most bacterial SPFH proteins remain unclear. Here, we report that the HflK-HflC complex in <i>Escherichia coli</i> is required for growth under high aeration. The absence of this complex causes a growth defect at high oxygen levels due to a reduced abundance of IspG, an essential iron-sulfur cluster enzyme in the isoprenoid biosynthetic pathway. This reduction might be related to lower stability of IspG and several other proteins, including the iron siderophore transporter TonB, in the absence of the HflK-HflC complex. Our results suggest that decreased IspG activity leads to lower levels of ubiquinone and misregulated expression of multiple respiratory enzymes, including cytochrome oxidases, and consequently reduced respiration and lower ATP levels. This impact of the <i>hflK hflC</i> deletion on aerobic respiration resembles the mitochondrial respiratory defects caused by the inactivation of prohibitins in mammalian and yeast cells, indicating functional parallels between these bacterial and eukaryotic SPFH proteins.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/plosbiology/NewArticles">PLoS Biology</a> on 2025-04-07 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/plosbiology/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/plosbiology/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/plosbiology/NewArticles&title=PLoS Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3003063">Completing the BASEL phage collection to unlock hidden diversity for systematic exploration of phage–host interactions</a></h4>

              
                <p>by Dorentina Humolli, Damien Piel, Enea Maffei, Yannik Heyer, Elia Agustoni, Aisylu Shaidullina, Luc Willi, Patrick Imwinkelried, Fabienne Estermann, Aline Cuénod, Dominik P. Buser, Carola Alampi, Mohamed Chami, Adrian Egli, Sebastian Hiller, Matthew Dunne, Alexander Harms</p>

Research on bacteriophages, the viruses infecting bacteria, has fueled the development of modern molecular biology and inspired their therapeutic application to combat bacterial multidrug resistance. However, most work has so far focused on a few model phages which impedes direct applications of these findings in clinics and suggests that a vast potential of powerful molecular biology has remained untapped. We have therefore recently composed the BASEL collection of <i>Escherichia coli</i> phages (BActeriophage SElection for your Laboratory), which made a relevant diversity of phages infecting the <i>E. coli</i> K-12 laboratory strain accessible to the community. These phages are widely used, but their assorted diversity has remained limited by the <i>E. coli</i> K-12 host. We have therefore now genetically overcome the two major limitations of <i>E. coli</i> K-12, its lack of O-antigen glycans and the presence of resident bacterial immunity. Restoring O-antigen expression resulted in the isolation of diverse additional viral groups like <i>Kagunavirus</i>, <i>Nonanavirus</i>, <i>Gordonclarkvirinae</i>, and <i>Gamaleyavirus</i>, while eliminating all known antiviral defenses of <i>E. coli</i> K-12 additionally enabled us to isolate phages of <i>Wifcevirus</i> genus. Even though some of these viral groups appear to be common in nature, no phages from any of them had previously been isolated using <i>E. coli</i> laboratory strains, and they had thus remained largely understudied. Overall, 37 new phage isolates have been added to complete the BASEL collection. These phages were deeply characterized genomically and phenotypically with regard to host receptors, sensitivity to antiviral defense systems, and host range. These results highlighted dominant roles of the O-antigen barrier for viral host recognition and of restriction-modification systems in bacterial immunity. We anticipate that the completed BASEL collection will propel research on phage–host interactions and their molecular mechanisms, deepening our understanding of viral ecology and fostering innovations in biotechnology and antimicrobial therapy.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/plosbiology/NewArticles">PLoS Biology</a> on 2025-04-07 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/13-940/v3?src=rss">Integration of Democratic Values ​​in Natural Sciences Education: a Review of the Literature of the Last 50 Years [version 3; peer review: 1 approved with reservations, 1 not approved]</a></h4>

              
                This study examines how democratic values have been promoted through natural sciences education over the last 50 years, providing a comprehensive analysis based on a systematic review of relevant literature. The central problem addressed is understanding the role of natural science education in fostering democratic values such as equity, participation, critical thinking, and ethical responsibility. This research aims to identify and analyze strategies, methodologies, and transformative experiences that contribute to the promotion of democratic values. This study employs the PRISMA methodology to ensure a rigorous and structured systematic review. Data were collected from multiple databases using detailed Boolean equations. Tools such as ScientoPy and VOSviewer were used for data preprocessing, clustering, and network visualization, followed by qualitative analysis to categorize the findings. Educational programs in natural sciences have increasingly integrated democratic values, fostering a culture of inclusivity and participation; the incorporation of ICT has enhanced equity and participation, while civic education has been fundamental in developing critical and informed citizens; citizen science initiatives have empowered students to engage in democratic deliberation and address epistemic injustice; cooperative learning methods in science classes have effectively promoted gender equity and inclusion; and emphasis on sustainability and environmental justice in science education has promoted democratic values and empowered students to take action on global challenges. In conclusion, natural science education is an effective vehicle for promoting democratic values, but it is an understudied field. By integrating practices that emphasize inclusivity, critical thinking, and ethical responsibility, science education not only enhances students’ scientific understanding but also prepares them to be active, informed, and responsible citizens.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 10:18:58 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/12-339/v3?src=rss">Comparing the effectiveness of pterostilbene and sitagliptin on modulating inflammatory levels and inducing autophagy to improve atherosclerosis outcome: A preclinical study in rabbits [version 3; peer review: 1 approved, 1 approved with reservations]</a></h4>

              
                Background: Inflammation is the key contributor to the development of atherosclerotic plague. This study aims to evaluate the protective and autophagy induction properties of pterostilbene and sitagliptin on modulating the degree of atherosclerosis in rabbit models treated with an atherogenic diet. Methods: 80 rabbits were randomly placed into one of four study groups (20 in each group): normal control diet (NC) fed normal diet for eight weeks, atherogenic control (AC) fed atherogenic diet for eight weeks, pterostilbene treated group (PT) fed atherogenic diet with pterostilbene (at 10 mg/kg/day) orally daily for eight weeks, and sitagliptin treated group (ST) fed atherogenic diet with sitagliptin (at 12 mg/kg/day) orally daily for eight weeks. Results: While serum lipids and F2-isoprostane were elevated significantly in the AC study cohort compared to NC study cohort, (P < 0.001), both pterostilbene and sitagliptin supplementations provided significant improvements in serum lipid parameters and F2-isoprostane in the PT study cohort and ST study cohort, respectively, when compared to the AC study cohort, (P<0.001). Total cholesterol, triglycerides and LDL levels were significantly reduced among the PT and ST study cohorts as compared to the AC study cohort. This was coupled with a significant rise in LC3B levels (marker of tissue autophagy) among the PT study cohort and the ST study cohort, as compared to the AC study cohort, (P < 0.001). The RNA expression of mTORC1 was reduced significantly at both PT study cohort and ST study cohort, (P<0.001). Pterostilbene supplementation induced a significant reduction in tissue expression of PI3K and AKT, (P<0.01), while sitagliptin induced significant reduction in 5’ adenosine monophosphate-activated protein kinase (AMPK) levels, (P<0.001). Conclusions: The results indicate that pterostilbene and/or sitagliptin supplementation can significantly improve the outcome of atherosclerosis due to their effects on the inflammatory pathways which hinder the progression of atherosclerotic plaque formation.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 09:41:46 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/13-673/v2?src=rss">Association of cancer and outcomes of patients hospitalized for COVID-19 between 2020 and 2023 [version 2; peer review: 1 not approved]</a></h4>

              
                Background The coronavirus disease 2019 (COVID-19) has caused substantial morbidity and mortality on a global scale. A strong correlation has been found between COVID-19 treatment outcomes and noncommunicable diseases such as cancers. However, there is limited information on the outcomes of cancer patients who were hospitalised for COVID-19. Methods We conducted an analysis on data collected in a large prospective cohort study set-up by the International Severe Acute Respiratory and Emerging Infection Consortium (ISARIC). All patients with laboratory-confirmed or clinically-diagnosed SARS-CoV-2 infection were included. Cancer was defined as having a current solid organ or haematological malignancy. The following outcomes were assessed; 30-day in-hospital mortality, intensive care unit (ICU) admission, length of hospitalization and receipt of higher-level care. Results Of the 560,547 hospitalised individuals who were analysed, 27,243 (4.9%) had cancer. Overall, cancer patients were older and had more comorbidities than non-cancer patients. Patients with cancer had higher 30-day in-hospital mortality than non-cancer patients (29.1.3% vs 18.0%) and longer hospital stays (median of 12 days vs 8 days). However, patients with cancer were admitted less often to intensive care units than non-cancer patients (12.6% vs 17.1%) and received less invasive mechanical ventilation than non-cancer patients (4.5% vs 7.6%). The hazard ratio of dying from cancer, adjusted for age, sex and country income level was 1.18 (95%CI: 1.15-1.2). Conclusions This study's findings underscore the heightened vulnerability of hospitalized COVID-19 patients with cancer, revealing a higher mortality rate, longer hospital stays, and an unstructured pattern of care that reflects the complexity of managing severely ill patients during a public health crisis like the COVID-19 pandemic.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 09:39:37 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-408/v1?src=rss">β-nicotinamide mononucleotide production in Vibrio natriegens: a preliminary study [version 1; peer review: awaiting peer review]</a></h4>

              
                Background Nicotinamide mononucleotide (NMN), is a promising nutraceutical attracting much attention for its pharmacological and anti-aging efficacies. However, NMN-containing commercial products are very high-priced due to the lack of efficient and facile methods for industrial-scale production. To date, various metabolic engineering strategies have been successfully applied to produce NMN in Escherichia coli. Recently, Vibrio natriegens has become a promising host in the bioindustry thanks to its rapid growth and capabilities of broad substrate utilization. This study aims to evaluate the NMN biosynthesis capability of V. natriegens. Methods Firstly, a mutant V. natriegens strain (Δdns::araC-T7RNAP-KanR ΔpncC::FtnadE-SmR ΔnadR) was generated via multiplex genome editing by natural transformation (MuGENT). Nampt genes encoding nicotinamide phosphoribosyltransferase from Chitinophaga pinensis, Sphingopyxis sp. C-1, Haemophilus ducreyi, and Vibrio phage KVP40 were codon-optimized and cloned into pACYCDuet™-1 under the control of T7 promoter. The recombinant plasmids were electroporated into the mutant strain. The expression of recombinant NAMPTs in V. natriegens was evaluated by SDS-PAGE analysis and the intracellular NMN concentrations were quantified by HPLC. Results After two rounds of MuGENT, V. natriegens V54-33 strain (Δdns::araC-T7RNAP-KanR ΔpncC::FtnadE-SmR ΔnadR) was successfully generated. SDS-PAGE analysis demonstrated that all NAMPTs were strongly expressed in the V54-33 strain. HPLC analysis revealed that the highest intracellular NMN concentration was obtained with NAMPT from Chitinophaga pinensis (44.5 μM), followed by NAMPT from Vibrio phage KVP40 (23.3 μM). Conclusion This study demonstrated the feasibility of NMN biosynthesis in V. natriegens.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 09:35:04 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-407/v1?src=rss">Red rice bran polyphenols suppress invasive properties of HepG2 cells, possibly through Wnt/β-catenin-mediated EMT reversal [version 1; peer review: awaiting peer review]</a></h4>

              
                Background Hepatocellular carcinoma (HCC) is the most invasive liver cancer, with high incidence rates and resistance to chemotherapy. Polyphenols from red rice bran extract (RRBE) exert diverse anti-cancer effects on certain cancers. This study, for the first time, examined the suppressive effects of RRBE on proliferation, migration, and invasion in HepG2 cells with a preliminary investigation into how its inhibitory effects are linked to reversing the EMT process via the Wnt/β-catenin pathway. Methods The anti-proliferative and cytotoxic effects of RRBE on HepG2 were determined by the resazurin cell viability assay. Its effects on the cell cycle distribution and the apoptosis induction in HepG2 cells were assessed by flow cytometry. Its ability to impede migration was tested using the scratch assay and the transwell migration assay. An initial study was conducted to explore the link between its inhibitory effect and the reversal of EMT via the Wnt/β-catenin signaling pathway using Western blot analysis. Polyphenols and flavonoids were quantified using the Folin-Ciocalteu assay and the aluminum chloride colorimetric assay, respectively. Results RRBE exhibited specific anti-proliferative and cytotoxic effects against HepG2 liver cancer cells compared to normal BNL CL2 liver cells in a dose- and time-dependent manner. RRBE caused a halt in the progression of HepG2 cells at the G2/M phase, resulting in subsequent apoptosis with variations in DNA content. The non-toxic doses of RRBE fully blocked the invasive migration of HepG2 cells after 24 h of its exposure as opposed to the untreated cells. Moreover, E-cadherin protein levels in HepG2 cells dose-dependently increased after 48 h of RRBE exposure, while the levels of β-catenin, vimentin, and MMP-9 proteins dose-dependently decreased compared to their respective untreated cells. The results indicated that RRBE-treated HepG2 cells showed a transition to epithelial characteristics along with diminished malignant features. Preliminary findings from the analysis of these proteins suggest that the inhibitory effects of RRBE may be connected to reversing EMT by regulating the Wnt/β-catenin signaling pathway. The phytochemical analysis revealed that RRBE contained high levels of polyphenols and flavonoids. Conclusions Red rice bran polyphenols demonstrated anti-proliferative, anti-migratory, and anti-invasive effects, which are possibly through Wnt/β-catenin-driven EMT reversal.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 09:33:27 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-406/v1?src=rss">Review the role of oxygen-delivering nanobubbles in stroke therapy: A novel approach [version 1; peer review: awaiting peer review]</a></h4>

              
                Abstract* Stroke remains a leading cause of mortality and long-term disability worldwide, necessitating innovative therapeutic strategies. The advent of nanotechnology, particularly oxygen-delivering nanobubbles (ODNBs), has introduced a promising avenue for enhancing stroke therapy. ODNBs have demonstrated the ability to improve oxygen delivery, enhance therapeutic efficacy, and provide diagnostic advantages through imaging contrast enhancement. However, challenges such as toxicity, off-target effects, and regulatory hurdles must be addressed before clinical translation. This review synthesizes the latest findings on ODNBs in stroke therapy, highlights their key benefits and challenges, and explores future applications, including gene therapy and brain tissue regeneration. By addressing these aspects, this review aims to provide a comprehensive understanding of the potential of ODNBs in revolutionizing stroke treatment.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 09:31:01 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-405/v1?src=rss">Assessing artificial intelligence knowledge among Al-Zahraa university students: A cross-sectional study [version 1; peer review: awaiting peer review]</a></h4>

              
                Introduction University educators’ knowledge of artificial intelligence (AI) helps them to effectively utilize these latest technological resources, significantly raising the quality of the teaching and learning process. Objective to evaluate a sample of Al-Zahraa university students' level of AI knowledge. Method From 5August 2024 to 28November 2024, data from Al-Zahraa University for Women students was obtained through an online questionnaire in a cross-sectional survey study. Data was downloaded to an Excel file from Google Forms following it was gathered. The questionnaire's quantitative data was imported and analyzed. Results The total number of participants was 498 participants; however, 89 students refused to answer the questions, which reduced the sample size to 409. Most of the students (90%) reported to be familiar or somewhat familiar with Artificial Intelligence. More than one half of the From 5August 2024 to 28November 2024, data from Al-Zahraa University for Women students was obtained through an online questionnaire in a cross-sectional survey study. Most Al-Zahraa Medical School students got an anonymous online survey. Using a pre-validated, semi-structured questionnaire, 419 medical students in Al-Zahraa university for women engaged in a cross-sectional study.students (57.7%) know AI, whereas a smaller percentage (42.7%) reported to know AI medical application. only one quarter (25.7%) reported having knowledge about machine deep learning. one half (49%) of the students considered it as extremely important in the medical field. Conclusions The study discovered that while Al-Zahraa students understand artificial intelligence (AI) well, they know not much about deep learning, machine learning, and AI applications in radiology and pathology.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 09:28:59 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/12-771/v2?src=rss">Fatalism and knowledge associated with type 2 diabetes mellitus risk perception in the Mexican population during COVID-19 confinement [version 2; peer review: 2 not approved]</a></h4>

              
                Background Individuals with fatalistic beliefs and lack of knowledge of healthy lifestyles tend to show conformism and resignation due to their state of health. Even individuals with fatalism consider that contracting a disease is divine punishment, and this is reflected when they perform unhealthy behaviours. The relationship between fatalistic beliefs, knowledge about diabetes risk factors, and perceived risk of T2DM in Mexican adults during the COVID-19 pandemic were evaluated. Objective The aim of this research was to examine the association between fatalistic beliefs, knowledge of T2DM risk factors, and perceived risk of T2DM among Mexican adults during the COVID-19 pandemic. Methods The study design was cross-sectional and correlational. The sample size was 222 individuals between 30 and 70 years of age, from a rural community in Guanajuato, Mexico. We used the Multidimensional Fatalism Scale in Spanish, Risk Perception Survey for Developing Diabetes. For the correlation analysis, Spearman’s coefficient and multiple linear regression were used to explain the perceived risk of T2DM. Results The average age was 29.2 years (SD = 10.5), 55% were women, 52.3% were single, 36.2% had university studies and 57.3% were working. Perceived risk of T2DM was inversely related to fatalistic beliefs. Knowledge level was positively correlated to perceived risk of T2DM. In multiple linear regression, pessimism and divine control dimensions were the only predictors of perceived risk of T2DM. Conclusions In young adults, knowledge increases regarding perceived risk of T2DM. But the higher the fatalistic beliefs the lower the level of knowledge and the lower the perceived risk of T2DM.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 09:23:06 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/13-996/v2?src=rss">Analyzing the Relationship Between Organizational Politics, Emotional Intelligence, and Employee Behavior: A PLS-SEM Modeling [version 2; peer review: 1 approved, 1 approved with reservations]</a></h4>

              
                Background This study explores the complex relationship between organizational politics, emotional intelligence and employee behavior in contemporary settings. Organizational politics, which is widespread in organizational settings, has a substantial impact on different aspects of employee behavior. Emotional Intelligence has become a vital factor in individuals’ capacity to successfully traverse the complexities of an organization. This study consolidates current material to offer insights into the impact of organizational politics on employee behavior and the mediating role of emotional intelligence in this connection. Methods A survey was undertaken with a sample size of 500 employees hailing from several Information Technology (IT) and Information Technology Enabled Services (ITES) firms in the Dehradun (Uttarakhand) region. The study employed quantitative methodologies to investigate the correlations between Emotional Intelligence, Perceived Organizational Politics and Employee Behavior. The data were evaluated using Partial Least Squares Structural Equation Modelling (PLS-SEM) to examine the hypothesized correlations and mediation effects. Results The study identified substantial correlations between Emotional Intelligence, Perceived Organizational Politics, and Employee Behavior. Emotional Intelligence was discovered to have a positive impact on Behavioral Dynamics. Perceived Organizational Politics had a substantial influence on both Emotional Intelligence and Behavioral Dynamics. Factors such as Need for Power, Relationship Conflict, Resource Constraints, Role Conflict, and Workforce Diversity were discovered to have an impact on Perceived Organizational Politics. Conclusions The results confirm strong connections between organizational politics, emotional intelligence, and employee behavior, highlighting the significance of these concepts in comprehending employee behavior in organizational environments. The study proposes that enhancing employees' emotional intelligence can alleviate the adverse effects of organizational politics, resulting in enhanced behavioral dynamics. The study also addresses the limits, outlines potential areas for further research, and highlights the managerial implications. It suggests that firms should prioritize the development of emotional intelligence to cultivate a favorable work environment.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 09:20:36 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/13-1043/v2?src=rss">The quantified method for blood clot detection in the extraction socket [version 2; peer review: 1 approved, 2 approved with reservations]</a></h4>

              
                Abstract* Background Currently, there is no objective and quantified measurement for detecting blood clots during extraction socket hemostasis. It has relied solely on clinical observation, even when conducting clinical research by using extraction sockets as samples. This study aimed to assess the in vitro reliability and clinical-relevant validity of a new objective measurement providing quantified data called blood clot detection (BCD) using a standard capillary tube. Methods The in vitro part of the study was conducted using surplus blood samples from ten healthy participants. Two identical sets of blood samples in simulation reservoirs mimicking bleeding sockets were prepared for reliability tests. Then, the capillary tubes were concurrently placed in the reservoirs. The blood-filled distances were measured. The part of clinical-relevant validity study was conducted in sixteen extraction sockets from each healthy participant. Clinical observation and BCD measurement were evaluated by two calibrated assessors. The total duration of the assessment was a 30-minute. Results The distances of the blood-filled capillary tube were decreased by time. Test and retest reliability analysis of the BCD measurement showed an excellent intraclass correlation coefficient of 0.980 (0.968 to 0.988). The medians of blood-filled distance categorized by clinical observation into active bleeding, sluggish oozing, and clot formation were 13.0 mm (Q1 = 11.7, Q3 = 13.8), 5.6 mm (Q1 = 4.3, Q3 = 7.0), and 0.9 mm (Q1 = 0.5, Q3 = 1.3), respectively. The blood-filled distance of the clot formation group was significantly less than the active bleeding and sluggish oozing (p<0.001). Therefore, the BCD measurement also significantly indicated the completion of extraction socket hemostasis Conclusions A distance of blood-filled in capillary tube of 0.9 mm from the BCD measurement significantly ensured complete clot formation. The BCD measurement proved to be a quantified tool for objectively measuring hemostasis of bleeding socket.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 09:18:54 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-180/v2?src=rss">Exploring the relationship between macroeconomic indicators and sectoral indices of Indian stock market [version 2; peer review: 1 approved, 1 approved with reservations]</a></h4>

              
                Background of the study The influence of macroeconomic indicators makes it important to study the relationship between macroeconomic indicators and stock market return. On further analysis it can be observed that different sectors respond differently to change in the macroeconomic indicator that is important for investors, researchers and policy makers. Methods The autoregressive distributed lag (ARDL) model is applied to study influence of macroeconomic indicators on sectoral return of NSE from April 2012 to August 2024. Results Findings of the study show that macroeconomic indicators influence sectoral return in the short run as well as long run and the influence is differential. The analysis of long run relationship shows that Foreign Institutional Investment (FII) significantly affects all the sectoral indices except IT. Index of industrial production (IIP) have significant relationship with Auto, IT, Media, Metal and Pharma. Money supply (MS) significantly affects Bank, FMCG and IT in the long run. Wholesale Price Index (WPI) has significant relationship with Auto, FMCG and Media in the long run. Economic Policy Uncertainty Index (EPU) affects Auto, FMCG and Pharma in the long run. Crude oil price (COP) has significant effect only on Media in the long run. Exchange rate (ER) does not have significant effect on any of the sectoral index. Conclusion In the long run FII, IIP, EPU, MS and WIP are major determinants of stock market return. In the short run FII, ER and COP are major determinants of stock market return.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 09:16:29 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/13-1502/v2?src=rss">Does gold open-access publishing increase journal’s impact? An assessment of hybrid versus gold open-access publishing models of medical journals on their impact [version 2; peer review: 2 not approved]</a></h4>

              
                Background Hybrid and gold open-access (OA) are common publishing models. The latter requires fees to allow full-text visibility, whereas hybrid journals offer the option to publish gold OA or for free (subscription-based) where users with access can get the full-text. We evaluated the impact of the publishing model and other factors on medical journals’ impact. Methods A sample of hybrid and gold OA medical journals indexed in Web of Science (WOS) and Scopus were included. The effect of the publishing model and other factors on journals’ impact factor (IF), CiteScore, quartile, and number of citations was assessed. Results 402 journals were included, 201 in each group. Hybrid and gold OA journals had a median age of 32 and 21 years, respectively (P<0.001). While gold OA journals had a slightly higher median IF (3.3 vs. 2.9; P=0.021), more hybrid journals had Q1 ranking in Scopus (57.7% vs. 43.8%; P=0.091). Publishing model, journal’s age, being of an organization/society, and EMBASE indexation didn’t affect IF, CiteScore, number of citations, and WOS quartile (P>0.05). However, gold OA model wasn’t significantly associated with Scopus Q1 ranking (OR, 0.49; 95% CI, 0.25-0.94; P=0.032), indicating that hybrid journals were more likely to have such ranking. Conclusion These findings indicate that gold OA publishing doesn’t result in higher impact or journals ranking. Therefore, authors can continue to submit their work to high impact hybrid journals if their interest is getting published in reputable prestigious journals regardless of their publishing model. Gold OA journals are encouraged to reduce their fees to facilitate global research access.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 09:14:30 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-404/v1?src=rss">Avocado (Persea americana) in diabetes management: a systematic review of preclinical evidence and its translational potential [version 1; peer review: awaiting peer review]</a></h4>

              
                Background Diabetes remains a major health challenge in low- and middle-income countries like Uganda, where access to effective treatment is often limited. Avocado (Persea americana) has demonstrated antidiabetic potential in global preclinical studies, but research on its pharmacological benefits within Uganda’s healthcare context is scarce. This review synthesizes global evidence on avocado’s bioactive compounds and therapeutic effects, highlighting gaps that warrant further investigation. Methods A systematic search was conducted across PubMed, Scopus, and Web of Science to identify peer-reviewed English-language studies on avocado’s antidiabetic properties. Eligible studies included preclinical models and intervention studies. This review adheres to PRISMA 2020 guidelines. Data extraction and quality assessment were performed independently by two reviewers. Of the 266 articles initially identified, 45 met the inclusion criteria. Results Avocado extracts significantly improve metabolic markers, with 30 studies reporting statistically significant reductions in blood glucose levels (p < 0.05). The bioactive compounds modulate key diabetes-related pathways, suggesting a multi-target therapeutic approach. However, most evidence originates from global research, with limited studies conducted in African populations or within Uganda’s healthcare setting. While toxicity assessments indicate a favorable safety profile, challenges such as standardization of extraction methods, dosage optimization, and clinical validation persist. Conclusion Avocado demonstrates strong antidiabetic potential, but region-specific research is essential to validate its efficacy and practical application in Uganda. Strengthening local research efforts could bridge the gap between global evidence and Uganda’s healthcare needs, facilitating its evidence-based integration into traditional and modern diabetes treatment strategies.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 09:11:13 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/13-1073/v3?src=rss">A novel hybrid genetic algorithm and Nelder-Mead approach and it’s application for parameter estimation [version 3; peer review: 1 approved, 1 approved with reservations]</a></h4>

              
                Background Traditional optimization methods often struggle to balance global exploration and local refinement, particularly in complex real-world problems. To address this challenge, we introduce a novel hybrid optimization strategy that integrates the Nelder-Mead (NM) technique and the Genetic Algorithm (GA), named the Genetic and Nelder-Mead Algorithm (GANMA). This hybrid approach aims to enhance performance across various benchmark functions and parameter estimation tasks. Methods GANMA combines the global search capabilities of GA with the local refinement strength of NM. It is first tested on 15 benchmark functions commonly used to evaluate optimization strategies. The effectiveness of GANMA is also demonstrated through its application to parameter estimation problems, showcasing its practical utility in real-world scenarios. Results GANMA outperforms traditional optimization methods in terms of robustness, convergence speed, and solution quality. The hybrid algorithm excels across different function landscapes, including those with high dimensionality and multimodality, which are often encountered in real-world optimization issues. Additionally, GANMA improves model accuracy and interpretability in parameter estimation tasks, enhancing both model fitting and prediction. Conclusions GANMA proves to be a flexible and powerful optimization method suitable for both benchmark optimization and real-world parameter estimation challenges. Its capability to efficiently explore parameter spaces and refine solutions makes it a promising tool for scientific, engineering, and economic applications. GANMA offers a valuable solution for improving model performance and effectively handling complex optimization problems.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 09:09:21 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/12-1055/v3?src=rss">Case Report: Synchronous primary location of gastrointestinal stromal tumors (GIST) and adenocarcinoma of the colon: an unusual occurrence [version 3; peer review: 1 approved with reservations]</a></h4>

              
                Background We have little knowledge about the synchronous occurrence of gastrointestinal stromal tumors (GISTs) and other types of histologic tumors. This association is very rare. Case presentation We describe a case of synchronous stromal tumor and adenocarcinoma of the left side colonic localization. Immunohistochemistry identified c-Kit expression. The discovery of colonic adenocarcinoma was on operative specimen after histologic examination. The patient underwent left oncologic colectomy with stoma. Follow-up at one year postoperatively did not detect tumor recurrence. Discussion Clinical implications of the association between these two neoplasms are not clearly described. Treatment depends on the the most aggressive histologic type or obviously highest stage. Knowledge of the genetic data of this association offers opportunity of treatment with the new targeted-therapy molecules. Surgical resection, may remain the curative treatment. Conclusions Synchronous adenocarcinoma and GIST has been more commonly described in the stomach.  The pathogeneses of tumorigenesis may not be the same for the two tumors. More studies seem be necessary to clarify a potential role of different genes in the development of adenocarcinomas. And therefore, above all their therapeutic implications.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 09:07:06 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/12-1464/v3?src=rss">Lived experiences of women who survived from pre-eclampsia and eclampsia in public hospitals of Shashemene Town, Oromia Region, Ethiopia: a qualitative study. [version 3; peer review: 1 approved, 1 approved with reservations]</a></h4>

              
                Objective The main purpose of this study was to explore experiences of women who survived pre-eclampsia and eclampsia in Shashemene referral hospital and Melka Oda general hospital, 2021. Design An institution-based exploratory qualitative study with a descriptive phenomenological study design Setting The present study was conducted in Shashemene referral hospital and Melka Oda general hospital from March 03 to May 18, 2021. Participants A total of 17 individual in-depth interviews (IDIs) were conducted with women who experienced and survived pre-eclampsia and eclampsia. Data were analyzed through thematic analysis using Atlas-ti software. Result The present study revealed that women’s level of awareness on raised blood pressure and or convulsion occurring during pregnancy was poor. Barriers that limit pregnant women from getting treatment at the earliest point included misconception, lack of insight, failure to accept counseling from health care providers, low income, and influence from husbands. Discussions with the women in this study showed that these mothers were not counseled on danger signs during antenatal care check-ups. Almost all of the women were very concerned and frustrated due to their diagnosis. Conclusions Women’s awareness of pre-eclampsia and eclampsia was poor. Improving awareness of hypertensive disorders in pregnancy and enhancing the quality of antenatal care is essential.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 09:02:43 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-403/v1?src=rss">Correlation of transcutaneous and serum bilirubin levels in late preterm and term neonates at a tertiary care center in south India [version 1; peer review: awaiting peer review]</a></h4>

              
                Background Neonatal jaundice is one of the most prevalent condition during first week of life causing morbidity and even mortality in few, especially in low – middle income countries. Although visual inspection for jaundice has been a time tested method, serum bilirubin is the gold standard investigation of choice. Due to this newborn receive many heel or vein pricks for testing, hence transcutaneous bilirubinometer can be a helpful non-invasive tool for diagnosing jaundice requiring phototherapy. Methods This prospective study was carried out in a tertiary care hospital in Mangalore, Karnataka to compare a non invasive method of detecting bilirubin levels and serum bilirubin levels. Performance of a transcutaneous bilirubinometer Dräger Jaundice Meter JM-105 was assessed against routine venous serum bilirubin testing before phototherapy during neonatal care in the first two weeks of life. Results were derived by analysing the correlation coefficient between two methods and direct agreement was analysed using Bland Altman analysis. Results Total of 271 neonates (>35 weeks) were included in the study. Transcutaneous bilirubinometry and serum bilirubin values were done on all of them in the first week of life. Correlation analysis showed significant relationship with a Pearson correlation coefficient of 0.629. Values of transcutaneous bilirubinometer showed excellent agreement with venous serum bilirubin concentration in Bland Altman analysis. Conclusions The transcutaneous bilirubinometer is a reliable tool to screen neonates and identify those needing phototherapy there by reducing invasive blood sampling.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-07 08:58:06 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro&title=PNAS" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.pnas.org/doi/abs/10.1073/pnas.2409985122?af=R">Whole-brain white matter variation across childhood environments</a></h4>

              
                Proceedings of the National Academy of Sciences, Volume 122, Issue 15, April 2025. <br/>SignificanceThe richness of a child’s early-life environment robustly predicts later cognitive abilities. White matter is likely also a key mediator of this link, given that communication along myelinated fiber tracts drives the maturation of brain ...
              

              <p class="text-muted">in <a href ="https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro">PNAS</a> on 2025-04-07 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro&title=PNAS" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.pnas.org/doi/abs/10.1073/pnas.2506044122?af=R">Correction for He et al., Radiation-induced cellular plasticity primes glioblastoma for forskolin-mediated differentiation</a></h4>

              
                Proceedings of the National Academy of Sciences, Volume 122, Issue 15, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro">PNAS</a> on 2025-04-07 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00459.2024?af=R">Single and paired TMS pulses engage spatially distinct corticomotor representations in human pericentral cortex</a></h4>

              
                Journal of Neurophysiology, Ahead of Print. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-07 01:31:24 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00282-7?rss=yes">ARHGAP12 suppresses F-actin assembly to control epithelial tight junction mechanics and paracellular leak pathway permeability</a></h4>

              
                Tambrin et al. show that the Cdc42/Rac GAP ARHGAP12 promotes the flux of macromolecules across tight junctions by suppressing junctional actin assembly and tension via N-WASP and the Arp2/3 complex. This study reveals a mechanism by which epithelial cells can fine-tune the permeability of the tight junction leak pathway.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00286-4?rss=yes">Development of antibiotic resistome in premature infants</a></h4>

              
                Keshet et al. investigate the impact of antibiotic treatment on the gut microbiome and resistome of preterm infants, highlighting the protective role of human milk in reducing antibiotic resistance and fostering beneficial microbiota. The study emphasizes balancing neonatal antibiotic use with nutritional strategies for healthy gut development.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00306-7?rss=yes">SFRP1 upregulation causes hippocampal synaptic dysfunction and memory impairment</a></h4>

              
                Pereyra et al. reveal that accumulation of the glial-derived matrisome protein, SFRP1, causes age-dependent synaptic and cognitive deficits by disrupting the turnover of structural synaptic proteins, thereby mirroring traits present in the brain of Alzheimer’s disease patients.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00279-7?rss=yes">Age-dependent glial heterogeneity and traumatic injury responses in a vertebrate brain structure</a></h4>

              
                Qin et al. present a remarkable single-cell transcriptome atlas of the three major glia types, exploring both physiological condition and the impacts of traumatic brain injury (TBI) throughout various post-embryonic life stages. The findings offer insights into the molecular and cellular foundations of age-related glial changes in TBI vertebrate brains.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/inpress.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/inpress.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/inpress.rss&title=Cell Reports: In press" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00275-X?rss=yes">Multiphasic protein condensation governed by shape and valency</a></h4>

              
                Pandey et al. present a computational model that reproduces various forms of liquid-liquid phase separation that occur in a mixture of four synaptic proteins. In particular, a characteristic phase-in-phase structure appears if CaMKII has a short linker and high valency and when GluN2B and STG competitively bind to PSD-95.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/inpress.rss">Cell Reports: In press</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/inpress.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/inpress.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/inpress.rss&title=Cell Reports: In press" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00211-6?rss=yes">Newly generated striatal neurons rescue motor circuitry in a Huntington’s disease mouse model</a></h4>

              
                Cano and colleagues use rabies tracing, optogenetics, calcium imaging, and chemogenetics to show that new neurons are recruited from the SVZ to the neostriatum after i.c.v. infusion of BDNF and Noggin and that the new neurons can restore cortico-striato-pallidal circuits and mediate motor behavior in a Huntington’s disease mouse model.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/inpress.rss">Cell Reports: In press</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41586-025-08924-0">Author Correction: Recapitulation of premature ageing with iPSCs from Hutchinson–Gilford progeria syndrome</a></h4>

              
                <p>Nature, Published online: 07 April 2025; <a href="https://www.nature.com/articles/s41586-025-08924-0">doi:10.1038/s41586-025-08924-0</a></p>Author Correction: Recapitulation of premature ageing with iPSCs from Hutchinson–Gilford progeria syndrome
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-01070-7">Studying seabirds with a cactus as a research assistant</a></h4>

              
                <p>Nature, Published online: 07 April 2025; <a href="https://www.nature.com/articles/d41586-025-01070-7">doi:10.1038/d41586-025-01070-7</a></p>Ecologist Cecilia Soldatini tracks the movements of magnificent frigatebirds.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00960-0">From Hippocrates to COVID-19: the scientific fight to prove diseases can be airborne</a></h4>

              
                <p>Nature, Published online: 07 April 2025; <a href="https://www.nature.com/articles/d41586-025-00960-0">doi:10.1038/d41586-025-00960-0</a></p>Science journalist and author Carl Zimmer joins us to talk about his new book Air-borne.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-01063-6">Why more AI researchers should collaborate with governments</a></h4>

              
                <p>Nature, Published online: 07 April 2025; <a href="https://www.nature.com/articles/d41586-025-01063-6">doi:10.1038/d41586-025-01063-6</a></p>Academics can drive policy innovation — but they must shift their focus from publishing papers to creating practical products.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00860-3">Creating an ‘all comers’ research group in quantitative history</a></h4>

              
                <p>Nature, Published online: 07 April 2025; <a href="https://www.nature.com/articles/d41586-025-00860-3">doi:10.1038/d41586-025-00860-3</a></p>Nathan Alexander's laboratory welcomes ‘anyone and everyone’ to use a mix of mathematics and computing to solve challenges in society.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-01069-0">AI for research: the ultimate guide to choosing the right tool</a></h4>

              
                <p>Nature, Published online: 07 April 2025; <a href="https://www.nature.com/articles/d41586-025-01069-0">doi:10.1038/d41586-025-01069-0</a></p>Curious about using artificial intelligence to boost your research? Here are the programs you shouldn’t miss.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-00995-3">Long COVID activists fought Trump team’s research cuts and won ― for now</a></h4>

              
                <p>Nature, Published online: 07 April 2025; <a href="https://www.nature.com/articles/d41586-025-00995-3">doi:10.1038/d41586-025-00995-3</a></p>Success gives hope to scientists and advocates who managed to get millions of dollars in grants restored.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-01033-y">AI race in 2025 is tighter than ever before</a></h4>

              
                <p>Nature, Published online: 07 April 2025; <a href="https://www.nature.com/articles/d41586-025-01033-y">doi:10.1038/d41586-025-01033-y</a></p>State of the industry report also shows that 2024 was a breakthrough year for small, sleek models to rival the behemoths.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-01045-8">Where do proteins go in cells? Next-generation methods map the molecules’ hidden lives</a></h4>

              
                <p>Nature, Published online: 07 April 2025; <a href="https://www.nature.com/articles/d41586-025-01045-8">doi:10.1038/d41586-025-01045-8</a></p>Spatial proteomics is helping biologists to uncover how cells work by mapping where proteins operate.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-01067-2">Why an overreliance on AI-driven modelling is bad for science</a></h4>

              
                <p>Nature, Published online: 07 April 2025; <a href="https://www.nature.com/articles/d41586-025-01067-2">doi:10.1038/d41586-025-01067-2</a></p>Without clear protocols to catch errors, artificial intelligence’s growing role in science could do more harm than good.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/neuro/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/neuro/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/neuro/rss/current&title=Nature Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41593-025-01931-4">Microglia heterogeneity, modeling and cell-state annotation in development and neurodegeneration</a></h4>

              
                <p>Nature Neuroscience, Published online: 07 April 2025; <a href="https://www.nature.com/articles/s41593-025-01931-4">doi:10.1038/s41593-025-01931-4</a></p>Microglia have key roles in CNS development and neurodegeneration. Here, the authors provide an overview of microglia heterogeneity, cell-state annotation and model systems.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/neuro/rss/current">Nature Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58603-x">Author Correction: Brain milieu induces early microglial maturation through the BAX-Notch axis</a></h4>

              
                <p>Nature Communications, Published online: 07 April 2025; <a href="https://www.nature.com/articles/s41467-025-58603-x">doi:10.1038/s41467-025-58603-x</a></p>Author Correction: Brain milieu induces early microglial maturation through the BAX-Notch axis
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58610-y">Author Correction: ITGB4/CD104 mediates zika virus attachment and infection</a></h4>

              
                <p>Nature Communications, Published online: 07 April 2025; <a href="https://www.nature.com/articles/s41467-025-58610-y">doi:10.1038/s41467-025-58610-y</a></p>Author Correction: ITGB4/CD104 mediates zika virus attachment and infection
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58702-9">Author Correction: Operando TEM study of a working copper catalyst during ethylene oxidation</a></h4>

              
                <p>Nature Communications, Published online: 07 April 2025; <a href="https://www.nature.com/articles/s41467-025-58702-9">doi:10.1038/s41467-025-58702-9</a></p>Author Correction: <i>Operando</i> TEM study of a working copper catalyst during ethylene oxidation
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58611-x">Publisher Correction: Reply to: On the giant deformation and ferroelectricity of guanidinium nitrate</a></h4>

              
                <p>Nature Communications, Published online: 07 April 2025; <a href="https://www.nature.com/articles/s41467-025-58611-x">doi:10.1038/s41467-025-58611-x</a></p>Publisher Correction: Reply to: On the giant deformation and ferroelectricity of guanidinium nitrate
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58701-w">Author Correction: Accelerating discovery of bioactive ligands with pharmacophore-informed generative models</a></h4>

              
                <p>Nature Communications, Published online: 07 April 2025; <a href="https://www.nature.com/articles/s41467-025-58701-w">doi:10.1038/s41467-025-58701-w</a></p>Author Correction: Accelerating discovery of bioactive ligands with pharmacophore-informed generative models
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58598-5">Author Correction: Assessing demographic vulnerability and weather impacts on utility disconnections in California</a></h4>

              
                <p>Nature Communications, Published online: 07 April 2025; <a href="https://www.nature.com/articles/s41467-025-58598-5">doi:10.1038/s41467-025-58598-5</a></p>Author Correction: Assessing demographic vulnerability and weather impacts on utility disconnections in California
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/ncomms/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/ncomms/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/ncomms/rss/current&title=Nature Communications" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41467-025-58596-7">Author Correction: Quantifying the distinct role of plasmon enhancement mechanisms in prototypical antenna-reactor photocatalysts</a></h4>

              
                <p>Nature Communications, Published online: 07 April 2025; <a href="https://www.nature.com/articles/s41467-025-58596-7">doi:10.1038/s41467-025-58596-7</a></p>Author Correction: Quantifying the distinct role of plasmon enhancement mechanisms in prototypical antenna-reactor photocatalysts
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/ncomms/rss/current">Nature Communications</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nphys/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nphys/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nphys/rss/current&title=Nature Physics" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41567-025-02840-w">Marangoni flows underlie symmetry breaking in early mouse gastruloids</a></h4>

              
                <p>Nature Physics, Published online: 07 April 2025; <a href="https://www.nature.com/articles/s41567-025-02840-w">doi:10.1038/s41567-025-02840-w</a></p>Spherical aggregates of mouse stem cells exhibit symmetry breaking by forming an elongated axis. This extension is driven by a recirculating Marangoni-like tissue flow, providing insights into the tissue mechanics underlying embryonic development.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nphys/rss/current">Nature Physics</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nphys/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nphys/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nphys/rss/current&title=Nature Physics" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41567-025-02874-0">Observation of polarization density waves in SrTiO3</a></h4>

              
                <p>Nature Physics, Published online: 07 April 2025; <a href="https://www.nature.com/articles/s41567-025-02874-0">doi:10.1038/s41567-025-02874-0</a></p>Despite exhibiting ferroelectric features, SrTiO3 fails to display long-range polar order at low temperatures due to quantum fluctuations. An ultrafast X-ray diffraction experiment now probes polar dynamics of this material at the nanometre scale.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nphys/rss/current">Nature Physics</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nphys/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nphys/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nphys/rss/current&title=Nature Physics" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41567-025-02799-8">The strongly driven Fermi polaron</a></h4>

              
                <p>Nature Physics, Published online: 07 April 2025; <a href="https://www.nature.com/articles/s41567-025-02799-8">doi:10.1038/s41567-025-02799-8</a></p>Fermi polarons are quasiparticles formed by impurities immersed in a Fermi gas. An experiment in an ultracold fermionic gas now shows how to control their properties with a tunable radio-frequency field.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nphys/rss/current">Nature Physics</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04918-9">Electoral precinct-level database for Mexican municipal elections</a></h4>

              
                <p>Scientific Data, Published online: 07 April 2025; <a href="https://www.nature.com/articles/s41597-025-04918-9">doi:10.1038/s41597-025-04918-9</a></p>Electoral precinct-level database for Mexican municipal elections
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/105352">Improving gender equity in academia depends on the workplace environment</a></h4>

              
                The iDiv Female Scientists initiative at the German Centre for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig in Germany was set up to connect women in science and to raise awareness of gender inequity. In this article we discuss the impact of the workplace environment on women in academia. Our experiences indicate that supportive workplace environments are more likely to discuss gender inequity, enabling “bottom-up” approaches where individual researchers can propose solutions to the problem. In contrast, unsupportive environments are less receptive to such discussions, so “top-down” approaches driven by legislation policies are required to improve the situation. We also make recommendations of actions that can be taken by individuals, institutions and policymakers to promote gender equity.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/103109">Cellular evolution of the hypothalamic preoptic area of behaviorally divergent deer mice</a></h4>

              
                Genetic variation is known to contribute to the variation of animal social behavior, but the molecular mechanisms that lead to behavioral differences are still not fully understood. Here, we investigate the cellular evolution of the hypothalamic preoptic area (POA), a brain region that plays a critical role in social behavior, across two sister species of deer mice (<i>Peromyscus maniculatus</i> and <i>P. polionotus</i>) with divergent social systems. These two species exhibit large differences in mating and parental care behavior across species and sex. Using single-nucleus RNA-sequencing, we build a cellular atlas of the POA for males and females of both <i>Peromyscus</i> species. We identify four cell types that are differentially abundant across species, two of which may account for species differences in parental care behavior based on known functions of these cell types. Our data further implicate two sex-biased cell types to be important for the evolution of sex-specific behavior. Finally, we show a remarkable reduction of sex-biased gene expression in <i>P. polionotus</i>, a monogamous species that also exhibits reduced sexual dimorphism in parental care behavior. Our POA atlas is a powerful resource to investigate how molecular neuronal traits may be evolving to give rise to innate differences in social behavior across animal species.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/101043">Synaptic cell adhesion molecule Cdh6 identifies a class of sensory neurons with novel functions in colonic motility</a></h4>

              
                Intrinsic sensory neurons are an essential part of the enteric nervous system (ENS) and play a crucial role in gastrointestinal tract motility and digestion. Neuronal subtypes in the ENS have been distinguished by their electrophysiological properties, morphology, and expression of characteristic markers, notably neurotransmitters and neuropeptides. Here, we investigated synaptic cell adhesion molecules as novel cell-type markers in the ENS. Our work identifies two type II classic cadherins, <i>Cdh6</i> and <i>Cdh8,</i> specific to sensory neurons in the mouse colon. We show that <i>Cdh6+</i> neurons demonstrate all other distinguishing classifications of enteric sensory neurons including marker expression of <i>Calcb</i> and <i>Nmu</i>, Dogiel type II morphology and AH-type electrophysiology and <i>I<sub>H</sub></i> current. Optogenetic activation of <i>Cdh6+</i> sensory neurons in distal colon evokes retrograde colonic motor complexes (CMCs), while pharmacologic blockade of rhythmicity-associated current <i>I<sub>H</sub></i> disrupts the spontaneous generation of CMCs. These findings provide the first demonstration of selective activation of a single neurochemical and functional class of enteric neurons and demonstrate a functional and critical role for sensory neurons in the generation of CMCs.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/90139">Ab initio prediction of specific phospholipid complexes and membrane association of HIV-1 MPER antibodies by multi-scale simulations</a></h4>

              
                A potent class of HIV-1 broadly neutralizing antibodies (bnAbs) targets the envelope glycoprotein’s membrane proximal exposed region (MPER) through a proposed mechanism where hypervariable loops embed into lipid bilayers and engage headgroup moieties alongside the epitope. We address the feasibility and determinant molecular features of this mechanism using multi-scale modeling. All-atom simulations of 4E10, PGZL1, 10E8, and LN01 docked onto HIV-like membranes consistently form phospholipid complexes at key complementarity-determining region loop sites, solidifying that stable and specific lipid interactions anchor bnAbs to membrane surfaces. Ancillary protein-lipid contacts reveal surprising contributions from antibody framework regions. Coarse-grained simulations effectively capture antibodies embedding into membranes. Simulations estimating protein-membrane interaction strength for PGZL1 variants along an inferred maturation pathway show bilayer affinity is evolved and correlates with neutralization potency. The modeling demonstrated here uncovers insights into lipid participation in antibodies’ recognition of membrane proteins and highlights antibody features to prioritize in vaccine design.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/103245">Preclinical systematic review of CCR5 antagonists as cerebroprotective and stroke recovery enhancing agents</a></h4>

              
                C-C chemokine receptor type 5 (CCR5) antagonists may improve both acute stroke outcome and long-term recovery. Despite their evaluation in ongoing clinical trials, gaps remain in the evidence supporting their use. With a panel of patients with lived experiences of stroke, we performed a systematic review of animal models of stroke that administered a CCR5 antagonist and assessed infarct size or behavioural outcomes. MEDLINE, Web of Science, and Embase were searched. Article screening and data extraction were completed in duplicate. We pooled outcomes using random effects meta-analyses. We assessed risk of bias using the Systematic Review Centre for Laboratory Animal Experimentation (SYRCLE) tool and alignment with the Stroke Treatment Academic Industry Roundtable (STAIR) and Stroke Recovery and Rehabilitation Roundtable (SRRR) recommendations. Five studies representing 10 experiments were included. CCR5 antagonists reduced infarct volume (standard mean difference −1.02; 95% confidence interval −1.58 to −0.46) when compared to stroke-only controls. Varied timing of CCR5 administration (pre- or post-stroke induction) produced similar benefit. CCR5 antagonists significantly improved 11 of 16 behavioural outcomes reported. High risk of bias was present in all studies and critical knowledge gaps in the preclinical evidence were identified using STAIR/SRRR. CCR5 antagonists demonstrate promise; however, rigorously designed preclinical studies that better align with STAIR/SRRR recommendations and downstream clinical trials are warranted. Prospective Register of Systematic Reviews (PROSPERO CRD42023393438).
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/101626">N6-methyladenosine in DNA promotes genome stability</a></h4>

              
                DNA base lesions, such as incorporation of uracil into DNA or base mismatches, can be mutagenic and toxic to replicating cells. To discover factors in repair of genomic uracil, we performed a CRISPR knockout screen in the presence of floxuridine, a chemotherapeutic agent that incorporates uracil and fluorouracil into DNA. We identified known factors, such as uracil DNA N-glycosylase (UNG), and unknown factors, such as the N6-adenosine methyltransferase, METTL3, as required to overcome floxuridine-driven cytotoxicity. Visualized with immunofluorescence, the product of METTL3 activity, N6-methyladenosine, formed nuclear foci in cells treated with floxuridine. The observed N6-methyladenosine was embedded in DNA, called 6mA, and these results were confirmed using an orthogonal approach, liquid chromatography coupled to tandem mass spectrometry. METTL3 and 6mA were required for repair of lesions driven by additional base-damaging agents, including raltitrexed, gemcitabine, and hydroxyurea. Our results establish a role for METTL3 and 6mA in promoting genome stability in mammalian cells, especially in response to base damage.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/104199">FtsK is critical for the assembly of the unique divisome complex of the FtsZ-less Chlamydia trachomatis</a></h4>

              
                <i>Chlamydia trachomatis</i> serovar L2 (<i>Ct),</i> an obligate intracellular bacterium that does not encode FtsZ, divides by a polarized budding process. In the absence of FtsZ, we show that FtsK, a chromosomal translocase, is critical for divisome assembly in <i>Ct</i>. Chlamydial FtsK forms discrete foci at the septum and at the base of the progenitor mother cell, and our data indicate that FtsK foci at the base of the mother cell mark the location of nascent divisome complexes that form at the site where a daughter cell will emerge in the next round of division. The divisome in <i>Ct</i> has a hybrid composition, containing elements of the divisome and elongasome from other bacteria, and FtsK is recruited to nascent divisomes prior to the other chlamydial divisome proteins assayed, including the PBP2 and PBP3 transpeptidases, and MreB and MreC. Knocking down FtsK prevents divisome assembly in <i>Ct</i> and inhibits cell division and septal peptidoglycan synthesis. We further show that MreB does not function like FtsZ and serve as a scaffold for the assembly of the <i>Ct</i> divisome. Rather, MreB is one of the last proteins recruited to the chlamydial divisome, and it is necessary for the formation of septal peptidoglycan rings. Our studies illustrate the critical role of chlamydial FtsK in coordinating divisome assembly and peptidoglycan synthesis in this obligate intracellular bacterial pathogen.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/102819">A single-cell atlas of spatial and temporal gene expression in the mouse cranial neural plate</a></h4>

              
                The formation of the mammalian brain requires regionalization and morphogenesis of the cranial neural plate, which transforms from an epithelial sheet into a closed tube that provides the structural foundation for neural patterning and circuit formation. Sonic hedgehog (SHH) signaling is important for cranial neural plate patterning and closure, but the transcriptional changes that give rise to the spatially regulated cell fates and behaviors that build the cranial neural tube have not been systematically analyzed. Here, we used single-cell RNA sequencing to generate an atlas of gene expression at six consecutive stages of cranial neural tube closure in the mouse embryo. Ordering transcriptional profiles relative to the major axes of gene expression predicted spatially regulated expression of 870 genes along the anterior-posterior and mediolateral axes of the cranial neural plate and reproduced known expression patterns with over 85% accuracy. Single-cell RNA sequencing of embryos with activated SHH signaling revealed distinct SHH-regulated transcriptional programs in the developing forebrain, midbrain, and hindbrain, suggesting a complex interplay between anterior-posterior and mediolateral patterning systems. These results define a spatiotemporally resolved map of gene expression during cranial neural tube closure and provide a resource for investigating the transcriptional events that drive early mammalian brain development.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/101143">A novel method (RIM-Deep) for enhancing imaging depth and resolution stability of deep cleared tissue in inverted confocal microscopy</a></h4>

              
                The increasing use of tissue clearing techniques underscores the urgent need for cost-effective and simplified deep imaging methods. While traditional inverted confocal microscopes excel in high-resolution imaging of tissue sections and cultured cells, they face limitations in deep imaging of cleared tissues due to refractive index mismatches between the immersion media of objectives and sample container. To overcome these challenges, the RIM-Deep was developed to significantly improve deep imaging capabilities without compromising the normal function of the confocal microscope. This system facilitates deep immunofluorescence imaging of the prefrontal cortex in cleared macaque tissue, extending imaging depth from 2 mm to 5 mm. Applied to an intact and cleared Thy1-EGFP mouse brain, the system allowed for clear axonal visualization at high imaging depth. Moreover, this advancement enables large-scale, deep 3D imaging of intact tissues. In principle, this concept can be extended to any imaging modality, including existing inverted wide-field, confocal, and two-photon microscopy. This would significantly upgrade traditional laboratory configurations and facilitate the study of connectomes in the brain and other tissues.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/104280">Load-based divergence in the dynamic allostery of two TCRs recognizing the same pMHC</a></h4>

              
                Increasing evidence suggests that mechanical load on the αβ T-cell receptor (TCR) is crucial for recognizing the antigenic peptide-bound major histocompatibility complex (pMHC) molecule. Our recent all-atom molecular dynamics (MD) simulations revealed that the inter-domain motion of the TCR is responsible for the load-induced catch bond behavior of the TCR-pMHC complex and peptide discrimination (Chang-Gonzalez et al., 2024). To further examine the generality of the mechanism, we perform all-atom MD simulations of the B7 TCR under different conditions for comparison with our previous simulations of the A6 TCR. The two TCRs recognize the same pMHC and have similar interfaces with pMHC in crystal structures. We find that the B7 TCR-pMHC interface stabilizes under ∼15 pN load using a conserved dynamic allostery mechanism that involves the asymmetric motion of the TCR chassis. However, despite forming comparable contacts with pMHC as A6 in the crystal structure, B7 has fewer high-occupancy contacts with pMHC and exhibits higher mechanical compliance during the simulation. These results indicate that the dynamic allostery common to the TCRαβ chassis can amplify slight differences in interfacial contacts into distinctive mechanical responses and nuanced biological outcomes.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/97760">Effort drives saccade selection</a></h4>

              
                What determines where to move the eyes? We recently showed that pupil size, a well-established marker of effort, also reflects the effort associated with making a saccade (‘saccade costs’). Here, we demonstrate saccade costs to critically drive saccade selection: when choosing between any two saccade directions, the least costly direction was consistently preferred. Strikingly, this principle even held during search in natural scenes in two additional experiments. When increasing cognitive demand experimentally through an auditory counting task, participants made fewer saccades and especially cut costly directions. This suggests that the eye-movement system and other cognitive operations consume similar resources that are flexibly allocated among each other as cognitive demand changes. Together, we argue that eye-movement behavior is tuned to adaptively minimize saccade-inherent effort.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://elifesciences.org/rss/recent.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://elifesciences.org/rss/recent.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://elifesciences.org/rss/recent.xml&title=eLife" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://elifesciences.org/articles/105039">SAM transmethylation pathway and adenosine recycling to ATP are essential for systemic regulation and immune response</a></h4>

              
                During parasitoid wasp infection, activated immune cells of <i>Drosophila melanogaster</i> larvae release adenosine to conserve nutrients for immune response. S-adenosylmethionine (SAM) is a methyl group donor for most methylations in the cell and is synthesized from methionine and ATP. After methylation, SAM is converted to S-adenosylhomocysteine, which is further metabolized to adenosine and homocysteine. Here, we show that the SAM transmethylation pathway is up-regulated during immune cell activation and that the adenosine produced by this pathway in immune cells acts as a systemic signal to delay <i>Drosophila</i> larval development and ensure sufficient nutrient supply to the immune system. We further show that the up-regulation of the SAM transmethylation pathway and the efficiency of the immune response also depend on the recycling of adenosine back to ATP by adenosine kinase and adenylate kinase. We therefore hypothesize that adenosine may act as a sensitive sensor of the balance between cell activity, represented by the sum of methylation events in the cell, and nutrient supply. If the supply of nutrients is insufficient for a given activity, adenosine may not be effectively recycled back into ATP and may be pushed out of the cell to serve as a signal to demand more nutrients.
              

              <p class="text-muted">in <a href ="http://elifesciences.org/rss/recent.xml">eLife</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.05.647394v1?rss=1
">Low affinity noradrenergic signaling promotes passive coping during reinforcement behavior</a></h4>

              
                Background: The fundamental decision processes that allow us to avoid negative outcomes (e.g., injury) and seek relevant positive outcomes (e.g., food) can be disrupted by stress. A critical component of the stress response is increased noradrenergic signaling, particularly at lower affinity noradrenergic receptors (1 and {beta}). It has been shown that noradrenergic signaling is essential for cue-driven reward seeking and that stress can impair positive reinforcement behavior in males. Whether noradrenergic component of the stress response alter negative reinforcement behavior is still unknown. Methods: We manipulated noradrenergic signaling at low affinity (1, 2) and high affinity adrenergic receptors ({beta}) during an active avoidance and reward seeking task in male and female rats. Results: We found that modulating noradrenergic activity on low affinity 1 and {beta} receptors but not high affinity 2 receptors decreased reinforcement behavior. Increased 1, but to a greater extent {beta} receptor activity shifted negative reinforcement behavior away from active avoidance and towards passive coping by increasing decision thresholds. We found sex differences in the impact of increased 1 and {beta} receptor activity on reward seeking where only males decreased their positive reinforcement behavior. Conclusions: Stress related noradrenergic signaling deceased active avoidance behavior by shifting strategy towards passive coping. Reward seeking was more robust behavior after noradrenergic modulation in females but not males. Collectively these results identify sex differences in noradrenergic stress response behaviors and highlight low affinity adrenoreceptors as relevant therapeutic targets to mitigate the impact of stress on goal-directed behavior.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.647619v1?rss=1
">Knockout of P2Y12 receptor facilitates microglia-neuron body-to-body interactions and accelerates prion disease</a></h4>

              
                Microglia continuously monitor neuronal health through somatic purinergic junctions, where microglial processes establish dynamic contacts with neuronal cell bodies. The P2Y12 receptor is a key component of these junctions, essential for intercellular communication between ramified microglia and neurons under homeostatic conditions. While P2Y12 has long been considered a marker of homeostatic microglia, its potential role in reactive microglia during neurodegenerative disease remains largely unexplored. In this study, we demonstrate that P2Y12 deletion significantly reduces microglia-neuron process-to-body contacts in adult mice, consistent with previous findings. However, unexpectedly, P2Y12 loss markedly increases microglia-neuron body-to-body contacts, revealing an alternative mode of microglia-neuron communication independent of P2Y12. In prion-infected mice, P2Y12 expression persists in reactive, amoeboid microglia during advanced disease stages, including those engaging in extensive neuronal envelopment. Notably, P2Y12 loss increases the prevalence of envelopment events and accelerates disease progression. These findings redefine the role of P2Y12 in neurodegeneration, suggesting that its progressive decline lowers the threshold for microglia-neuron body-to-body interactions, ultimately influencing disease trajectory.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.06.646695v1?rss=1
">A data-driven approach to identifying and evaluating connectivity-based neural correlates of consciousness</a></h4>

              
                Identifying the neural correlates of consciousness remains a major challenge in neuroscience, requiring theories that bridge between subjective experience and measurable neural correlates. However, theoretical interpretation of empirical evidence is often post hoc and susceptible to confirmation bias. Building upon the adversarial collaboration mediated by the COGITATE Consortium, we present a generalizable approach for the data-driven identification, evaluation, and theoretical modeling of connectivity-based neural correlates of consciousness. Using the same magnetoencephalography (MEG) dataset and accompanying pre-registered hypotheses from the COGITATE Consortium, we systematically compared 246 functional connectivity (FC) measures between regions predicted to underlie conscious vision by Integrated Information Theory (IIT) and/or Global Neuronal Workspace Theory (GNWT). We identified a family of FC measures based on the barycenter--tracking the 'center of mass' between two signals--as the top-performing stimulus decoding measures that generalize across regions central to predictions of both IIT and GNWT. To interpret these findings within a theoretical framework, we developed neural mass models that recapitulate the neural dynamics hypothesized to underlie conscious perception by each theory. Comparing simulated barycenter values from these models against empirically measured MEG data revealed that the GNWT-based model, featuring delayed ignition dynamics, better captured observed connectivity patterns than the IIT-based model, which relied on highly synchronized sensory dynamics. Beyond dataset-specific conclusions and limitations, we introduce a framework for systematically identifying and testing candidate neural correlates of consciousness in an unbiased and interpretable manner.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.646944v1?rss=1
">Harmonizing Inter-Site Differences in T1-Weighted Images Using CycleGAN</a></h4>

              
                Introduction When neuroimaging studies using magnetic resonance imaging (MRI) are conducted across multiple centers, they often encounter inter-site differences in MRI equipment and protocols leading to biases and confounding effects in MRI measurements. There are existing techniques for correcting these site effects, i.e., harmonization, but they have limitations, including the need for preprocessing of MRI data, which involves processes such as spatial normalization. Deep learning-based methods have emerged as potential alternatives that can handle site effects without the need for preprocessing steps. In this study, we propose a novel method based on the generative adversarial network (GAN) framework, CycleGAN, that effectively addresses inter-site differences in T1-weighted images with minimal preprocessing requirements. We compare the harmonization efficacy of CycleGAN with that of the commonly used method ComBat. Methods We trained the proposed CycleGAN method and the comparative ComBat method using data from 40 subjects at each of two sites. To evaluate the effectiveness of the two methods, we used data from nine subjects who underwent imaging at both sites. We assessed harmonization performance at the image level using the structural similarity index (SSIM) and peak signal-to-noise ratio (PSNR). Additionally, we evaluated harmonization results at the feature level by analyzing regional cortical thickness and volume data. Cohen's d was employed to quantify the differences between feature values. Results At the image level, the ComBat method decreased the median baseline SSIM value from 0.86 (interquartile range [IQR], 0.02) to 0.84 (IQR, 0.02), whereas the proposed CycleGAN method maintained the SSIM value at 0.86 (IQR, 0.02). For PSNR, the baseline value was 18.33 (IQR, 1.78), which decreased to 15.30 (IQR, 2.20) after applying ComBat, but increased to 19.58 (IQR, 3.12) with the proposed CycleGAN method. These findings indicate that CycleGAN preserved the structural and signal similarity of the images. At the feature level, the effect size for cortical thickness decreased from 0.97 (IQR, 1.79) to 0.91 (IQR, 1.54) after applying ComBat, whereas the proposed CycleGAN method yielded an effect size of 1.05 (IQR, 1.14). For cortical volume, the effect size decreased from 0.95 (IQR, 1.78) to 0.69 (IQR, 1.00) after applying ComBat, and decreased to 0.88 (IQR, 0.74) with the CycleGAN method. Compared with baseline, Cohen's d was significantly lower with both ComBat (p = 0.000002) and CycleGAN (p = 0.028) with no significant difference between the two methods, indicating similar performance of the two methods under the study conditions. Conclusion The results underscore the ability of CycleGAN to harmonize data without explicit normalization and emphasize the potential impact of the normalization process on harmonization procedures. Our findings suggest that CycleGAN holds promise as a harmonization technique in multi-site neuroimaging studies.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.646449v1?rss=1
">High-throughput measurements of neuronal activity in single human iPSC-derived glutamate neurons</a></h4>

              
                Induced pluripotent stem cell (iPSC)-derived neurons provide a promising platform for studying neuronal function and modeling central nervous system (CNS) diseases. However, functional analysis of large populations of iPSC-derived neurons has been challenging. Here, we developed a high throughput strategy targeting N-methyl-D-aspartate receptors (NMDA-R) to enhance neuronal activity and reveal functional phenotypes in human iPSC-induced glutamatergic neurons (iGlut). Using a genetically encoded calcium indicator (GCaMP8f), we first demonstrate that using artificial cerebrospinal fluid (ACSF) lacking Mg2+; (Mg2+;-free) significantly increases neuronal firing, and that firing is enhanced by a potentiator (glycine) but inhibited by the NMDA-R antagonist AP-V. Similarly, multi-electrode array (MEA) recordings also show robust firing in Mg2+;-free ACSF. Lastly, single-cell patch-clamp electrophysiology confirms the high firing rates in Mg2+;-free ACSF across multiple iPSC donor lines and also reveals iPSC donor-specific tonic and bursting firing phenotypes. This new methodology provides a scalable, high-throughput method to study neuronal activity in iGlut neurons while preserving single-cell resolution. The strategy also reveals different functional phenotypes, enabling detailed characterization of iGlut neurons in diverse applications such as CNS disease modeling and drug screening. These findings establish a versatile framework for future studies of neuronal network dynamics and individual excitability in iPSC-derived neuronal cultures.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.647631v1?rss=1
">Connectomic traces of Hebbian plasticity in the entorhinal-hippocampal system</a></h4>

              
                The key model of how we learn and memorize is Hebbian learning in the hippocampus, via long-term potentiation of synapses, allowing the storage of associations, linkage to places, and their consolidation into imprinted episodes. Learning is therefore expected to change synaptic weights. With the notion of hippocampal circuits being the primary site of learning in the mammalian brain, it has been assumed that all synapses in these circuits are constantly exposed to synaptic plasticity, and possibly in a learned state. However, a testing of these hypotheses is so far missing. In particular, the systematic mapping of synaptic weight distributions, and their relation to Hebbian preconditions has not been achieved yet in the hippocampal-entorhinal system. Here, we report such a systematic connectomic mapping of synaptic weight distributions and their relation to same-axon same-dendrite paired synaptic configurations across the hippocampal-entorhinal system. By analyzing millions of synapses and tens of thousands of paired synaptic configurations from 3D EM-based automated circuit reconstructions in hippocampal areas CA3, CA1, and layers 2 and 3 of the medial entorhinal cortex (MEC), we found systematic and unique synaptic weight distributions, with almost 50% (but not 100%) of synaptic weights in CA1 being in a Hebbian-consistent state, CA3 uniquely exhibiting small synaptic weights with indications of learned states, and MEC resembling previous data from other isocortices, with only up to 20% learned synaptic configurations. We further analyzed the sublayer-specificity of these weight distributions, finding the molecular layer of CA1 and the lower layers of CA3 being the most unique site of potential learning. Together, this data provides a first systematic synaptic weight analysis of the key neuronal system involved in memory formation in mammalian brains.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.647541v1?rss=1
">Differential Effects of Short-term and Long-term Deep Brain Stimulation on Striatal Neuronal Excitability in a Dystonia Animal Model</a></h4>

              
                Deep brain stimulation (DBS) is, by now, one of the standard treatment options for movement disorders like dystonia or Parkinsons Disease. Although its clinical effectiveness is established, the exact mechanisms by which DBS influences neural motor networks are not fully understood. The present study explores the development of adaptive network mechanisms with DBS in the dtsz hamster model, an in-vivo model exhibiting spontaneous dystonic episodes, by comparing functional impacts of short-term and long-term DBS on medium spiny neurons (MSNs) and synaptic transmission in the striatum. In this electrophysiological study, we uncovered contrasting changes in neuronal excitability and synaptic dynamics following short-term versus long-term DBS. Short-term DBS enhanced neuronal firing responses, while long-term DBS diminished them. Regarding synaptic alterations, both short-term and long-term DBS significantly shifted spontaneous EPSC occurrences to longer intervals, with this effect, however, being more pronounced in short-term DBS, leading to a significant decrease in mEPSC frequency. Notably, acetylcholine application effectively reversed this effect, restoring mEPSC frequency more efficiently again in tissue subjected to short-term DBS compared to long-term DBS. These observations indicate that the therapeutic benefits of DBS in dystonia may involve both immediate and adaptive mechanisms, which has implications for improving stimulation parameters and treatment protocols. The findings shed light on the temporal specificity of DBS effects and highlight the importance of understanding synaptic mechanisms to enhance therapeutic outcomes for dystonia patients.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.04.647167v1?rss=1
">Gaze-centered spatial coding of touch on a hand-held tool</a></h4>

              
                Humans possess the remarkable ability to project tactile sensations outside their body and onto a hand-held tool that they are using. Despite nearly a century of research, the computations underlying this projection have not been adequately addressed. In the present study, we employed model-driven psychophysics to fill this gap. We hypothesized that tool-based sensory projection involves the remapping of touch from sensory feedback in the hand into an egocentric coordinate system. We first formalized the computational steps underlying tactile remapping. We designed a novel tool-sensing experiment that allowed us to rigorously test this model. In this task, participants contacted an object with a hand-held rod and then judged whether the object was above or below where they were currently looking. This comparison would only be possible if touch on the tool was projected outside the hand. Crucially, both object location and gaze position varied independently, allowing us to characterize the hand-to-space-to-gaze remapping process. Model-based curve fitting provided strong evidence that all participants in our task projected touch outside their body and into gaze-centered coordinates. Crucially, the resolution of this projection was similar to what has been found for touch on the body. These findings provide the first step towards characterizing the computations underlying the spatial projection of touch on external objects, highlighting the incredible versatility of the sensorimotor system.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.647335v1?rss=1
">Decoding the Moving Mind: Multi-Subject fMRI-to-Video Retrieval with MLLM Semantic Grounding</a></h4>

              
                Decoding dynamic visual information from brain activity remains challenging due to inter subject neural heterogeneity, limited per subject data availability, and the substantial temporal resolution gap between fMRI signals (0.5 Hz) and video dynamics (30 Hz). Current approaches face persistent challenges in addressing these temporal mismatches, demonstrate limited capacity to integrate subject specific neural patterns with shared representational frameworks, and lack adequate semantic granularity for aligning neural responses with visual content. To bridge these gaps, we propose a framework addressing these limitations through three innovations: (1) a Dynamic Temporal Alignment module that resolves temporal mismatches via exponentially weighted multiframe fusion with adaptive decay coefficients; (2) a Brain Mixture of Experts architecture that combines subject specific extractors with shared expert layers through parameter efficient tri modal contrastive learning; and (3) a Multi-perspective Semantic Hyper Anchoring module that resolves cross subject attention bias via multi-dimensional semantic decomposition, leveraging multimodal LLMs for fine grained video semantic extraction enabling the model to match individual attention patterns as different subjects naturally focus on distinct aspects of the same visual stimulus. This module boosts Top 10/Top 100 retrieval by 17.7%/6.6%. Experiments on two video fMRI datasets demonstrate state of the art performance, with 39%/30% improvements in Top 10/Top 100 accuracy over single subject baselines and 27% gains against multi subject models. The framework exhibits remarkable few shot adaptability, retaining 97% performance when using only 10% training data for new subjects. Visualization analysis confirms this generalization capability stems from effective disentanglement of subject specific and shared neural representations.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.647118v1?rss=1
">Cell-type specific repertoire of responses to natural scenes in primate retinal ganglion cells</a></h4>

              
                At least 20 distinct retinal ganglion cells (RGC) types have been identified morphologically in the primate retina, but our understanding of the distinctive visual messages they send to various targets in the brain remains limited. Here, we use large-scale multi-electrode array recordings to examine how multiple functionally-distinct RGC types in the macaque retina respond to flashed natural images. Responses to white noise visual stimulation were used to functionally identify 936 RGCs of 12 types in three recordings. Each cell type was confirmed by the mosaic organization of receptive fields, and 7 cell types were cross-identified between recordings. The average kinetics of light response in each RGC type as well as the repertoire of distinct firing patterns that each type produces were examined across thousands of natural images. The kinetics of the average response across images were highly stereotyped among cells of each cell type and distinct for cells of different types. Moreover, the full repertoires of firing patterns produced by different cell types, assessed by their latency and duration, were generally quite distinct with only a few exceptions. Together these data provide an overview of the range of responses to natural images transmitted from the primate retina to the brain.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.05.647398v1?rss=1
">Learning evoked centrality dynamics in the schizophrenia brain: Entropy, heterogeneity and inflexibility of brain networks</a></h4>

              
                Background: Brain network dynamics are responsive to task induced fluctuations, but such responsivity may not hold in schizophrenia (SCZ). We introduce and implement Centrality Dynamics (CD), a method developed specifically to capture task driven dynamic changes in graph theoretic measures of centrality. We applied CD to fMRI data in SCZ and Healthy Controls (HC) acquired during a learning paradigm. Methods: fMRI (3T Siemens Verio) was acquired in 88 participants (49 SCZ). Time series were extracted from 246 functionally defined cerebral nodes. We applied a dynamic widowing technique to estimate 280 partially overlapping connectomes (30,135 region-pairs in each connectome). In each connectome we calculated every node's Betweenness Centrality (BC) before building 246 unique time series (representing a node's CD) from a node's BC in successive connectomes. Next, in each group nodes were clustered based on similarities in CD. Results: Clustering gave rise to fewer sub-networks in SCZ, and these were formed by nodes with greater functional heterogeneity. These sub-networks also showed greater ApEn (indicating greater stochasticity) but lower amplitude variability (suggesting less adaptability to task-induced dynamics). Higher ApEn was associated with worse clinical symptoms. Limitations: Centrality Dynamics is a new method for network discovery in health and schizophrenia but will need further extension to other tasks and psychiatric conditions, before we achieve a fuller understanding of its promise. Conclusion: The brain's functional connectome is not static under task-driven conditions, and characterizing the dynamics of the connectome will provide new insight on the dysconnection syndrome that is schizophrenia. Centrality Dynamics provides novel characterization of task-induced changes in the brain's connectome and shows that in the schizophrenia brain, learning evoked sub-network dynamics were less responsive to learning evoked changes and showed greater stochasticity.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.647271v1?rss=1
">YPEL controls synapse development through p62-Nrf2 antioxidant pathway in Drosophila neuromuscular junction</a></h4>

              
                The Yippee-like (YPEL) genes are highly conserved among all eukaryotic species, yet the molecular and cellular pathways that YPEL regulate are poorly understood. Human and animal studies suggest that YPEL is involved in nervous system functions. Here we report that YPEL is necessary for synapse development in neuromuscular junction and motor functions in Drosophila. YPEL interacts with the Drosophila p62, Refractory to sigma P, which forms cytoplasmic proteinaceous bodies for selective autophagy and signaling. YPEL overexpression decreased p62 bodies, while increased p62 bodies were observed in YPEL mutant neurons. Suppressing p62 bodies by reducing p62 gene dosage significantly alleviated both synaptic and locomotion defects in YPEL mutants, suggesting that YPEL acts through p62 for synapse development. On the other hand, suppressing p62 bodies via autophagy did not restore synapse development in YPEL mutants. Interestingly, reduced levels of reactive oxygen species were found in YPEL mutants, which is consistent with the role of p62 in promoting the nuclear factor erythroid 2-related factor 2 (Nrf2) antioxidant pathway. Overexpressing Drosophila Nrf2, Cap 'n' collar (Cnc), phenocopied the synaptic and locomotion deficits in YPEL mutants. Importantly, both synaptic and locomotion defects were completely rescued by knocking down Cnc in YPEL mutant motor neurons. Taken together, our study demonstrates that YPEL negatively controls p62 - Nrf2 antioxidant pathway for neuromuscular synapse development and locomotion.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.647298v1?rss=1
">Strategies used by two memories to share space in a common neural network</a></h4>

              
                How distinct memories are encoded into the same network space without destructive interference is not well-understood. Here, we utilized Tritonia diomedea's escape swim network to explore how two sequentially acquired forms of non-associative learning, sensitization and habituation, are encoded into the same network. Behavioral experiments showed them to alter non-identical sets of behavioral features, suggesting they utilize somewhat independent sites of plasticity within the network. Large-scale voltage-sensitive dye recordings revealed two findings. First, both forms of learning, which occur sequentially in the 10-trial training protocol used, act to produce a change in the number of pedal neurons firing during the dorsal phase of the motor program, with sensitization producing an increase, and habituation a decrease in their number. The number of neurons participating in the ventral phase was unaffected. Second, sensitization produced an enhancement of burst intensity specific to the ventral phase neurons, while habituation was associated with a decrease in burst intensity in both phases. Using injected current pulses, intracellular recordings revealed that sensitization acted to increase the excitability of neurons firing in both phases, whereas habituation only acted to reduce excitability in ventral phase neurons. These excitability changes were associated with different mechanisms - reduced spike frequency accommodation in the ventral phase neurons, and depolarization of the resting potential in the dorsal phase neurons. These findings of partially different storage sites and mechanisms for two different non-associative memories illuminate a potential network strategy for minimizing destructive interference when storing multiple memories into the same network.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.647582v1?rss=1
">Optogenetic disruption of neural dynamics in the prefrontal cortex impaired spatial learning</a></h4>

              
                Cortical neural activity is highly dynamical at several temporal scales, property that has been postulated to be critical for the emergence of specific patterns supporting cognitive operations. During spatial learning, task-associated activity patterns gradually develop in the medial prefrontal cortex (mPFC) as the subject acquires experience. If neural activity dynamics is required in the mPFC for spatial learning is still unclear. Here we show that optogenetic entrainment of neural activity the mPFC disrupted local oscillatory and single-neuron dynamics. When applied during spatial training, optogenetic entrainment impaired behavioral performance and navigation strategy progression, a hallmark of spatial learning supported by the mPFC. Also, optogenetic entrainment blocked the emergence of learning-related activity patterns such as cross-frequency coupling and firing patterns signaling efficient goal approaching. Importantly, during memory retrieval, training-stimulated mice showed impaired performance in the absence of optogenetic stimulation. This evidence show that neural activity dynamics in the mPFC is crucial for spatial learning.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.647510v1?rss=1
">Connecting the Dots: A Computational Framework linking Molecular Regulation, Synaptic Plasticity, and Brain Disorders</a></h4>

              
                The complexity of brain function emerges from multiscale interactions spanning molecular, synaptic, and circuit levels. While substantial progress has been made in each domain, a unified framework linking molecular perturbations to large-scale brain dysfunctions remains elusive. Here, we introduce a computational framework that bridges molecular regulation and neuronal plasticity, providing mechanistic insights into disease pathogenesis. By creating a Boolean network model of molecular regulatory pathways, we identify two stable states-synaptic potentiation and depression-governed by AMPA receptor trafficking and consistent with Hebbian learning principles. Through in silico knockout simulations mimicking genetic anomalies, our model demonstrates how molecular perturbations cascade across scales, disrupting synaptic plasticity and ultimately driving diverse neurological dysfunctions. The model's predictions align with a broad range of experimental and clinical data. Our framework establishes a mechanistic foundation for understanding the hierarchical causality of brain disorders, offering a powerful tool for dissecting disease mechanisms and informing targeted therapies.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.647589v1?rss=1
">Competition between tool and hand motion impairs movement planning in limb apraxia</a></h4>

              
                Tool use is a complex motor planning problem. Prior research suggests that planning to use tools involves resolving competition between different action representations. We reasoned that competition may be exacerbated with tools for which the motions of the tool and the hand are incongruent, such as pinching the fingers to open a clothespin. If this hypothesis is correct, then we should observe particularly marked deficits in planning the use of incongruent as compared to congruent tools in individuals with limb apraxia, a common disorder after left-hemisphere stroke (LCVA) that is associated with abnormal action competition. In a first experiment, we asked 34 individuals with chronic LCVA and 16 matched neurotypical controls to use novel tools that we developed in which the correspondence between the motions of the hand and tool-tip were either congruent or incongruent. Individuals with LCVA also completed background assessments to quantify apraxia severity. We observed increased planning time for incongruent as compared to congruent tools as a function of the severity of apraxia, particularly when the tools were first introduced. Exploratory lesion-symptom mapping analyses revealed that lesions to posterior portions of the tool-use network were associated with impaired planning for incongruent tools. A second experiment on the same individuals with LCVA revealed that the ability to demonstrate the use of conventional tools was impaired for tools rated as more incongruent by a normative sample. Together, these findings suggest tool-hand incongruence evokes action competition and influences the tool-use difficulties experienced by people with apraxia.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.05.647403v1?rss=1
">Visual pathway origins: an electron microscopic volume for connectomic analysis of the human foveal retina</a></h4>

              
                With over 1014 synapses, the human brain presents a seemingly insurmountable challenge to a nano-scale circuit-level understanding of its diverse neural systems. The foveal retina however presents a feasible site for a complete connectome of a key human CNS structure. Foveal cells and circuits are miniaturized and compressed to densely sample the visual image at highest resolution to initiate form, color and motion perception. Here we use computational methods first applied to the fly brain to provide a draft connectome of all neurons in a foveal volume. We found synaptic connections distinct to humans linking short-wavelength sensitive cones to color vision pathways. Moreover, by reconstructing excitatory synaptic pathways arising from cone photoreceptors we found that 96% of foveal ganglion cells contribute to only three major pathways to the brain. This new resource reveals unique features of a human neural system and opens a door to its complete connectome.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.647497v1?rss=1
">Optimizing Biophysical Large-Scale Brain Circuit Models With Deep Neural Networks</a></h4>

              
                Biophysical modeling provides mechanistic insights into brain function, from single-neuron dynamics to large-scale circuit models bridging macro-scale brain activity with microscale measurements. Biophysical models are governed by biologically meaningful parameters, many of which can be experimentally measured. Some parameters are unknown, and optimizing their values can dramatically improve adherence to experimental data, significantly enhancing biological plausibility. Previous optimization methods -- such as exhaustive search, gradient descent, evolutionary strategies and Bayesian optimization -- require repeated, computationally expensive numerical integration of biophysical differential equations, limiting scalability to population-level datasets. Here, we introduce DELSSOME (DEep Learning for Surrogate Statistics Optimization in MEan field modeling), a framework that bypasses numerical integration by directly predicting whether model parameters produce realistic brain dynamics. When applied to the widely used feedback inhibition control (FIC) mean field model, DELSSOME achieves a 2000x speedup over Euler integration. By embedding DELSSOME within an evolutionary optimization strategy, trained models generalize to new datasets without additional tuning, enabling a 50x speedup in FIC model estimation while preserving neurobiological insights. The massive acceleration facilitates large-scale mechanistic modeling in population-level neuroscience, unlocking new opportunities for understanding brain function.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.646998v1?rss=1
">Nociceptor clock genes control excitability and pain perception in a sex- and time-dependent manner</a></h4>

              
                Nociception is critical for pain perception and survival, and begins with the activation of nociceptors, specialized sensory neurons located in the dorsal root ganglia (DRGs). Both sex and circadian rhythms, governed by clock genes, seem to play a significant role in modulating pain perception. However, the potential interaction between circadian rhythms and sex differences in nociception at the peripheral level has been largely overlooked. Here we first report that DRGs from mice express clock genes in a time- and sex-dependent manner. Using whole-cell recordings in whole-mounted DRGs and optogenetic stimulation of Nav1.8-expressing neurons, we demonstrate that male nociceptors exhibit reduced excitability during the night, while female nociceptor excitability remains stable across time points. Disruption of the core clock gene Bmal1 in Nav1.8-expressing neurons not only diminished nociceptor activity but also abolished the nighttime reduction in heat sensitivity, highlighting a pivotal role for the molecular clock in regulating nociception. Transcriptomic analyses, voltage-clamp recordings, and pharmacological experiments identified the voltage-gated chloride channel ClC-2, controlled by Bmal1, as a key mediator for the observed fluctuations in male nociceptor excitability. This work opens new avenues for chronobiology-inspired strategies in pain management, tailored to sex-specific mechanisms.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.647612v1?rss=1
">Scalp EEG predicts intracranial brain activity in humans</a></h4>

              
                Inferring deep brain activity from noninvasive scalp recordings remains a fundamental challenge in neuroscience. Here, we analyzed concurrent scalp and intracranial recordings from 1918 electrode contacts across 20 patients affected by drug-resistant epilepsy undergoing intracranial depth electrode monitoring for pre-surgical evaluation to establish predictive relationships between surface and deep brain signals. Using regularized and cross-validated linear regression within subjects, we demonstrate that scalp recordings can predict spontaneous intracranial activity, with accuracy varying by region, depth, and frequency. Low-frequency signals (<12 Hz) were most predictable, with our models explaining approximately 10% of intracranial signal variance across contacts. Prediction accuracy decreased with contact depth, particularly for high-frequency signals. Using Bayesian modeling with leave-one-patient-out cross-validation, we observed generalizable prediction of activity in mesial temporal, prefrontal, and orbitofrontal cortices, explaining 10-12% of low-frequency signal variance. This scalp-to-intracranial mapping derived from spontaneous activity was further validated by its correlation with scalp responses evoked by direct electrical stimulation. These findings support the development of improved inverse models of brain activity and potentially more accurate scalp-based markers of disease and treatment response.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.647615v1?rss=1
">Bioinformatics-Driven Identification and Prioritization of PTSD Targets Based on Published Multi-omic Data</a></h4>

              
                Efforts in recent years to uncover neurobiological mechanisms underlying post-traumatic stress disorder (PTSD) have yielded an expanding candidate pool of targets from genomic and transcriptomic data. However, not all candidates are disease-causing, related to pathological mechanisms, clinically relevant, nor druggable by conventional means. An effective method to systematically identify and prioritize high-con[fi]dence, high-impact targets in the central nervous system (CNS) is required to de-risk resource-intensive experimental validation of disease mechanisms and accelerate the development of novel treatments. Here, we describe methods and implementation of a novel 3-phased, biologically rationalized, and quantitative prioritization strategy to identify and rank PTSD-associated targets based on con[fi]dence of association to PTSD and estimated CNSrelevant pathogenicity. Phase 1 was designed to identify and advance targets con[fi]dently associated with PTSD through their expression in CNS tissues. Putative targets derived from 29 transcriptomic and genomic analyses of PTSD were evaluated for: 1. Replication in independent cohorts, 2. Observation of differential expression in PTSD CNS tissues, and 3. Demonstration of consistent direction of effect. This strategy resulted in 177 targets that passed criteria for advancement. Phase 1-selected targets were enriched for PTSD relevant traits including irritability, emotional symptoms, and insomnia (FDR <0.05). Phase 2 advanced targets with additional evidence of association to pathological CNS phenotypes. DisGeNET gene-disease association scores were applied to each Phase 1-selected target to assign a con[fi]dence score indicating that a target was associated with CNS-relevant pathology using criteria for moderate or strong evidence of CNS disease association. Phase 2 advanced 55 of the 177 (31.1%) targets. The Phase 2 target pool was enriched for CNS phenotypic abnormalities (FDR<0.05). Finally, Phase 3 enabled target prioritization by annotating targets with a composite pathogenicity score. Components of the pathogenicity score included metrics derived from drug trial databases, predicted loss-of-function intolerance, and connectivity within a protein-protein interaction network de[fi]ned by PTSD-associated targets. The resulting 55 targets were ultimately prioritized by the sum of Phase 2 and Phase 3 scores, where top-ranked targets had strong evidence to support both association with PTSD in brain and high pathogenicity estimates in a CNS-relevant context. Biologically, top-ranked targets implicate transmitter systems (GABA, histamine, and estrogen), structural regulation of neurites, and protein homeostasis. Future work will be required to experimentally validate the utility of the high priority PTSD targets we identi[fi]ed as well as to demonstrate the general applicability of this methodology. Ultimately, we anticipate that the three phased approach will enable ef[fi]cient de-risking of PTSD and other poorly understood CNS disorders.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.647574v1?rss=1
">Sulfotransferase 4A1 (SULT4a1): A Novel Neuroprotective Protein in Stroke</a></h4>

              
                SULT4a1, a member of the cytosolic sulfotransferase family, is predominantly expressed in neurons and plays potentially vital roles in regulating neural survival and function. SULT4a1 protects against mitochondrial dysfunction and oxidative stress. SULT4a1 levels decrease in experimental stroke models and may play a critical neuroprotective role in mitigating neuronal injury caused by oxygen-glucose deprivation (OGD) and ischemic stroke, as shown in a transient middle cerebral artery occlusion (tMCAO) mouse model. In this study, we investigated the neuroprotective role of SULT4a1 in OGD and tMCAO and highlighted its expression pattern and involvement in maintaining mitochondrial function and reducing oxidative stress, two early pathophysiological features in stroke and related neuronal injury. Our data show that decreased SULT4a1 expression in OGD conditions and in the tMCAO mouse brain leads to enhanced neuronal damage, emphasizing the importance of SULT4a1 in preserving neuronal integrity. Loss of SULT4a1 alone was sufficient to decrease mitochondrial function in mouse cortical neurons. Notably, overexpression of SULT4a1 preserved mitochondrial function, reduced the loss of mitochondrial membrane potential, and diminished oxidative stress, as evidenced by lower reactive oxygen species (ROS) production and reduced protein carbonylation. These results indicate that modulating SULT4a1 expression in stroke could offer a promising strategy for preventing neuronal damage. Indeed, overexpression of SULT4a1 via stereotaxic injection of AAV9 into the mouse brain mitigated tMCAO-related brain injury and functional deficits over time. The findings of this study indicate that SULT4a1 may protect neurons in stroke and related brain injury, possibly by maintaining mitochondrial function and redox homeostasis through mechanisms that are still unknown. It is likely that SULT4a1 regulates neuroprotective processes in both the mitochondria and the cytosol. However, further research is needed to clarify the specific molecular pathways involved in its neuroprotective function.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.647618v1?rss=1
">Mapping individualized multi-scale hierarchical brain functional networks from fMRI by self-supervised deep learning</a></h4>

              
                The brain's multi-scale hierarchical organization supports functional segregation and integration. Characterizing the hierarchy of individualized multi-scale functional networks (FNs) is crucial for understanding these fundamental brain processes. It provides promising opportunities for both basic neuroscience and translational research in neuropsychiatric illness. However, current methods typically compute individualized FNs at a single scale and are not equipped to quantify any possible hierarchical organization. To address this limitation, we present a self-supervised deep learning (DL) framework that simultaneously computes multi-scale FNs and characterizes their across-scale hierarchical structure at the individual level. Our method learns intrinsic representations of fMRI data in a low-dimensional latent space to effectively encode multi-scale FNs and their hierarchical structure by optimizing functional homogeneity of FNs across scales jointly in an end-to-end learning manner. A DL model trained on fMRI scans from the Human Connectome Project successfully identified individualized multi-scale hierarchical FNs for unseen individuals and generalized to two external cohorts. Furthermore, the individualized hierarchical structure of FNs was significantly associated with biological phenotypes, including sex, brain development, and brain health. Our framework provides an effective method to compute multi-scale FNs and to characterize the inter-scale hierarchy of FNs for individuals, facilitating a comprehensive understanding of brain functional organization and its inter-individual variation.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.647542v1?rss=1
">Unreal? A Behavioral, Physiological, and Computational Model of the Sense of Reality</a></h4>

              
                An intriguing aspect of the human mind is our knowledge that our perceptions may be false. Our frequent exposure to non-veridical perceptions such as those found in dreams, illusions and hallucinations cause us to examine the actuality of our sensory experiences. As such humans continuously monitor the veridicality of their perceptions in a process termed the Sense of Reality (SoR). Moreover, the Sense of Reality is a central criterion in the assessment of neurological and psychiatric health. The scientific study of hallucinatory experiences has been hindered by their transitory, subjective, ineffable nature and by the fact that they typically co-occur with psychiatric, medical or psychedelic states. Despite the critical role of SoR in daily life and in pathologies little is known regarding its cognitive, physiological and computational underpinnings. Here we employed a novel immersive virtual reality paradigm to induce Virtual Hallucinations (VH) simulating the phenomenology of visual hallucinations found in psychiatric, neurological, and pharmacological conditions. Combining psychophysics, physiological recordings and computational modeling in one exploratory (n = 31) and one preregistered experiment (n = 32) we examined responses to VH of varying magnitudes and domains. Judgments of SoR varied depending on the magnitude and domain of VH. These were accompanied by distinct motor, pupillometric and cardiac responses. Finally, sense of reality can be well explained by a computational model in which decisions of reality are based on comparison of current sensory experience to an internal model of the world. Our results shed some light on the age-old question: how do we know what is real?
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.647435v1?rss=1
">Leukemia Inhibitory Factor as a late-stage treatment for delayed white matter loss in concussive head injury</a></h4>

              
                Leukemia Inhibitory Factor (LIF) is an injury-induced cytokine that peaks 48 hours after a traumatic brain injury (TBI). Juvenile LIF haplodeficient mice exhibit desynchronized glial responses, increased neurodegeneration, decreased axonal conductivity and behavioral deficits after a concussive head injury. Given the necessity of LIF during the acute recovery phase after injury, we hypothesized that intranasal (IN) LIF treatment would prevent neurodegeneration when administered during the chronic recovery period from a mild TBI (mTBI). Here, young adult male CD1 mice were subjected to a midline, closed-head frontal cortex injury using a flat metal impactor with a 3mm tip to induce a mTBI. In the 6-8 weeks post-mTBI, known to precede axonal atrophy in this mTBI model, two doses of 40 ng and 100 ng of LIF were administered twice daily, 5 days/week for two consecutive weeks. Sensorimotor functions were assessed at 4 and 8 weeks post mTBI, followed by ex-vivo brain magnetic resonance imaging at 9.4T and histopathology. mTBI mice showed sensorimotor deficits at 4 weeks, which worsened by 8 weeks post-injury. IN-LIF treatment prevented the progressive sensorimotor loss seen in the vehicle-treated controls. Increased mean diffusivity (MD) and decreased fractional anisotropy (FA) were observed in the corpus callosum and prefrontal cortex of mTBI brains. In a dose-dependent manner, IN-LIF prevented the mTBI-induced MD increase and FA decrease. Histologically, there was significantly less astrogliosis, microgliosis and axonal injury in the IN-LIF treated mice vs. controls. These results support the therapeutic potential of IN-LIF to reduce delayed neurodegeneration and improve neurological outcomes after mTBIs.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.646926v1?rss=1
">Microglia and myeloid cell populations of the developing mouse retina</a></h4>

              
                Microglia make important contributions to central nervous system (CNS) development, but the breadth of their distinct developmental functions remain poorly understood. The mouse retina has been a key model system for understanding fundamental mechanisms controlling assembly of the CNS. To gain insight into where and how microglia might influence retinal development, here we identified molecularly unique myeloid cell populations that are selectively present during development, and characterized their anatomical locations. Development-specific transcriptional states were identified using single-cell and single-nucleus RNA-sequencing (RNA-seq) of purified retinal myeloid cells across multiple timepoints. Transcriptional states were validated in vivo by histological staining for key RNA and/or protein markers. Several of these development-specific myeloid populations have been described before in brain RNA-seq atlases but not validated in vivo, while others are unique to our retinal dataset. We identify two closely-related microglial populations, labeled by the Spp1 and Hmox1 genes, that are distinguished mainly by activity of the NRF2 transcription factor. Both types are present selectively within the developing retinal nerve fiber layer where they engulf neurons and astrocytes undergoing developmental cell death. Hmox1+ microglia were also localized selectively at the wavefront of developing vasculature during retinal angiogenesis, suggesting that developmental events associated with angiogenesis modulate NRF2 activity and thereby induce microglia to switch between the Spp1+ and Hmox1+ states. Overall, our results identify transcriptional profiles that define specific populations of retinal microglia, opening the way to future investigations of how these programs support microglial functions during development.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.06.647503v1?rss=1
">Fast-spiking neurons in monkey orbitofrontal cortex underlie economic value computation</a></h4>

              
                Inhibitory interneurons are fundamental constituents of cortical circuits that process information to shape economic behaviors. However, the role of inhibitory interneurons in this process remains elusive at the core cortical reward-region, orbitofrontal cortex (OFC). Here, we show that presumed parvalbumin-containing GABAergic interneurons (fast-spiking neurons, FSNs) cooperate with presumed regular-spiking pyramidal neurons (RSNs) during economic-values computation. While monkeys perceived a visual lottery for probability and magnitude of rewards, identified FSNs occupied a small subset of OFC neurons (12%) with high-frequency firing-rates and wide dynamic-ranges, both are key intrinsic cellular characteristics to regulate cortical computation. We found that FSNs showed higher sensitivity to the probability and magnitude of rewards than RSNs. Unambiguously, both neural populations signaled expected values (i.e., probability times magnitude), but FSNs processed these reward's information strongly governed by the dynamic range. Thus, cooperative information processing between FSNs and RSNs provides a common cortical framework for computing economic values.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.07.647511v1?rss=1
">Altered Neural Processing in Middle Frontal Gyrus and Cerebellum During Temporal Recalibration of Action-Outcome Predictions in Schizophrenia Spectrum Disorders</a></h4>

              
                A key function of the perceptual system is to predict the (multi)sensory outcomes of actions and recalibrate these predictions in response to changing conditions. In schizophrenia spectrum disorders (SSD), impairments in this ability have been linked to difficulties in self-other distinction. This study investigated the neural correlates of the recalibration of action-outcome predictions to delays, the transfer of this process across sensory modality, and whether patients with SSD exhibit alterations in the underlying neural processes. Patients and healthy controls (HC) underwent fMRI while exposed to delays between active or passive button presses and auditory outcomes. A delay detection task assessed recalibration effects on auditory perception (unimodal trials) and its transfer to visual perception (cross-modal trials). In unimodal trials, HC exhibited reduced activation in left middle frontal gyrus (MFG) after recalibration, particularly for active movements, whereas this effect was reversed in SSD. In cross-modal trials, recalibration was linked to increased activation in bilateral cerebellum in HC, especially for active movements, a pattern significantly reduced in SSD. These findings suggest that unimodal temporal recalibration of action-outcome predictions in HC is reflected in reduced prediction error-related MFG activity, which is significantly reduced in SSD revealing potentially disrupted recalibration processes. Additionally, cerebellar engagement appears crucial for cross-modal transfer of recalibrated action-outcome timings, a process that may be impaired in SSD, leading to severe perceptual disturbances like hallucinations.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.03.647115v1?rss=1
">Distinct cerebrospinal fluid DNA methylation signatures linked to Alzheimer's disease</a></h4>

              
                Alzheimer's disease (AD) accounts for more than 60% of the dementia cases and currently there is no curative treatment for it. With the emergence of potentially disease modifying treatments, early diagnosis is key to identify patient groups that would benefit from such treatments, aiming to prevent severe cognitive decline. We previously identified a set of DNA methylation signatures that allow for accurate diagnosis of AD in cortical neurons and brain tissue, even before clinical manifestation of the disease [1]. Here we investigate 11 of these signature regions via targeted next-generation sequencing in cell-free DNA (cfDNA) isolated from cerebrospinal fluid (CSF) of AD patients homozygous for APOE4 (n=4) and sporadic AD (n=5) cases compared to age-matched control samples (n=5). Our analyses demonstrated that 6/11 of the tested DNA methylation signatures that had initially been identified in cortical neurons and brain tissue were also validated in cfDNA. The remainder of the tested regions either showed opposite trends (3/11) or did not result in any differences (2/11) between control and AD cases. Thus, this presents a direct approach allowing to test for these DNA methylation signatures in CSF-derived cfDNA, and bypasses the need to generate induced pluripotent stem cell-derived cortical neurons from patients.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience&title=bioRxiv: Neuroscience" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="
https://www.biorxiv.org/content/10.1101/2025.04.01.646593v1?rss=1
">Neural and Behavioral Changes in Older Adults from Auditory-Cognitive Training</a></h4>

              
                Speech perception in noisy environments is a common challenge among older adults, even for those with clinically normal hearing. Cognitive decline may be one of the contributing factors, and, as such, auditory-cognitive training may enhance speech perception in these conditions. This study aims to determine if auditory-cognitive training can improve speech-in-noise listening in normal-hearing, older adults using neural and behavioral measures, supplemented with comparisons across younger and older adults. Neural responses were obtained using magnetoencephalography (MEG) while participants listened to long, narrative passages (60 s) under four noise conditions. Neural measures employed reverse correlation using encoding and decoding models, via the temporal response function (TRF) framework, to predict neural responses and reconstruct stimulus features, respectively, with the boosting algorithm to enforce sparsity. Behavioral measures, such as working memory (reading span; RSPAN), speech perception in noise (SPIN), and nonlinguistic auditory stream segregation (stochastic figure- ground; SFG) showed improvement post-training, along with neural and subjective ratings for listening effort. Additionally, auditory-cognitive training may enhance the neural contrast between the selectively attended and unattended stimulus reconstructions, and pre-training SFG performance may predict the extent of this neuroplasticity change. These results provide promising, additional insight into the effects of auditory-cognitive training, both perceptually and neurally.
              

              <p class="text-muted">in <a href ="http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience">bioRxiv: Neuroscience</a> on 2025-04-07 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00304-3?rss=yes">Quantifying the evolutionary paths to endomembranes</a></h4>

              
                Schavemaker and Lynch derived functions that relate the cell biology of endomembranes to cellular fitness. Applied to the pinocytosis of small-molecule nutrients and the insertion of membrane proteins by a proto-endoplasmic reticulum, the proto-endoplasmic reticulum is revealed to be the more likely path to complex endomembranes in the origin of eukaryotes.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-08010-1">Targeting alveolar epithelial cells with lipid micelle-encapsulated necroptosis inhibitors to alleviate acute lung injury</a></h4>

              
                <p>Communications Biology, Published online: 06 April 2025; <a href="https://www.nature.com/articles/s42003-025-08010-1">doi:10.1038/s42003-025-08010-1</a></p>A lipid micelle-encapsulated drug targeting MLKL in alveolar type II epithelial cells successfully applied in ALI treatment. This nanoparticle selectively inhibited necroptosis, protecting epithelial cells and reducing inflammatory injury
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-08015-w">Spatially informed graph transformers for spatially resolved transcriptomics</a></h4>

              
                <p>Communications Biology, Published online: 06 April 2025; <a href="https://www.nature.com/articles/s42003-025-08015-w">doi:10.1038/s42003-025-08015-w</a></p>Spatially informed Graph Transformer integrates gene expression and spatial context to accurately denoise data and identify fine-grained tissue domains.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-08014-x">GRASPs link Reelin to the Golgi during neocortical development to control neuronal migration and dendritogenesis</a></h4>

              
                <p>Communications Biology, Published online: 06 April 2025; <a href="https://www.nature.com/articles/s42003-025-08014-x">doi:10.1038/s42003-025-08014-x</a></p>GRASPs act as downstream effectors for Reelin, influencing the positioning and deployment of the Golgi apparatus, directing neuronal migration, and regulating dendrite formation during the development of the neocortex.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-07995-z">Uncovering critical transitions and molecule mechanisms in disease progressions using Gaussian graphical optimal transport</a></h4>

              
                <p>Communications Biology, Published online: 06 April 2025; <a href="https://www.nature.com/articles/s42003-025-07995-z">doi:10.1038/s42003-025-07995-z</a></p>Gaussian Graphical Optimal Transport (GGOT) captures network instability to analyze disease progression, identify tipping points, and key regulatory molecules, offering insights into critical transitions across complex diseases.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-04-06 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00300-6?rss=yes">Amino acids in cancer: Understanding metabolic plasticity and divergence for better therapeutic approaches</a></h4>

              
                Tumors exhibit substantial metabolic complexity. Do et al. discuss the metabolic properties of tumors with a focus on amino acid metabolism, particularly how the heterogeneity and flexibility of amino acid metabolism in cancer influence therapeutic responses. Understanding these intricacies can help us develop strategies for metabolic reprogramming in cancer treatment.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00312-2?rss=yes">PLK1 blockade enhances the anti-tumor effect of MAPK inhibition in pancreatic ductal adenocarcinoma</a></h4>

              
                Zhao et al. demonstrate that elevated expression of Polo-like kinase 1 (PLK1) in PDAC is associated with resistance to MEK inhibition. MEK inhibition suppresses JNK activity, resulting in the inactivation of c-JUN, cell death. The combined inhibition of MEK and PLK1 lifts the repression of JNK, leading to increased cell death.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.cell.com/cell-reports/current.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.cell.com/cell-reports/current.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.cell.com/cell-reports/current.rss&title=Cell Reports: Current Issue" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(25)00277-3?rss=yes">STAT3 signaling enhances tissue expansion during postimplantation mouse development</a></h4>

              
                Azami et al. reveal that STAT3 deletion depletes epiblast cells at implantation, causing a persistent developmental delay of around a day. STAT3 null embryonic stem cells contribute to all embryonic tissues in chimeras except blood. Mixing with wild-type cells for in vitro erythroid differentiation causes the outcompetition of STAT3 null cells.
              

              <p class="text-muted">in <a href ="https://www.cell.com/cell-reports/current.rss">Cell Reports: Current Issue</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.nature.com/nature/rss/current>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.nature.com/nature/rss/current" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.nature.com/nature/rss/current&title=Nature" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/d41586-025-01038-7">Obesity-drug pioneers and 13,508 physicists win US$3-million Breakthrough Prizes</a></h4>

              
                <p>Nature, Published online: 05 April 2025; <a href="https://www.nature.com/articles/d41586-025-01038-7">doi:10.1038/d41586-025-01038-7</a></p>Advances recognized by science’s most lucrative awards include Large Hadron Collider experiments and groundbreaking weight-loss treatments.
              

              <p class="text-muted">in <a href ="http://feeds.nature.com/nature/rss/current">Nature</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04868-2">An open-access database of nature-based carbon offset project boundaries</a></h4>

              
                <p>Scientific Data, Published online: 05 April 2025; <a href="https://www.nature.com/articles/s41597-025-04868-2">doi:10.1038/s41597-025-04868-2</a></p>An open-access database of nature-based carbon offset project boundaries
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04914-z">An improved long-term high-resolution surface pCO2 data product for the Indian Ocean using machine learning</a></h4>

              
                <p>Scientific Data, Published online: 05 April 2025; <a href="https://www.nature.com/articles/s41597-025-04914-z">doi:10.1038/s41597-025-04914-z</a></p>An improved long-term high-resolution surface <i>p</i>CO<sub>2</sub> data product for the Indian Ocean using machine learning
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04903-2">Seven years of time-tracking data capturing collaboration and failure dynamics: the Gryzzly dataset</a></h4>

              
                <p>Scientific Data, Published online: 05 April 2025; <a href="https://www.nature.com/articles/s41597-025-04903-2">doi:10.1038/s41597-025-04903-2</a></p>Seven years of time-tracking data capturing collaboration and failure dynamics: the Gryzzly dataset
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04854-8">Diploid chromosome-level genome assembly and annotation for Lycorma delicatula</a></h4>

              
                <p>Scientific Data, Published online: 05 April 2025; <a href="https://www.nature.com/articles/s41597-025-04854-8">doi:10.1038/s41597-025-04854-8</a></p>Diploid chromosome-level genome assembly and annotation for <i>Lycorma delicatula</i>
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/sdata.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/sdata.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/sdata.rss&title=Nature scientific data" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s41597-025-04753-y">Enabling disaggregation of Asian American subgroups: a dataset of Wikidata names for disparity estimation</a></h4>

              
                <p>Scientific Data, Published online: 05 April 2025; <a href="https://www.nature.com/articles/s41597-025-04753-y">doi:10.1038/s41597-025-04753-y</a></p>Enabling disaggregation of Asian American subgroups: a dataset of Wikidata names for disparity estimation
              

              <p class="text-muted">in <a href ="https://www.nature.com/sdata.rss">Nature scientific data</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-07993-1">Direct comparisons of neural activity during placebo analgesia and nocebo hyperalgesia between humans and rats</a></h4>

              
                <p>Communications Biology, Published online: 05 April 2025; <a href="https://www.nature.com/articles/s42003-025-07993-1">doi:10.1038/s42003-025-07993-1</a></p>A novel framework for cross-species analysis reveals the conserved and species-specific neural circuitry underlying placebo analgesia and nocebo hyperalgesia.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-08001-2">Trans-ethnic GWAS meta-analysis of idiopathic spermatogenic failure highlights the immune-mediated nature of Sertoli cell-only syndrome</a></h4>

              
                <p>Communications Biology, Published online: 05 April 2025; <a href="https://www.nature.com/articles/s42003-025-08001-2">doi:10.1038/s42003-025-08001-2</a></p>A large-scale genetic study conducted on European and Asian populations reveals shared genetic variants associated with the most severe form of spermatogenic failure, offering valuable insights into the underlying mechanisms of male infertility.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-07534-w">TransBind allows precise detection of DNA-binding proteins and residues using language models and deep learning</a></h4>

              
                <p>Communications Biology, Published online: 05 April 2025; <a href="https://www.nature.com/articles/s42003-025-07534-w">doi:10.1038/s42003-025-07534-w</a></p>An alignment-free deep learning framework predicts DNA-binding proteins and residues from primary sequences, eliminating multiple sequence alignments (MSAs). It uses protein language models and class-weighted training to address data imbalance and improve accuracy.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.nature.com/commsbio.rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.nature.com/commsbio.rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.nature.com/commsbio.rss&title=Nature communications biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.nature.com/articles/s42003-025-07940-0">Network-informed analysis of a multivariate trait-space reveals optimal trait selection</a></h4>

              
                <p>Communications Biology, Published online: 05 April 2025; <a href="https://www.nature.com/articles/s42003-025-07940-0">doi:10.1038/s42003-025-07940-0</a></p>Using network analysis, the authors propose a trait reduction procedure to identify the most informative traits while preserving the functional complexity of trait space.
              

              <p class="text-muted">in <a href ="https://www.nature.com/commsbio.rss">Nature communications biology</a> on 2025-04-05 00:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.eneuro.org/rss/current.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.eneuro.org/rss/current.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.eneuro.org/rss/current.xml&title=eNeuro" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="http://www.eneuro.org/cgi/content/short/12/4/ENEURO.0423-24.2025?rss=1">Maturational Stage-Dependent Contributions of the Cav3.2 T-Type Calcium Channel to Dentate Gyrus Granule Cell Excitability</a></h4>

              
                <p>T-Type calcium channels shape neuronal excitability driving burst firing, plasticity, and neuronal oscillations that influence circuit activity. The three biophysically distinct T-type channel subtypes (Cav3.1, Cav3.2, Cav3.3) are differentially expressed in the brain, contributing to divergent physiological processes. Cav3.2 channels are highly expressed in the dentate gyrus (DG) of the hippocampus, and mice lacking Cav3.2 [knock-out (KO)] exhibit impairments in hippocampal dependent learning and memory tasks, as well as attenuated development of pilocarpine induced epilepsy. Owing to neurogenesis, granule cells (GCs) are continuously added to the DG, generating a heterogeneous population of maturational stages with distinct excitability. While initial studies identified the role of Cav3.2 in mature GC burst firing, its functional relevance in the intrinsic excitability of different GC subpopulations has not yet been examined. In this study, we used juvenile Cav3.2 KO mice to examine the contributions of Cav3.2 channels to GC excitability at three different stages of maturation. We recorded from cells throughout the GC layer using their electrophysiological and morphological features to allocate GCs into immature, intermediate, and mature groups. In immature GCs, loss of Cav3.2 channels reduced the proportion of cells that fired low-threshold calcium spikes. Conversely, Cav3.2 KO increased excitability in regular spiking intermediate and mature GCs, enabling higher-frequency firing, with little impact on the frequency-dependent response. Overall, this study shows that Cav3.2 channels differentially regulate GC excitability throughout maturation and suggest that calcium influx via Cav3.2 may have maturation-dependent contributions to DG processes such as GC survival, integration, and memory encoding.</p>
              

              <p class="text-muted">in <a href ="https://www.eneuro.org/rss/current.xml">eNeuro</a> on 2025-04-04 16:30:15 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.eneuro.org/rss/current.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.eneuro.org/rss/current.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.eneuro.org/rss/current.xml&title=eNeuro" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="http://www.eneuro.org/cgi/content/short/12/4/ENEURO.0561-24.2025?rss=1">The Effect of Speech Masking on the Human Subcortical Response to Continuous Speech</a></h4>

              
                <p>Auditory masking&mdash;the interference of the encoding and processing of an acoustic stimulus imposed by one or more competing stimuli&mdash;is nearly omnipresent in daily life and presents a critical barrier to many listeners, including people with hearing loss, users of hearing aids and cochlear implants, and people with auditory processing disorders. The perceptual aspects of masking have been actively studied for several decades, and particular emphasis has been placed on masking of speech by other speech sounds. The neural effects of such masking, especially at the subcortical level, have been much less studied, in large part due to the technical limitations of making such measurements. Recent work has allowed estimation of the auditory brainstem response (ABR), whose characteristic waves are linked to specific subcortical areas, to naturalistic speech. In this study, we used those techniques to measure the encoding of speech stimuli that were masked by one or more simultaneous other speech stimuli. We presented listeners with simultaneous speech from one, two, three, or five simultaneous talkers, corresponding to a range of signal-to-noise ratios (clean, 0, &ndash;3, and &ndash;6&nbsp;dB), and derived the ABR to each talker in the mixture. Each talker in a mixture was treated in turn as a target sound masked by other talkers, making the response quicker to acquire. We found consistently across listeners that ABR Wave V amplitudes decreased and latencies increased as the number of competing talkers increased.</p>
              

              <p class="text-muted">in <a href ="https://www.eneuro.org/rss/current.xml">eNeuro</a> on 2025-04-04 16:30:15 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-234/v2?src=rss">Women's studies in the Latin American context: a bibliometric approach [version 2; peer review: 1 approved with reservations]</a></h4>

              
                Background Women’s studies in Latin America are transcendental because they make visible and challenge gender inequalities to achieve a more just and equitable society. However, despite this, there are still many research gaps, for which the objective is to examine the trends in research on women’s studies in Latin America. Methods An exploratory methodology based on bibliometric analysis is proposed to evaluate the scientific literature, based on the parameters of the PRISMA-2020 declaration. Results The bibliometric analysis reveals a growth in women’s studies in Latin America, reflecting its importance and relevance. Scientific production has experienced exponential growth, with leading researchers and journals. The United States and Canada lead scientific production. A change is observed in the topics addressed, focusing more on gender and equality. The thematic clusters identified highlight priority areas such as politics, institutions and representation. Conclusions Emerging keywords include neoliberalism, gender violence, political participation, female empowerment, and femicide, reflecting new concerns and challenges addressed in gender studies in Latin America.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-04 14:51:09 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-395/v1?src=rss">Serum sampling proximal to the joint or after exercise influences biomarker concentration and potentially improves understanding of joint microenvironment: a pilot case-control study [version 1; peer review: awaiting peer review]</a></h4>

              
                Background Osteoarthritis (OA) is a common, heterogeneous whole-joint disease contributing to significant morbidity with variable trajectories of symptom progression. Early detrimental joint changes may be identified using molecular biomarkers in serum or synovial fluid. When measuring the former, biomarkers are at risk of systemic dilution, with the latter technically challenging with increased associated risks. In addition, biomarker concentrations are also potentially influenced by extrinsic factors, including exercise. This case-control pilot study aims to understand the influence of sampling location and sub-maximal exercise on serum biomarker concentrations. Methods Recruited participants either had a recent lower-limb musculoskeletal injury (INJ), established knee OA (KOA) or were controls (CON). Serum was taken from their antecubital fossa (ACF, ‘Arm 1’), their great saphenous vein local to their ipsilateral knee (‘Knee’), and after a ten-minute non-weight-bearing exercise, their ACF again (‘Arm 2’). Serum biomarker assays were performed, with results analysed for differences between condition (CON v KOA v INJ), location (Arm 1 v Knee) and pre-post exercise (Arm 1 v Arm 2). Results Thirty-two participants were recruited (CON n=12, KOA n=10, INJ n=10), 81% male, median age 28 (IQR: 25-43) and BMI 23.9 (IQR: 22.5-26.3). Interleukin-(IL)-1β concentration was lower in the KOA group (p=0.004). IL-6 (1.65 and 0.96 ng/L) and leptin (12.65 and 7.01 ug/L) were higher in the Knee sample than Arm 1 (both p<0.001). COMP (173.06 and 190.27 ug/L, p=0.005) and CTX-II (0.68 and 0.83 ug/L, p=0.02) increased following exercise. Further differences were noted when the location and exercise intervention analyses were stratified by condition. Conclusions These novel sampling techniques and easy, accessible 10-minute exercise protocol are suitable for a wide range of settings. Both sampling location and precedent non-weight-bearing sub-maximal exercise influenced serum biomarker concentrations in this pilot case-control study. Optimising these sampling variables could improve the sensitivity of biomarker analysis following validation in a larger population.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-04 14:49:20 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012959">Seizure and redox rescue in a model of glucose transport deficiency</a></h4>

              
                <p>by Jay S. Coggan, Polina Shichkova, Henry Markram, Daniel Keller</p>

Disruptions of energy supply to the brain are associated with many neurodegenerative pathologies and are difficult to study due to numerous interlinked metabolic pathways. We explored the effects of diminished energy supply on brain metabolism using a computational model of the neuro-glia-vasculature ensemble, in the form of a neuron, an astrocyte and local blood supply. As a case study, we investigated the glucose transporter type-1 deficiency syndrome (GLUT1-DS), a childhood affliction characterized by impaired glucose utilization and associated with phenotypes including seizures. Compared to neurons, astrocytes exhibited markedly higher metabolite concentration variabilities for all but a few redox species. This effect could signal a role for astrocytes in absorbing the shock of blood nutrient fluctuations. Redox balances were disrupted in GLUT1-DS with lower levels of reducing equivalent carriers NADH and ATP. The best non-glucose nutrient or pharmacotherapies for re-establishing redox normalcy involved lactate, the keto-diet (β-hydroxybutyrate), NAD and Q10 supplementation, suggesting a possible glucose sparing mechanism. GLUT1-DS seizures resulted from after-discharge neuronal firing caused by post-stimulus ATP reductions and impaired Na<sup>+</sup>/K<sup>+</sup>-ATPase, which can be rescued by restoring either normal glucose or by relatively small increases in neuronal ATP.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012906">Quantifying the impact of contact tracing interview prioritisation strategies on disease transmission: A modelling study</a></h4>

              
                <p>by Logan Wu, Christopher M Baker, Nicholas Tierney, Kylie Carville, Jodie McVernon, Nick Golding, James M McCaw, Freya M Shearer</p>

Contact tracing is an important public health measure used to reduce transmission of infectious diseases. Contact tracers typically conduct telephone interviews with cases to identify contacts and direct them to quarantine, with the aim of preventing onward transmission. However, in situations where caseloads exceed the capacity of the public health system, timely interviews may not be feasible for all cases. Here we present a modelling framework for assessing the impact of different case interview prioritisation strategies on disease transmission. Our model is based on Australian contact tracing procedures and informed by contact tracing data on COVID-19 cases notified in Australia from 2020 to 2021. Our results demonstrate that last-in-first-out strategies (where cases with the most recent swab or notification dates are interviewed first) are more effective at reducing transmission than first-in-first-out strategies (where cases with the oldest swab or notification dates are interviewed first) or strategies with no explicit prioritisation. To maximise the public health benefit from a given case interview capacity, public health practitioners may consider our findings when designing case interview prioritisation protocols for outbreak response.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012904">scTrans: Sparse attention powers fast and accurate cell type annotation in single-cell RNA-seq data</a></h4>

              
                <p>by Zhiyi Zou, Ying Liu, Yuting Bai, Jiawei Luo, Zhaolei Zhang</p>

Cell type annotation is crucial in single-cell RNA sequencing data analysis because it enables significant biological discoveries and deepens our understanding of tissue biology. Given the high-dimensional and highly sparse nature of single-cell RNA sequencing data, most existing annotation tools focus on highly variable genes to reduce dimensionality and computational load. However, this approach inevitably results in information loss, potentially weakening the model’s generalization performance and adaptability to novel datasets. To mitigate this issue, we developed scTrans, a single cell Transformer-based model, which employs sparse attention to utilize all non-zero genes, thereby effectively reducing the input data dimensionality while minimizing information loss. We validated the speed and accuracy of scTrans by performing cell type annotation on 31 different tissues within the Mouse Cell Atlas. Remarkably, even with datasets nearing a million cells, scTrans efficiently perform cell type annotation in limited computational resources. Furthermore, scTrans demonstrates strong generalization capabilities, accurately annotating cells in novel datasets and generating high-quality latent representations, which are essential for precise clustering and trajectory analysis.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012888">Ten simple rules for getting started with knowledge mobilization</a></h4>

              
                <p>by Rime Diany, Laura K. Fitzgibbon-Collins, Sarah A. Gagliano Taliun</p>
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012874">Segmentation of dense and multi-species bacterial colonies using models trained on synthetic microscopy images</a></h4>

              
                <p>by Vincent Hickl, Abid Khan, René M. Rossi, Bruno F. B. Silva, Katharina Maniura-Weber</p>

The spread of microbial infections is governed by the self-organization of bacteria on surfaces. Bacterial interactions in clinically relevant settings remain challenging to quantify, especially in systems with multiple species or varied material properties. Quantitative image analysis methods based on machine learning show promise to overcome this challenge and support the development of novel antimicrobial treatments, but are limited by a lack of high-quality training data. Here, novel experimental and image analysis techniques for high-fidelity single-cell segmentation of bacterial colonies are developed. Machine learning-based segmentation models are trained solely using synthetic microscopy images that are processed to look realistic using a state-of-the-art image-to-image translation method (cycleGAN), requiring no biophysical modeling. Accurate single-cell segmentation is achieved for densely packed single-species colonies and multi-species colonies of common pathogenic bacteria, even under suboptimal imaging conditions and for both brightfield and confocal laser scanning microscopy. The resulting data provide quantitative insights into the self-organization of bacteria on soft surfaces. Thanks to their high adaptability and relatively simple implementation, these methods promise to greatly facilitate quantitative descriptions of bacterial infections in varied environments, and may be used for the development of rapid diagnostic tools in clinical settings.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012864">Logic-based modeling of biological networks with Netflux</a></h4>

              
                <p>by Alexander P. Clark, Mukti Chowkwale, Alexander Paap, Stephen Dang, Jeffrey J. Saucerman</p>

Molecular signaling networks drive a diverse range of cellular decisions, including whether to proliferate, how and when to die, and many processes in between. Such networks often connect hundreds of proteins, genes, and processes. Understanding these complex networks is aided by computational modeling, but these tools require extensive programming knowledge. In this article, we describe a user-friendly, programming-free network simulation tool called Netflux. Over the last decade, Netflux has been used to construct numerous predictive network models that have deepened our understanding of how complex biological networks make cell decisions. Here, we provide a Netflux tutorial that covers how to construct a network model and then simulate network responses to perturbations. Upon completion of this tutorial, you will be able to construct your own model in Netflux and simulate how perturbations to proteins and genes propagate through signaling and gene-regulatory networks.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012771">Why the growth of arboviral diseases necessitates a new generation of global risk maps and future projections</a></h4>

              
                <p>by Oliver J. Brady, Leonardo S. Bastos, Jamie M. Caldwell, Simon Cauchemez, Hannah E. Clapham, Illaria Dorigatti, Katy A. M. Gaythorpe, Wenbiao Hu, Laith Hussain-Alkhateeb, Michael A. Johansson, Ahyoung Lim, Velma K. Lopez, Richard James Maude, Jane P. Messina, Erin A. Mordecai, Andrew Townsend Peterson, Isabel Rodriquez-Barraquer, Ingrid B. Rabe, Diana P. Rojas, Sadie J. Ryan, Henrik Salje, Jan C. Semenza, Quan Minh Tran</p>
Author summary: Global risk maps are an important tool for assessing the global threat of mosquito and tick-transmitted arboviral diseases. Public health officials increasingly rely on risk maps to understand the drivers of transmission, forecast spread, identify gaps in surveillance, estimate disease burden, and target and evaluate the impact of interventions. Here, we describe how current approaches to mapping arboviral diseases have become unnecessarily siloed, ignoring the strengths and weaknesses of different data types and methods. This places limits on data and model output comparability, uncertainty estimation and generalisation that limit the answers they can provide to some of the most pressing questions in arbovirus control. We argue for a new generation of risk mapping models that jointly infer risk from multiple data types. We outline how this can be achieved conceptually and show how this new framework creates opportunities to better integrate epidemiological understanding and uncertainty quantification. We advocate for more co-development of risk maps among modellers and end-users to better enable risk maps to inform public health decisions. Prospective validation of risk maps for specific applications can inform further targeted data collection and subsequent model refinement in an iterative manner. If the expanding use of arbovirus risk maps for control is to continue, methods must develop and adapt to changing questions, interventions and data availability.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012593">TIPP3 and TIPP3-fast: Improved abundance profiling in metagenomics</a></h4>

              
                <p>by Chengze Shen, Eleanor Wedell, Mihai Pop, Tandy Warnow</p>

We present TIPP3 and TIPP3-fast, new tools for abundance profiling in metagenomic datasets. Like its predecessor, TIPP2, the TIPP3 pipeline uses a maximum likelihood approach to place reads into labeled taxonomies using marker genes, but it achieves superior accuracy to TIPP2 by enabling the use of much larger taxonomies through improved algorithmic techniques. We show that TIPP3 is generally more accurate than leading methods for abundance profiling in two important contexts: when reads come from genomes not already in a public database (i.e., novel genomes) and when reads contain sequencing errors. We also show that TIPP3-fast has slightly lower accuracy than TIPP3, but is also generally more accurate than other leading methods and uses a small fraction of TIPP3’s runtime. Additionally, we highlight the potential benefits of restricting abundance profiling methods to those reads that map to marker genes (i.e., using a filtered marker-gene based analysis), which we show typically improves accuracy. TIPP3 is freely available at https://github.com/c5shen/TIPP3.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/ploscompbiol/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/ploscompbiol/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/ploscompbiol/NewArticles&title=PLoS Computational Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1012493">Molecular dynamics simulations of human cohesin subunits identify DNA binding sites and their potential roles in DNA loop extrusion</a></h4>

              
                <p>by Chenyang Gu, Shoji Takada, Giovanni B. Brandani, Tsuyoshi Terakawa</p>

The SMC complex cohesin mediates interphase chromatin structural formation in eukaryotic cells through DNA loop extrusion. Here, we sought to investigate its mechanism using molecular dynamics simulations. To achieve this, we first constructed the amino-acid-residue-resolution structural models of the cohesin subunits, SMC1, SMC3, STAG1, and NIPBL. By simulating these subunits with double-stranded DNA molecules, we predicted DNA binding patches on each subunit and quantified the affinities of these patches to DNA using their dissociation rate constants as a proxy. Then, we constructed the structural model of the whole cohesin complex and mapped the predicted high-affinity DNA binding patches on the structure. From the spatial relations of the predicted patches, we identified that multiple patches on the SMC1, SMC3, STAG1, and NIPBL subunits form a DNA clamping patch group. The simulations of the whole complex with double-stranded DNA molecules suggest that this patch group facilitates DNA bending and helps capture a DNA segment in the cohesin ring formed by the SMC1 and SMC3 subunits. In previous studies, these have been identified as critical steps in DNA loop extrusion. Therefore, this study provides experimentally testable predictions of DNA binding sites implicated in previously proposed DNA loop extrusion mechanisms and highlights the essential roles of the accessory subunits STAG1 and NIPBL in the mechanism.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/ploscompbiol/NewArticles">PLoS Computational Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/plosbiology/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/plosbiology/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/plosbiology/NewArticles&title=PLoS Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3003125">Correction: Predation drives complex eco-evolutionary dynamics in sexually selected traits</a></h4>

              
                <p>by Brian A. Lerch, Maria R. Servedio</p>
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/plosbiology/NewArticles">PLoS Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/plosbiology/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/plosbiology/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/plosbiology/NewArticles&title=PLoS Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3003101">Refining the sleep circuits one neuron at a time</a></h4>

              
                <p>by Giorgio F. Gilestro</p>

The neural basis of sleep regulation remains elusive. A new study in PLOS Biology refines the key neuronal circuits involved in the regulation of sleep in fruit flies, confirming <i>Drosophila melanogaster</i> as the model of choice for unraveling the systems neuroscience of such a mysterious phenomenon.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/plosbiology/NewArticles">PLoS Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.plos.org/plosbiology/NewArticles>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.plos.org/plosbiology/NewArticles" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.plos.org/plosbiology/NewArticles&title=PLoS Biology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.3002649">Combining nanobody labeling with STED microscopy reveals input-specific and layer-specific organization of neocortical synapses</a></h4>

              
                <p>by Yeasmin Akter, Grace Jones, Grant J. Daskivich, Victoria Shifflett, Karina J. Vargas, Martin Hruska</p>

The discovery of synaptic nanostructures revealed key insights into the molecular logic of synaptic function and plasticity. Yet, our understanding of how diverse synapses in the brain organize their nano-architecture remains elusive, largely due to the limitations of super-resolution imaging in complex brain tissue. Here, we characterized single-domain camelid nanobodies for the 3D quantitative multiplex imaging of synaptic nano-organization sing tau-STED nanoscopy in cryosections from the mouse primary somatosensory cortex. We focused on thalamocortical (TC) and corticocortical (CC) synapses along the apical-basal axis of layer five pyramidal neurons as models of functionally diverse glutamatergic synapses in the brain. Spines receiving TC input were larger than those receiving CC input in all layers examined. However, the nano-architecture of TC synapses varied with dendritic location. TC afferents on apical dendrites frequently contacted spines with multiple aligned PSD-95/Bassoon nanomodules of constant size. In contrast, TC spines on basal dendrites predominantly contained a single aligned nanomodule, with PSD-95 nanocluster sizes scaling proportionally with spine volume. The nano-organization of CC synapses did not change across cortical layers and resembled modular architecture defined in vitro. These findings highlight the nanoscale diversity of synaptic architecture in the brain, that is, shaped by both the source of afferent input and the subcellular localization of individual synaptic contacts.
              

              <p class="text-muted">in <a href ="http://feeds.plos.org/plosbiology/NewArticles">PLoS Biology</a> on 2025-04-04 14:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/13-447/v3?src=rss">Case Report: Methimazole-Induced Parotitis - An Unusual Presentation [version 3; peer review: 1 approved, 1 approved with reservations, 2 not approved]</a></h4>

              
                A 56-year-old female with a medical history of unspecified hyperthyroidism and a recent thyroid storm presented to the ED with tachycardia, hypertension, and bilateral enlarged parotid glands. During a previous hospitalization, she was diagnosed with unspecified hyperthyroidism and started on methimazole. During hospitalization, laboratory findings suggested Graves’ disease with an acute thyroid storm. The patient also complained of enlarged parotid glands bilaterally. CT tomography of the neck revealed no calculi of the parotid glands but showed extensive fatty replacement, possibly related to methimazole use. Treatment with propranolol and IV hydrocortisone improved thyroid function. Due to the suspicion of methimazole-induced parotitis, she was transitioned to a reduced methimazole dosage for treatment of Graves’ disease, which subsequently improved her parotitis. Methimazole, the standard initial treatment for Graves’ disease, is generally well-tolerated. It can cause adverse reactions; however, parotitis is very rare and has been documented in only a few case reports. Owing to the limited number of reports, its incidence is currently unknown. Here, we present a case of methimazole-induced parotitis as an unusual presentation of thyroid storm. Drug-induced reactions can only be considered once common causes of parotitis such as viral infection, obstruction, and autoimmune diseases are ruled out. Treatment involves dosage adjustments and supportive care. Methimazole-induced parotitis is often misdiagnosed and overlooked because of the lack of reported cases. This necessitates future research into the reaction mechanisms and optimal treatment.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-04 12:59:10 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-394/v1?src=rss">The association between HBA1C level and the outcome of hernia surgery in diabetic patients managed at a tertiary center: A retrospective cohort study [version 1; peer review: awaiting peer review]</a></h4>

              
                Background The literature indicates a link between HbA1C levels, infections, and postoperative complications across various surgical procedures. However, its specific association with hernioplasty outcomes remains uncertain. This study aims to evaluate the relationship between HbA1C levels and the outcomes of hernia surgery in diabetic patients treated at a tertiary care center. Methods Data were collected from 301 diabetic patients, categorized into controlled (HbA1C < 7%) and uncontrolled (HbA1C ≥ 7%) groups, who underwent hernioplasty at King Abdulaziz Medical City (KAMC) between September 1, 2017, and September 1, 2022. Patient records were analyzed to explore the association between diabetes and hernia surgery outcomes, including complications, infections, and recurrence, as well as to compare the outcomes of different surgical techniques (open and laparoscopic). Results Of the 301 patients, this study included only 140 (46.5%) diabetic individuals who underwent hernia surgery at the Ministry of National Guard Health Affairs (MNGHA) in Riyadh between 2017 and 2022. Preoperative uncontrolled HbA1C levels were not statistically significant for wound infection (p = 0.706) or hernia recurrence (p = 0.075) within a one-year follow-up period. However, a statistically significant difference was observed in the prevalence of hypertension between the controlled and uncontrolled HbA1C groups (p = 0.039). Conclusion This study indicates that HbA1C levels do not significantly influence complications or infections following hernia surgery, particularly regarding wound infections and recurrence. However, HTN was found to be statistically significant and warrants further investigation. Additionally, no statistical significance was identified between elevated BMI (>30 kg/m2) and HbA1C groups (p = 0.081), which may be attributed to the limited sample size of this study.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-04 12:17:22 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-393/v1?src=rss">Comparing two chemotherapeutic regimens VDC/IE (or VAC/IE) v/s VIDE for patients suffering with Ewing sarcoma: A systematic review on survivorship and regime difference across borders [version 1; peer review: awaiting peer review]</a></h4>

              
                Background Ewing sarcoma is the second most common primary bone cancer in adolescents and young adults, with a poor prognosis. It is usually treated with a combination of treatment options, including but not limited to chemotherapy, surgery, and radiation therapy. In this systematic review, we focus on comparing two different chemotherapeutic regimens i.e., VDC/IE (vincristine, doxorubicin, and cyclophosphamide alternating with ifosfamide and etoposide) and VIDE (vincristine, ifosfamide, doxorubicin, and etoposide), for patients with Ewing Sarcoma, with Overall survival as the primary outcome and event-free survival as the secondary outcome. Methods MEDLINE, Embase (via Ovid), and Web of Science were searched methodically to identify studies which satisfied our inclusion and exclusion criteria. After obtaining the studies, deduplication was performed (manually and automatically). Once duplicates were removed, the studies were scanned against the inclusion and exclusion criteria predefined in the registered Prospero protocol. The process of screening was carried out in two phases: in the first stage, the Titles and Abstracts were scanned, and those which were included were scanned in the second stage of full-eligibility screening. The studies obtained were assessed for Risk of Bias using the JBI’s checklist. Results Post Risk of Bias Assessment, three studies were deemed suitable for inclusion. Through analysis of the data and text of these studies, we conclude that the VDC/IE regime yields better survival outcomes when compared to the VIDE regime, and hence should tentatively be used as standard of care for treating Ewing Sarcoma.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-04 12:13:34 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00282.2024?af=R">Neuroscientific Approaches to Adolescent Social Media Use: A Review of Neural correlates and Potential Associations to Social Media Behaviors</a></h4>

              
                Journal of Neurophysiology, Ahead of Print. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-04 11:59:53 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00042.2025?af=R">Convergent effects of peptides on the initiation of feeding motor programs in the mollusk Aplysia</a></h4>

              
                Journal of Neurophysiology, Ahead of Print. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-04 11:39:57 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00108.2024?af=R">Third-order entropy for spatiotemporal neural network characterization</a></h4>

              
                Journal of Neurophysiology, <a href="https://journals.physiology.org/toc/jn/133/4">Volume 133, Issue 4</a>, Page 1234-1244, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-04 10:59:32 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00480.2023?af=R">Action complexity modulates motor performance in the emotional oddball task</a></h4>

              
                Journal of Neurophysiology, <a href="https://journals.physiology.org/toc/jn/133/4">Volume 133, Issue 4</a>, Page 1245-1255, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-04 10:59:31 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00016.2025?af=R">A novel approach to the assessment of motor-unit identification accuracy and its radical incongruence with conventional methodology</a></h4>

              
                Journal of Neurophysiology, <a href="https://journals.physiology.org/toc/jn/133/4">Volume 133, Issue 4</a>, Page 1256-1265, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-04 10:59:30 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn&title=Journal of Neurophysiology" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://journals.physiology.org/doi/abs/10.1152/jn.00034.2025?af=R">Dual task reactive balance control in older adults with mild cognitive impairment: Does the cognitive task domain make a difference?</a></h4>

              
                Journal of Neurophysiology, Ahead of Print. <br/>
              

              <p class="text-muted">in <a href ="https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn">Journal of Neurophysiology</a> on 2025-04-04 10:49:58 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/14-88/v2?src=rss">Structuring data analysis projects in the Open Science era with Kerblam! [version 2; peer review: 2 approved]</a></h4>

              
                Background Structuring data analysis projects, that is, defining the layout of files and folders needed to analyze data using existing tools and novel code, largely follows personal preferences. Open Science calls for more accessible, transparent and understandable research. We believe that Open Science principles can be applied to the way data analysis projects are structured. Methods We examine the structure of several data analysis project templates by analyzing project template repositories present in GitHub. Through visualization of the resulting consensus structure, we draw observations regarding how the ecosystem of project structures is shaped, and what salient characteristics it has. Results Project templates show little overlap, but many distinct practices can be highlighted. We take them into account with the wider Open Science philosophy to draw a few fundamental Design Principles to guide researchers when designing a project space. We present Kerblam!, a project management tool that can work with such a project structure to expedite data handling, execute workflow managers, and share the resulting workflow and analysis outputs with others. Conclusions We hope that, by following these principles and using Kerblam!, the landscape of data analysis projects can become more transparent, understandable, and ultimately useful to the wider community.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-04 10:21:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/13-1128/v2?src=rss">Fostering healthcare system sustainability through efficient practices: Can adopting biosimilars ease the financial burden of rheumatoid arthritis? [version 2; peer review: 2 approved]</a></h4>

              
                Immune-mediated inflammatory diseases like rheumatoid arthritis (RA) have been successfully treated using biologic disease-modifying antirheumatic drugs. These medications are not utilized as first-line treatment, in part because of their high cost, but they are frequently seen to be cost-effective for RA patient populations that do not respond adequately to conventional disease-modifying antirheumatic drugs. Moreover, not all RA patients who meet clinical eligibility criteria can access biologics, not even as second-line therapy. Recently, there has been an increasing interest in biosimilars that are highly comparable to their originator biologics in terms of efficacy and safety but generally come at a lower price. This review summarizes the potential role of biosimilars in reducing RA expenditure and increasing RA patient access to biologic therapies. As the global landscape for biosimilars continues to evolve, it is essential to consider the unique challenges and opportunities in different healthcare systems. By leveraging the potential of biosimilars, healthcare systems can improve RA management, ease its economic burden and ensure that patients have access to effective and affordable treatments. The future of RA treatment lies in the integration of biosimilars into clinical practice, offering hope for more sustainable and equitable healthcare systems.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-04 10:06:47 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml&title=Physical Review E: Biological physics" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="http://link.aps.org/doi/10.1103/PhysRevE.111.044404">Microbial populations hardly ever grow logistically and never sublinearly</a></h4>

              
                <p>Author(s): José Camacho-Mateu, Aniello Lampo, Mario Castro, and José A. Cuesta</p><p>Challenging the conventional logistic model, this study shows that microbial growth follows a generalized <math xmlns="http://www.w3.org/1998/Math/MathML" display="inline"><mi>θ</mi></math>-logistic model, accounting for environmental fluctuations. The findings show microbial growth is never sublinear, with significant implications for macroecological patterns and the mechanisms governing microbial ecosystem stability.</p>
<p>#BiophysicsSpotlight #TimelyTopic #AdvancingField</p><img src="//cdn.journals.aps.org/journals/PRE/key_images/10.1103/PhysRevE.111.044404.png" width="200" height=\"100\"><br/><p>[Phys. Rev. E 111, 044404] Published Fri Apr 04, 2025</p>
              

              <p class="text-muted">in <a href ="http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml">Physical Review E: Biological physics</a> on 2025-04-04 10:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://f1000research.com/published/rss>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://f1000research.com/published/rss" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://f1000research.com/published/rss&title=F1000Research" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://f1000research.com/articles/12-142/v4?src=rss">Comparison of the antimicrobial reduction effect of photodynamic inactivation with the addition of chlorophyll and curcumin photosensitizer in Aggregatibacter actinomycetemcomitans and Enterococcus faecalis [version 4; peer review: 3 approved with reserva</a></h4>

              
                Background Aggregatibacter actinomycetemcomitans and Enterococcus faecalis are pathogenic bacteria of the oral cavity that cause various diseases such as periodontitis and endodontics. These bacteria are easily resistant to antibiotics. Photodynamic inactivation (PDI) is a method of inactivating microorganisms that utilizes light to activate a photosensitizer agent (PS) that produces reactive oxygen species causing cell lysis. Methods This study used the PDI method with a 405 nm diode laser at various energy density with the addition PS curcumin or chlorophyll Alfalfa, as much as 1.6 mg/ml on A. actinomycetemcomitans and E. faecalis bacteria. Results The study on E. faecalis bacteria showed that the energy density diode laser irradiation of 1.59 J/cm² gave the percentage of E. faecalis bacteria death 36.7% without PS, 69.30% with the addition of chlorophyll Medicago sativa L and 89.42% with the addition of curcumin. Meanwhile, the bacteria A. actinomycetemcomitans showed that the energy density diode laser irradiation of 1.59 J/cm² gave the percentage of bacterial death 35.81% without PS, 64.39% with the addition of chlorophyll Medicago sativa L and 89.82% with the addition of curcumin. PS was critical to the success of the PDI. Conclusions The addition of PS curcumin increased the effectiveness of reducing bacteria E. faecalis and A. actinomycetemcomitans compared to chlorophyll Medicago sativa L.
              

              <p class="text-muted">in <a href ="https://f1000research.com/published/rss">F1000Research</a> on 2025-04-04 08:09:26 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.ads1325?af=R">Nonlinear sound-sheet microscopy: Imaging opaque organs at the capillary and cellular scale</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/388/6742">Volume 388, Issue 6742</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.ads4539?af=R">Exogenous RNA surveillance by proton-sensing TRIM25</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/388/6742">Volume 388, Issue 6742</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.ads7913?af=R">Homogeneous-heterogeneous bifunctionality in Pd-catalyzed vinyl acetate synthesis</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/388/6742">Volume 388, Issue 6742</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adt0548?af=R">Osteoarthritis treatment via the GLP-1–mediated gut-joint axis targets intestinal FXR signaling</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/388/6742">Volume 388, Issue 6742</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adp1853?af=R">A geological timescale for bacterial evolution and oxygen adaptation</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/388/6742">Volume 388, Issue 6742</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adr3675?af=R">Human high-order thalamic nuclei gate conscious perception through the thalamofrontal loop</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/388/6742">Volume 388, Issue 6742</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science&title=Science" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/science.adq2110?af=R">Integrating multiple evidence streams to understand insect biodiversity change</a></h4>

              
                Science, <a href="https://www.science.org/toc/science/388/6742">Volume 388, Issue 6742</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science">Science</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ado8268?af=R">Robust semi-automatic vessel tracing in the human retinal image by an instance segmentation neural network</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ads9711?af=R">PIP2 promotes the incorporation of CD43, PSGL-1, and CD44 into nascent HIV-1 particles</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adu4391?af=R">A high-throughput experimentation platform for data-driven discovery in electrochemistry</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adt4666?af=R">Galvanostatic cycling of a micron-sized solid-state battery: Visually linking void evolution to electrochemistry</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adt9526?af=R">Topological elastic liquid diode</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adv7218?af=R">Experimental quantification of nuclear quantum effects on the hydrogen bond of liquid water</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adr7134?af=R">MIST: An interpretable and flexible deep learning framework for single–T cell transcriptome and receptor analysis</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ads6094?af=R">Architecture and conformational dynamics of the BAM-SurA holo insertase complex</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ads7475?af=R">Complex-valued matrix-vector multiplication using a scalable coherent photonic processor</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adu0601?af=R">Mg2+ influx mediated by TRPM7 triggers the initiation of muscle stem cell activation</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adr9038?af=R">Leveraging data mining, active learning, and domain adaptation for efficient discovery of advanced oxygen evolution electrocatalysts</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adt1691?af=R">Observation of the magnonic Dicke superradiant phase transition</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adv1878?af=R">Melt-mixed superlayer cocrystal formation using symmetric and unsymmetric organic semiconductors</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adq1575?af=R">Atf3 controls transitioning in female mitochondrial cardiomyopathy as identified by spatial and single-cell transcriptomics</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.adu4268?af=R">An autonomous snapper featuring adaptive actuation and embodied intelligence</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          
            <li class="list-group-item">
              <p class="text-right">-&nbsp;<a href="https://app.wallabag.it/bookmarklet?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv>" target="wallabagit"><small>Wallabag.it!</small></a>&nbsp;-&nbsp;<a href="http://instapaper.com/hello2?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv" target="instapaper"><small>Save to Instapaper</small></a>&nbsp;-&nbsp;<a href="http://getpocket.com/save?url=https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv&title=Science Advances" target="pocket"><small>Save to Pocket</small></a>&nbsp;-</p>
              <h4><a href="https://www.science.org/doi/abs/10.1126/sciadv.ads7191?af=R">Cholinergic signaling to CA1 astrocytes controls fear extinction</a></h4>

              
                Science Advances, <a href="https://www.science.org/toc/sciadv/11/14">Volume 11, Issue 14</a>, April 2025. <br/>
              

              <p class="text-muted">in <a href ="https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv">Science Advances</a> on 2025-04-04 07:00:00 UTC.
              </p>
            </li>
          <!-- each item -->
        </ul>
      </div>
      <div class="col-md-3">
        <div class="panel panel-default">
          <div class="panel-heading">
            <h3 class="panel-title">Feed list</h3>
          </div>
          <ul class="list-group">
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/computational-neuroscience'>Frontiers in Computational Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/aging-neuroscience'>Frontiers in Ageing Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/evolutionary-neuroscience/rss'>Frontiers in Evolutionary Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/human-neuroscience/rss'>Frontiers in Human Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/molecular-neuroscience/rss'>Frontiers in Molecular Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/neuroenergetics/rss'>Frontiers in Neurogenetics</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/neuroengineering/rss'>Frontiers in Neuroengineering</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/systems-neuroscience/rss'>Frontiers in Systems Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/synaptic-neuroscience/rss'>Frontiers in Synaptic Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/cellular-neuroscience/rss'>Frontiers in Cellular Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/integrative-neuroscience/rss'>Frontiers in Integrative Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/neural-circuits/rss'>Frontiers in Neural Circuits</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/neuroanatomy/rss'>Frontiers in Neuroanatomy</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.frontiersin.org/journals/neuroinformatics/rss'>Frontiers in Neuroinformatics</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://www.cell.com/trends/neurosciences/inpress.rss'>Trends in Neurosciences: In press</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://www.cell.com/neuron/current.rss'>Neuron: Current Issue</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://www.cell.com/neuron/inpress.rss'>Neuron: In press</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.cell.com/cell-reports/current.rss'>Cell Reports: Current Issue</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.cell.com/cell-reports/inpress.rss'>Cell Reports: In press</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.plos.org/ploscompbiol/NewArticles'>PLoS Computational Biology</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.plos.org/plosbiology/NewArticles'>PLoS Biology</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/nature/rss/current'>Nature</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/neuro/rss/current'>Nature Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/nrn/rss/current'>Nature Reviews</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/nphoton/rss/current'>Nature Photomics</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/nmeth/rss/current'>Nature Methods</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/ncomms/rss/current'>Nature Communications</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.nature.com/nphys/rss/current'>Nature Physics</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.nature.com/sdata.rss'>Nature scientific data</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.nature.com/commsbio.rss'>Nature communications biology</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://elifesciences.org/rss/recent.xml'>eLife</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://onlinelibrary.wiley.com/action/showFeed?jc=15318249&type=etoc&feed=rss'>Annals of Neurology</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://onlinelibrary.wiley.com/feed/10969861/most-recent'>Journal of Comparative Neurology</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://onlinelibrary.wiley.com/action/showFeed?jc=10974547&type=etoc&feed=rss'>Journal of Neuroscience Research</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.jneurosci.org/rss/current.xml'>Journal of Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://bmcneurosci.biomedcentral.com/articles/most-recent/rss.xml'>BMC Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.pnas.org/action/showFeed?type=searchTopic&taxonomyCode=topic&tagCode=neuro'>PNAS</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://direct.mit.edu/neco'>Neural Computation</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://direct.mit.edu/netn'>Network Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://connect.biorxiv.org/biorxiv_xml.php?subject=neuroscience'>bioRxiv: Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://export.arxiv.org/rss/q-bio.NC/'>arXiv: Quantitative Biology: Neurons and Cognition</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://export.arxiv.org/rss/cs.NE'>arXiv: Computer Science: Neural and Evolutionary Computing</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://rss.sciencedirect.com/publication/science/09594388'>Current Opinion in Neurobiology</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://rss.sciencedirect.com/publication/science/08936080'>Neural Networks</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://academic.oup.com/rss/site_5122/3011.xml'>Cerebral Cortex</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.eneuro.org/rss/current.xml'>eNeuro</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://journals.physiology.org/action/showFeed?type=etoc&feed=rss&jc=jn'>Journal of Neurophysiology</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=science'>Science</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.science.org/action/showFeed?type=etoc&feed=rss&jc=sciadv'>Science Advances</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://onlinelibrary.wiley.com/feed/10981063/most-recent'>Hippocampus</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://rescience.github.io/feed'>ReScience C</a>
            </li>
           
            <li class="list-group-item">
              <a href='http://feeds.aps.org/rss/tocsec/PRE-Biologicalphysics.xml'>Physical Review E: Biological physics</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://mathematical-neuroscience.springeropen.com/articles/most-recent/rss.xml'>The Journal of Mathematical Neuroscience</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://behavioralandbrainfunctions.biomedcentral.com/articles/most-recent/rss.xml'>Behavioural and Brain Functions</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://www.mdpi.com/rss/journal/brainsci'>Brain Sciences</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://f1000research.com/published/rss'>F1000Research</a>
            </li>
           
            <li class="list-group-item">
              <a href='https://journals.sagepub.com/action/showFeed?ui=0&mi=ehikzz&ai=2b4&jc=nroa&type=etoc&feed=rss'>The Neuroscientist</a>
            </li>
           
           </ul>
        </div>
      </div>
    </div>
    <footer id="site-footer">
      <p class="text-center text-muted">All content on this page is owned by their respective owners. The source code used to generate this page can be found <a href="https://github.com/neurofedora/planet-neuroscience">here</a>.</p>
      </div>
    </footer>
    <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <script type="text/javascript">
      $( document ).ready(function(){
        $("img").addClass("img-responsive");
        $("figure").addClass("img-responsive");
      });
    </script>
    <!-- Include all compiled plugins (below), or include individual files as needed -->
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
  </body>
</html>

